# trait used to check
## species
### NA
extant flowering plants (angiosperms) between tropical and non-tropical zones, and into and out of the major tropical regions of the world. we then calculate rates of speciation and extinction taking into account incomplete taxonomic sampling. we use a recently published fossil calibrated phylogeny and apply novel bioinformatic tools to code species into user-defined polygons. we reconstruct biogeographic history using stochastic character mapping to compute relative numbers of range shifts in proportion to the number of available lineages through time. our results, based on the analysis of c. 22,600 species and c. 20 million geo-", 
"d occurrence records, show no significant differences

recently published fossil calibrated phylogeny and apply novel bioinformatic tools to code species into user-defined polygons. we reconstruct biogeographic history using stochastic character mapping to compute relative numbers of range shifts in proportion to the number of available lineages through time. our results, based on the analysis of c. 22,600 species and c. 20 million geo-", 
"d occurrence records, show no significant differences between the speciation and extinction of tropical and non-tropical angiosperms. this suggests that at least in plants, the latitudinal biodiversity gradient primarily derives from other factors than differential rates of diversification. in contrast, the outstanding species richness found

c. 22,600 species and c. 20 million geo-", 
"d occurrence records, show no significant differences between the speciation and extinction of tropical and non-tropical angiosperms. this suggests that at least in plants, the latitudinal biodiversity gradient primarily derives from other factors than differential rates of diversification. in contrast, the outstanding species richness found today in the american tropics (the neotropics), as compared to tropical africa and tropical asia, is associated with significantly higher speciation and extinction rates. this suggests an exceedingly rapid evolutionary turnover, i.e., neotropical species being formed and replaced by one another at unparalleled rates. in addition, tropical america

primarily derives from other factors than differential rates of diversification. in contrast, the outstanding species richness found today in the american tropics (the neotropics), as compared to tropical africa and tropical asia, is associated with significantly higher speciation and extinction rates. this suggests an exceedingly rapid evolutionary turnover, i.e., neotropical species being formed and replaced by one another at unparalleled rates. in addition, tropical america stands out from other continents by having \"pumped out\" more species than it received through most of the last 66 million years. these results imply that the neotropics have acted as an engine for global plant

compared to tropical africa and tropical asia, is associated with significantly higher speciation and extinction rates. this suggests an exceedingly rapid evolutionary turnover, i.e., neotropical species being formed and replaced by one another at unparalleled rates. in addition, tropical america stands out from other continents by having \"pumped out\" more species than it received through most of the last 66 million years. these results imply that the neotropics have acted as an engine for global plant diversity. keywords: angiosperms, biogeography, diversification rates, latitudinal diversity gradient, out-of-the-tropics, phylogenetics, tropical biodiversity introduction the world's biodiversity is unevenly distributed, and most species are found

out\" more species than it received through most of the last 66 million years. these results imply that the neotropics have acted as an engine for global plant diversity. keywords: angiosperms, biogeography, diversification rates, latitudinal diversity gradient, out-of-the-tropics, phylogenetics, tropical biodiversity introduction the world's biodiversity is unevenly distributed, and most species are found in the tropical regions of asia (including australasia), africa, and the americas. understanding the underlying causes for the latitudinal biodiversity gradient - the decrease of taxonomic diversity away from the equator - has fostered extensive and integrative research, and its formation still constitutes a matter of debate 1

angiosperms or were there phases of markedly different range shift rates and directionality? (2) is high diversity correlated with high speciation and/or low extinction? more specifically, were there significant differences in speciation and extinction rates between tropical and non-tropical zones, and among tropical regions? in such case, are the most species rich regions also those with highest speciation and/or lowest extinction? in evolutionary biology and biogeography (see e.g., pianka, 1966; hillebrand, 2004; jablonski et al., 2006; wiens et al., 2006; brown, 2014; huang et al., 2014; kerkhoff et al., 2014; mannion et al., 2014; rolland et al., 2014). there are three

intrinsic eco-physiological constraints (niche conservatism; kerkhoff et al., 2014). a second striking feature of tropical biodiversity, besides being consistently higher than in non-tropical regions, is its uneven distribution among the three tropical regions of the world. for instance, it has been suggested that the american tropics (the neotropics) comprise more species of seed plants than tropical africa and tropical asia together, with similar patterns for other organismal groups such as amphibians, mammals, birds, nymphalid butterflies, and reptiles (govaerts, 2001; antonelli, 2008; antonelli and sanmartín, 2011 and", 
"s therein). the underlying causes for these inter-continental differences are poorly understood, and could be

collections (amongst others) could also play important roles. evaluating the validity and relative roles of the factors driving these fundamental biodiversity differences requires combining evidence from several sources and disciplines, such as palaeontology, ecology and molecular phylogenetics. among these, two main components stand out as essential in this pursuit: understanding species diversification (i.e., the interplay between speciation and extinction) and the geographic history of lineages. in this study we explore these two components at a global and continental scale. we focus on the cenozoic history (i.e., the last 66 ma) of flowering plants (angiosperms), which form the dominant structure of tropical

compare rates of speciation and extinction between tropical and non-tropical zones and among the world's three tropical regions (in africa, asia, and the americas), and we infer the timing and direction of range shifts into and out of each tropical region. material and methods data compilation fossils, molecular phylogenies, and species occurrences constitute diverse data sources that, taken together, can be used to infer diversity trends through time and space. here we explore the feasibility of using both neontological and palaeontological data for addressing the questions outlined in this study. fossils we explored whether fossils could be used to infer diversity

lower cretaceous to today), according to the geological time scale of gradstein et al. (2012). unfortunately, a visual inspection of the data (figure 1) showed severe spatial and temporal biases. these biases precluded any sensible analyses of diversity changes in tropical regions, and we were therefore forced to rely on species distribution and molecular data alone. species occurrences we downloaded all geo-", 
"d (i.e., provided with a longitude and latitude) species occurrences of angiosperms available at the global biodiversity information facility (gbif, http://www.gbif.org; downloaded in june 2014). records flagged to contain \"known coordinate issues\" were excluded prior to the download. one

the geological time scale of gradstein et al. (2012). unfortunately, a visual inspection of the data (figure 1) showed severe spatial and temporal biases. these biases precluded any sensible analyses of diversity changes in tropical regions, and we were therefore forced to rely on species distribution and molecular data alone. species occurrences we downloaded all geo-", 
"d (i.e., provided with a longitude and latitude) species occurrences of angiosperms available at the global biodiversity information facility (gbif, http://www.gbif.org; downloaded in june 2014). records flagged to contain \"known coordinate issues\" were excluded prior to the download. one record per location per species was

the data (figure 1) showed severe spatial and temporal biases. these biases precluded any sensible analyses of diversity changes in tropical regions, and we were therefore forced to rely on species distribution and molecular data alone. species occurrences we downloaded all geo-", 
"d (i.e., provided with a longitude and latitude) species occurrences of angiosperms available at the global biodiversity information facility (gbif, http://www.gbif.org; downloaded in june 2014). records flagged to contain \"known coordinate issues\" were excluded prior to the download. one record per location per species was retained. we then applied basic data cleaning steps on the full data set (c.

alone. species occurrences we downloaded all geo-", 
"d (i.e., provided with a longitude and latitude) species occurrences of angiosperms available at the global biodiversity information facility (gbif, http://www.gbif.org; downloaded in june 2014). records flagged to contain \"known coordinate issues\" were excluded prior to the download. one record per location per species was retained. we then applied basic data cleaning steps on the full data set (c. 40 gigabytes) for identifying and excluding obviously erroneous data points, such as records with non-numeric coordinates or missing species names, records with identical latitude and longitude, and latitudes or longitudes equal to zero (which we

to contain \"known coordinate issues\" were excluded prior to the download. one record per location per species was retained. we then applied basic data cleaning steps on the full data set (c. 40 gigabytes) for identifying and excluding obviously erroneous data points, such as records with non-numeric coordinates or missing species names, records with identical latitude and longitude, and latitudes or longitudes equal to zero (which we considered to have been left in blank during data entry). for these steps (1) have the tropics as a whole, and each tropical region separately, been mainly a sink or a source of angiosperm

c. 23◦ south). although smaller operational units would have been interesting from a biological perspective, e.g., separating rain forests and savannas, it would inevitably incur a considerable loss of data and statistical power for the subsequent analyses. we utilized the software package speciesgeocoder v.1.0 (töpel et al., 2014) to code species into operational units. the resulting polygons can be retrieved from the authors upon request. to further identify potential biases caused by erroneous geo", 
"s (e.g., due to wrong coordinates or species identifications), we applied a set of arbitrary thresholds in order for a species to be coded as \"present\" in

statistical power for the subsequent analyses. we utilized the software package speciesgeocoder v.1.0 (töpel et al., 2014) to code species into operational units. the resulting polygons can be retrieved from the authors upon request. to further identify potential biases caused by erroneous geo", 
"s (e.g., due to wrong coordinates or species identifications), we applied a set of arbitrary thresholds in order for a species to be coded as \"present\" in a certain operational unit. three filters were defined, with increasingly more strict criteria, as outlined in table 1. we implemented functions and scripts to carry out this data filtering in r

(töpel et al., 2014) to code species into operational units. the resulting polygons can be retrieved from the authors upon request. to further identify potential biases caused by erroneous geo", 
"s (e.g., due to wrong coordinates or species identifications), we applied a set of arbitrary thresholds in order for a species to be coded as \"present\" in a certain operational unit. three filters were defined, with increasingly more strict criteria, as outlined in table 1. we implemented functions and scripts to carry out this data filtering in r (scripts available from the authors). there was no major loss of occurrence records

the diversification rate analyses using the filter 3 data set, due to the fact that the method we employed cannot handle widespread taxa (see below). we used a modified version of the scripts by zanne et al. (2014) implemented in r (r core team, 2014). geographic assignments we coded each species for its presence and absence in four large regions or operational units (figure 2): tropical america (the neotropics), tropical africa (the afrotropics), tropical asia (including australasia), and all other (non-tropical) regions combined. we delimited those regions by following the same boundaries for biomes and ecoregions as adopted by the world

other. we therefore used the recent fossil-calibrated molecular phylogeny of angiosperms from figure 2 | the four regions (operational units) used in this study. tropical regions are shown in red (dark red: rain forests, light red: savannas), non-tropical regions are shown in blue. dots indicate frontiers in genetics | www.frontiersin.org species occurrence records of angiosperms (c. 20 million) used in this study and obtained from the global biodiversity information facility (gbif). 4 april 2015 | volume 6 | article 130 antonelli et al. biogeography and diversification of tropical angiosperms table 1 | automated criteria for recording presence of each species in

www.frontiersin.org species occurrence records of angiosperms (c. 20 million) used in this study and obtained from the global biodiversity information facility (gbif). 4 april 2015 | volume 6 | article 130 antonelli et al. biogeography and diversification of tropical angiosperms table 1 | automated criteria for recording presence of each species in each operational unit defined in figure 2, departing from raw gbif species occurrence data. table 2 | proportion of species included in the phylogeny for each plant order analyzed. order filter total # spp # in diversification sampling sampling analyses in tropics outside minimum minimum presence on more #

and obtained from the global biodiversity information facility (gbif). 4 april 2015 | volume 6 | article 130 antonelli et al. biogeography and diversification of tropical angiosperms table 1 | automated criteria for recording presence of each species in each operational unit defined in figure 2, departing from raw gbif species occurrence data. table 2 | proportion of species included in the phylogeny for each plant order analyzed. order filter total # spp # in diversification sampling sampling analyses in tropics outside minimum minimum presence on more # records % records than one polygon none 1 - allowed apiales 3114 478

(gbif). 4 april 2015 | volume 6 | article 130 antonelli et al. biogeography and diversification of tropical angiosperms table 1 | automated criteria for recording presence of each species in each operational unit defined in figure 2, departing from raw gbif species occurrence data. table 2 | proportion of species included in the phylogeny for each plant order analyzed. order filter total # spp # in diversification sampling sampling analyses in tropics outside minimum minimum presence on more # records % records than one polygon none 1 - allowed apiales 3114 478 0.07 1 3 - allowed asparagales 13956 1601

0.18 fabales 15049 2050 0.09 0.13 gentianales 11583 1052 0.08 0.07 lamiales 14813 1397 0.04 0.12 malpighiales 10882 1219 0.09 0.09 malvales 4398 446 0.07 0.10 0.12 tropics *under this filter, presence was only coded in the region with the highest number of records. zanne et al. (2014), with 30,535 species. the phylogeny was based on data from seven gene regions and families and orders were constrained to the apg iii classification system (bremer et al., 2009). to evaluate whether the level of taxonomic representation was consistent among regions, which could otherwise bias our subsequent analyses, we calculated the ratio between

based on data from seven gene regions and families and orders were constrained to the apg iii classification system (bremer et al., 2009). to evaluate whether the level of taxonomic representation was consistent among regions, which could otherwise bias our subsequent analyses, we calculated the ratio between the number of species sampled in the phylogeny and the total number of species recorded in each of the four regions in the gbif database. tropical conservatism 0.17 myrtales 8439 825 0.06 poales 13872 2190 0.06 0.18 ranunculales 2776 468 0.07 0.18 rosales 6396 852 0.09 0.13 sapindales 4991 667 0.10 0.12 solanales 3298

orders were constrained to the apg iii classification system (bremer et al., 2009). to evaluate whether the level of taxonomic representation was consistent among regions, which could otherwise bias our subsequent analyses, we calculated the ratio between the number of species sampled in the phylogeny and the total number of species recorded in each of the four regions in the gbif database. tropical conservatism 0.17 myrtales 8439 825 0.06 poales 13872 2190 0.06 0.18 ranunculales 2776 468 0.07 0.18 rosales 6396 852 0.09 0.13 sapindales 4991 667 0.10 0.12 solanales 3298 512 0.10 0.14 species numbers follow those in the gbif

phylogeny and the total number of species recorded in each of the four regions in the gbif database. tropical conservatism 0.17 myrtales 8439 825 0.06 poales 13872 2190 0.06 0.18 ranunculales 2776 468 0.07 0.18 rosales 6396 852 0.09 0.13 sapindales 4991 667 0.10 0.12 solanales 3298 512 0.10 0.14 species numbers follow those in the gbif backbone taxonomy. (sampling in tropics = number of species in the diversification analyses classified as tropical divided by the total number of species classified as tropical; sampling outside the tropics = number of species in the diversification analyses classified as outside the tropics divided

the gbif database. tropical conservatism 0.17 myrtales 8439 825 0.06 poales 13872 2190 0.06 0.18 ranunculales 2776 468 0.07 0.18 rosales 6396 852 0.09 0.13 sapindales 4991 667 0.10 0.12 solanales 3298 512 0.10 0.14 species numbers follow those in the gbif backbone taxonomy. (sampling in tropics = number of species in the diversification analyses classified as tropical divided by the total number of species classified as tropical; sampling outside the tropics = number of species in the diversification analyses classified as outside the tropics divided by the total number of species classified as outside the tropics). we tested whether species

0.18 ranunculales 2776 468 0.07 0.18 rosales 6396 852 0.09 0.13 sapindales 4991 667 0.10 0.12 solanales 3298 512 0.10 0.14 species numbers follow those in the gbif backbone taxonomy. (sampling in tropics = number of species in the diversification analyses classified as tropical divided by the total number of species classified as tropical; sampling outside the tropics = number of species in the diversification analyses classified as outside the tropics divided by the total number of species classified as outside the tropics). we tested whether species in each of the regions defined (figure 2) were clustered in the angiosperm phylogeny

sapindales 4991 667 0.10 0.12 solanales 3298 512 0.10 0.14 species numbers follow those in the gbif backbone taxonomy. (sampling in tropics = number of species in the diversification analyses classified as tropical divided by the total number of species classified as tropical; sampling outside the tropics = number of species in the diversification analyses classified as outside the tropics divided by the total number of species classified as outside the tropics). we tested whether species in each of the regions defined (figure 2) were clustered in the angiosperm phylogeny (i.e., showed strong phylogenetic signal) using bayesian tipsignificance testing implemented in

gbif backbone taxonomy. (sampling in tropics = number of species in the diversification analyses classified as tropical divided by the total number of species classified as tropical; sampling outside the tropics = number of species in the diversification analyses classified as outside the tropics divided by the total number of species classified as outside the tropics). we tested whether species in each of the regions defined (figure 2) were clustered in the angiosperm phylogeny (i.e., showed strong phylogenetic signal) using bayesian tipsignificance testing implemented in the software bats v. 1.0 (parker et al., 2008). we compared the observed distribution of states

species in the diversification analyses classified as tropical divided by the total number of species classified as tropical; sampling outside the tropics = number of species in the diversification analyses classified as outside the tropics divided by the total number of species classified as outside the tropics). we tested whether species in each of the regions defined (figure 2) were clustered in the angiosperm phylogeny (i.e., showed strong phylogenetic signal) using bayesian tipsignificance testing implemented in the software bats v. 1.0 (parker et al., 2008). we compared the observed distribution of states in the ", 
" phylogeny against 100 randomized replicates,

relative number of transitions through time were obtained by simulating 100 stochastic histories of geographic range evolution. we optimized the original scripts implementing this method and implemented them in we calculated rates of speciation (λ) and extinction (µ) for each tropical region separately, as well as for tropical and non-tropical species. for these analyses we used the multiple state speciation and extinction method (musse) as implemented in diversitree (fitzjohn, 2012). we analyzed 17 subclades separately (table 2), which we chose to correspond to plant orders. this division was necessary due to computational limitations in analysing the full tree under this method,

the rates in all regions analyzed. in all analyses, we used mean values of rates. frontiers in genetics | www.frontiersin.org 5 april 2015 | volume 6 | article 130 antonelli et al. biogeography and diversification of tropical angiosperms results (e.g., synonymisation and different taxonomic circumscriptions), and the fact that numerous species did not occur in both data sets. despite these issues, a total of 27,585 species could be fully matched between the molecular phylogeny and the occurrence data set, representing 14.6% of the total number of currently accepted species of angiosperms (273,174 species, according to http://www.theplantlist.org; accessed september 2014). the data

frontiers in genetics | www.frontiersin.org 5 april 2015 | volume 6 | article 130 antonelli et al. biogeography and diversification of tropical angiosperms results (e.g., synonymisation and different taxonomic circumscriptions), and the fact that numerous species did not occur in both data sets. despite these issues, a total of 27,585 species could be fully matched between the molecular phylogeny and the occurrence data set, representing 14.6% of the total number of currently accepted species of angiosperms (273,174 species, according to http://www.theplantlist.org; accessed september 2014). the data set generated under filter 2, used for all analyses except musse and bats, comprised a

results (e.g., synonymisation and different taxonomic circumscriptions), and the fact that numerous species did not occur in both data sets. despite these issues, a total of 27,585 species could be fully matched between the molecular phylogeny and the occurrence data set, representing 14.6% of the total number of currently accepted species of angiosperms (273,174 species, according to http://www.theplantlist.org; accessed september 2014). the data set generated under filter 2, used for all analyses except musse and bats, comprised a total of c. 20 million occurrence points and between c. 500 to 6600 species per region (figure 3). the proportion between species with

different taxonomic circumscriptions), and the fact that numerous species did not occur in both data sets. despite these issues, a total of 27,585 species could be fully matched between the molecular phylogeny and the occurrence data set, representing 14.6% of the total number of currently accepted species of angiosperms (273,174 species, according to http://www.theplantlist.org; accessed september 2014). the data set generated under filter 2, used for all analyses except musse and bats, comprised a total of c. 20 million occurrence points and between c. 500 to 6600 species per region (figure 3). the proportion between species with geo-", 
"s and species

representing 14.6% of the total number of currently accepted species of angiosperms (273,174 species, according to http://www.theplantlist.org; accessed september 2014). the data set generated under filter 2, used for all analyses except musse and bats, comprised a total of c. 20 million occurrence points and between c. 500 to 6600 species per region (figure 3). the proportion between species with geo-", 
"s and species in the phylogeny ranged from c. 8 to 15% among regions data compilation figure 3 shows the number of species and occurrences coded into each of the regions defined, the number of those that were also present

accepted species of angiosperms (273,174 species, according to http://www.theplantlist.org; accessed september 2014). the data set generated under filter 2, used for all analyses except musse and bats, comprised a total of c. 20 million occurrence points and between c. 500 to 6600 species per region (figure 3). the proportion between species with geo-", 
"s and species in the phylogeny ranged from c. 8 to 15% among regions data compilation figure 3 shows the number of species and occurrences coded into each of the regions defined, the number of those that were also present in the phylogeny, and the influence of each

species, according to http://www.theplantlist.org; accessed september 2014). the data set generated under filter 2, used for all analyses except musse and bats, comprised a total of c. 20 million occurrence points and between c. 500 to 6600 species per region (figure 3). the proportion between species with geo-", 
"s and species in the phylogeny ranged from c. 8 to 15% among regions data compilation figure 3 shows the number of species and occurrences coded into each of the regions defined, the number of those that were also present in the phylogeny, and the influence of each filter applied. the raw data

and bats, comprised a total of c. 20 million occurrence points and between c. 500 to 6600 species per region (figure 3). the proportion between species with geo-", 
"s and species in the phylogeny ranged from c. 8 to 15% among regions data compilation figure 3 shows the number of species and occurrences coded into each of the regions defined, the number of those that were also present in the phylogeny, and the influence of each filter applied. the raw data set of species occurrence points (after applying the basic cleaning steps described above) comprised a total of 24,908,478 records pertaining

phylogeny ranged from c. 8 to 15% among regions data compilation figure 3 shows the number of species and occurrences coded into each of the regions defined, the number of those that were also present in the phylogeny, and the influence of each filter applied. the raw data set of species occurrence points (after applying the basic cleaning steps described above) comprised a total of 24,908,478 records pertaining to 188,655 species (purple bars, figure 3). many species could not be matched between the species occurrence data set and the molecular phylogeny used, due to taxonomic issues that could not be easily

coded into each of the regions defined, the number of those that were also present in the phylogeny, and the influence of each filter applied. the raw data set of species occurrence points (after applying the basic cleaning steps described above) comprised a total of 24,908,478 records pertaining to 188,655 species (purple bars, figure 3). many species could not be matched between the species occurrence data set and the molecular phylogeny used, due to taxonomic issues that could not be easily solved figure 3 | number of angiosperm species and occurrences in the four regions defined in this study. the bars

defined, the number of those that were also present in the phylogeny, and the influence of each filter applied. the raw data set of species occurrence points (after applying the basic cleaning steps described above) comprised a total of 24,908,478 records pertaining to 188,655 species (purple bars, figure 3). many species could not be matched between the species occurrence data set and the molecular phylogeny used, due to taxonomic issues that could not be easily solved figure 3 | number of angiosperm species and occurrences in the four regions defined in this study. the bars show the influence of different cleaning

also present in the phylogeny, and the influence of each filter applied. the raw data set of species occurrence points (after applying the basic cleaning steps described above) comprised a total of 24,908,478 records pertaining to 188,655 species (purple bars, figure 3). many species could not be matched between the species occurrence data set and the molecular phylogeny used, due to taxonomic issues that could not be easily solved figure 3 | number of angiosperm species and occurrences in the four regions defined in this study. the bars show the influence of different cleaning steps on the data set (see also

steps described above) comprised a total of 24,908,478 records pertaining to 188,655 species (purple bars, figure 3). many species could not be matched between the species occurrence data set and the molecular phylogeny used, due to taxonomic issues that could not be easily solved figure 3 | number of angiosperm species and occurrences in the four regions defined in this study. the bars show the influence of different cleaning steps on the data set (see also table 1). (a) number of species per dataset and geographic region, (b) number of occurrence points per dataset and geographic region, (c) number of species

molecular phylogeny used, due to taxonomic issues that could not be easily solved figure 3 | number of angiosperm species and occurrences in the four regions defined in this study. the bars show the influence of different cleaning steps on the data set (see also table 1). (a) number of species per dataset and geographic region, (b) number of occurrence points per dataset and geographic region, (c) number of species per dataset and geographic region (tropical vs. non-tropical), (d) number of occurrence records per dataset and geographic region frontiers in genetics | www.frontiersin.org (tropical vs. non-tropical). purple: gbif download; blue: species

species and occurrences in the four regions defined in this study. the bars show the influence of different cleaning steps on the data set (see also table 1). (a) number of species per dataset and geographic region, (b) number of occurrence points per dataset and geographic region, (c) number of species per dataset and geographic region (tropical vs. non-tropical), (d) number of occurrence records per dataset and geographic region frontiers in genetics | www.frontiersin.org (tropical vs. non-tropical). purple: gbif download; blue: species that are included (and could be matched) in the phylogeny; dark green: filter 1 (minimum 3 occurrences to be

species per dataset and geographic region, (b) number of occurrence points per dataset and geographic region, (c) number of species per dataset and geographic region (tropical vs. non-tropical), (d) number of occurrence records per dataset and geographic region frontiers in genetics | www.frontiersin.org (tropical vs. non-tropical). purple: gbif download; blue: species that are included (and could be matched) in the phylogeny; dark green: filter 1 (minimum 3 occurrences to be coded as present in a given region); light green: filter 2 (additionally 10% of all occurrences per species needed to be coded as present); orange: filter 3 (additionally widespread species restricted

region frontiers in genetics | www.frontiersin.org (tropical vs. non-tropical). purple: gbif download; blue: species that are included (and could be matched) in the phylogeny; dark green: filter 1 (minimum 3 occurrences to be coded as present in a given region); light green: filter 2 (additionally 10% of all occurrences per species needed to be coded as present); orange: filter 3 (additionally widespread species restricted to one region). the filter 2 data set was used for all analyses except for musse and bats. 6 april 2015 | volume 6 | article 130 antonelli et al. biogeography and diversification of tropical angiosperms discussion

blue: species that are included (and could be matched) in the phylogeny; dark green: filter 1 (minimum 3 occurrences to be coded as present in a given region); light green: filter 2 (additionally 10% of all occurrences per species needed to be coded as present); orange: filter 3 (additionally widespread species restricted to one region). the filter 2 data set was used for all analyses except for musse and bats. 6 april 2015 | volume 6 | article 130 antonelli et al. biogeography and diversification of tropical angiosperms discussion (table 3). all tropical regions were similarly represented in the phylogeny, with

recent meta-analysis of 111 dated phylogenies, including seven clades of angiosperms (jansson et al., 2013), and also reflects the directionality observed from the fossil record of marine bivalves for the last 11 ma (jablonski et al., 2006, phylogeny-based analyses figure 4a shows the angiosperm phylogeny and the coding of each species as occurring in each of the four regions defined, whereas figure 4b shows the coding in tropical and non-tropical regions. the bayesian tip-significance testing indicated that species in all regions (figure 2) are highly clustered phylogenetically (p < 0.001 for all three statistical tests implemented in bats: parsimony score, association

bivalves for the last 11 ma (jablonski et al., 2006, phylogeny-based analyses figure 4a shows the angiosperm phylogeny and the coding of each species as occurring in each of the four regions defined, whereas figure 4b shows the coding in tropical and non-tropical regions. the bayesian tip-significance testing indicated that species in all regions (figure 2) are highly clustered phylogenetically (p < 0.001 for all three statistical tests implemented in bats: parsimony score, association index and maximum exclusive single-state clade). the results from the range shift analyses are summarized in figure 5. confidence intervals of range shift rates were generally large

and are not statistically different (figures 6a,b). in contrast, both the speciation and the extinction rates estimated for tropical america were significantly higher than those estimates for tropical africa and tropical asia (figures 6c,d, p < 0.05 for speciation, and p < 0.001 for extinction). table 3 | number of species recorded in each of the regions defined for the analyses (for which geo", 
"d data were available from gbif), number of those species that could be included in the range shift analysis (after applying data filter 2; see table 1 and figure 3), and their sampling fraction. region total #

than those estimates for tropical africa and tropical asia (figures 6c,d, p < 0.05 for speciation, and p < 0.001 for extinction). table 3 | number of species recorded in each of the regions defined for the analyses (for which geo", 
"d data were available from gbif), number of those species that could be included in the range shift analysis (after applying data filter 2; see table 1 and figure 3), and their sampling fraction. region total # # in range shift sampling spp (gbif) analyses fraction african tropics 26194 2460 0.09 american tropics 66844 5342 0.08 asian tropics 26854 2686

american tropics 66844 5342 0.08 asian tropics 26854 2686 0.10 total tropics 115196 9913 0.09 non tropical 103682 15666 0.15 frontiers in genetics | www.frontiersin.org figure 4 | angiosperm phylogeny used for the range shift and diversification analyses, pruned from zanne et al. (2014). the tree contains c. 22,600 terminal species and shows (a) the codification into each one of the continental-level regions defined in figure 2, and (b) the codification of all species as tropical or non-tropical. species in each of the regions defined are highly clustered phylogenetically according to bayesian tip-significance testing (p < 0.001). 7 april 2015 |

| www.frontiersin.org figure 4 | angiosperm phylogeny used for the range shift and diversification analyses, pruned from zanne et al. (2014). the tree contains c. 22,600 terminal species and shows (a) the codification into each one of the continental-level regions defined in figure 2, and (b) the codification of all species as tropical or non-tropical. species in each of the regions defined are highly clustered phylogenetically according to bayesian tip-significance testing (p < 0.001). 7 april 2015 | volume 6 | article 130 antonelli et al. biogeography and diversification of tropical angiosperms confidence intervals are shown as shaded areas in all

angiosperm phylogeny used for the range shift and diversification analyses, pruned from zanne et al. (2014). the tree contains c. 22,600 terminal species and shows (a) the codification into each one of the continental-level regions defined in figure 2, and (b) the codification of all species as tropical or non-tropical. species in each of the regions defined are highly clustered phylogenetically according to bayesian tip-significance testing (p < 0.001). 7 april 2015 | volume 6 | article 130 antonelli et al. biogeography and diversification of tropical angiosperms confidence intervals are shown as shaded areas in all plots. in (a), a global

least in part, account for the inferred peak. the high rate of range shifts out of the neotropics is particularly noteworthy in comparison to the other tropical regions, where we did not find this difference between immigration and emigration. thus, our results suggest that the neotropics have functioned as a \"species pump\" for the rest of the world during 9 april 2015 | volume 6 | article 130 antonelli et al. biogeography and diversification of tropical angiosperms (richardson et al., 2001; erkens et al., 2007). diversification in the region has been linked to the substantial changes in the landscape in the

for a slow decrease in shifts entering the neotropics (which, if confirmed, could also be linked to the global temperature decline at eocene/oligocene transition). reliability of results: pushing the limits of biological data evolutionary biology and biogeography are now experiencing a tremendous accumulation of data, including molecular sequences, fossils, and species occurrences, with a hitherto unrealized scientific potential. an emerging question, however, is to what extent available data and methods are sufficient to provide us with reliable answers to some of the most fundamental questions in biology. a critical evaluation of the data, methods and assumptions is therefore crucial but often

continental scale, lack of data is particularly critical for africa, southeast asia and australasia; but even within relatively well-sampled continents (such as europe and south america) there are strong regional biases among countries. similar to the case of fossil data, there is general skepticism concerning the use of publicly available species occurrences for understanding species distributions, especially from nonverified databases such as gbif. distribution data have been shown to contain important taxonomic, temporal and spatial biases (boakes et al., 2010). the question of whether bioinformatic tools may correctly infer biodiversity patterns despite those biases remains largely unanswered, and will also depend

data is particularly critical for africa, southeast asia and australasia; but even within relatively well-sampled continents (such as europe and south america) there are strong regional biases among countries. similar to the case of fossil data, there is general skepticism concerning the use of publicly available species occurrences for understanding species distributions, especially from nonverified databases such as gbif. distribution data have been shown to contain important taxonomic, temporal and spatial biases (boakes et al., 2010). the question of whether bioinformatic tools may correctly infer biodiversity patterns despite those biases remains largely unanswered, and will also depend on the scale and

recent studies suggest that automated data handling procedures are able to yield biologically realistic results, if enough care and appropriate techniques are employed (zanne et al., 2014; engemann et al., 2015; maldonado et al., accepted). in other cases, the manual validation by taxonomists appears crucial, e.g., for the assessment of species' conservation status for the iucn red list of threatened species (hjarding et al., 2014). our approach of automatically coding species into regions and calculating sampling fractions using gbif data and polygons is not intended to replace the time-consuming work by taxonomists. however, it constitutes an additional, dataderived and spatially explicit

to yield biologically realistic results, if enough care and appropriate techniques are employed (zanne et al., 2014; engemann et al., 2015; maldonado et al., accepted). in other cases, the manual validation by taxonomists appears crucial, e.g., for the assessment of species' conservation status for the iucn red list of threatened species (hjarding et al., 2014). our approach of automatically coding species into regions and calculating sampling fractions using gbif data and polygons is not intended to replace the time-consuming work by taxonomists. however, it constitutes an additional, dataderived and spatially explicit approach that deserves further exploration and validation. estimating global and

techniques are employed (zanne et al., 2014; engemann et al., 2015; maldonado et al., accepted). in other cases, the manual validation by taxonomists appears crucial, e.g., for the assessment of species' conservation status for the iucn red list of threatened species (hjarding et al., 2014). our approach of automatically coding species into regions and calculating sampling fractions using gbif data and polygons is not intended to replace the time-consuming work by taxonomists. however, it constitutes an additional, dataderived and spatially explicit approach that deserves further exploration and validation. estimating global and regional patterns of species richness and biodiversity remains a notoriously

2014). our approach of automatically coding species into regions and calculating sampling fractions using gbif data and polygons is not intended to replace the time-consuming work by taxonomists. however, it constitutes an additional, dataderived and spatially explicit approach that deserves further exploration and validation. estimating global and regional patterns of species richness and biodiversity remains a notoriously difficult and contentious topic, with no consensus reached (govaerts, 2001; crane, 2004; ungricht, 2004; wortley and scotland, 2004; chapman, 2009; mora et al., 2011). in addition, building up tropical biodiversity our phylogeny-based estimates of speciation and extinction rates (figure 6) showed that angiosperms in

neotropical angiosperms speciated on average about 2 - 2.5 times faster than angiosperms in tropical asia and tropical africa. however, they also went extinct about 2 - 2.5 times faster than in tropical asia. these high rates of speciation and extinction in the neotropics indicate a rapid evolutionary turnover, i.e., species being formed and replacing each other at an unparalleled rate. this result is also in accordance to the observation that south american plant diversity is characterized by a relatively large number of recent, species-rich radiations, for instance in the tropical andes (hughes and eastwood, 2006; drummond et al., 2012; madriñán

these high rates of speciation and extinction in the neotropics indicate a rapid evolutionary turnover, i.e., species being formed and replacing each other at an unparalleled rate. this result is also in accordance to the observation that south american plant diversity is characterized by a relatively large number of recent, species-rich radiations, for instance in the tropical andes (hughes and eastwood, 2006; drummond et al., 2012; madriñán et al., 2013) and amazonia frontiers in genetics | www.frontiersin.org 10 april 2015 | volume 6 | article 130 antonelli et al. biogeography and diversification of tropical angiosperms future prospects: more data, improved methods

agreement on how to best define, delimit and name biogeographical regions (kreft and jetz, 2010; holt et al., 2013; vilhena and antonelli, 2014), with the implication for this study that the world's three tropical regions are differently circumscribed in the literature. our study suggests that a relatively stable assignment of species to large regions (as in figure 2) may be attained through simple, automated filtering steps, in which the addition of increasingly restrictive criteria for coding species results in relatively small differences (figure 3). the reconstruction of ancestral character states (such as morphology and geographic distribution) along phylogenies is now common

for this study that the world's three tropical regions are differently circumscribed in the literature. our study suggests that a relatively stable assignment of species to large regions (as in figure 2) may be attained through simple, automated filtering steps, in which the addition of increasingly restrictive criteria for coding species results in relatively small differences (figure 3). the reconstruction of ancestral character states (such as morphology and geographic distribution) along phylogenies is now common practice in evolutionary studies, but only make sense when the traits analyzed are phylogenetically structured - i.e., they are not randomly distributed across the tree. since

3). the reconstruction of ancestral character states (such as morphology and geographic distribution) along phylogenies is now common practice in evolutionary studies, but only make sense when the traits analyzed are phylogenetically structured - i.e., they are not randomly distributed across the tree. since we found highly significant clustering of species pertaining to the same geographic assignment in each of the regions defined (figure 4), we consider that the geographic coding and reconstruction analyses using stochastic mapping are suitable for the goals of this study. the low taxonomic sampling in the phylogeny (tables 2, 3) may influence the calculation of range

among the regions outlined. this is because biological sampling is far from being random, with an over-representation of deep nodes that reflect morphological and geographical variations in taxa (hohna et al., 2011; ter steege et al., 2011; cusimano et al., 2012). in other words, even if only a couple of species were sampled from a speciesrich but strictly african clade, our analyses should be able to detect when that clade arrived in africa. further simulations would be helpful to assess at which sampling levels the calculation of continental-level range shifts stabilize and become fully reliable. diversification rates of angiosperms have varied

angiosperm phylogeny was at or below 10% for all tropical regions (figure 3, table 3). sampling levels already below c. 80% are bound to flaw diversification rate estimates under current methods, often showing slowdowns in net diversification that represent methodological artifacts (cusimano and renner, 2010). expectations on how the missing species are distributed in a phylogeny depending on the sampling scheme may increase the accuracy of diversification analyses (stadler and bokma, 2013). however, no method has been developed so far that is capable of confidently dealing with the level of taxonomic sampling observed in the angiosperm phylogeny we used. the musse

the inevitable incompleteness of the fossil record represents a limit to macro-evolutionary analyses that can be carried out using currently available data. however, the development of new methods has shown that even incomplete fossil data can provide essential information in estimating trends of phenotypic evolution (slater and harmon, 2013) and species diversification dynamics (silvestro et al., 2014). such models should be ideally extended to historical biogeography and might shed new light on the dynamics of migration of lineages through time and among regions. in particular, fossils provide an important resource for improving biogeographic reconstructions, as they provide information on past species

species diversification dynamics (silvestro et al., 2014). such models should be ideally extended to historical biogeography and might shed new light on the dynamics of migration of lineages through time and among regions. in particular, fossils provide an important resource for improving biogeographic reconstructions, as they provide information on past species ranges and may therefore further refine or validate ancestral range analyses as performed here (ronquist et al., 2012; wood et al., 2013; lawing and matzke, 2014). although correct fossil placement on phylogenies can be problematic, their potential in this area is still insufficiently explored (wood et al., 2013). phylogeny-based diversification

genetically), or methods currently used successfully with e.g., mammals (morlon et al., 2011; stadler, 2011) are adapted and validated for plants, we remain with limited power to assess the dynamics of diversification rates through time and across clades. conclusions here we have shown that currently available biological data - including species occurrences and dated phylogenetic trees - hold the potential of providing novel and important insights into large-scale patterns of species diversification and biogeography. the geographic history of angiosperms involved a large number of range transitions between tropical and non-tropical zones, as well as into and out of the world's three

plants, we remain with limited power to assess the dynamics of diversification rates through time and across clades. conclusions here we have shown that currently available biological data - including species occurrences and dated phylogenetic trees - hold the potential of providing novel and important insights into large-scale patterns of species diversification and biogeography. the geographic history of angiosperms involved a large number of range transitions between tropical and non-tropical zones, as well as into and out of the world's three tropical regions. global climatic changes and major geological events are likely to have influenced some of the observed changes in

constitute better models for explaining tropical angiosperm diversity. continental differences in tropical angiosperm diversity show clearer patterns, adding to our knowledge on the global patterns of plant diversity (kier et al., 2005; barthlott et al., 2007; kreft and jetz, 2007; kreft et al., 2010; mutke et al., 2011). the outstanding species richness of angiosperms found today in the neotropics as compared to tropical africa and tropical asia is associated with significantly higher speciation and extinction rates in the neotropics (figures 6c,d) - and thereby higher species turnover and shorter average longevity of species. the causes underlying these differences remain elusive, but

2007; kreft and jetz, 2007; kreft et al., 2010; mutke et al., 2011). the outstanding species richness of angiosperms found today in the neotropics as compared to tropical africa and tropical asia is associated with significantly higher speciation and extinction rates in the neotropics (figures 6c,d) - and thereby higher species turnover and shorter average longevity of species. the causes underlying these differences remain elusive, but might be associated with the substantial landscape dynamics that have affected northern south america since the miocene, among other continent-specific differences such as biome sizes, niche space, and climatic history. our results also show that

al., 2010; mutke et al., 2011). the outstanding species richness of angiosperms found today in the neotropics as compared to tropical africa and tropical asia is associated with significantly higher speciation and extinction rates in the neotropics (figures 6c,d) - and thereby higher species turnover and shorter average longevity of species. the causes underlying these differences remain elusive, but might be associated with the substantial landscape dynamics that have affected northern south america since the miocene, among other continent-specific differences such as biome sizes, niche space, and climatic history. our results also show that neotropical diversity, once generated in situ, was

### NA
2018 citation: bruy d, hattermann t, barrabé l, mouly a, barthélémy d and isnard s (2018) evolution of plant architecture, functional diversification and divergent evolution in the genus atractocarpus (rubiaceae) for new caledonia. front. plant sci. 9:1775. doi: 10.3389/fpls.2018.01775 the diversification of ecological roles and related adaptations in closely related species within a lineage is one of the most important processes linking plant evolution and ecology. plant architecture offers a robust framework to study these processes as it can highlight how plant structure influences plant diversification and ecological strategies. we investigated a case of gradual evolution of branching architecture in atractocarpus

index that assesses branching incidence on functional traits influencing both assimilation and exploration functions. we showed that architectural transitions correlate with ecologically important functional traits. variation in ecologically important traits among closely relatives, as supported by the architectural analysis, is suggestive of intense competition that favored divergence among locally coexisting species. we propose that pleistocene climatic fluctuations causing expansion and contraction of rainforest could also have offered ecological opportunities for colonizers in addition to the process of divergent evolution. keywords: branching index, convergence, corner's rules, gardenieae, island, rainforest, treelet, understory introduction convergent and divergent evolution are widely recognized as important drivers

al., 1978). in the context of this diversity of possible models to explain evolution in plant architecture, detailed studies within individual clades that combine results from molecular phylogenetic work with detailed architectural analysis may be particularly informative. atractocarpus schltr. & k. krause (gardenieae, rubiaceae), a pacific genus of about 40 species, of which 32 are endemics to new caledonia (mouly et al., under review), is ideally suited to study the evolution of plant architecture and to test hypotheses regarding the role of architecture in plant ecological diversification. especially since a recently published phylogeny based on comprehensive sampling is available for the

sampling is available for the genus (mouly et al., under review). most members of this genus are treelets occurring in the understory of tropical and subtropical rainforests, and they exhibit substantial variation in branching pattern (tirvengadum and sastre, 1979; robbrecht and puff, 1986), ranging from monocaulous (i.e., unbranched) to well-branched species, including various intermediate states (figure 1). the new caledonia members of atractocarpus, which originated from a single colonization event that occurred ca. 2.4 myr ago (mouly et al., under review), has the highest diversification rate of any genus studied to date (1.17 species species−1 myr−1 ), approaching that of the

ranging from monocaulous (i.e., unbranched) to well-branched species, including various intermediate states (figure 1). the new caledonia members of atractocarpus, which originated from a single colonization event that occurred ca. 2.4 myr ago (mouly et al., under review), has the highest diversification rate of any genus studied to date (1.17 species species−1 myr−1 ), approaching that of the well-known hawaiian lobelioid genus cyanea on the island of maui (1.36 species species−1 myr−1 , givnish et al., 2009). the presence of a high diversification rate combined with architectural diversity is thought to provide an indication of ecological diversification, a situation that is

atractocarpus, which originated from a single colonization event that occurred ca. 2.4 myr ago (mouly et al., under review), has the highest diversification rate of any genus studied to date (1.17 species species−1 myr−1 ), approaching that of the well-known hawaiian lobelioid genus cyanea on the island of maui (1.36 species species−1 myr−1 , givnish et al., 2009). the presence of a high diversification rate combined with architectural diversity is thought to provide an indication of ecological diversification, a situation that is often associated with island colonization (silvertown, 2004; silvertown et al., 2005). in groups adapted to growing in the forest

(silvertown, 2004; silvertown et al., 2005). in groups adapted to growing in the forest understory, light is a variable and limiting resource, and in response, plants have deployed various growth strategies related to leaf photosynthesis and the economization of carbon. to quantify the architectural gradient from monocaulous (unbranched) to well-branched species, we developed a new \"functional branching index,\" mechanisms in lianas (sousa-baena et al., 2014, 2018), and succulence in arid environments (ogburn and edwards, 2010; arakaki et al., 2011). these and others examples demonstrate that similar habitats or micro-habitats can produce selective pressures that favor some morphological and physiological traits. another

and succulence in arid environments (ogburn and edwards, 2010; arakaki et al., 2011). these and others examples demonstrate that similar habitats or micro-habitats can produce selective pressures that favor some morphological and physiological traits. another evolutionary process, sometimes linked with convergence, is the diversification of ecological roles among closely related species, i.e., divergent evolution, which results in the exploitation of different ecological resources (e.g., bramwell, 1975; givnish et al., 2009, 2014). convergent and divergent evolution have been attributed to morpho-physiological traits, whose gradual evolution or rapid innovation can lead to adaptive radiation, sometimes involving increased diversification rates (givnish and sytsma, 1997;

evolutionary processes underlying plant evolution (bateman, 1994, 1999; sussex and kerk, 2001; meyerberthaud et al., 2010). architectural studies have shown that plants are modular organisms comprising elements that can differ in their organization and functions (hallé et al., 1978; barthélémy and caraglio, 2007). for instance, in many tree and treelet species, the trunk functions primarily for exploration and to provide support, while branches are involved in assimilation and reproduction. plant architecture thus influences spatial and temporal exploitation of resources (smith et al., 2014). some architectural traits have been shown to impact plant fitness, either directly (küppers, 1989; millet et al., 1999;

charlesdominique et al., 2017) or leaf-to-stem relation that trigger flowering (\"index of axialisation,\" lauri, 1988; lauri and kelner, 2001). frontiers in plant science | www.frontiersin.org 2 december 2018 | volume 9 | article 1775 bruy et al. evolution of plant architecture in atractocarpus figure 1 | photographs of different atractocarpus species in their environment showing variability in growth habit and lateral axis. monocaulous species: (a) a. confertus, (b) a. bracteatus, (c) a. bracteatus. intermediate species: (d) a. ngoyensis, (e) a. ngoyensis, (f) a. ngoyensis. branched species: (g) a. pseudoterminalis, (h) a. sp. nov. 10, (i) a. pseudoterminalis. materials and methods which

lauri, 1988; lauri and kelner, 2001). frontiers in plant science | www.frontiersin.org 2 december 2018 | volume 9 | article 1775 bruy et al. evolution of plant architecture in atractocarpus figure 1 | photographs of different atractocarpus species in their environment showing variability in growth habit and lateral axis. monocaulous species: (a) a. confertus, (b) a. bracteatus, (c) a. bracteatus. intermediate species: (d) a. ngoyensis, (e) a. ngoyensis, (f) a. ngoyensis. branched species: (g) a. pseudoterminalis, (h) a. sp. nov. 10, (i) a. pseudoterminalis. materials and methods which assesses branching incidence on functional traits that influence assimilation and exploration functions. considering

www.frontiersin.org 2 december 2018 | volume 9 | article 1775 bruy et al. evolution of plant architecture in atractocarpus figure 1 | photographs of different atractocarpus species in their environment showing variability in growth habit and lateral axis. monocaulous species: (a) a. confertus, (b) a. bracteatus, (c) a. bracteatus. intermediate species: (d) a. ngoyensis, (e) a. ngoyensis, (f) a. ngoyensis. branched species: (g) a. pseudoterminalis, (h) a. sp. nov. 10, (i) a. pseudoterminalis. materials and methods which assesses branching incidence on functional traits that influence assimilation and exploration functions. considering each understory species of atractocarpus in new caledonia, we analyzed correlations

et al. evolution of plant architecture in atractocarpus figure 1 | photographs of different atractocarpus species in their environment showing variability in growth habit and lateral axis. monocaulous species: (a) a. confertus, (b) a. bracteatus, (c) a. bracteatus. intermediate species: (d) a. ngoyensis, (e) a. ngoyensis, (f) a. ngoyensis. branched species: (g) a. pseudoterminalis, (h) a. sp. nov. 10, (i) a. pseudoterminalis. materials and methods which assesses branching incidence on functional traits that influence assimilation and exploration functions. considering each understory species of atractocarpus in new caledonia, we analyzed correlations between architecture and traits associated with key ecologically significant functions (viz.,

confertus, (b) a. bracteatus, (c) a. bracteatus. intermediate species: (d) a. ngoyensis, (e) a. ngoyensis, (f) a. ngoyensis. branched species: (g) a. pseudoterminalis, (h) a. sp. nov. 10, (i) a. pseudoterminalis. materials and methods which assesses branching incidence on functional traits that influence assimilation and exploration functions. considering each understory species of atractocarpus in new caledonia, we analyzed correlations between architecture and traits associated with key ecologically significant functions (viz., photosynthesis, hydraulic, mechanics, and dispersal). we investigated the evolution of plant architecture (branched vs. unbranched) in forest understory species and its impact on the exceptional diversification rate of the genus. finally,

on functional traits that influence assimilation and exploration functions. considering each understory species of atractocarpus in new caledonia, we analyzed correlations between architecture and traits associated with key ecologically significant functions (viz., photosynthesis, hydraulic, mechanics, and dispersal). we investigated the evolution of plant architecture (branched vs. unbranched) in forest understory species and its impact on the exceptional diversification rate of the genus. finally, we explored how ecological opportunities might stimulate the diversification of architecture through spatial differentiation in resource use (niche partitioning) in closely related species following the colonization of new caledonian rainforests. frontiers in plant science | www.frontiersin.org sampling we

and dispersal). we investigated the evolution of plant architecture (branched vs. unbranched) in forest understory species and its impact on the exceptional diversification rate of the genus. finally, we explored how ecological opportunities might stimulate the diversification of architecture through spatial differentiation in resource use (niche partitioning) in closely related species following the colonization of new caledonian rainforests. frontiers in plant science | www.frontiersin.org sampling we sampled all the 27 known rainforest species of atractocarpus occurring in new caledonia (list of taxa in appendix 1 in supplementary material). for ancestral character estimation (ace), three australian atractocarpus species and one species in

diversification rate of the genus. finally, we explored how ecological opportunities might stimulate the diversification of architecture through spatial differentiation in resource use (niche partitioning) in closely related species following the colonization of new caledonian rainforests. frontiers in plant science | www.frontiersin.org sampling we sampled all the 27 known rainforest species of atractocarpus occurring in new caledonia (list of taxa in appendix 1 in supplementary material). for ancestral character estimation (ace), three australian atractocarpus species and one species in each of the most closely related genera for which dna sequences were available (mouly et al., 2014, under review) were included as

partitioning) in closely related species following the colonization of new caledonian rainforests. frontiers in plant science | www.frontiersin.org sampling we sampled all the 27 known rainforest species of atractocarpus occurring in new caledonia (list of taxa in appendix 1 in supplementary material). for ancestral character estimation (ace), three australian atractocarpus species and one species in each of the most closely related genera for which dna sequences were available (mouly et al., 2014, under review) were included as outgroups. individuals were sampled in 20 rainforest sites occurring on grande terre, new caledonia's main island (appendix 1 in supplementary material). 3 december 2018

related species following the colonization of new caledonian rainforests. frontiers in plant science | www.frontiersin.org sampling we sampled all the 27 known rainforest species of atractocarpus occurring in new caledonia (list of taxa in appendix 1 in supplementary material). for ancestral character estimation (ace), three australian atractocarpus species and one species in each of the most closely related genera for which dna sequences were available (mouly et al., 2014, under review) were included as outgroups. individuals were sampled in 20 rainforest sites occurring on grande terre, new caledonia's main island (appendix 1 in supplementary material). 3 december 2018 | volume 9

architecture in atractocarpus branching index and plant architectural traits 12 due to small population sizes and habitat degradation, so these taxa were removed from this analysis. we concentrated our sampling in the apical part of the trunk because (i) this enabled standardization of physiological age and (ii) unbranched and branched species can be compared only on the basis of the main stem. moreover, variation in resource allocation and anatomy between unbranched and branched species might be more prevalent toward the apex, where large leaves are deployed (carlquist, 1974). stem and leaf measurements were standardized as follows. leaf traits (sla, leaf area)

from this analysis. we concentrated our sampling in the apical part of the trunk because (i) this enabled standardization of physiological age and (ii) unbranched and branched species can be compared only on the basis of the main stem. moreover, variation in resource allocation and anatomy between unbranched and branched species might be more prevalent toward the apex, where large leaves are deployed (carlquist, 1974). stem and leaf measurements were standardized as follows. leaf traits (sla, leaf area) were measured on the five youngest, fully expended leaves of the main axis (trunk). wood was collected below the terminal leaf tuft for

two dimensions are used to approximate fruit volume (table 1). a total of 592 fruits were measured, and only the five largest fruits per taxon (four for a. confertus and a. sessilifolius) were used to ensure that values from only mature fruits were included in our analyses. one of the species, a. sp. nov. 6, is a very rare species whose fruits have never been observed; it was consequently excluded from the fruit volume dataset. based on the observation that an observed architectural gradient might result from differential allocation in branch length and supported leaf area (figure 1), we used two

1). a total of 592 fruits were measured, and only the five largest fruits per taxon (four for a. confertus and a. sessilifolius) were used to ensure that values from only mature fruits were included in our analyses. one of the species, a. sp. nov. 6, is a very rare species whose fruits have never been observed; it was consequently excluded from the fruit volume dataset. based on the observation that an observed architectural gradient might result from differential allocation in branch length and supported leaf area (figure 1), we used two functional branching indexes based on the differentiation of function

the greater the functional role played by branches. we built a single integrative branching index that expresses the architectural gradient by combining these two variables using principal component analysis (correlation between photosynthetic and exploration branching indexes: rho = 0.86) employed in the ade4 package for r (dray and dufour, 2007). species were segregated into three architectural classes (monocaulous, intermediate, and branched) using wilcoxon signed-rank tests that enable detection of breaks in the distribution of the integrative branching index. using morphological criteria reviewed by barthélémy and caraglio (2007), we then described main traits commonly used in architectural analyses (growth process, branching pattern,

using wilcoxon signed-rank tests that enable detection of breaks in the distribution of the integrative branching index. using morphological criteria reviewed by barthélémy and caraglio (2007), we then described main traits commonly used in architectural analyses (growth process, branching pattern, position of reproductive structures, etc.) for the 27 new caledonian species of atractocarpus. the number of individuals studied varied from more than a hundred in some species (e.g., a. pseudoterminalis, a. ngoyensis and a. bracteatus) to 12 in a. sezitat, a rare species whose habitat is highly disturbed by introduced herbivores. all species were studied at different ontogenetic stages (from very

index. using morphological criteria reviewed by barthélémy and caraglio (2007), we then described main traits commonly used in architectural analyses (growth process, branching pattern, position of reproductive structures, etc.) for the 27 new caledonian species of atractocarpus. the number of individuals studied varied from more than a hundred in some species (e.g., a. pseudoterminalis, a. ngoyensis and a. bracteatus) to 12 in a. sezitat, a rare species whose habitat is highly disturbed by introduced herbivores. all species were studied at different ontogenetic stages (from very young plants to senescent adults) except a. sezitat and a. sp. nov. 12, for which no

used in architectural analyses (growth process, branching pattern, position of reproductive structures, etc.) for the 27 new caledonian species of atractocarpus. the number of individuals studied varied from more than a hundred in some species (e.g., a. pseudoterminalis, a. ngoyensis and a. bracteatus) to 12 in a. sezitat, a rare species whose habitat is highly disturbed by introduced herbivores. all species were studied at different ontogenetic stages (from very young plants to senescent adults) except a. sezitat and a. sp. nov. 12, for which no young plants could be found. data analysis phylogenetic tree the molecular phylogeny of mouly et al.

reproductive structures, etc.) for the 27 new caledonian species of atractocarpus. the number of individuals studied varied from more than a hundred in some species (e.g., a. pseudoterminalis, a. ngoyensis and a. bracteatus) to 12 in a. sezitat, a rare species whose habitat is highly disturbed by introduced herbivores. all species were studied at different ontogenetic stages (from very young plants to senescent adults) except a. sezitat and a. sp. nov. 12, for which no young plants could be found. data analysis phylogenetic tree the molecular phylogeny of mouly et al. (under review) was used for analysis. this ultrametric phylogenetic tree

which no young plants could be found. data analysis phylogenetic tree the molecular phylogeny of mouly et al. (under review) was used for analysis. this ultrametric phylogenetic tree was constructed under bayesian inference using one nuclear (its) and two chloroplastic (trntl, rpl32) loci. since the current study focused on rainforest species, we pruned the five dry forest and maquis species using the r software (ver. 3.4.3). phylogenetic signal we assessed phylogenetic independence of measured traits (felsenstein, 1985) using two complementary statistics calculated under a brownian motion (bm) model of evolution. pagel's lambda (pagel, 1999; freckleton et al., 2002) is widely used

phylogenetic tree the molecular phylogeny of mouly et al. (under review) was used for analysis. this ultrametric phylogenetic tree was constructed under bayesian inference using one nuclear (its) and two chloroplastic (trntl, rpl32) loci. since the current study focused on rainforest species, we pruned the five dry forest and maquis species using the r software (ver. 3.4.3). phylogenetic signal we assessed phylogenetic independence of measured traits (felsenstein, 1985) using two complementary statistics calculated under a brownian motion (bm) model of evolution. pagel's lambda (pagel, 1999; freckleton et al., 2002) is widely used for low rates of type i error and robustness,

2013). both statistics were calculated considering intraspecific variation using the phytools package for r (revell, 2012). plant functional traits we tested the correlation of 14 traits with branching index. the traits were selected to reflect important features of plant ecological strategies (table 1). they were measured on five individuals per species and, when possible (e.g., for leaf and internode traits), five times per individual. the selection of individuals was standardized for environment (rainforest understory) and for ontogeny following three criteria: (i) recently mature individuals, (ii) non reiterated individuals (see oldeman, 1974; barthélémy and caraglio, 2007), and (iii) nontraumatized trunks. it was

was not possible to locate individuals meeting all three conditions for a. sezitat and a. sp. nov. frontiers in plant science | www.frontiersin.org 4 december 2018 | volume 9 | article 1775 bruy et al. evolution of plant architecture in atractocarpus table 1 | functional traits measured for 25 atractocarpus species. trait abbreviation unit formula related function", 
"s sr plant height plant basal diameter geometric feature of buckling resistance almeras et al., 2004; niklas et al., 2006; homeier et al., 2010 wright et al., 2007; olson et al., 2009; pérez-harguindeguy et al., 2013 trunk traits slenderness ratio ø specific wood density

1983a cornelissen, 1999; cornelissen et al., 2003", 
"s are given to justify the interest of each trait for corresponding functions. in, internode; pom, point of measurement; lf, leaf; br, branch; tr, trunk. genus, ancestral character estimation (ace) was performed. we assigned an architectural class to each of the 11 outgroup species based on published descriptions (fosberg, 1987; smith and darwin, 1988; fosberg et al., 1993; puttock, 1999; wong, 2004; zahid and wong, 2004, 2010; tong et al., 2013), herbaria specimens (p, k, bm, e) and available photos. the ace were performed using a maximum likelihood method under the ape package for

the explanatory variables were all other functional traits (table 1). significance of relations was tested using a likelihood ratio test between the model and a model constrained with a slope of zero. to characterize the functional space of each architectural classes, we performed a principal component analysis. we used the species arithmetical mean of each functional traits (branching index excluded), and functional differences between architectural classes (monocaulous, intermediate, branched) were tested with a permanova (vegan package for r, oksanen et al., 2018). trait based diversification to test whether one of the architectural classes has contributed more than the others to the

of atractocarpus and infer the evolution of architecture in the frontiers in plant science | www.frontiersin.org 5 december 2018 | volume 9 | article 1775 bruy et al. evolution of plant architecture in atractocarpus for this analysis, outgroups were dropped from the phylogeny to consider only the 27 rainforest atractocarpus species. fifteen models of diversification were used, each differing in whether or not of speciation, extinction and transition rates were equal between classes (table 2). model selection was done according to aicc. the branching position of c1, which was continuous for some monocaulous species while diffuse for others. two main quantitative

to consider only the 27 rainforest atractocarpus species. fifteen models of diversification were used, each differing in whether or not of speciation, extinction and transition rates were equal between classes (table 2). model selection was done according to aicc. the branching position of c1, which was continuous for some monocaulous species while diffuse for others. two main quantitative architectural traits varied greatly among species, the number of modules per branch and the length of modules, which decreased in monocaulous species (table 3, figures 3, 4). for example, a. longistipitatus is a monocaulous species whose c2 comprise many very short modules (apart

used, each differing in whether or not of speciation, extinction and transition rates were equal between classes (table 2). model selection was done according to aicc. the branching position of c1, which was continuous for some monocaulous species while diffuse for others. two main quantitative architectural traits varied greatly among species, the number of modules per branch and the length of modules, which decreased in monocaulous species (table 3, figures 3, 4). for example, a. longistipitatus is a monocaulous species whose c2 comprise many very short modules (apart from the first one). by contrast, a. bracteatus is also monocaulous but its

classes (table 2). model selection was done according to aicc. the branching position of c1, which was continuous for some monocaulous species while diffuse for others. two main quantitative architectural traits varied greatly among species, the number of modules per branch and the length of modules, which decreased in monocaulous species (table 3, figures 3, 4). for example, a. longistipitatus is a monocaulous species whose c2 comprise many very short modules (apart from the first one). by contrast, a. bracteatus is also monocaulous but its c2 comprise a single long module (figure 4). the number of internodes per module vary little

of c1, which was continuous for some monocaulous species while diffuse for others. two main quantitative architectural traits varied greatly among species, the number of modules per branch and the length of modules, which decreased in monocaulous species (table 3, figures 3, 4). for example, a. longistipitatus is a monocaulous species whose c2 comprise many very short modules (apart from the first one). by contrast, a. bracteatus is also monocaulous but its c2 comprise a single long module (figure 4). the number of internodes per module vary little in the genus but tend to be reduced in monocaulous species, with an

a monocaulous species whose c2 comprise many very short modules (apart from the first one). by contrast, a. bracteatus is also monocaulous but its c2 comprise a single long module (figure 4). the number of internodes per module vary little in the genus but tend to be reduced in monocaulous species, with an extreme case of one node per module in a. confertus (table 3). all species showed the ability to develop delayed reiterate on damaged or aged individuals. results branching index branching indexes confirmed the existence of a morphofunctional gradient from branched to monocaulous species of atractocarpus, for both exploration

contrast, a. bracteatus is also monocaulous but its c2 comprise a single long module (figure 4). the number of internodes per module vary little in the genus but tend to be reduced in monocaulous species, with an extreme case of one node per module in a. confertus (table 3). all species showed the ability to develop delayed reiterate on damaged or aged individuals. results branching index branching indexes confirmed the existence of a morphofunctional gradient from branched to monocaulous species of atractocarpus, for both exploration and photosynthetic functions (figures 2a,b). for both indexes, interspecific variation was too gradual to allow the

to be reduced in monocaulous species, with an extreme case of one node per module in a. confertus (table 3). all species showed the ability to develop delayed reiterate on damaged or aged individuals. results branching index branching indexes confirmed the existence of a morphofunctional gradient from branched to monocaulous species of atractocarpus, for both exploration and photosynthetic functions (figures 2a,b). for both indexes, interspecific variation was too gradual to allow the partitioning in distinct classes, but when combined (i.e., using the branching index), three distinct architectural groups could be differentiated (wilcoxon tests: p < 0.1, figure 2c), viz. the branched,

(figures 2a,b). for both indexes, interspecific variation was too gradual to allow the partitioning in distinct classes, but when combined (i.e., using the branching index), three distinct architectural groups could be differentiated (wilcoxon tests: p < 0.1, figure 2c), viz. the branched, intermediate and monocaulous architectural classes. on average, branched species had 82% of photosynthesis and 76% of exploration provided by branches, vs. 36 and 45%, respectively, for intermediate species, and 3 and 16% for monocaulous species. functional characterization phylogenetic signal varied substantially depending on which statistic was used (lambda or kappa, table 4). this was probably due to the small

combined (i.e., using the branching index), three distinct architectural groups could be differentiated (wilcoxon tests: p < 0.1, figure 2c), viz. the branched, intermediate and monocaulous architectural classes. on average, branched species had 82% of photosynthesis and 76% of exploration provided by branches, vs. 36 and 45%, respectively, for intermediate species, and 3 and 16% for monocaulous species. functional characterization phylogenetic signal varied substantially depending on which statistic was used (lambda or kappa, table 4). this was probably due to the small size of our phylogeny (25 terminals), since the statistics differ in sensitivity to the number of terminals (freckleton et

distinct architectural groups could be differentiated (wilcoxon tests: p < 0.1, figure 2c), viz. the branched, intermediate and monocaulous architectural classes. on average, branched species had 82% of photosynthesis and 76% of exploration provided by branches, vs. 36 and 45%, respectively, for intermediate species, and 3 and 16% for monocaulous species. functional characterization phylogenetic signal varied substantially depending on which statistic was used (lambda or kappa, table 4). this was probably due to the small size of our phylogeny (25 terminals), since the statistics differ in sensitivity to the number of terminals (freckleton et al., 2002; kamilar and cooper, 2013). blomberg's

diameter) as well as photosynthesis (e.g., leaf area, sla), hydraulics (e.g., huber value), and even dispersal (fruit volume). these correlations between architecture and functions were confirmed by pca and permanova (p = 0.016), which showed significant differences between the functional space occupied by the three architectural classes (figure 5). intermediate species were confounded between the two other architectural classes, but monocaulous and branched species appeared to be functionally very different. in comparison to monocaulous species, branched taxa tend to have numerous small leaves, higher sla, smaller fruits and, smaller internode diameters with a lower proportion of wood, but a higher wood

and even dispersal (fruit volume). these correlations between architecture and functions were confirmed by pca and permanova (p = 0.016), which showed significant differences between the functional space occupied by the three architectural classes (figure 5). intermediate species were confounded between the two other architectural classes, but monocaulous and branched species appeared to be functionally very different. in comparison to monocaulous species, branched taxa tend to have numerous small leaves, higher sla, smaller fruits and, smaller internode diameters with a lower proportion of wood, but a higher wood area-leaf area ratio (huber value). the monocaulous a. confertus (the lowest point on

were confirmed by pca and permanova (p = 0.016), which showed significant differences between the functional space occupied by the three architectural classes (figure 5). intermediate species were confounded between the two other architectural classes, but monocaulous and branched species appeared to be functionally very different. in comparison to monocaulous species, branched taxa tend to have numerous small leaves, higher sla, smaller fruits and, smaller internode diameters with a lower proportion of wood, but a higher wood area-leaf area ratio (huber value). the monocaulous a. confertus (the lowest point on figure 5) was distinct from the other atractocarpus species examined in

to monocaulous species, branched taxa tend to have numerous small leaves, higher sla, smaller fruits and, smaller internode diameters with a lower proportion of wood, but a higher wood area-leaf area ratio (huber value). the monocaulous a. confertus (the lowest point on figure 5) was distinct from the other atractocarpus species examined in having numerous very large leaves, leading to a disproportionately higher total leaf area (up to 2.4 m² vs. a mean of 0.5 m² for the other species). architectural characterization the architecture of new caledonian atractocarpus species involves two axis categories: (i) a trunk (c1), i.e., an orthotropic monopodium

area-leaf area ratio (huber value). the monocaulous a. confertus (the lowest point on figure 5) was distinct from the other atractocarpus species examined in having numerous very large leaves, leading to a disproportionately higher total leaf area (up to 2.4 m² vs. a mean of 0.5 m² for the other species). architectural characterization the architecture of new caledonian atractocarpus species involves two axis categories: (i) a trunk (c1), i.e., an orthotropic monopodium with continuous indeterminate growth, and (ii) \"branches\" (c2), i.e., orthotropic sympodia with terminal sexuality conferring rhythmic growth (table 3, figure 3). the sympodial branching of c2 always originates in

(the lowest point on figure 5) was distinct from the other atractocarpus species examined in having numerous very large leaves, leading to a disproportionately higher total leaf area (up to 2.4 m² vs. a mean of 0.5 m² for the other species). architectural characterization the architecture of new caledonian atractocarpus species involves two axis categories: (i) a trunk (c1), i.e., an orthotropic monopodium with continuous indeterminate growth, and (ii) \"branches\" (c2), i.e., orthotropic sympodia with terminal sexuality conferring rhythmic growth (table 3, figure 3). the sympodial branching of c2 always originates in a hypotonic (i.e., on the lower surface) or amphitonic

growth, and (ii) \"branches\" (c2), i.e., orthotropic sympodia with terminal sexuality conferring rhythmic growth (table 3, figure 3). the sympodial branching of c2 always originates in a hypotonic (i.e., on the lower surface) or amphitonic position, resulting in upward flower exposure (figure 1). the only qualitative architectural variation observed between species concerned table 2 | models fitted for trait based diversification analysis. model 1 2 3 4 5 6 7 8a 8b 8c 9a 9b 9c 10 11 speciation rates extinction rates transition rates 6= 6= 6= −46.72 = = = −50.59 6= = = −49.59 6= 6= = −49.56 6=

i; b 6= 0 = = −49.59 m = i; b 6= m = b; i 6= i = b; m 6= m = b; i 6= 0 0 aicc = −50.58 = −50.50 0 = −49.59 = = −50.59 6= 0 = −49.59 ancestral character estimation all considered outgroup species belonged to the branched architectural class, except for the fijian sukunia pentagonoides (seem.) a.c.sm, which is monocaulous. among the three possible models of evolution that fitted the data, the best supported one was that of equal transition rates between classes (er), with an aicc of 84.4 vs. 96.1 and 104.2

branched. frontiers in plant science | www.frontiersin.org 6 december 2018 | volume 9 | article 1775 bruy et al. evolution of plant architecture in atractocarpus figure 2 | branching indexes calculated on the base of (a) exploration function, (b) photosynthetic function and (c) the combination of both, for 25 atractocarpus species. letters in (c) correspond to the result of the wilcoxon test; species with shared letters are not significantly different for a given risk of error. trait based diversification as \"monocaulous clade a\" and \"monocaulous clade b\" in figure 6), ca. 1.4 and 0.9 - 0.6 myr ago respectively. in each

9 | article 1775 bruy et al. evolution of plant architecture in atractocarpus figure 2 | branching indexes calculated on the base of (a) exploration function, (b) photosynthetic function and (c) the combination of both, for 25 atractocarpus species. letters in (c) correspond to the result of the wilcoxon test; species with shared letters are not significantly different for a given risk of error. trait based diversification as \"monocaulous clade a\" and \"monocaulous clade b\" in figure 6), ca. 1.4 and 0.9 - 0.6 myr ago respectively. in each clade, species with an intermediate architecture are closely related to monocaulous species,

in (c) correspond to the result of the wilcoxon test; species with shared letters are not significantly different for a given risk of error. trait based diversification as \"monocaulous clade a\" and \"monocaulous clade b\" in figure 6), ca. 1.4 and 0.9 - 0.6 myr ago respectively. in each clade, species with an intermediate architecture are closely related to monocaulous species, the only exception being a. brandzeanus, whose closest relatives are branched. conversely, pairs of sister or closely related species never showed branched and monocaulous habit. frontiers in plant science | www.frontiersin.org following the iacc, models 2 and 10 are the

species with shared letters are not significantly different for a given risk of error. trait based diversification as \"monocaulous clade a\" and \"monocaulous clade b\" in figure 6), ca. 1.4 and 0.9 - 0.6 myr ago respectively. in each clade, species with an intermediate architecture are closely related to monocaulous species, the only exception being a. brandzeanus, whose closest relatives are branched. conversely, pairs of sister or closely related species never showed branched and monocaulous habit. frontiers in plant science | www.frontiersin.org following the iacc, models 2 and 10 are the best fitted, in which speciation and transition rates are equal

clade a\" and \"monocaulous clade b\" in figure 6), ca. 1.4 and 0.9 - 0.6 myr ago respectively. in each clade, species with an intermediate architecture are closely related to monocaulous species, the only exception being a. brandzeanus, whose closest relatives are branched. conversely, pairs of sister or closely related species never showed branched and monocaulous habit. frontiers in plant science | www.frontiersin.org following the iacc, models 2 and 10 are the best fitted, in which speciation and transition rates are equal for each state and extinction rates are equal or null (table 2). this means that diversification was not greater

sp2 growth pattern 2 (-3) a. aragoensis 2 a. ngoyensis intermediate a. brandzeanus 2 a. sp7 branched a. nigricans ac, axis category; c1, first axis category (trunk); c2, second axis category (\"branches\"). c2 c1 number of ac architectural class taxon a. sp6 table 3 | architectural traits of 27 atractocarpus species. a. conferta 1.7 1 1 bruy et al. evolution of plant architecture in atractocarpus december 2018 | volume 9 | article 1775 bruy et al. evolution of plant architecture in atractocarpus figure 3 | schematic representation of the three main architectural classes found in the rainforest understory species of atractocarpus:

27 atractocarpus species. a. conferta 1.7 1 1 bruy et al. evolution of plant architecture in atractocarpus december 2018 | volume 9 | article 1775 bruy et al. evolution of plant architecture in atractocarpus figure 3 | schematic representation of the three main architectural classes found in the rainforest understory species of atractocarpus: (a) monocaulous, (b) intermediate (c) branched. figure 4 | photographs of branches with their axillary leaves (i.e., from trunk) for height atractocarpus species with different branching degrees. arrows indicate apical death, i.e., flowering sites (for a. longistipitatus (e), only half of apical death has been represented). branched species:

bruy et al. evolution of plant architecture in atractocarpus figure 3 | schematic representation of the three main architectural classes found in the rainforest understory species of atractocarpus: (a) monocaulous, (b) intermediate (c) branched. figure 4 | photographs of branches with their axillary leaves (i.e., from trunk) for height atractocarpus species with different branching degrees. arrows indicate apical death, i.e., flowering sites (for a. longistipitatus (e), only half of apical death has been represented). branched species: (a) a. pseudoterminalis, (b) a. sp4. intermediate species: (c) a. ngoyensis, (d) a. brandzeanus. monocaulous species: (e) a. longistipitatus, (f) a. pterocarpon, (g) a. bracteatus,

species of atractocarpus: (a) monocaulous, (b) intermediate (c) branched. figure 4 | photographs of branches with their axillary leaves (i.e., from trunk) for height atractocarpus species with different branching degrees. arrows indicate apical death, i.e., flowering sites (for a. longistipitatus (e), only half of apical death has been represented). branched species: (a) a. pseudoterminalis, (b) a. sp4. intermediate species: (c) a. ngoyensis, (d) a. brandzeanus. monocaulous species: (e) a. longistipitatus, (f) a. pterocarpon, (g) a. bracteatus, (h) a. confertus. discussion the measurement of functionally explicit variables (branch length and leaf area) that link plant architecture and function (lauri, 1988; lauri and

branched. figure 4 | photographs of branches with their axillary leaves (i.e., from trunk) for height atractocarpus species with different branching degrees. arrows indicate apical death, i.e., flowering sites (for a. longistipitatus (e), only half of apical death has been represented). branched species: (a) a. pseudoterminalis, (b) a. sp4. intermediate species: (c) a. ngoyensis, (d) a. brandzeanus. monocaulous species: (e) a. longistipitatus, (f) a. pterocarpon, (g) a. bracteatus, (h) a. confertus. discussion the measurement of functionally explicit variables (branch length and leaf area) that link plant architecture and function (lauri, 1988; lauri and kelner, 2001; charles-dominique et al., 2017) showed that

their axillary leaves (i.e., from trunk) for height atractocarpus species with different branching degrees. arrows indicate apical death, i.e., flowering sites (for a. longistipitatus (e), only half of apical death has been represented). branched species: (a) a. pseudoterminalis, (b) a. sp4. intermediate species: (c) a. ngoyensis, (d) a. brandzeanus. monocaulous species: (e) a. longistipitatus, (f) a. pterocarpon, (g) a. bracteatus, (h) a. confertus. discussion the measurement of functionally explicit variables (branch length and leaf area) that link plant architecture and function (lauri, 1988; lauri and kelner, 2001; charles-dominique et al., 2017) showed that variation of growth habit in atractocarpus is correlated

(g) a. bracteatus, (h) a. confertus. discussion the measurement of functionally explicit variables (branch length and leaf area) that link plant architecture and function (lauri, 1988; lauri and kelner, 2001; charles-dominique et al., 2017) showed that variation of growth habit in atractocarpus is correlated with a gradual variation from branched species, for which photosynthesis and exploration are largely assumed by branches, toward monocaulous species, for which these functions are assumed by the trunk. the reduced c2 in monocaulous species supports a limited number of small leaves (on average 4 leaves of 9 cm2 ) and are dedicated almost evolution of plant

(branch length and leaf area) that link plant architecture and function (lauri, 1988; lauri and kelner, 2001; charles-dominique et al., 2017) showed that variation of growth habit in atractocarpus is correlated with a gradual variation from branched species, for which photosynthesis and exploration are largely assumed by branches, toward monocaulous species, for which these functions are assumed by the trunk. the reduced c2 in monocaulous species supports a limited number of small leaves (on average 4 leaves of 9 cm2 ) and are dedicated almost evolution of plant architecture despite the diversity of growth habits in the genus atractocarpus, ranging from

kelner, 2001; charles-dominique et al., 2017) showed that variation of growth habit in atractocarpus is correlated with a gradual variation from branched species, for which photosynthesis and exploration are largely assumed by branches, toward monocaulous species, for which these functions are assumed by the trunk. the reduced c2 in monocaulous species supports a limited number of small leaves (on average 4 leaves of 9 cm2 ) and are dedicated almost evolution of plant architecture despite the diversity of growth habits in the genus atractocarpus, ranging from well branched to monocaulous species, the architectural construction of members of the group is fairly

are assumed by the trunk. the reduced c2 in monocaulous species supports a limited number of small leaves (on average 4 leaves of 9 cm2 ) and are dedicated almost evolution of plant architecture despite the diversity of growth habits in the genus atractocarpus, ranging from well branched to monocaulous species, the architectural construction of members of the group is fairly homogenous. differences between species mainly result from quantitative variations in which the number of modules per branches and module length tend to increase with increasing branching index. frontiers in plant science | www.frontiersin.org 9 december 2018 | volume 9 |

number of small leaves (on average 4 leaves of 9 cm2 ) and are dedicated almost evolution of plant architecture despite the diversity of growth habits in the genus atractocarpus, ranging from well branched to monocaulous species, the architectural construction of members of the group is fairly homogenous. differences between species mainly result from quantitative variations in which the number of modules per branches and module length tend to increase with increasing branching index. frontiers in plant science | www.frontiersin.org 9 december 2018 | volume 9 | article 1775 bruy et al. evolution of plant architecture in atractocarpus exclusively to reproduction.

atractocarpus and our results indicated at least two independent origins of monocauly in new caledonia. other examples of derived monocauly have recently been revealed in flowering plants (chomicki et al., 2017; barrabé et al., 2018). the intermediate architectural class is evolutionary closer to the monocaulous habit since intermediate and monocaulous species are always sister taxa or very close relatives. there is a high probability that the putative ancestor of \"monocaulous clade b,\" had an intermediate architecture. this insight, as well as the architectural gradient observed among extant species, suggests a gradual evolutionary transition from the branched architecture toward monocauly through an

architectural class is evolutionary closer to the monocaulous habit since intermediate and monocaulous species are always sister taxa or very close relatives. there is a high probability that the putative ancestor of \"monocaulous clade b,\" had an intermediate architecture. this insight, as well as the architectural gradient observed among extant species, suggests a gradual evolutionary transition from the branched architecture toward monocauly through an intermediate architectural class. reversion from a monocaulous architecture toward an intermediate one appears to have been possible (e.g., atractocarpus sp. nov. 7) but full reversion to a branched architecture was not observed. the similarity of developmental origin

one appears to have been possible (e.g., atractocarpus sp. nov. 7) but full reversion to a branched architecture was not observed. the similarity of developmental origin and the gradual quantitative architectural variation of lateral axes (c2) across closely related taxa suggest a homologous origin of these axes in the atractocarpus species studied here (see nozeran, 1955; rossignol and rossignol, 1985; haicour et al., 1989). a homologous origin between vegetative and inflorescence axes has been suspected in several groups such as ranunculaceae and papaveraceae (stebbins, 1973), as well as several families of alismatales (posluszny and charlton, 1993, 1999) and pandanales (rudall and

1999) and pandanales (rudall and bateman, 2006), though none of these studies combined architectural analysis with molecular phylogenetic results. only a few studies to date have explored evolutionary links between architectural models (see chomicki et al., 2017). in new caledonia atractocarpus, we found recurrent transitions from stone's model for branched species (with c2 functioning as branches) to corner's model for monocaulous species (with c2 functioning as inflorescences), although the limit between them remains fuzzy, as illustrated by species whose architecture is intermediate. the variation observed in branching degree across species illustrates an \"architectural continuum\" (oldeman, 1974; barthélémy et al., 1989) rather

studies combined architectural analysis with molecular phylogenetic results. only a few studies to date have explored evolutionary links between architectural models (see chomicki et al., 2017). in new caledonia atractocarpus, we found recurrent transitions from stone's model for branched species (with c2 functioning as branches) to corner's model for monocaulous species (with c2 functioning as inflorescences), although the limit between them remains fuzzy, as illustrated by species whose architecture is intermediate. the variation observed in branching degree across species illustrates an \"architectural continuum\" (oldeman, 1974; barthélémy et al., 1989) rather than saltational evolution involving rapid and profound change in architecture (bateman

evolutionary links between architectural models (see chomicki et al., 2017). in new caledonia atractocarpus, we found recurrent transitions from stone's model for branched species (with c2 functioning as branches) to corner's model for monocaulous species (with c2 functioning as inflorescences), although the limit between them remains fuzzy, as illustrated by species whose architecture is intermediate. the variation observed in branching degree across species illustrates an \"architectural continuum\" (oldeman, 1974; barthélémy et al., 1989) rather than saltational evolution involving rapid and profound change in architecture (bateman and dimichele, 1994; bateman, 1999). nothing suggests that the intermediate architectural class confers lower fitness (i.e.,

caledonia atractocarpus, we found recurrent transitions from stone's model for branched species (with c2 functioning as branches) to corner's model for monocaulous species (with c2 functioning as inflorescences), although the limit between them remains fuzzy, as illustrated by species whose architecture is intermediate. the variation observed in branching degree across species illustrates an \"architectural continuum\" (oldeman, 1974; barthélémy et al., 1989) rather than saltational evolution involving rapid and profound change in architecture (bateman and dimichele, 1994; bateman, 1999). nothing suggests that the intermediate architectural class confers lower fitness (i.e., represents a fitness valley) compared to monocaulous or branched architecture, particularly in

has already been noted (hallé et al., 1978), suggesting that preferential transition between models might occur. the two independent origins of monocauly inferred in atractocarpus illustrate two different evolutionary pathways frontiers in plant science | www.frontiersin.org table 4 | phylogenetic signals of measured traits across the phylogeny of 25 atractocarpus species (24 for fruit volume). trait lambda p-value kappa p-value branching index 1.013 <0.001 1.132 0.002 slenderness ratio 0.748 1 0.251 0.539 specific wood density 0.447 1 0.429 0.242 specific stem density 0.813 0.265 0.607 0.029 internode diameter 0.076 0.442 0.047 0.533 internode length 0 1 0.127 0.946 proportion of pith

branching index following a 0.05 risk of error. involved in the transition from branches into inflorescences. in \"monocaulous clade a,\" reduction in the branching index occurred through a reduction in the number of modules, while in \"monocaulous clade b,\" module length is involved. architectural analysis actually showed that the two species with intermediate architecture and the two monocaulous species belonging to the clade b are those with the higher number of modules (up to 46 for the intermediate a. sessilifolius and 26 the for monocaulous a. longistipitatus, i.e., as much as certain branched species) and with shorter module length (excluding the

involved in the transition from branches into inflorescences. in \"monocaulous clade a,\" reduction in the branching index occurred through a reduction in the number of modules, while in \"monocaulous clade b,\" module length is involved. architectural analysis actually showed that the two species with intermediate architecture and the two monocaulous species belonging to the clade b are those with the higher number of modules (up to 46 for the intermediate a. sessilifolius and 26 the for monocaulous a. longistipitatus, i.e., as much as certain branched species) and with shorter module length (excluding the monocaulous a. confertus). this type of morphological reduction

architectural analysis actually showed that the two species with intermediate architecture and the two monocaulous species belonging to the clade b are those with the higher number of modules (up to 46 for the intermediate a. sessilifolius and 26 the for monocaulous a. longistipitatus, i.e., as much as certain branched species) and with shorter module length (excluding the monocaulous a. confertus). this type of morphological reduction observed at the module level in atractocarpus could affect any elementary level 10 december 2018 | volume 9 | article 1775 bruy et al. evolution of plant architecture in atractocarpus figure 5 | projection of

with shorter module length (excluding the monocaulous a. confertus). this type of morphological reduction observed at the module level in atractocarpus could affect any elementary level 10 december 2018 | volume 9 | article 1775 bruy et al. evolution of plant architecture in atractocarpus figure 5 | projection of (a) species and (b) traits on the two first axis of principal component analysis (see table 1 for trait abbreviations). ellipses represent the 95% confidence interval for each architectural classes. functional differences between architectural classes were tested with permanova. 2001; westoby et al., 2002; preston and ackerly, 2003; westoby and wright, 2003;

architectural classes. functional differences between architectural classes were tested with permanova. 2001; westoby et al., 2002; preston and ackerly, 2003; westoby and wright, 2003; pickup et al., 2005; sun et al., 2006; normand et al., 2008). large leaves and fruit carried by a large single stem, as seen in monocaulous species, illustrate one extremity of this spectrum. the second rule (diminution of ramification), has received much less consideration (but see white, 1983a; ackerly and donoghue, 1998). our results agree with this statement, as we have shown that internode diameter, as well as fruit and leaf size, are negatively correlated with branching

between branching index and several functional traits related to various ecological strategies in areas with a similar habitat (i.e., rainforest). we found that branching index tends to be correlated with ecologically important dimensions (mooney, 1972; grime, 1974; grime et al., 1988; westoby et al., 2002). the large leaves of monocaulous species increase light interception in shady understory (poorter, 1999; rozendaal et al., 2006), their low sla and huber value are likely associated with low photosynthetic capacity (field and mooney, 1986; shipley et al., 2005; poorter and bongers, 2006; rozendaal et al., 2006; pérez-harguindeguy et al., 2013) and their small internode length

associated with low photosynthetic capacity (field and mooney, 1986; shipley et al., 2005; poorter and bongers, 2006; rozendaal et al., 2006; pérez-harguindeguy et al., 2013) and their small internode length can result from a slow growth rate (mooney, 1972; chuah, 1977; grime et al., 1988; westoby et al., 2002). monocaulous species are likely skewed toward a high resource conservation strategy (grime et al., 1997; díaz et al., 2004, 2016) suited to the shady understory of rainforest. in sparsely branched to unbranched species, a distal part of the stem with thicker diameter and higher wood proportion was found to be suited to

result from a slow growth rate (mooney, 1972; chuah, 1977; grime et al., 1988; westoby et al., 2002). monocaulous species are likely skewed toward a high resource conservation strategy (grime et al., 1997; díaz et al., 2004, 2016) suited to the shady understory of rainforest. in sparsely branched to unbranched species, a distal part of the stem with thicker diameter and higher wood proportion was found to be suited to supporting large and numerous energetically costly leaves (higher leaf area and sla). however, higher wood area does not fully compensate for high total leaf area of the main stem since the

deep characterization of architectural units (edelin, 1977, 1984), combining qualitative and quantitative traits. branching index, corner's rules, and ecological strategies in the original durian theory, corner (1949, 1953-1954) proposed two fundamental rules governing plant morphology: (i) axial conformity, stipulating that \"the stouter, or more massive, the axis in a given species, the larger and more complicated its appendages\"; and (ii) diminution on ramification, stipulating that \"the greater the ramification, the smaller become the branches and their appendages\". axial conformity is by far the most studied since the leaf size - twig size and fruit size - leaf size spectra are among

clades in new caledonia have been shown to result from recent colonization (e.g., palms, pintaud et al., 2001; pillon, 2012). similarities in form and physiology among close relatives, as suggested by our character state reconstruction and architectural analysis, are suggested to involve intense competition that favors divergence among locally coexisting species (givnish, 2016). in support of this, we found notable variation in module length and number between sisters species, even when they belong to the same architectural class. in atractocarpus, variation in growth habit is correlated with a gradual switch in assimilation function from branches to trunk, and is associated with

pillon, 2012). similarities in form and physiology among close relatives, as suggested by our character state reconstruction and architectural analysis, are suggested to involve intense competition that favors divergence among locally coexisting species (givnish, 2016). in support of this, we found notable variation in module length and number between sisters species, even when they belong to the same architectural class. in atractocarpus, variation in growth habit is correlated with a gradual switch in assimilation function from branches to trunk, and is associated with a vertical differentiation of major functions. leaf arrangement in monocaulous species results in important overlap that impacts photosynthesis,

in module length and number between sisters species, even when they belong to the same architectural class. in atractocarpus, variation in growth habit is correlated with a gradual switch in assimilation function from branches to trunk, and is associated with a vertical differentiation of major functions. leaf arrangement in monocaulous species results in important overlap that impacts photosynthesis, while stem slenderness suggests a rather small structural investment in vertical support. branched species tend to increase light harvesting and reduce leaf overlap (via increase branch length), a strategy that require more investment in stem tissue (givnish, 1995), as allocation to non-leaf structures

growth habit is correlated with a gradual switch in assimilation function from branches to trunk, and is associated with a vertical differentiation of major functions. leaf arrangement in monocaulous species results in important overlap that impacts photosynthesis, while stem slenderness suggests a rather small structural investment in vertical support. branched species tend to increase light harvesting and reduce leaf overlap (via increase branch length), a strategy that require more investment in stem tissue (givnish, 1995), as allocation to non-leaf structures might increase with ramification. atractocarpus species thus exhibit a gradient of foraging for light in which leaf size and disposition vary

impacts photosynthesis, while stem slenderness suggests a rather small structural investment in vertical support. branched species tend to increase light harvesting and reduce leaf overlap (via increase branch length), a strategy that require more investment in stem tissue (givnish, 1995), as allocation to non-leaf structures might increase with ramification. atractocarpus species thus exhibit a gradient of foraging for light in which leaf size and disposition vary with the degree of ramification. new caledonian rainforests are characterized by a low canopy (ca. 20 m) and trees with small crowns (blanchard et al., 2016), two structural features that increase the number and intensity

(givnish, 2016). new caledonia exhibits both components of divergent selection: (i) climatically permissive rainforests and (ii) ecological opportunities offered by recent climatic fluctuations. divergent selection, caused by competition among closely related taxa, leads to adaptive radiation, i.e., the rise of a diversity of ecological roles and related adaptations in different species among a lineage (givnish, 2016). the theory of niche preemption holds that adaptive radiation creates a barrier that inhibits the establishment of closely related taxa, thus reducing the likelihood of repeated colonization. clearly highlights evolutionary processes behind architectural transitions and their link to plant ecological strategy and perhaps also to

the implications of architectural variation on plant fitness under different environmental conditions. based on such an index, our study provides a functional definition of monocauly that is ecologically and evolutionary more explicit than one based solely on physiognomy (cf. hallé et al., 1978; chomicki et al., 2017): we define monocaulous species as woody plants whose cardinal functions (e.g., assimilation and exploration) rely on a single apparent stem. although our work focused on a small clade, it opens new perspectives and proposes a general framework for further understanding evolution of plant architecture and its functional implications in other plant groups and other

### NA
and ecosystem dynamics, university of amsterdam, p.o. box 94248, 1090 ge amsterdam, netherlands e-mail: wdkissling@gmail.com †these authors have joint ﬁrst authorship. tropical rain forests (trf) are the most diverse terrestrial biome on earth, but the diversiﬁcation dynamics of their constituent growth forms remain largely unexplored. climbing plants contribute signiﬁcantly to species diversity and ecosystem processes in trf. we investigate the broad-scale patterns and drivers of species richness as well as the diversiﬁcation history of climbing and non-climbing palms (arecaceae). we quantify to what extent macroecological diversity patterns are related to contemporary climate, forest canopy height, and paleoclimatic changes. we test whether

†these authors have joint ﬁrst authorship. tropical rain forests (trf) are the most diverse terrestrial biome on earth, but the diversiﬁcation dynamics of their constituent growth forms remain largely unexplored. climbing plants contribute signiﬁcantly to species diversity and ecosystem processes in trf. we investigate the broad-scale patterns and drivers of species richness as well as the diversiﬁcation history of climbing and non-climbing palms (arecaceae). we quantify to what extent macroecological diversity patterns are related to contemporary climate, forest canopy height, and paleoclimatic changes. we test whether diversiﬁcation rates are higher for climbing than non-climbing palms and estimate the origin of the

and non-climbing palms (arecaceae). we quantify to what extent macroecological diversity patterns are related to contemporary climate, forest canopy height, and paleoclimatic changes. we test whether diversiﬁcation rates are higher for climbing than non-climbing palms and estimate the origin of the climbing habit. climbers account for 22% of global palm species diversity, mostly concentrated in southeast asia. global variation in climbing palm species richness can be partly explained by past and present-day climate and rain forest canopy height, but regional differences in residual species richness after accounting for current and past differences in environment suggest a strong role of historical contingencies

are related to contemporary climate, forest canopy height, and paleoclimatic changes. we test whether diversiﬁcation rates are higher for climbing than non-climbing palms and estimate the origin of the climbing habit. climbers account for 22% of global palm species diversity, mostly concentrated in southeast asia. global variation in climbing palm species richness can be partly explained by past and present-day climate and rain forest canopy height, but regional differences in residual species richness after accounting for current and past differences in environment suggest a strong role of historical contingencies in climbing palm diversiﬁcation. climbing palms show a higher net diversiﬁcation rate

non-climbing palms and estimate the origin of the climbing habit. climbers account for 22% of global palm species diversity, mostly concentrated in southeast asia. global variation in climbing palm species richness can be partly explained by past and present-day climate and rain forest canopy height, but regional differences in residual species richness after accounting for current and past differences in environment suggest a strong role of historical contingencies in climbing palm diversiﬁcation. climbing palms show a higher net diversiﬁcation rate than non-climbers. diversiﬁcation analyses of palms detected a diversiﬁcation rate increase along the branches leading to the most speciesrich clade of

to the most speciesrich clade of climbers. ancestral character reconstructions revealed that the climbing habit originated between early eocene and miocene. these results imply that changes from non-climbing to climbing habits may have played an important role in palm diversiﬁcation, resulting in the origin of one ﬁfth of all palm species. we suggest that, in addition to current climate and paleoclimatic changes after the late neogene, present-day diversity of climbing palms can be explained by morpho-anatomical innovations, the biogeographic history of southeast asia, and/or ecological opportunities due to the diversiﬁcation of high-stature dipterocarps in asian trfs. keywords: classe, bamm, growth form,

indicate that understanding changes in the abundance and biomass of speciﬁc growth forms has important implications for future community and ecosystem dynamics in trfs (phillips et al., 2002; schnitzer and bongers, 2011). the climbing growth form (lianas and vines) constitutes a key component of tropical forests worldwide, contributing considerably to species diversity (between 10 - 50%), stem density (∼25%), and ecosystem processes such as forest transpiration and carbon sequestration (gentry, 1991; schnitzer and bongers, 2002). the climbing habit is present in more than 130 plant families and has evolved independently numerous times within angiosperms january 2015 | volume 5 | article

ecosystem processes such as forest transpiration and carbon sequestration (gentry, 1991; schnitzer and bongers, 2002). the climbing habit is present in more than 130 plant families and has evolved independently numerous times within angiosperms january 2015 | volume 5 | article 452 | 1 couvreur et al. (gentry, 1991). across species from different woody plant families, abundance of climbers in tropical forests is negatively correlated with mean annual precipitation and positively with seasonality, peaking in tropical dry forests (schnitzer, 2005). however, regional studies of climber abundance do not necessarily support such results and suggest that structural characteristics of the forests can

among climbing plants (tomlinson and fisher, 2000; rowe et al., 2004) that could potentially mask environmental controls of speciﬁc lineages in cross-taxon analyses. despite their ecological importance, few studies have investigated the role of the climbing habit in the evolution and diversiﬁcation of trf. based on sister group comparisons of species richness between climbing and non-climbing clades within 48 angiosperm families, gianoli (2004) inferred that the climbing habit is a key innovation within ﬂowering plants, leading to higher species richness than non-climbing sister groups. wang et al. (2012) used a generic level dated phylogeny of the largely climbing trf family menispermaceae

studies have investigated the role of the climbing habit in the evolution and diversiﬁcation of trf. based on sister group comparisons of species richness between climbing and non-climbing clades within 48 angiosperm families, gianoli (2004) inferred that the climbing habit is a key innovation within ﬂowering plants, leading to higher species richness than non-climbing sister groups. wang et al. (2012) used a generic level dated phylogeny of the largely climbing trf family menispermaceae and found evidence for a burst of diversiﬁcation shortly after the cretaceous - paleogene boundary, suggesting an important role of the climbing habit in the diversiﬁcation of trfs

in the diversiﬁcation of trfs throughout the cenozoic (this is also found in ferns; schneider et al., 2004). moreover, biogeographic analyses of the neotropical tribe bignonieae (bignoniaceae) further showed that drivers of climber diversiﬁcation are possibly related to climate drying and andean orogeny (lohmann et al., 2013). with about 2,500 species, palms (arecaceae) are a speciesrich, monocotyledonous plant family characteristic of tropical and subtropical ecosystems (dransﬁeld et al., 2008; couvreur and baker, 2013). palms have limited ability to survive in areas with cold and arid climates due to structural constraints (tomlinson, 2006). as a consequence, palm species richness generally peaks in

2013). with about 2,500 species, palms (arecaceae) are a speciesrich, monocotyledonous plant family characteristic of tropical and subtropical ecosystems (dransﬁeld et al., 2008; couvreur and baker, 2013). palms have limited ability to survive in areas with cold and arid climates due to structural constraints (tomlinson, 2006). as a consequence, palm species richness generally peaks in warm and humid areas with low seasonality (bjorholm et al., 2005; kissling et al., 2012a). however, historical legacies related to evolutionary history of speciﬁc lineages (baker and couvreur, 2013a,b), dispersal limitation (bjorholm et al., 2006; kissling et al., 2012a; eiserhardt et al., 2013), and the unique

couvreur, 2013a,b), dispersal limitation (bjorholm et al., 2006; kissling et al., 2012a; eiserhardt et al., 2013), and the unique history of biogeographic regions (bjorholm et al., 2006; kissling et al., 2012a; blach-overgaard et al., 2013; rakotoarinivo et al., 2013) also play an important role in shaping global patterns of palm species richness and distribution. a large diversity of growth forms has evolved within palms, including tree palms, palms with clustered stems, acaulescent palms, and climbing palms (dransﬁeld, 1978; dransﬁeld et al., 2008; balslev et al., 2011). the climbing habit has evolved independently several times (baker et al., 2000a), most notably within

within palms, including tree palms, palms with clustered stems, acaulescent palms, and climbing palms (dransﬁeld, 1978; dransﬁeld et al., 2008; balslev et al., 2011). the climbing habit has evolved independently several times (baker et al., 2000a), most notably within subfamily calamoideae (table 1), but also in relatively small sets of species within subfamily arecoideae in the neotropical genera desmoncus and chamaedorea, and in the madagascan genus dypsis (table 1). the mainly southeast asian calamus is the most species-rich genus of palms and is one of the most diverse genera of climbing plants (gentry, 1991). climbing in palms is typically facilitated by

has evolved independently several times (baker et al., 2000a), most notably within subfamily calamoideae (table 1), but also in relatively small sets of species within subfamily arecoideae in the neotropical genera desmoncus and chamaedorea, and in the madagascan genus dypsis (table 1). the mainly southeast asian calamus is the most species-rich genus of palms and is one of the most diverse genera of climbing plants (gentry, 1991). climbing in palms is typically facilitated by traits such as elongated stems that are stiffened by cylindrical leaf sheaths distally, the presence of spines on almost frontiers in genetics | evolutionary and population genetics

and ask what role the climbing habit has played in the diversiﬁcation of palms through time. speciﬁcally, we test the following hypotheses and corresponding predictions: (1) the diversity of climbing palms correlates with aseasonal tropical climates and peaks in high-stature forests: (a) due to physiological and functional adaptations of palms, species richness of climbing palms is positively correlated with temperature and precipitation and negatively with temperature seasonality. (b) differences in canopy height among trfs explain global variation in species richness of climbing palms, with tall forests having more climbing species than short-stature forests. (2) the climbing habit has played an important

of climbing palms correlates with aseasonal tropical climates and peaks in high-stature forests: (a) due to physiological and functional adaptations of palms, species richness of climbing palms is positively correlated with temperature and precipitation and negatively with temperature seasonality. (b) differences in canopy height among trfs explain global variation in species richness of climbing palms, with tall forests having more climbing species than short-stature forests. (2) the climbing habit has played an important role in palm diversiﬁcation: (a) diversiﬁcation rates are higher for climbing than nonclimbing palms lineages. (b) the origin of the climbing habit correlates with increases in diversiﬁcation rates.

high-stature forests: (a) due to physiological and functional adaptations of palms, species richness of climbing palms is positively correlated with temperature and precipitation and negatively with temperature seasonality. (b) differences in canopy height among trfs explain global variation in species richness of climbing palms, with tall forests having more climbing species than short-stature forests. (2) the climbing habit has played an important role in palm diversiﬁcation: (a) diversiﬁcation rates are higher for climbing than nonclimbing palms lineages. (b) the origin of the climbing habit correlates with increases in diversiﬁcation rates. january 2015 | volume 5 | article 452 | 2 couvreur

an important role in palm diversiﬁcation: (a) diversiﬁcation rates are higher for climbing than nonclimbing palms lineages. (b) the origin of the climbing habit correlates with increases in diversiﬁcation rates. january 2015 | volume 5 | article 452 | 2 couvreur et al. evolution of climbing palms table 1 | species diversity of climbers within palm genera. genus (# spp./total #) % climber subfamily regional distribution calamus (348/381) 91 calamoideae afrotropics, indomalaya, australasia, oceania ceratolobus (6/6) 100 calamoideae indomalaya chamaedorea (1/107) 0.01 arecoideae neotropics daemonorops (92/102) 90 calamoideae indomalaya, australasia desmoncus (12/12) 100 arecoideae neotropics dypsis (2/144) 0.01 arecoideae afrotropics (madagascar)

calamoideae afrotropics plectocomia (16/16) 100 calamoideae indomalaya plectocomiopsis (5/5) 100 calamoideae indomalaya pogonotium (2/3) 67 calamoideae indomalaya retispatha (1/1) 100 calamoideae indomalaya total (535/2445) 22 data are based on the world checklist of palms (govaerts et al., 2014). data for the genus dypsis have been additionally updated (now two climbing species). materials and methods palm species distribution data we used data on global palm species distributions from an exhaustive, authoritative checklist of the world's palm species (govaerts et al., 2014, data used here accessed march 2009). though coarse in resolution, this dataset currently represents the most complete and reliable source on

calamoideae indomalaya plectocomiopsis (5/5) 100 calamoideae indomalaya pogonotium (2/3) 67 calamoideae indomalaya retispatha (1/1) 100 calamoideae indomalaya total (535/2445) 22 data are based on the world checklist of palms (govaerts et al., 2014). data for the genus dypsis have been additionally updated (now two climbing species). materials and methods palm species distribution data we used data on global palm species distributions from an exhaustive, authoritative checklist of the world's palm species (govaerts et al., 2014, data used here accessed march 2009). though coarse in resolution, this dataset currently represents the most complete and reliable source on species distributions of all palms

67 calamoideae indomalaya retispatha (1/1) 100 calamoideae indomalaya total (535/2445) 22 data are based on the world checklist of palms (govaerts et al., 2014). data for the genus dypsis have been additionally updated (now two climbing species). materials and methods palm species distribution data we used data on global palm species distributions from an exhaustive, authoritative checklist of the world's palm species (govaerts et al., 2014, data used here accessed march 2009). though coarse in resolution, this dataset currently represents the most complete and reliable source on species distributions of all palms worldwide. the dataset records palm species presences and absences

data are based on the world checklist of palms (govaerts et al., 2014). data for the genus dypsis have been additionally updated (now two climbing species). materials and methods palm species distribution data we used data on global palm species distributions from an exhaustive, authoritative checklist of the world's palm species (govaerts et al., 2014, data used here accessed march 2009). though coarse in resolution, this dataset currently represents the most complete and reliable source on species distributions of all palms worldwide. the dataset records palm species presences and absences across the world within the level 3 geographic units as deﬁned

materials and methods palm species distribution data we used data on global palm species distributions from an exhaustive, authoritative checklist of the world's palm species (govaerts et al., 2014, data used here accessed march 2009). though coarse in resolution, this dataset currently represents the most complete and reliable source on species distributions of all palms worldwide. the dataset records palm species presences and absences across the world within the level 3 geographic units as deﬁned by the international working group on taxonomic databases (tdwg; brummitt, 2001). these tdwg level 3 units mostly correspond to countries and/or major islands, such as borneo,

on global palm species distributions from an exhaustive, authoritative checklist of the world's palm species (govaerts et al., 2014, data used here accessed march 2009). though coarse in resolution, this dataset currently represents the most complete and reliable source on species distributions of all palms worldwide. the dataset records palm species presences and absences across the world within the level 3 geographic units as deﬁned by the international working group on taxonomic databases (tdwg; brummitt, 2001). these tdwg level 3 units mostly correspond to countries and/or major islands, such as borneo, madagascar, and new guinea, but very large countries such as

units mostly correspond to countries and/or major islands, such as borneo, madagascar, and new guinea, but very large countries such as usa, brazil, and china are subdivided into smaller units. we included only native palm occurrences, excluding introduced occurrences, and doubtful as well as erroneous records. to derive estimates of species richness, we summed all presences of palm species within each tdwg level 3 unit, and did this separately for climbing and non-climbing palms (see below). the dataset contained a total of 2,445 accepted palm species names and 5,027 native occurrence records within 194 tdwg level 3 units (kissling et al.,

such as borneo, madagascar, and new guinea, but very large countries such as usa, brazil, and china are subdivided into smaller units. we included only native palm occurrences, excluding introduced occurrences, and doubtful as well as erroneous records. to derive estimates of species richness, we summed all presences of palm species within each tdwg level 3 unit, and did this separately for climbing and non-climbing palms (see below). the dataset contained a total of 2,445 accepted palm species names and 5,027 native occurrence records within 194 tdwg level 3 units (kissling et al., 2012a). it does not include a recently described

occurrences, excluding introduced occurrences, and doubtful as well as erroneous records. to derive estimates of species richness, we summed all presences of palm species within each tdwg level 3 unit, and did this separately for climbing and non-climbing palms (see below). the dataset contained a total of 2,445 accepted palm species names and 5,027 native occurrence records within 194 tdwg level 3 units (kissling et al., 2012a). it does not include a recently described climbing species of dypsis (rakotoarinivo and dransﬁeld, 2010) which would raise the total number of liana species in this genus to two. the addition of this extra

each tdwg level 3 unit, and did this separately for climbing and non-climbing palms (see below). the dataset contained a total of 2,445 accepted palm species names and 5,027 native occurrence records within 194 tdwg level 3 units (kissling et al., 2012a). it does not include a recently described climbing species of dypsis (rakotoarinivo and dransﬁeld, 2010) which would raise the total number of liana species in this genus to two. the addition of this extra species would not impact the results presented here, but this new information was taken into account throughout the discussion and in table 1. categorization of

below). the dataset contained a total of 2,445 accepted palm species names and 5,027 native occurrence records within 194 tdwg level 3 units (kissling et al., 2012a). it does not include a recently described climbing species of dypsis (rakotoarinivo and dransﬁeld, 2010) which would raise the total number of liana species in this genus to two. the addition of this extra species would not impact the results presented here, but this new information was taken into account throughout the discussion and in table 1. categorization of growth forms we classiﬁed all palm species into two growth forms: (1) climbers, and (2)

names and 5,027 native occurrence records within 194 tdwg level 3 units (kissling et al., 2012a). it does not include a recently described climbing species of dypsis (rakotoarinivo and dransﬁeld, 2010) which would raise the total number of liana species in this genus to two. the addition of this extra species would not impact the results presented here, but this new information was taken into account throughout the discussion and in table 1. categorization of growth forms we classiﬁed all palm species into two growth forms: (1) climbers, and (2) non-climbers (including all other growth forms such as stemmed and acaulescent

which would raise the total number of liana species in this genus to two. the addition of this extra species would not impact the results presented here, but this new information was taken into account throughout the discussion and in table 1. categorization of growth forms we classiﬁed all palm species into two growth forms: (1) climbers, and (2) non-climbers (including all other growth forms such as stemmed and acaulescent palms). palm species www.frontiersin.org that show leaning growth forms such as some bactris species (not really climbers or trees) were considered as non-climbers. for climbers, we also included palm species that

impact the results presented here, but this new information was taken into account throughout the discussion and in table 1. categorization of growth forms we classiﬁed all palm species into two growth forms: (1) climbers, and (2) non-climbers (including all other growth forms such as stemmed and acaulescent palms). palm species www.frontiersin.org that show leaning growth forms such as some bactris species (not really climbers or trees) were considered as non-climbers. for climbers, we also included palm species that can, within the same species, show both climbing and non-climbing habits (only 16 species). the remaining species (n = 2431) were exclusively

into account throughout the discussion and in table 1. categorization of growth forms we classiﬁed all palm species into two growth forms: (1) climbers, and (2) non-climbers (including all other growth forms such as stemmed and acaulescent palms). palm species www.frontiersin.org that show leaning growth forms such as some bactris species (not really climbers or trees) were considered as non-climbers. for climbers, we also included palm species that can, within the same species, show both climbing and non-climbing habits (only 16 species). the remaining species (n = 2431) were exclusively climbers or non-climbers. information on climbing habit was derived from the

palm species into two growth forms: (1) climbers, and (2) non-climbers (including all other growth forms such as stemmed and acaulescent palms). palm species www.frontiersin.org that show leaning growth forms such as some bactris species (not really climbers or trees) were considered as non-climbers. for climbers, we also included palm species that can, within the same species, show both climbing and non-climbing habits (only 16 species). the remaining species (n = 2431) were exclusively climbers or non-climbers. information on climbing habit was derived from the literature (russell, 1968; dransﬁeld, 1979, 1986; dransﬁeld and beentje, 1995; henderson et al., 1995; henderson, 2002,

(1) climbers, and (2) non-climbers (including all other growth forms such as stemmed and acaulescent palms). palm species www.frontiersin.org that show leaning growth forms such as some bactris species (not really climbers or trees) were considered as non-climbers. for climbers, we also included palm species that can, within the same species, show both climbing and non-climbing habits (only 16 species). the remaining species (n = 2431) were exclusively climbers or non-climbers. information on climbing habit was derived from the literature (russell, 1968; dransﬁeld, 1979, 1986; dransﬁeld and beentje, 1995; henderson et al., 1995; henderson, 2002, 2009; dransﬁeld et al., 2008; dowe,

forms such as stemmed and acaulescent palms). palm species www.frontiersin.org that show leaning growth forms such as some bactris species (not really climbers or trees) were considered as non-climbers. for climbers, we also included palm species that can, within the same species, show both climbing and non-climbing habits (only 16 species). the remaining species (n = 2431) were exclusively climbers or non-climbers. information on climbing habit was derived from the literature (russell, 1968; dransﬁeld, 1979, 1986; dransﬁeld and beentje, 1995; henderson et al., 1995; henderson, 2002, 2009; dransﬁeld et al., 2008; dowe, 2010) and for a few species supplemented with expert

stemmed and acaulescent palms). palm species www.frontiersin.org that show leaning growth forms such as some bactris species (not really climbers or trees) were considered as non-climbers. for climbers, we also included palm species that can, within the same species, show both climbing and non-climbing habits (only 16 species). the remaining species (n = 2431) were exclusively climbers or non-climbers. information on climbing habit was derived from the literature (russell, 1968; dransﬁeld, 1979, 1986; dransﬁeld and beentje, 1995; henderson et al., 1995; henderson, 2002, 2009; dransﬁeld et al., 2008; dowe, 2010) and for a few species supplemented with expert knowledge. environmental determinants

habits (only 16 species). the remaining species (n = 2431) were exclusively climbers or non-climbers. information on climbing habit was derived from the literature (russell, 1968; dransﬁeld, 1979, 1986; dransﬁeld and beentje, 1995; henderson et al., 1995; henderson, 2002, 2009; dransﬁeld et al., 2008; dowe, 2010) and for a few species supplemented with expert knowledge. environmental determinants of climber and non-climber species richness we tested 14 predictor variables as potential determinants of species richness in climbing and non-climbing palms. these variables reﬂected contemporary climate (six variables), paleoclimate (six variables), canopy height (one variable), and biogeographic region (one variable). present and past

exclusively climbers or non-climbers. information on climbing habit was derived from the literature (russell, 1968; dransﬁeld, 1979, 1986; dransﬁeld and beentje, 1995; henderson et al., 1995; henderson, 2002, 2009; dransﬁeld et al., 2008; dowe, 2010) and for a few species supplemented with expert knowledge. environmental determinants of climber and non-climber species richness we tested 14 predictor variables as potential determinants of species richness in climbing and non-climbing palms. these variables reﬂected contemporary climate (six variables), paleoclimate (six variables), canopy height (one variable), and biogeographic region (one variable). present and past climates (kissling et al., 2012a; blach-overgaard et al., 2013; rakotoarinivo et

the literature (russell, 1968; dransﬁeld, 1979, 1986; dransﬁeld and beentje, 1995; henderson et al., 1995; henderson, 2002, 2009; dransﬁeld et al., 2008; dowe, 2010) and for a few species supplemented with expert knowledge. environmental determinants of climber and non-climber species richness we tested 14 predictor variables as potential determinants of species richness in climbing and non-climbing palms. these variables reﬂected contemporary climate (six variables), paleoclimate (six variables), canopy height (one variable), and biogeographic region (one variable). present and past climates (kissling et al., 2012a; blach-overgaard et al., 2013; rakotoarinivo et al., 2013) as well as biogeographic history (baker and couvreur, 2013a,b)

palms. these variables reﬂected contemporary climate (six variables), paleoclimate (six variables), canopy height (one variable), and biogeographic region (one variable). present and past climates (kissling et al., 2012a; blach-overgaard et al., 2013; rakotoarinivo et al., 2013) as well as biogeographic history (baker and couvreur, 2013a,b) are important drivers of broad-scale species distributions and diversity patterns in palms. extracted at a coarse resolution (averaged within tdwg level 3 units), these predictor variables allow assessing how broad-scale trends in environmental conditions are related to geographic differences in species numbers of palms worldwide. we acknowledge that ﬁne-scale heterogeneity (e.g. in climates, canopy heights etc.)

2013) as well as biogeographic history (baker and couvreur, 2013a,b) are important drivers of broad-scale species distributions and diversity patterns in palms. extracted at a coarse resolution (averaged within tdwg level 3 units), these predictor variables allow assessing how broad-scale trends in environmental conditions are related to geographic differences in species numbers of palms worldwide. we acknowledge that ﬁne-scale heterogeneity (e.g. in climates, canopy heights etc.) within tdwg level units might not be well captured by such coarse-grained data, but analyses with new species distribution datasets at high resolution could incorporate some of this heterogeneity in the future. january 2015 |

these predictor variables allow assessing how broad-scale trends in environmental conditions are related to geographic differences in species numbers of palms worldwide. we acknowledge that ﬁne-scale heterogeneity (e.g. in climates, canopy heights etc.) within tdwg level units might not be well captured by such coarse-grained data, but analyses with new species distribution datasets at high resolution could incorporate some of this heterogeneity in the future. january 2015 | volume 5 | article 452 | 3 couvreur et al. contemporary climate to represent contemporary climate, we chose six climatic predictor variables from the worldclim dataset (version 1.4; www.worldclim.org), a set of global

(temp cold, in ◦ c × 10). data extraction and geoprocessing of climate data are described in more detail in kissling et al. (2012a). several of these variables, including prec and various temperature measures (temp, temp seas, temp cold), have been identiﬁed as important drivers of the global range and species richness of palms (kissling et al., 2012a). other climate variables (e.g., minimum or maximum monthly precipitation and temperature) are highly correlated with these climatic predictors and were hence not included here. paleoclimate to represent paleoclimatic changes over the neogene and quaternary epochs, we compiled both temperature and precipitation data from

present-day climate (i.e., contemporary climate minus paleoclimate). large positive anomaly values indicate a higher precipitation and temperature in the present than in the past whereas small or negative anomaly values indicate the opposite, i.e., higher precipitation and temperature in the past than in the present. hence, a negative relationship between species richness and climate anomalies indicates that species richness is higher in areas that were relatively wetter or warmer in the past than 1 http://pmip2.lsce.ipsl.fr/ frontiers in genetics | evolutionary and population genetics evolution of climbing palms today. tdwg level 3 unit values for lgm precipitation and temperature were calculated as

large positive anomaly values indicate a higher precipitation and temperature in the present than in the past whereas small or negative anomaly values indicate the opposite, i.e., higher precipitation and temperature in the past than in the present. hence, a negative relationship between species richness and climate anomalies indicates that species richness is higher in areas that were relatively wetter or warmer in the past than 1 http://pmip2.lsce.ipsl.fr/ frontiers in genetics | evolutionary and population genetics evolution of climbing palms today. tdwg level 3 unit values for lgm precipitation and temperature were calculated as mean values across two paleoclimatic simulations (ccsm3,

anomalies since the lgm to the present (lgmtemp ) can be considered roughly representative for climatic oscillations of the whole quaternary (the last 2.6 million years; jansson, 2003; kissling et al., 2012a). canopy height we included canopy height (canopy) of tropical rain forests as a predictor variable to test whether species richness of climbing palms increases with the tallness of forests. forest canopy height data were derived from a recent global map of forest heights (lefsky, 2010), derived from lidar and multispectral remote sensing data for forest patches across the world (average patch size of approximately 25 ± 50 km2 ).

in tropical and subtropical forests. biogeographic history we derived a biogeographic variable (region) to capture potential effects related to the long-term history of biogeographic regions. this categorical variable distinguished seven major regions: afrotropics, australasia, indomalaya, nearctic, neotropics, oceania, and palaearctic (kissling et al., 2012a). it can capture major differences in species richness and clade distributions between regions (kissling et al., 2012a,b; baker and couvreur, 2013a,b) and permits examination of how species richness varies among realms once present-day environment and paleoclimate have been statistically accounted for (kissling et al., 2012a). statistical analysis of determinants of species richness we assessed the environmental determinants

long-term history of biogeographic regions. this categorical variable distinguished seven major regions: afrotropics, australasia, indomalaya, nearctic, neotropics, oceania, and palaearctic (kissling et al., 2012a). it can capture major differences in species richness and clade distributions between regions (kissling et al., 2012a,b; baker and couvreur, 2013a,b) and permits examination of how species richness varies among realms once present-day environment and paleoclimate have been statistically accounted for (kissling et al., 2012a). statistical analysis of determinants of species richness we assessed the environmental determinants of species richness of both climbers and non-climbers with separate multi-predictor regression models. we only included tdwg level 3 units

it can capture major differences in species richness and clade distributions between regions (kissling et al., 2012a,b; baker and couvreur, 2013a,b) and permits examination of how species richness varies among realms once present-day environment and paleoclimate have been statistically accounted for (kissling et al., 2012a). statistical analysis of determinants of species richness we assessed the environmental determinants of species richness of both climbers and non-climbers with separate multi-predictor regression models. we only included tdwg level 3 units where species richness >0 and for which environmental data were available. a number of smaller islands had to be excluded for nonclimbing palms because

and clade distributions between regions (kissling et al., 2012a,b; baker and couvreur, 2013a,b) and permits examination of how species richness varies among realms once present-day environment and paleoclimate have been statistically accounted for (kissling et al., 2012a). statistical analysis of determinants of species richness we assessed the environmental determinants of species richness of both climbers and non-climbers with separate multi-predictor regression models. we only included tdwg level 3 units where species richness >0 and for which environmental data were available. a number of smaller islands had to be excluded for nonclimbing palms because canopy height data were not available for those.

varies among realms once present-day environment and paleoclimate have been statistically accounted for (kissling et al., 2012a). statistical analysis of determinants of species richness we assessed the environmental determinants of species richness of both climbers and non-climbers with separate multi-predictor regression models. we only included tdwg level 3 units where species richness >0 and for which environmental data were available. a number of smaller islands had to be excluded for nonclimbing palms because canopy height data were not available for those. hence, ﬁnal sample sizes for the statistical analysis were 82 and 164 tdwg level 3 units for climbers and non-climbers,

canopy height data were not available for those. hence, ﬁnal sample sizes for the statistical analysis were 82 and 164 tdwg level 3 units for climbers and non-climbers, respectively. we used generalized linear models (glm) with a gaussian error distribution and included all 14 predictor variables as well as either species richness of climbers or non-climbers as response variable. we then applied a model selection based on the akaike information criterion (aic) to derive a minimum adequate model that had the smallest possible number of predictor variables (i.e., the lowest aic value; burnham and anderson, 2002). we used the variance inﬂation

5 | article 452 | 4 couvreur et al. temp cold for climber richness, temp cold and prec dry for non-climber richness). all continuous predictor variables were scaled before the analysis (standardized to mean = 0 and sd = 1). we compared the relative importance of predictor variables to explain species richness of climbing and non-climbing palms with semi-standardized coefﬁcients (for continuous variables only). both response variables as well as several predictor variables (temp, prec seas, temp seas, prec dry, lgmtemp ) were log-transformed to improve normality. we included second order polynomials to account for non-linear relationships if single predictor models

with polynomials showed statistically signiﬁcant improvements over models without polynomials (using an anova model comparison at p < 0.05; crawley, 2007). the ", 
" level of the categorical variable region was set to the indomalayan region because this allowed us compare all other regions to the region with the highest species richness of climbing palms (i.e. indomalaya). for climbing palms, tdwg level 3 sample sizes for the nearctic, oceania, and the palaearctic were small (n ≤ 1), and statistical analyses of climbing palm species richness were therefore restricted to the afrotropics, australasia, indomalaya, and the neotropics. to test for a potential

the indomalayan region because this allowed us compare all other regions to the region with the highest species richness of climbing palms (i.e. indomalaya). for climbing palms, tdwg level 3 sample sizes for the nearctic, oceania, and the palaearctic were small (n ≤ 1), and statistical analyses of climbing palm species richness were therefore restricted to the afrotropics, australasia, indomalaya, and the neotropics. to test for a potential inﬂuence of spatial autocorrelation (kissling and carl, 2008), we further calculated moran's i values on the residuals of our minimum adequate glms. the geographic distance for calculating moran's i values was based on

i values of the residuals of both minimum adequate glms were not statistically signiﬁcant (see results) and residual spatial autocorrelation was therefore considered to be unimportant in our analyses. hence, there was no need to additionally implement spatial regression models (kissling and carl, 2008). all statistical analyses on determinants of species richness were done with the r version 3.0.2 (r development core team, 2013). moran's i analyses were conducted using the r library 'spdep.' diversification analyses beyond examining the determinants of species richness of climbers and non-climbers, we performed diversiﬁcation analyses to test whether the evolution of climbers has an impact

was no need to additionally implement spatial regression models (kissling and carl, 2008). all statistical analyses on determinants of species richness were done with the r version 3.0.2 (r development core team, 2013). moran's i analyses were conducted using the r library 'spdep.' diversification analyses beyond examining the determinants of species richness of climbers and non-climbers, we performed diversiﬁcation analyses to test whether the evolution of climbers has an impact on diversiﬁcation rates as a whole and for speciﬁc clades. overall diversiﬁcation analysis of climbing palms we used the cladogenetic state speciation and extinction model (classe, goldberg and igić, 2012) to

and non-climbers, we performed diversiﬁcation analyses to test whether the evolution of climbers has an impact on diversiﬁcation rates as a whole and for speciﬁc clades. overall diversiﬁcation analysis of climbing palms we used the cladogenetic state speciation and extinction model (classe, goldberg and igić, 2012) to test if climbing species have overall higher diversiﬁcation rates than non-climbing palms. we generated a species-level phylogeny of palms (i.e., a 2,445 tip dated phylogeny), based on the genus-level, fossil-calibrated tree of couvreur et al. (2011a) derived from the global palm supertree of baker et al. (2009) as an initial backbone. this phylogenetic tree

climbers has an impact on diversiﬁcation rates as a whole and for speciﬁc clades. overall diversiﬁcation analysis of climbing palms we used the cladogenetic state speciation and extinction model (classe, goldberg and igić, 2012) to test if climbing species have overall higher diversiﬁcation rates than non-climbing palms. we generated a species-level phylogeny of palms (i.e., a 2,445 tip dated phylogeny), based on the genus-level, fossil-calibrated tree of couvreur et al. (2011a) derived from the global palm supertree of baker et al. (2009) as an initial backbone. this phylogenetic tree provides the most robust hypothesis of relationships between palm genera to date.

a 2,445 tip dated phylogeny), based on the genus-level, fossil-calibrated tree of couvreur et al. (2011a) derived from the global palm supertree of baker et al. (2009) as an initial backbone. this phylogenetic tree provides the most robust hypothesis of relationships between palm genera to date. we then simulated random species phylogenies at the tips (i.e., the genera) of this backbone tree using known species diversity for each genus (see above). this was done 100 times to get 100 species level trees. we used the script \"gratfmissingtaxa\" www.frontiersin.org evolution of climbing palms written by françois michonneau2 and modiﬁed it to simulate

al. (2011a) derived from the global palm supertree of baker et al. (2009) as an initial backbone. this phylogenetic tree provides the most robust hypothesis of relationships between palm genera to date. we then simulated random species phylogenies at the tips (i.e., the genera) of this backbone tree using known species diversity for each genus (see above). this was done 100 times to get 100 species level trees. we used the script \"gratfmissingtaxa\" www.frontiersin.org evolution of climbing palms written by françois michonneau2 and modiﬁed it to simulate phylogenies under a pure birth model (sim.bdtree function in ape, paradis et al., 2004).

initial backbone. this phylogenetic tree provides the most robust hypothesis of relationships between palm genera to date. we then simulated random species phylogenies at the tips (i.e., the genera) of this backbone tree using known species diversity for each genus (see above). this was done 100 times to get 100 species level trees. we used the script \"gratfmissingtaxa\" www.frontiersin.org evolution of climbing palms written by françois michonneau2 and modiﬁed it to simulate phylogenies under a pure birth model (sim.bdtree function in ape, paradis et al., 2004). for each genus, the crown node of the simulated species-level phylogeny was inserted either (i)

100 times to get 100 species level trees. we used the script \"gratfmissingtaxa\" www.frontiersin.org evolution of climbing palms written by françois michonneau2 and modiﬁed it to simulate phylogenies under a pure birth model (sim.bdtree function in ape, paradis et al., 2004). for each genus, the crown node of the simulated species-level phylogeny was inserted either (i) randomly along the whole length of the subtending branch of the genus (referred to as the 'non-constrained analysis'), or (ii) following a uniform distribution between 0.2 and 0.8 (i.e., between 20 and 80% of the whole length of the subtending branch, referred to as 'constrained

whole length of the subtending branch, referred to as 'constrained analysis'). this latter approach was used to test the effect of extremely young or old inferred crown node ages on the results. different proportions of climbers and non-climbers within genera (see table 1) can create a problem when simulating random species relationships at the tips of the phylogeny because specieslevel relationships are unknown. a preliminary analysis suggested that coding all species within climbing genera as climbers (generic level coding) or coding the observed proportions within each genus (i.e., coding each species as is) had little effect on the results. we therefore

of extremely young or old inferred crown node ages on the results. different proportions of climbers and non-climbers within genera (see table 1) can create a problem when simulating random species relationships at the tips of the phylogeny because specieslevel relationships are unknown. a preliminary analysis suggested that coding all species within climbing genera as climbers (generic level coding) or coding the observed proportions within each genus (i.e., coding each species as is) had little effect on the results. we therefore used the generic level coding approach for subsequent analyses. the 100 trees generated above were combined with the trait dataset

(see table 1) can create a problem when simulating random species relationships at the tips of the phylogeny because specieslevel relationships are unknown. a preliminary analysis suggested that coding all species within climbing genera as climbers (generic level coding) or coding the observed proportions within each genus (i.e., coding each species as is) had little effect on the results. we therefore used the generic level coding approach for subsequent analyses. the 100 trees generated above were combined with the trait dataset of growth forms. the classe model is a derived model of the binary state speciation and extinction model (bisse) (maddison

λ110 , λ001 , λ100 , λ011 ; at the node) and two extinctions (μ1 and μ2 ). see text for details. estimate the conﬁdence intervals of the parameters because these are highly dependent on each individual tree shape and thus have no real meaning in our context (randomly generated species level trees). in contrast, the maximum likelihood (ml) analyses are averaged over all trees and here provide a better way to take the phylogenetic uncertainty into account. clade speciﬁc diversiﬁcation analyses in order to test for possible shifts in diversiﬁcation rates (speciation minus extinction) associated with the evolution of the

diversiﬁcation (rabosky, 2014). bamm has been shown to better identify increases/decreases in diversiﬁcation shifts when compared to other methods such as medusa (rabosky, 2014). bamm accommodates incomplete taxon sampling under a phylogenetically structured sampling. in our phylogeny, we have included most palm genera, each one being represented by a single species. we thus provided bamm with the proportion of species sampled per genus (i.e., 1/number of species in genus). we used the chronogram of palm genera as the input tree (couvreur et al., 2011a). priors were estimated with bammtools (rabosky et al., 2014b) using the function \"setbammpriors\". a compound poisson process

identify increases/decreases in diversiﬁcation shifts when compared to other methods such as medusa (rabosky, 2014). bamm accommodates incomplete taxon sampling under a phylogenetically structured sampling. in our phylogeny, we have included most palm genera, each one being represented by a single species. we thus provided bamm with the proportion of species sampled per genus (i.e., 1/number of species in genus). we used the chronogram of palm genera as the input tree (couvreur et al., 2011a). priors were estimated with bammtools (rabosky et al., 2014b) using the function \"setbammpriors\". a compound poisson process is implemented in bamm for the prior probability of

to other methods such as medusa (rabosky, 2014). bamm accommodates incomplete taxon sampling under a phylogenetically structured sampling. in our phylogeny, we have included most palm genera, each one being represented by a single species. we thus provided bamm with the proportion of species sampled per genus (i.e., 1/number of species in genus). we used the chronogram of palm genera as the input tree (couvreur et al., 2011a). priors were estimated with bammtools (rabosky et al., 2014b) using the function \"setbammpriors\". a compound poisson process is implemented in bamm for the prior probability of a rate shift along any branch. a

we used the generic level chronogram of couvreur et al. (2011a; 183 tips) and the genus-level coded dataset (derived from the monomorphic dataset used for the diversiﬁcation analyses above). this approach does not take into account the evolution of climbers within mainly non-climbing genera such as dypsis and chamaedorea (three species only). we also coded the genera calamus and daemonorops (subfamily calamoideae; subtribe calaminae) as ancestrally climbing, even though a few species within each genus are non-climbers (table 1). ancestral states were estimated using the make.simmap function in the r package phytools (revell, 2012). because specifying incorrect prior values can inﬂuence

monomorphic dataset used for the diversiﬁcation analyses above). this approach does not take into account the evolution of climbers within mainly non-climbing genera such as dypsis and chamaedorea (three species only). we also coded the genera calamus and daemonorops (subfamily calamoideae; subtribe calaminae) as ancestrally climbing, even though a few species within each genus are non-climbers (table 1). ancestral states were estimated using the make.simmap function in the r package phytools (revell, 2012). because specifying incorrect prior values can inﬂuence posterior mapping results (couvreur et al., 2010) we used the empirical approach function of make.simmap where the priors (α and β)

with q being the transition matrix between both states. this was achieved using the use.empirical = true option in make.simmap. we undertook 10,000 generations sampling every 100 steps. we used the function 'densitymap' in phytools to depict the changes in posterior probabilities (pp) along branches of the phylogeny. results global species richness of climbing and non-climbing palms out of 2,446 palm species in our dataset, a total of 535 species (22%) were classiﬁed as climbers. the majority of climbing palm species are found within the genera calamus (348 species) and daemonorops (92 species). in most cases, genera with climbing habits have

achieved using the use.empirical = true option in make.simmap. we undertook 10,000 generations sampling every 100 steps. we used the function 'densitymap' in phytools to depict the changes in posterior probabilities (pp) along branches of the phylogeny. results global species richness of climbing and non-climbing palms out of 2,446 palm species in our dataset, a total of 535 species (22%) were classiﬁed as climbers. the majority of climbing palm species are found within the genera calamus (348 species) and daemonorops (92 species). in most cases, genera with climbing habits have a high (≥90%) proportion of climbing species (table 1). geographically, climbing

make.simmap. we undertook 10,000 generations sampling every 100 steps. we used the function 'densitymap' in phytools to depict the changes in posterior probabilities (pp) along branches of the phylogeny. results global species richness of climbing and non-climbing palms out of 2,446 palm species in our dataset, a total of 535 species (22%) were classiﬁed as climbers. the majority of climbing palm species are found within the genera calamus (348 species) and daemonorops (92 species). in most cases, genera with climbing habits have a high (≥90%) proportion of climbing species (table 1). geographically, climbing palm species occur in all four major tropical

the function 'densitymap' in phytools to depict the changes in posterior probabilities (pp) along branches of the phylogeny. results global species richness of climbing and non-climbing palms out of 2,446 palm species in our dataset, a total of 535 species (22%) were classiﬁed as climbers. the majority of climbing palm species are found within the genera calamus (348 species) and daemonorops (92 species). in most cases, genera with climbing habits have a high (≥90%) proportion of climbing species (table 1). geographically, climbing palm species occur in all four major tropical regions (afrotropics, australasia, indomalaya, neotropics; table 1; figure 2) and two

changes in posterior probabilities (pp) along branches of the phylogeny. results global species richness of climbing and non-climbing palms out of 2,446 palm species in our dataset, a total of 535 species (22%) were classiﬁed as climbers. the majority of climbing palm species are found within the genera calamus (348 species) and daemonorops (92 species). in most cases, genera with climbing habits have a high (≥90%) proportion of climbing species (table 1). geographically, climbing palm species occur in all four major tropical regions (afrotropics, australasia, indomalaya, neotropics; table 1; figure 2) and two species in calamus even reach oceania. climber species

(pp) along branches of the phylogeny. results global species richness of climbing and non-climbing palms out of 2,446 palm species in our dataset, a total of 535 species (22%) were classiﬁed as climbers. the majority of climbing palm species are found within the genera calamus (348 species) and daemonorops (92 species). in most cases, genera with climbing habits have a high (≥90%) proportion of climbing species (table 1). geographically, climbing palm species occur in all four major tropical regions (afrotropics, australasia, indomalaya, neotropics; table 1; figure 2) and two species in calamus even reach oceania. climber species richness peaked in indomalaya,

out of 2,446 palm species in our dataset, a total of 535 species (22%) were classiﬁed as climbers. the majority of climbing palm species are found within the genera calamus (348 species) and daemonorops (92 species). in most cases, genera with climbing habits have a high (≥90%) proportion of climbing species (table 1). geographically, climbing palm species occur in all four major tropical regions (afrotropics, australasia, indomalaya, neotropics; table 1; figure 2) and two species in calamus even reach oceania. climber species richness peaked in indomalaya, closely followed by australasia, with the neotropics showing the lowest climber richness (figure 2a). this

our dataset, a total of 535 species (22%) were classiﬁed as climbers. the majority of climbing palm species are found within the genera calamus (348 species) and daemonorops (92 species). in most cases, genera with climbing habits have a high (≥90%) proportion of climbing species (table 1). geographically, climbing palm species occur in all four major tropical regions (afrotropics, australasia, indomalaya, neotropics; table 1; figure 2) and two species in calamus even reach oceania. climber species richness peaked in indomalaya, closely followed by australasia, with the neotropics showing the lowest climber richness (figure 2a). this contrasted to the species richness of

are found within the genera calamus (348 species) and daemonorops (92 species). in most cases, genera with climbing habits have a high (≥90%) proportion of climbing species (table 1). geographically, climbing palm species occur in all four major tropical regions (afrotropics, australasia, indomalaya, neotropics; table 1; figure 2) and two species in calamus even reach oceania. climber species richness peaked in indomalaya, closely followed by australasia, with the neotropics showing the lowest climber richness (figure 2a). this contrasted to the species richness of non-climbing palms, which, when compared to other tropical regions, was highest in the neotropics and lowest in the

species) and daemonorops (92 species). in most cases, genera with climbing habits have a high (≥90%) proportion of climbing species (table 1). geographically, climbing palm species occur in all four major tropical regions (afrotropics, australasia, indomalaya, neotropics; table 1; figure 2) and two species in calamus even reach oceania. climber species richness peaked in indomalaya, closely followed by australasia, with the neotropics showing the lowest climber richness (figure 2a). this contrasted to the species richness of non-climbing palms, which, when compared to other tropical regions, was highest in the neotropics and lowest in the afrotropics (figure 2b). determinants of species richness

climbing palm species occur in all four major tropical regions (afrotropics, australasia, indomalaya, neotropics; table 1; figure 2) and two species in calamus even reach oceania. climber species richness peaked in indomalaya, closely followed by australasia, with the neotropics showing the lowest climber richness (figure 2a). this contrasted to the species richness of non-climbing palms, which, when compared to other tropical regions, was highest in the neotropics and lowest in the afrotropics (figure 2b). determinants of species richness in climbing palms among contemporary climatic variables, temp (positive effect) and prec seas (negative effect) were the most important variables to explain climber

climber species richness peaked in indomalaya, closely followed by australasia, with the neotropics showing the lowest climber richness (figure 2a). this contrasted to the species richness of non-climbing palms, which, when compared to other tropical regions, was highest in the neotropics and lowest in the afrotropics (figure 2b). determinants of species richness in climbing palms among contemporary climatic variables, temp (positive effect) and prec seas (negative effect) were the most important variables to explain climber species richness in the minimum adequate models (figures 3b,c; table 2). interestingly, prec seas showed contrasting effects for climbers vs. non-climbers (negative vs. positive sign). two

richness of non-climbing palms, which, when compared to other tropical regions, was highest in the neotropics and lowest in the afrotropics (figure 2b). determinants of species richness in climbing palms among contemporary climatic variables, temp (positive effect) and prec seas (negative effect) were the most important variables to explain climber species richness in the minimum adequate models (figures 3b,c; table 2). interestingly, prec seas showed contrasting effects for climbers vs. non-climbers (negative vs. positive sign). two additional contemporary climatic variables (prec, temp seas) were of further importance to explain species richness of non-climbing palms, but not climbers (table 2). paleoclimatic changes

seas (negative effect) were the most important variables to explain climber species richness in the minimum adequate models (figures 3b,c; table 2). interestingly, prec seas showed contrasting effects for climbers vs. non-climbers (negative vs. positive sign). two additional contemporary climatic variables (prec, temp seas) were of further importance to explain species richness of non-climbing palms, but not climbers (table 2). paleoclimatic changes (anomalies) since the pliocene (plioprec ) and miocene (mioprec ) showed negative effects on species richness of both climbers (figures 3d,e) and non-climbers (table 2). this indicated that areas that were relatively wetter during the late pliocene or late

contrasting effects for climbers vs. non-climbers (negative vs. positive sign). two additional contemporary climatic variables (prec, temp seas) were of further importance to explain species richness of non-climbing palms, but not climbers (table 2). paleoclimatic changes (anomalies) since the pliocene (plioprec ) and miocene (mioprec ) showed negative effects on species richness of both climbers (figures 3d,e) and non-climbers (table 2). this indicated that areas that were relatively wetter during the late pliocene or late miocene tend to have more palm species today than areas that were relatively drier in the past. in contrast to pliocene and miocene climate variables, precipitation

climbers (table 2). paleoclimatic changes (anomalies) since the pliocene (plioprec ) and miocene (mioprec ) showed negative effects on species richness of both climbers (figures 3d,e) and non-climbers (table 2). this indicated that areas that were relatively wetter during the late pliocene or late miocene tend to have more palm species today than areas that were relatively drier in the past. in contrast to pliocene and miocene climate variables, precipitation and temperature anomalies since the lgm were not important to explain species richness of climbing palms, but the lgmtemp effect was important for non-climbing palms (table 2). the minimum adequate models

indicated that areas that were relatively wetter during the late pliocene or late miocene tend to have more palm species today than areas that were relatively drier in the past. in contrast to pliocene and miocene climate variables, precipitation and temperature anomalies since the lgm were not important to explain species richness of climbing palms, but the lgmtemp effect was important for non-climbing palms (table 2). the minimum adequate models included canopy as an important predictor variable for the species richness of both non-climbing and climbing palms, but the effect was stronger for climbers than for non-climbing palms (table 2). the

past. in contrast to pliocene and miocene climate variables, precipitation and temperature anomalies since the lgm were not important to explain species richness of climbing palms, but the lgmtemp effect was important for non-climbing palms (table 2). the minimum adequate models included canopy as an important predictor variable for the species richness of both non-climbing and climbing palms, but the effect was stronger for climbers than for non-climbing palms (table 2). the canopy effect on climbing palm species richness was positive (figure 3a), supporting the hypothesis that richness increases with canopy height of forests. in addition to canopy and contemporary climate

the lgmtemp effect was important for non-climbing palms (table 2). the minimum adequate models included canopy as an important predictor variable for the species richness of both non-climbing and climbing palms, but the effect was stronger for climbers than for non-climbing palms (table 2). the canopy effect on climbing palm species richness was positive (figure 3a), supporting the hypothesis that richness increases with canopy height of forests. in addition to canopy and contemporary climate as well as paleo-climatic variables, biogeographic region (region) was also selected in the minimum adequate models (table 2). climbers showed a signiﬁcantly higher species richness in indomalaya

on climbing palm species richness was positive (figure 3a), supporting the hypothesis that richness increases with canopy height of forests. in addition to canopy and contemporary climate as well as paleo-climatic variables, biogeographic region (region) was also selected in the minimum adequate models (table 2). climbers showed a signiﬁcantly higher species richness in indomalaya relative to other tropical regions (figure 3f) whereas non-climbers showed a signiﬁcantly lower january 2015 | volume 5 | article 452 | 7 couvreur et al. figure 2 | the global distribution of species richness in (a) climbing and (b) non-climbing palms. global maps (in behrmann projection)

selected in the minimum adequate models (table 2). climbers showed a signiﬁcantly higher species richness in indomalaya relative to other tropical regions (figure 3f) whereas non-climbers showed a signiﬁcantly lower january 2015 | volume 5 | article 452 | 7 couvreur et al. figure 2 | the global distribution of species richness in (a) climbing and (b) non-climbing palms. global maps (in behrmann projection) show palm species richness for the mass centroids of geographic units as deﬁned by the international working group on taxonomic databases (tdwg level 3 units). differences in species richness between major regions are illustrated in boxplots (left).

indomalaya relative to other tropical regions (figure 3f) whereas non-climbers showed a signiﬁcantly lower january 2015 | volume 5 | article 452 | 7 couvreur et al. figure 2 | the global distribution of species richness in (a) climbing and (b) non-climbing palms. global maps (in behrmann projection) show palm species richness for the mass centroids of geographic units as deﬁned by the international working group on taxonomic databases (tdwg level 3 units). differences in species richness between major regions are illustrated in boxplots (left). species richness in the afrotropics relative to indomalaya and the neotropics (table 2). these results were

et al. figure 2 | the global distribution of species richness in (a) climbing and (b) non-climbing palms. global maps (in behrmann projection) show palm species richness for the mass centroids of geographic units as deﬁned by the international working group on taxonomic databases (tdwg level 3 units). differences in species richness between major regions are illustrated in boxplots (left). species richness in the afrotropics relative to indomalaya and the neotropics (table 2). these results were similar to the trends in raw species richness among realms (figure 2), but they additionally accounted for major differences in contemporary climate, canopy height, and

richness in (a) climbing and (b) non-climbing palms. global maps (in behrmann projection) show palm species richness for the mass centroids of geographic units as deﬁned by the international working group on taxonomic databases (tdwg level 3 units). differences in species richness between major regions are illustrated in boxplots (left). species richness in the afrotropics relative to indomalaya and the neotropics (table 2). these results were similar to the trends in raw species richness among realms (figure 2), but they additionally accounted for major differences in contemporary climate, canopy height, and paleoclimate. this suggests that deep-time historical effects beyond those driven

geographic units as deﬁned by the international working group on taxonomic databases (tdwg level 3 units). differences in species richness between major regions are illustrated in boxplots (left). species richness in the afrotropics relative to indomalaya and the neotropics (table 2). these results were similar to the trends in raw species richness among realms (figure 2), but they additionally accounted for major differences in contemporary climate, canopy height, and paleoclimate. this suggests that deep-time historical effects beyond those driven by quaternary and late neogene climate contribute substantially to the major differences in species richness of climbers and non-climbers among regions. diversification

results were similar to the trends in raw species richness among realms (figure 2), but they additionally accounted for major differences in contemporary climate, canopy height, and paleoclimate. this suggests that deep-time historical effects beyond those driven by quaternary and late neogene climate contribute substantially to the major differences in species richness of climbers and non-climbers among regions. diversification analyses comparing the ten classe diversiﬁcation models revealed that model 7, the cladogenetic speciation and extinction model, was the best ﬁtting model based on average aicc values from the 100 trees (table 3). this was not only the case for the unconstrained

have changed 9, 8, or 10 times across the history of palms. the phylorate plot shows an increase in mean diversiﬁcation rates at the crown node of subtribe calaminae (depicted by an arrow in figure 4a). calaminae is the largest clade of climbing palms containing around 20% of all palms species (dransﬁeld et al., 2008). we can also note an increase in diversiﬁcation rates for subtribe bactridinae, which contains one genus of climbing palms (desmoncus). other climber dominated clades do not show such increases. the six most probable css's (with a cumulative pp of 0.614, figure 5) show that signiﬁcant rate

of containing a rate shift. the shift in probability for subtribe calaminae is visible in both cases but not signiﬁcant when compared to other branches across palms. january 2015 | volume 5 | article 452 | 8 couvreur et al. figure 3 | key determinants of the global distribution of species richness of climbing palms. partial residual plots show the effects of continuous (a - e) and categorical (f) predictor variables from the minimum adequate model (table 2) on climbing palm species richness ancestral character reconstruction posterior mapping identiﬁed 4.9 average transformations from the non-climbing to the climbing state across the

5 | article 452 | 8 couvreur et al. figure 3 | key determinants of the global distribution of species richness of climbing palms. partial residual plots show the effects of continuous (a - e) and categorical (f) predictor variables from the minimum adequate model (table 2) on climbing palm species richness ancestral character reconstruction posterior mapping identiﬁed 4.9 average transformations from the non-climbing to the climbing state across the 100 trees, and 0.61 transformations in the opposite direction (reversals). these results do not include the three climbing species within the genera dypsis and chamaedorea, but in terms of timing of

predictor variables from the minimum adequate model (table 2) on climbing palm species richness ancestral character reconstruction posterior mapping identiﬁed 4.9 average transformations from the non-climbing to the climbing state across the 100 trees, and 0.61 transformations in the opposite direction (reversals). these results do not include the three climbing species within the genera dypsis and chamaedorea, but in terms of timing of the origin of climbers in palms this omission has no effect on the results. the ancestral state of the crown nodes of the subtribes calaminae, plectocomiinae, and ancistrophyllinae were strongly supported as climbing [pp(1) = 1.00; 0.99; 0.98,

leading to the crown nodes of the subtribes plectocomiinae and calaminae. the timing of the two last origins could not be estimated, having evolved along the stem node of the south american genus desmoncus and the southeast asian genus korthalsia (figure 4b). discussion environmental and geographic correlates of climbing palm species richness our results show that climbing palm species richness is associated with present-day climate (temperature, precipitation seasonality) and paleoclimatic changes since the late neogene (miocene and www.frontiersin.org evolution of climbing palms once all other variables in the model are statistically controlled for. sampling units as in figure 2. details of

plectocomiinae and calaminae. the timing of the two last origins could not be estimated, having evolved along the stem node of the south american genus desmoncus and the southeast asian genus korthalsia (figure 4b). discussion environmental and geographic correlates of climbing palm species richness our results show that climbing palm species richness is associated with present-day climate (temperature, precipitation seasonality) and paleoclimatic changes since the late neogene (miocene and www.frontiersin.org evolution of climbing palms once all other variables in the model are statistically controlled for. sampling units as in figure 2. details of boxplots as in figure 2. region: ind, indomalaya;

late neogene (miocene and www.frontiersin.org evolution of climbing palms once all other variables in the model are statistically controlled for. sampling units as in figure 2. details of boxplots as in figure 2. region: ind, indomalaya; afr, afrotropics; aus, australasia; neo, neotropics. pliocene; table 2; figure 3). an increase in species richness with higher temperatures was observed for both climbers and nonclimbers which is in line with our hypothesis (prediction 1a) and consistent with ﬁndings on global palm diversity patterns (kissling et al., 2012a). this relationship reﬂects the limited physiological and functional adaptations of palms, which reduce their survival in areas

consistent with ﬁndings on global palm diversity patterns (kissling et al., 2012a). this relationship reﬂects the limited physiological and functional adaptations of palms, which reduce their survival in areas with cold climates (tomlinson, 2006). however, the missing effect of temperature seasonality and the observed negative effect of precipitation seasonality on species richness of climbers are opposite to that of non-climbers (table 2). the negative correlation with precipitation seasonality also contrasts with woody climbing plants, which increase (rather than decrease) in diversity with precipitation seasonality (schnitzer, 2005). these trends could be explained by climbing palms being mostly found within calamoideae, a subfamily

to warm and humid environments (baker et al., 2000c; couvreur et al., 2011a). the effect of precipitation changes (rather than absolute levels of contemporary precipitation) is also supported by the paleoclimatic effects in our regression models, which showed that miocene and pliocene precipitation anomalies had negative effects on climbing palm species richness. hence, areas that were relatively wetter during the late miocene or late pliocene today have more climbing palm species than areas that were drier in the past. this suggests that multimillion-year non-equilibrium dynamics in diversity - climate relationships play a role in explaining present-day diversity of palms (blach-overgaard et

absolute levels of contemporary precipitation) is also supported by the paleoclimatic effects in our regression models, which showed that miocene and pliocene precipitation anomalies had negative effects on climbing palm species richness. hence, areas that were relatively wetter during the late miocene or late pliocene today have more climbing palm species than areas that were drier in the past. this suggests that multimillion-year non-equilibrium dynamics in diversity - climate relationships play a role in explaining present-day diversity of palms (blach-overgaard et al., 2013). january 2015 | volume 5 | article 452 | 9 couvreur et al. evolution of climbing palms table

this suggests that multimillion-year non-equilibrium dynamics in diversity - climate relationships play a role in explaining present-day diversity of palms (blach-overgaard et al., 2013). january 2015 | volume 5 | article 452 | 9 couvreur et al. evolution of climbing palms table 2 | multiple-predictor regression models to explain global species richness of climbing (n = 534) and non-climbing (n = 1911) palms. palm species richness climbers standard coefficient non-climbers p standard coefficient p intercept 2.798 *** 2.413 canopy 0.403 ** 0.227 * canopy2 na 0.063 n.s. prec − 0.271 n.s. temp 0.421 *** 0.539 *** * 0.219 n.s. 0.044 n.s.

in explaining present-day diversity of palms (blach-overgaard et al., 2013). january 2015 | volume 5 | article 452 | 9 couvreur et al. evolution of climbing palms table 2 | multiple-predictor regression models to explain global species richness of climbing (n = 534) and non-climbing (n = 1911) palms. palm species richness climbers standard coefficient non-climbers p standard coefficient p intercept 2.798 *** 2.413 canopy 0.403 ** 0.227 * canopy2 na 0.063 n.s. prec − 0.271 n.s. temp 0.421 *** 0.539 *** * 0.219 n.s. 0.044 n.s. prec seas −0.356 prec seas2 na temp seas − −0.507 temp seas2 − na

taxonomic databases (tdwg) level 3 units (n = 82 for climbers; n = 164 for non-climbers). ' - ' indicates not selected variables, 'na' not available variables or categorical levels. residual spatial autocorrelation was tested using moran's i values based on the closest neighbor of each tdwg level 3 unit. species richness and several continuous predictor variables (temp, prec seas, temp seas, prec dry, lgmtemp ) were log10 transformed. abbreviations of predictor variables: canopy, canopy height; prec, annual precipitation; temp, annual mean temperature; prec seas, precipitation seasonality; temp seas, temperature seasonality; prec dry, precipitation of driest quarter; temp cold, mean temperature

higher forests have a more patchy canopy because they have larger gaps in between tree trunks and fewer large trees reaching maximum heights (e.g., dipterocarp forests, see below, richards, 1996). second, larger trees can lead to larger tree-fall gaps, which has been shown to maintain a higher diversity of climber species in general (putz, 1984; schnitzer and carson, 2001). however, in contrast to nonmonocot lianas (schnitzer, 2005) the majority of climbing palms might be less tolerant of strong disturbances. this is probably also reﬂected in the negative relationship of palm species richness with prec seas (table 2), which is opposite to

has been shown to maintain a higher diversity of climber species in general (putz, 1984; schnitzer and carson, 2001). however, in contrast to nonmonocot lianas (schnitzer, 2005) the majority of climbing palms might be less tolerant of strong disturbances. this is probably also reﬂected in the negative relationship of palm species richness with prec seas (table 2), which is opposite to dicotyledonous lianas (schnitzer, 2005). third, climbing palms have developed particularly long and light stiff \"searcher-stems\" compared with many other woody climbers, and they also produce long and sharptoothed cirri and ﬂagella. these climbing traits are particularly effective for spanning large

they also produce long and sharptoothed cirri and ﬂagella. these climbing traits are particularly effective for spanning large gaps between large and tall canopy trees. as indicated by isnard and rowe (2008), there is relatively little developmental and anatomical difference between climbing and non-climbing palms compared with clades of woody species in which climbers can develop highly derived and complex stem structures via anomalous secondary growth. since the arguably more \"simple\" palm organization can develop effective \"liana\"-sized climbers it is intriguing that palms have not evolved the climbing habit more often and more widely, especially in the neotropics. with just seven

in which climbers can develop highly derived and complex stem structures via anomalous secondary growth. since the arguably more \"simple\" palm organization can develop effective \"liana\"-sized climbers it is intriguing that palms have not evolved the climbing habit more often and more widely, especially in the neotropics. with just seven species, the neotropics contain few representatives of the subfamily calamoideae (dransﬁeld et al., 2008) and the observed diversity anomaly among regions could thus be related to phylogenetic constraints within palms (bjorholm et al., 2006). compared with the calamoideae, few climbers have evolved within the arecoideae (the largest palm subfamily in general

(bjorholm et al., 2006). compared with the calamoideae, few climbers have evolved within the arecoideae (the largest palm subfamily in general and also within the neotropics, pintaud et al., 2008; figure 4). this is reﬂected in genera such as chamaedorea and dypsis (madagascar), which each contain one and two climbing species out of 110 and 140 species, respectively www.frontiersin.org (dransﬁeld et al., 2008). the failure to diversify as climbers might be related to the fact that they are non-spiny, in contrast to the majority of other climbers in the calamoideae. the only exception in arecoideae is the neotropical spiny genus desmoncus

the calamoideae, few climbers have evolved within the arecoideae (the largest palm subfamily in general and also within the neotropics, pintaud et al., 2008; figure 4). this is reﬂected in genera such as chamaedorea and dypsis (madagascar), which each contain one and two climbing species out of 110 and 140 species, respectively www.frontiersin.org (dransﬁeld et al., 2008). the failure to diversify as climbers might be related to the fact that they are non-spiny, in contrast to the majority of other climbers in the calamoideae. the only exception in arecoideae is the neotropical spiny genus desmoncus (figure 4), which has a currently

the failure to diversify as climbers might be related to the fact that they are non-spiny, in contrast to the majority of other climbers in the calamoideae. the only exception in arecoideae is the neotropical spiny genus desmoncus (figure 4), which has a currently reported diversity between 12 and 24 species (isnard et al., 2005; henderson, 2011). other tropical plant families show a similar pattern of low climber diversity in the neotropics compared to the paleotropics. for example, annonaceae, an important trf plant family (couvreur et al., 2011b), have a single climbing species in the neotropics compared to ∼500 species in

a currently reported diversity between 12 and 24 species (isnard et al., 2005; henderson, 2011). other tropical plant families show a similar pattern of low climber diversity in the neotropics compared to the paleotropics. for example, annonaceae, an important trf plant family (couvreur et al., 2011b), have a single climbing species in the neotropics compared to ∼500 species in the paleotropics. this might point toward potential phylogenetic constraints on certain plant families to successfully evolve and diversify as climbers in a given region. some families might be particularly successful in their evolution toward a climbing form in one region, for example,

24 species (isnard et al., 2005; henderson, 2011). other tropical plant families show a similar pattern of low climber diversity in the neotropics compared to the paleotropics. for example, annonaceae, an important trf plant family (couvreur et al., 2011b), have a single climbing species in the neotropics compared to ∼500 species in the paleotropics. this might point toward potential phylogenetic constraints on certain plant families to successfully evolve and diversify as climbers in a given region. some families might be particularly successful in their evolution toward a climbing form in one region, for example, the neotropics (e.g., bignoniaceae; lohmann et al.,

improved diversiﬁcation analysis approach (rabosky, 2014) conﬁrms this hypothesis. indeed, a total of nine rate shifts across palms were detected with the highest posterior probability. the analyses presented here suggest an important evolutionary role of climbers in explaining present-day palm diversity (hypothesis 2). our classe analyses indicated that across palms, species with a climbing habit diversiﬁed on average 1.3 times faster when compared to species with a non-climbing habit (λ111 > λ000 ; see table 3, prediction 2a). elevated diversiﬁcation rates for particular clades are highly dependent on young estimated crown node ages (linder, 2008). in our case, young crown node

rate shifts across palms were detected with the highest posterior probability. the analyses presented here suggest an important evolutionary role of climbers in explaining present-day palm diversity (hypothesis 2). our classe analyses indicated that across palms, species with a climbing habit diversiﬁed on average 1.3 times faster when compared to species with a non-climbing habit (λ111 > λ000 ; see table 3, prediction 2a). elevated diversiﬁcation rates for particular clades are highly dependent on young estimated crown node ages (linder, 2008). in our case, young crown node ages of large climbing genera such as calamus and daemonorops will strongly pull toward

the posterior (red = high diversiﬁcation rates; blue = low diversiﬁcation rates) resulting from the bamm output using the dated generic-level chronogram (couvreur et al., 2011a) in millions of years ago (mya). each terminal branch represents a genus. the ﬁgure also shows the phylogenetic distribution of genera that include climbing species either containing entirely or mainly climbing species (red circles) or mainly non-climbing species (one or two species of climbers, blue circles). all other non highlighted genera are strictly non-climbers. the arrow indicates the crown node of subtribe calaminae. numbers refer to subtribes or frontiers in genetics | evolutionary and population

blue = low diversiﬁcation rates) resulting from the bamm output using the dated generic-level chronogram (couvreur et al., 2011a) in millions of years ago (mya). each terminal branch represents a genus. the ﬁgure also shows the phylogenetic distribution of genera that include climbing species either containing entirely or mainly climbing species (red circles) or mainly non-climbing species (one or two species of climbers, blue circles). all other non highlighted genera are strictly non-climbers. the arrow indicates the crown node of subtribe calaminae. numbers refer to subtribes or frontiers in genetics | evolutionary and population genetics evolution of climbing palms genera (1:

from the bamm output using the dated generic-level chronogram (couvreur et al., 2011a) in millions of years ago (mya). each terminal branch represents a genus. the ﬁgure also shows the phylogenetic distribution of genera that include climbing species either containing entirely or mainly climbing species (red circles) or mainly non-climbing species (one or two species of climbers, blue circles). all other non highlighted genera are strictly non-climbers. the arrow indicates the crown node of subtribe calaminae. numbers refer to subtribes or frontiers in genetics | evolutionary and population genetics evolution of climbing palms genera (1: ancistrophyllinae; 2: korthalsiinae (korthalsia); 3: plectocomiinae;

using the dated generic-level chronogram (couvreur et al., 2011a) in millions of years ago (mya). each terminal branch represents a genus. the ﬁgure also shows the phylogenetic distribution of genera that include climbing species either containing entirely or mainly climbing species (red circles) or mainly non-climbing species (one or two species of climbers, blue circles). all other non highlighted genera are strictly non-climbers. the arrow indicates the crown node of subtribe calaminae. numbers refer to subtribes or frontiers in genetics | evolutionary and population genetics evolution of climbing palms genera (1: ancistrophyllinae; 2: korthalsiinae (korthalsia); 3: plectocomiinae; 4: calaminae; 5: desmoncus;

arrow indicates the crown node of subtribe calaminae. the ﬁve state changes from non-climbing to climbing are represented by a change in color from blue to red. photos illustrate typical climbing palms: (c) korthalsia zippelii from papua new guinea (photo: william j. baker, www.palmweb.org), and (d) laccosperma robustum, a frequent species throughout central africa (photo: thomas l. p. couvreur, www.palms.myspecies.info). january 2015 | volume 5 | article 452 | 12 couvreur et al. figure 5 | most probable combination of diversification rate shifts across palms. the six most credible rate shift sets (css) with the highest posterior probability using a bayes

higher diversiﬁcation rate associated to climbers, the impact of the climbing trait on the diversiﬁcation of speciﬁc clades is unclear (prediction 2a). based on the analysis of the bamm output, an increase in diversiﬁcation rates (figure 4a) is detected around the crown node of subtribe calaminae (subfamily calamoideae), the most species rich clade of climbing palms with around 500 southeast asian species (dransﬁeld et al., 2008). surprisingly, however, this increase is not signiﬁcant when compared to other diversiﬁcation rate shifts found across the family (figures 5 and 6). nevertheless, a previous www.frontiersin.org evolution of climbing palms circle size is proportional to

trait on the diversiﬁcation of speciﬁc clades is unclear (prediction 2a). based on the analysis of the bamm output, an increase in diversiﬁcation rates (figure 4a) is detected around the crown node of subtribe calaminae (subfamily calamoideae), the most species rich clade of climbing palms with around 500 southeast asian species (dransﬁeld et al., 2008). surprisingly, however, this increase is not signiﬁcant when compared to other diversiﬁcation rate shifts found across the family (figures 5 and 6). nevertheless, a previous www.frontiersin.org evolution of climbing palms circle size is proportional to the marginal probability of that shift. letters highlight clades that are

the family (figures 5 and 6). nevertheless, a previous www.frontiersin.org evolution of climbing palms circle size is proportional to the marginal probability of that shift. letters highlight clades that are associated with signiﬁcant rates shifts (ba, bactridinae; tr, trachycarpeae; ar, areceae). the increase in diversiﬁcation rates leading to the most species-rich clade of climbing palms, subtribe calaminae (ca), was not signiﬁcant within the ﬁrst six css. diversiﬁcation study using the same phylogenetic tree and data but based on a ml stepwise aic approach implemented in medusa identiﬁed two signiﬁcant rate increases at the stem nodes of calamus and daemonorops, both genera

study using the same phylogenetic tree and data but based on a ml stepwise aic approach implemented in medusa identiﬁed two signiﬁcant rate increases at the stem nodes of calamus and daemonorops, both genera belonging to calaminae (baker and couvreur, 2013b). in addition, calamus was found to have signiﬁcantly more species than expected under a constant birth - death rate model as well as have signiﬁcantly higher diversiﬁcation rates when compared to other genera under a high extinction rate assumption (baker and couvreur, 2013b). even though these results should be interpreted with caution, taken together they support the idea that the

diversification rate shifts across palms. the bayes factor (bf) (a) and marginal probabilities (b) of rate shifts are proportional to branch lengths. major palm clades associated with rate shifts are highlighted. for subtribe calaminae, the largest clade rates in palms resulting in the speciation of a ﬁfth of all palm species. the increase in diversiﬁcation rates in bactridinae does not appear to be related to the climbing habit that evolved in the genus desmoncus. indeed, this rate increase, which was also detected with the medusa analysis (rate shift 8 of baker and couvreur, 2013b that included desmoncus, bactris, and astrocaryum), concerns

climbing habit evolved a minimum of ﬁve independent times in palms: four times in calamoideae and once in arecoideae (figure 4b). these results are consistent with the inferences for calamoideae of baker et al. (2000a). however, the climbing habit arose at least seven independent times because of the three climbing species within dypsis and chamaedorea that were not taken into account in the analysis presented here (both genera coded as non-climbing). at least in chamaedorea, the single climbing species (c. elatior) is nested within the genus validating this assumption (cuenca and asmussen-lange, 2007). in addition, these independent evolutions will be valid

for calamoideae of baker et al. (2000a). however, the climbing habit arose at least seven independent times because of the three climbing species within dypsis and chamaedorea that were not taken into account in the analysis presented here (both genera coded as non-climbing). at least in chamaedorea, the single climbing species (c. elatior) is nested within the genus validating this assumption (cuenca and asmussen-lange, 2007). in addition, these independent evolutions will be valid if our coding assumptions of calamus and frontiers in genetics | evolutionary and population genetics evolution of climbing palms of climbing palms, a shift probability is visible along

both cases, but it is not signiﬁcant when compared to other rate shifts in palms. scale bars represent the value of the branch lengths for the bf (a) and the marginal probabilities (b). daemonorops as ancestrally \"climber\" are also correct. indeed, both these genera have a small proportion of non-climbing species (table 1), and in this study we did not take them into account (both genera coded as climbers). the phylogenetic relations within both genera and for the subtribe calaminae in general remain insufﬁciently understood and the exact placement of the non-climbing species is unresolved (baker et al., 2000b). our results

these genera have a small proportion of non-climbing species (table 1), and in this study we did not take them into account (both genera coded as climbers). the phylogenetic relations within both genera and for the subtribe calaminae in general remain insufﬁciently understood and the exact placement of the non-climbing species is unresolved (baker et al., 2000b). our results reveal an interesting pattern: the climbing habit appears to have had an impact on diversiﬁcation rates in calaminae, but not in other clades/genera where it evolved (ancistrophyllinae, plectocomiinae, korthalsia, desmoncus). for example, the climbing subtribes ancistrophyllinae and plectocomiinae (figure 4a) have relatively

reveal an interesting pattern: the climbing habit appears to have had an impact on diversiﬁcation rates in calaminae, but not in other clades/genera where it evolved (ancistrophyllinae, plectocomiinae, korthalsia, desmoncus). for example, the climbing subtribes ancistrophyllinae and plectocomiinae (figure 4a) have relatively old stem node ages associated with few extant species (dransﬁeld et al., 2008; sunderland, 2012; baker and couvreur, 2013a; faye et al., 2014). why did the climbing habit have such an important effect on the diversiﬁcation in a particular clade and not in other clades? one reason might be that different morphological adaptations are underlying the \"same\" trait (donoghue,

is associated with different combinations of morphological characters january 2015 | volume 5 | article 452 | 14 couvreur et al. (baker et al., 2000a; isnard, 2006). comparative studies of anatomical characters between calaminae (calamus and daemonorops) and desmoncus and subtribe plectocomiinae (although based on the study of a few species) suggest that the evolution of a more ﬂexible stem within subtribe calaminae could explain the outstanding diversiﬁcation of this group in terms of climbing mechanics (rowe et al., 2004; isnard, 2006). in addition, the evolution of a unique climbing organ (the ﬂagellum, a modiﬁed inﬂorescence) has taken place within calamus

a unique climbing organ (the ﬂagellum, a modiﬁed inﬂorescence) has taken place within calamus (dransﬁeld et al., 2008) and might have provided an additional advantage over other climbing structures in palms. moreover, the presence of a \"knee\" a swelling at the junction of the leaf sheath and petiole, in most species in subtribe calaminae and, to some extent, the african genus eremospatha has been suggested as a potentially important trait for enhancing leaf strength at this important stress point (isnard and rowe, 2008). overall, the repeated homoplasious occurrence of the climbing habit in the calamoideae ('clustered homoplasy') could also indicate a

grapnel spines on climbing organs. finally, some life history traits might also act against increasing diversiﬁcation rates. indeed, plectocomiinae and the genera korthalsia (korthalsiinae, figure 4c) and laccosperma (ancistrophyllinae, figure 4d) are hapaxanthic (individual stems die after a single ﬂowering event), a condition which is generally not associated with high species richness across palms (dransﬁeld et al., 2008). thus, several morphological novelties of subtribe calaminae absent from other climbing genera (more ﬂexible stems, the evolution of a ﬂagellum in calamus, the better mechanical role of the leaf sheath under stress, and the presence of a knee) possibly played decisive functional roles

the evolution of a ﬂagellum in calamus, the better mechanical role of the leaf sheath under stress, and the presence of a knee) possibly played decisive functional roles in explaining the diversiﬁcation of this group when compared to other climbing palms. explaining geographic differences in diversiﬁcation rates (e.g., numerous climbing species in southeast asia vs. few in the neotropics) might be related to the fact that a particular trait only increases diversiﬁcation rates under certain environmental conditions (de queiroz, 2002). for instance, a highly dynamic geographic setting linked to a complex biogeographic history of southeast asia (hall, 2009) might be important

al., 2012; thomas et al., 2012; bacon et al., 2013; buerki et al., 2013; richardson et al., 2014). www.frontiersin.org evolution of climbing palms one important and characteristic plant family in southeast asia is dipterocarpaceae (appanah and turnbull, 1998), with major peaks in diversity in malaysia and mainland southeast asia. its species are generally large emergent trees, which often dominate the upper canopy of the region's forests, contributing to the much taller stature of the forests in this region relative to, e.g., the neotropics (richards, 1996). interestingly, the ﬁrst record of the family dipterocarpaceae in southeast asia (borneo) after its dispersal from

al., 2011) as well as the start of its estimated radiation in the region (morley, 2000) correspond to the timing of increased diversiﬁcation rates within the calamoideae (oligocene to early miocene). we therefore hypothesize that the diversiﬁcation of dipterocarps in combination with the evolution of several morpho-anatomical traits in calaminae species could have triggered the radiation of climbing palms in this region. ecological opportunity responses of one clade based on the success of another have been suggested in other cases such as ferns (schneider et al., 2004) or ants (moreau et al., 2006) during the radiation of angiosperms. conclusion global diversity

based on the success of another have been suggested in other cases such as ferns (schneider et al., 2004) or ants (moreau et al., 2006) during the radiation of angiosperms. conclusion global diversity patterns of climbing palms show a diversity anomaly relative to other palms, with a strong peak of species richness in southeast asian rain forests and low species richness in other regions (e.g., the neotropics). present-day climate, forest canopy heights, and paleoclimatic changes in the neogene and quaternary can partly explain this pattern, but they do not provide a sufﬁcient explanation for the extraordinary diversiﬁcation of climbing palms in

in other cases such as ferns (schneider et al., 2004) or ants (moreau et al., 2006) during the radiation of angiosperms. conclusion global diversity patterns of climbing palms show a diversity anomaly relative to other palms, with a strong peak of species richness in southeast asian rain forests and low species richness in other regions (e.g., the neotropics). present-day climate, forest canopy heights, and paleoclimatic changes in the neogene and quaternary can partly explain this pattern, but they do not provide a sufﬁcient explanation for the extraordinary diversiﬁcation of climbing palms in southeast asia. an increase in diversiﬁcation rates in calaminae,

of southeast asia, and/or ecological opportunity responses to the regional presence and diversiﬁcation of tall canopy trees such as dipterocarps. we suggest that, in addition to climatic and paleoclimatic factors, such historical and evolutionary contingencies play an important role in explaining present-day biodiversity across trfs. new datasets (e.g., global high-quality species distribution data at ﬁne resolutions, well resolved species-level phylogenies, and additional morphological trait data) as well as novel analytical tools will likely increase our knowledge of palm diversiﬁcation and our understanding of tropical rain forest evolution in the future. acknowledgments we thank john dransﬁeld for species-level information about climbing palms

the regional presence and diversiﬁcation of tall canopy trees such as dipterocarps. we suggest that, in addition to climatic and paleoclimatic factors, such historical and evolutionary contingencies play an important role in explaining present-day biodiversity across trfs. new datasets (e.g., global high-quality species distribution data at ﬁne resolutions, well resolved species-level phylogenies, and additional morphological trait data) as well as novel analytical tools will likely increase our knowledge of palm diversiﬁcation and our understanding of tropical rain forest evolution in the future. acknowledgments we thank john dransﬁeld for species-level information about climbing palms and yanis bouchenak-khelladi for discussions and help with

datasets (e.g., global high-quality species distribution data at ﬁne resolutions, well resolved species-level phylogenies, and additional morphological trait data) as well as novel analytical tools will likely increase our knowledge of palm diversiﬁcation and our understanding of tropical rain forest evolution in the future. acknowledgments we thank john dransﬁeld for species-level information about climbing palms and yanis bouchenak-khelladi for discussions and help with r scripts and diversiﬁcation analyses. we are also grateful to two anonymous reviewers whose comments improved this manuscript. we thank james richardson, valenti rull, and toby pennington for inviting us to submit our work to this special issue.

### NA
e-mail rafa.rubiodecasas@gmail.com or dsoltis@ufl.edu 1 2  background and aims organismal evolution tends to be closely associated with ecological conditions. however, the extent to which this association constrains adaptation or diversification into new habitats remains unclear. we studied habitat evolution in the hyper-diverse angiosperm clade saxifragales.  methods we used species-level phylogenies for approx. 950 species to analyse the evolution of habitat shifts as well as their influence on plant diversification. we combined habitat characterization based on floristic assignments and state-of-the art phylogenetic comparative methods to estimate within- and across-habitat diversification patterns.  key results our analyses showed that saxifragales diversified

2  background and aims organismal evolution tends to be closely associated with ecological conditions. however, the extent to which this association constrains adaptation or diversification into new habitats remains unclear. we studied habitat evolution in the hyper-diverse angiosperm clade saxifragales.  methods we used species-level phylogenies for approx. 950 species to analyse the evolution of habitat shifts as well as their influence on plant diversification. we combined habitat characterization based on floristic assignments and state-of-the art phylogenetic comparative methods to estimate within- and across-habitat diversification patterns.  key results our analyses showed that saxifragales diversified into multiple habitats from a

### NA
centre, uppsala university, uppsala, sweden ‡naturalis biodiversity center, leiden, the netherlands §natural history museum, university of oslo, oslo, norway ¶universit\023e de lille 1, laboratoire g\023en\023etique et evolution des populations v\023eg\023etales, villeneuve d'ascq, france keywords: abstract angiosperms; dioecy; diversification; phylogenetic methods; sister clades. in angiosperms, dioecious clades tend to have fewer species than their nondioecious sister clades. this departure from the expected equal species richness in the standard sister clade test has been interpreted as implying that dioecious clades diversify less and has initiated a series of studies suggesting that dioecy might be an 'evolutionary dead end'. however, two of us recently

history museum, university of oslo, oslo, norway ¶universit\023e de lille 1, laboratoire g\023en\023etique et evolution des populations v\023eg\023etales, villeneuve d'ascq, france keywords: abstract angiosperms; dioecy; diversification; phylogenetic methods; sister clades. in angiosperms, dioecious clades tend to have fewer species than their nondioecious sister clades. this departure from the expected equal species richness in the standard sister clade test has been interpreted as implying that dioecious clades diversify less and has initiated a series of studies suggesting that dioecy might be an 'evolutionary dead end'. however, two of us recently showed that the 'equal species richness' null hypothesis is not valid in

clades. this departure from the expected equal species richness in the standard sister clade test has been interpreted as implying that dioecious clades diversify less and has initiated a series of studies suggesting that dioecy might be an 'evolutionary dead end'. however, two of us recently showed that the 'equal species richness' null hypothesis is not valid in the case of derived char acters, such as dioecy, and proposed a new test for sister clade comparisons; preliminary results, using a data set available in the litterature, indicated that dioecious clades migth diversify more than expected. however, it is crucial for this

into account in the available data set. here, we present a new data set that was obtained by searching the phylogenetic literature on more than 600 completely dioecious angiosperm genera and identifying 115 sister clade pairs for which dioecy is likely to be derived (including > 50% of the dioecious species). applying the new sister clade test to this new dataset, we confirm the preliminary result that dioecy is associated with an increased diversification rate, a result that does not support the idea that dioecy is an evolutionary dead end in angiosperms. the traits usually associated with dioecy, that is, an

result that does not support the idea that dioecy is an evolutionary dead end in angiosperms. the traits usually associated with dioecy, that is, an arborescent growth form, abiotic pollination, fleshy fruits or a tropical distribution, do not influence the diversification rate. rather than a low diversification rate, the observed species richness patterns of dioecious clades seem to be better explained by a low transition rate to dioecy and frequent losses. introduction dioecy, that is, having separate sexes, is the norm in many animal clades, but rather rare in flowering plants, occurring in only about 6% of all species (renner &

the observed species richness patterns of dioecious clades seem to be better explained by a low transition rate to dioecy and frequent losses. introduction dioecy, that is, having separate sexes, is the norm in many animal clades, but rather rare in flowering plants, occurring in only about 6% of all species (renner & ricklefs, 1995). indeed, dioecy comes with some costs (whitfield, 2004; otto, 2009; lehtonen et al., 2012). first, unisexual individuals lack the reproductive assurance that benefits self-compatible hermaphroditic specorrespondence: jos k€ afer, universit\002e lyon 1, centre national de la recherche scientifique, umr5558, laboratoire de biom\002etrie et biologie evolutive, villeurbanne,

level, either because fewer offspring are produced or because a large production of seeds is carried by a limited number of individuals, thus increasing offspring competition (janzen, 1971; maynard-smith, 1978; bawa, 1980; heilbuth et al., 2001; charlesworth, 2009). however, some of these costs are expected to be similar for animal species with separate sexes, which are nonetheless numerous, and the rareness of dioecious plants requires further explanation. apart from the debate over whether or not the ancestor of the angiosperms had bisexual flowers (e.g. endress & doyle, 2009), no reasonable doubt seems to exist about the fact that bisexual flowers occurred

dispersed by animals, a tropical distribution, abiotic pollination, small or inconspicuous flowers or inflorescences, and an arborescent growth form (e.g. bawa, 1980; renner & ricklefs, 1995), could be explained by the fact that selection for separate sexes should be particularly strong when associated with these traits. for example, abiotically pollinated species are thought to face a high risk of selfing (lloyd & yates, 1982; cox & grubb, 1991), which should increase selection for dioecy. nevertheless, the relative rareness of dioecy in plants helped establish the idea that it is deleterious for plants in the long run. an angiosperm-wide study of the

for dioecy. nevertheless, the relative rareness of dioecy in plants helped establish the idea that it is deleterious for plants in the long run. an angiosperm-wide study of the diversification of dioecious lineages, using sister clade comparisons, was conducted by heilbuth (2000), who found that dioecious clades often possess fewer species than their nondioecious sister clades and concluded that dioecious clades diversify less. further work, reinterpreting the associations of dioecy with the traits mentioned earlier, proposed that dioecious species suffer from higher extinction rates and that these traits could alleviate the 'handicaps' associated with dioecy (heilbuth et al., 2001; vamosi &

the diversification of dioecious lineages, using sister clade comparisons, was conducted by heilbuth (2000), who found that dioecious clades often possess fewer species than their nondioecious sister clades and concluded that dioecious clades diversify less. further work, reinterpreting the associations of dioecy with the traits mentioned earlier, proposed that dioecious species suffer from higher extinction rates and that these traits could alleviate the 'handicaps' associated with dioecy (heilbuth et al., 2001; vamosi & otto, 2002; vamosi et al., 2003). recently, k€afer & mousset (2014) revisited the relatively simple sister clade comparison framework for its applicability to derived traits, such as dioecy.

recently, k€afer & mousset (2014) revisited the relatively simple sister clade comparison framework for its applicability to derived traits, such as dioecy. heilbuth's (2000) conclusion that dioecious clades diversify less was based on the expectation that, under equal diversification rates, the two sister clades in each pair should be equally species rich. this expectation is indeed the one commonly used in sister clade comparisons (e.g. slowinski & guyer, 1993; wiegmann et al., 1993; mooers & moller, 1996; heilbuth, 2000; davies et al., 2004; davis et al., 2010): if two groups originate from the 1479 same ancestor and diversify at equal rates,

indeed the one commonly used in sister clade comparisons (e.g. slowinski & guyer, 1993; wiegmann et al., 1993; mooers & moller, 1996; heilbuth, 2000; davies et al., 2004; davis et al., 2010): if two groups originate from the 1479 same ancestor and diversify at equal rates, the expectation of equal species richness seems justified. however, if the criterion to select sister clade pairs is a trait that arose after the two clades split, unequal species richness can arise even if both clades diversify at the same rate (k€ afer & mousset, 2014). this typically is the case for derived characters, such

davies et al., 2004; davis et al., 2010): if two groups originate from the 1479 same ancestor and diversify at equal rates, the expectation of equal species richness seems justified. however, if the criterion to select sister clade pairs is a trait that arose after the two clades split, unequal species richness can arise even if both clades diversify at the same rate (k€ afer & mousset, 2014). this typically is the case for derived characters, such as dioecy: one of the sisters harbours the ancestral state and inherited this state from the common ancestor of the sisters, as represented in

expect that the dioecious clades will on average be smaller than their nondioecious sisters, even when their diversification rates are the same. k€afer & mousset (2014) developed a statistical test to take this selection bias into account and showed, using simulated trees, that the classical statistical tests (which expect equal species richness in the sister clades) have a very high type i error rate and thus easily lead to the conclusion that there are differences in the diversification rate, whereas in reality there are not. they applied the new method to the data set compiled by heilbuth (2000) and reached the

heilbuth's (2000) data did not distinguish between transitions to and reversions from dioecy, this could explain why k€afer & mousset detected increased diversification associated with dioecy. furthermore, heilbuth constructed her data set to be conservative in accordance with her hypothesis that dioecious clades diversify less, by excluding genera containing dioecious species that occur in the sister clades and by including hermaphrodites that occur in the dioecious clades. thus, when testing the inverse hypothesis that dioecious clades diversify more, heilbuth's data set is not conservative anymore as it tends to overestimate the species richness of the dioecious clades compared to their sisters.

dioecious clades diversify less, by excluding genera containing dioecious species that occur in the sister clades and by including hermaphrodites that occur in the dioecious clades. thus, when testing the inverse hypothesis that dioecious clades diversify more, heilbuth's data set is not conservative anymore as it tends to overestimate the species richness of the dioecious clades compared to their sisters. finally, more recent phylogenetic studies have changed some of the sister group relationships since the year 2000. the goal of this study was to construct a new data set of dioecious clades and their nearest nondioecious sisters to apply the test

preliminary result of k€afer & mousset. this data set furthermore offers some insights into the evolutionary history of dioecy in angiosperms. data and methods sister clade data set as starting point for a new data set of dioecious clades and their sisters, we used the list of genera containing dioecious species compiled by susanne renner (september 2004). only completely or predominantly dioecious genera were included, as resolving infra-generic sister clades appeared to be practically impossible at the scale of all angiosperms. for these genera (more than 600), we searched for the latest phylogenetic studies including the genus (a total of ~170

included, as resolving infra-generic sister clades appeared to be practically impossible at the scale of all angiosperms. for these genera (more than 600), we searched for the latest phylogenetic studies including the genus (a total of ~170 papers) to identify its sisters, and we checked for the occurrence of dioecious species in the sister genera. when a sister genus was not present in the list of dioecious genera, the presence or absence of dioecy was assessed using the phylogenetic literature and online floras. dioecious sister genera were merged, until a nondioecious sister group was found, to obtain two monophyletic sister clades,

clades, one dioecious and the other nondioecious. in several cases, nondioecious genera occurred within larger dioecious groups. in some cases, these are known reversals, but in the absence of literature, these genera were considered as reversals and included as such in the dioecious group based on the number of nondioecious species and their phylogenetic position. if the phylogenetic studies did not enable the unambiguous determination of a single sister clade because of ill-resolved nodes (e.g. nepenthaceae and sarcobatus), the dioecious clades were nevertheless included in the data set if none of the possible sister clades was dioecious, and information about the

data set if none of the possible sister clades was dioecious, and information about the smallest and largest possible sister clades was retained in the data set. in addition to the literature study, we performed an ancestral reconstruction using parsimony for the malpighiales, a large order with approximately 3400 dioecious species, based on a recent phylogeny (xi et al., 2012). data and methods are presented in data s1. species counts for dioecious groups were taken from renner's database, unless the phylogenetic studies indicated that the species count was different (e.g. dioscorea). species counts for sister groups were found in the literature

possible sister clades was retained in the data set. in addition to the literature study, we performed an ancestral reconstruction using parsimony for the malpighiales, a large order with approximately 3400 dioecious species, based on a recent phylogeny (xi et al., 2012). data and methods are presented in data s1. species counts for dioecious groups were taken from renner's database, unless the phylogenetic studies indicated that the species count was different (e.g. dioscorea). species counts for sister groups were found in the literature or looked up in tropicos (www.tropicos.org) and the plant list (www.theplantlist.org). for each pair of sister clades, we

an ancestral reconstruction using parsimony for the malpighiales, a large order with approximately 3400 dioecious species, based on a recent phylogeny (xi et al., 2012). data and methods are presented in data s1. species counts for dioecious groups were taken from renner's database, unless the phylogenetic studies indicated that the species count was different (e.g. dioscorea). species counts for sister groups were found in the literature or looked up in tropicos (www.tropicos.org) and the plant list (www.theplantlist.org). for each pair of sister clades, we thus have an estimate of the total number of species in the dioecious clade (d), the number

the malpighiales, a large order with approximately 3400 dioecious species, based on a recent phylogeny (xi et al., 2012). data and methods are presented in data s1. species counts for dioecious groups were taken from renner's database, unless the phylogenetic studies indicated that the species count was different (e.g. dioscorea). species counts for sister groups were found in the literature or looked up in tropicos (www.tropicos.org) and the plant list (www.theplantlist.org). for each pair of sister clades, we thus have an estimate of the total number of species in the dioecious clade (d), the number of dioecious species in the dioecious

unless the phylogenetic studies indicated that the species count was different (e.g. dioscorea). species counts for sister groups were found in the literature or looked up in tropicos (www.tropicos.org) and the plant list (www.theplantlist.org). for each pair of sister clades, we thus have an estimate of the total number of species in the dioecious clade (d), the number of dioecious species in the dioecious clade (d*), the number of species in the sister clade (s), and the number of nondioecious species in the sister clade (s*). in the remainder of the text, we will refer to the total number of species

different (e.g. dioscorea). species counts for sister groups were found in the literature or looked up in tropicos (www.tropicos.org) and the plant list (www.theplantlist.org). for each pair of sister clades, we thus have an estimate of the total number of species in the dioecious clade (d), the number of dioecious species in the dioecious clade (d*), the number of species in the sister clade (s), and the number of nondioecious species in the sister clade (s*). in the remainder of the text, we will refer to the total number of species in both clades m = d + s, and the

found in the literature or looked up in tropicos (www.tropicos.org) and the plant list (www.theplantlist.org). for each pair of sister clades, we thus have an estimate of the total number of species in the dioecious clade (d), the number of dioecious species in the dioecious clade (d*), the number of species in the sister clade (s), and the number of nondioecious species in the sister clade (s*). in the remainder of the text, we will refer to the total number of species in both clades m = d + s, and the total number of species in both clades when species

the plant list (www.theplantlist.org). for each pair of sister clades, we thus have an estimate of the total number of species in the dioecious clade (d), the number of dioecious species in the dioecious clade (d*), the number of species in the sister clade (s), and the number of nondioecious species in the sister clade (s*). in the remainder of the text, we will refer to the total number of species in both clades m = d + s, and the total number of species in both clades when species that arose after secondary transitions to dioecy or reversions to the

species in the dioecious clade (d), the number of dioecious species in the dioecious clade (d*), the number of species in the sister clade (s), and the number of nondioecious species in the sister clade (s*). in the remainder of the text, we will refer to the total number of species in both clades m = d + s, and the total number of species in both clades when species that arose after secondary transitions to dioecy or reversions to the ancestral state were excluded, m* = d* + s*. ª 2014 the authors. j. evol. biol. 27 (2014) 1478 -

clade (d*), the number of species in the sister clade (s), and the number of nondioecious species in the sister clade (s*). in the remainder of the text, we will refer to the total number of species in both clades m = d + s, and the total number of species in both clades when species that arose after secondary transitions to dioecy or reversions to the ancestral state were excluded, m* = d* + s*. ª 2014 the authors. j. evol. biol. 27 (2014) 1478 - 1490 journal of evolutionary biology ª 2014 european society for evolutionary biology dioecy and

species in the sister clade (s), and the number of nondioecious species in the sister clade (s*). in the remainder of the text, we will refer to the total number of species in both clades m = d + s, and the total number of species in both clades when species that arose after secondary transitions to dioecy or reversions to the ancestral state were excluded, m* = d* + s*. ª 2014 the authors. j. evol. biol. 27 (2014) 1478 - 1490 journal of evolutionary biology ª 2014 european society for evolutionary biology dioecy and diversification in angiosperms if the

ancestral state were excluded, m* = d* + s*. ª 2014 the authors. j. evol. biol. 27 (2014) 1478 - 1490 journal of evolutionary biology ª 2014 european society for evolutionary biology dioecy and diversification in angiosperms if the sister clade could not be identified unambiguously, or the number of species was not well known, the minimum and maximum clade sizes were used for the sister clade, smin and smax, respectively. the corresponding total sizes of the clade pair are thus mmax = smax + d and mmin = smin + d. the relative species richness of the dioecious clade within

identified unambiguously, or the number of species was not well known, the minimum and maximum clade sizes were used for the sister clade, smin and smax, respectively. the corresponding total sizes of the clade pair are thus mmax = smax + d and mmin = smin + d. the relative species richness of the dioecious clade within a clade pair is the fraction md . unless explicitly stated, the \001mean \003 relative species richness of the dioecious clades, md , is calculated using d and mmax throughout the text. analysis of diversification these data were analysed with the method for sister

clade, smin and smax, respectively. the corresponding total sizes of the clade pair are thus mmax = smax + d and mmin = smin + d. the relative species richness of the dioecious clade within a clade pair is the fraction md . unless explicitly stated, the \001mean \003 relative species richness of the dioecious clades, md , is calculated using d and mmax throughout the text. analysis of diversification these data were analysed with the method for sister clade comparisons described by k€afer & mousset (2014). in short, the method estimates the length of the stem branches of each of

is calculated using d and mmax throughout the text. analysis of diversification these data were analysed with the method for sister clade comparisons described by k€afer & mousset (2014). in short, the method estimates the length of the stem branches of each of the clades based on the number of species in the clades. as the probability of the occurrence of a transition is assumed to be proportional to the branch length, the ratio of the stem branch lengths can be used to calculate the relative probability that either one of the clades is dioecious, assuming equal diversification rates under the

to be proportional to the branch length, the ratio of the stem branch lengths can be used to calculate the relative probability that either one of the clades is dioecious, assuming equal diversification rates under the null hypothesis. the test statistic in this method is the mean relative \001 \003 species richness of the dioecious clade md . for each clade pair i, di is the number of species in the dioecious clade, and mi the number of species in both clades. the relative species richness for one pair, mdii , is thus a value \001 \003 p between 0 and

calculate the relative probability that either one of the clades is dioecious, assuming equal diversification rates under the null hypothesis. the test statistic in this method is the mean relative \001 \003 species richness of the dioecious clade md . for each clade pair i, di is the number of species in the dioecious clade, and mi the number of species in both clades. the relative species richness for one pair, mdii , is thus a value \001 \003 p between 0 and 1, as is the average md ¼ 1c ci¼1 mdii for all c clade pairs in the data

is dioecious, assuming equal diversification rates under the null hypothesis. the test statistic in this method is the mean relative \001 \003 species richness of the dioecious clade md . for each clade pair i, di is the number of species in the dioecious clade, and mi the number of species in both clades. the relative species richness for one pair, mdii , is thus a value \001 \003 p between 0 and 1, as is the average md ¼ 1c ci¼1 mdii for all c clade pairs in the data set. following k€ afer & mousset (2014), a resampling approach

under the null hypothesis. the test statistic in this method is the mean relative \001 \003 species richness of the dioecious clade md . for each clade pair i, di is the number of species in the dioecious clade, and mi the number of species in both clades. the relative species richness for one pair, mdii , is thus a value \001 \003 p between 0 and 1, as is the average md ¼ 1c ci¼1 mdii for all c clade pairs in the data set. following k€ afer & mousset (2014), a resampling approach is used to \001 \003 obtain

assumption that there are no differences in diversification and that transitions occur at a constant rate, we can compute the probability that a clade pair with total size m harbours a transition on one of its stem branches. this allows us to calculate, for each clade pair, an expected relative species richness of the dioecious clade and a p-value for the difference between the observed and expected relative species richness (data s2, eqns 4 and 5). when testing for differences in diversification within individ- 1481 ual clade pairs, we used two-sided p-values, as we do not expect a priori that all

compute the probability that a clade pair with total size m harbours a transition on one of its stem branches. this allows us to calculate, for each clade pair, an expected relative species richness of the dioecious clade and a p-value for the difference between the observed and expected relative species richness (data s2, eqns 4 and 5). when testing for differences in diversification within individ- 1481 ual clade pairs, we used two-sided p-values, as we do not expect a priori that all dioecious clades will be larger than expected. some could be smaller as well due to other factors influencing

ual clade pairs, we used two-sided p-values, as we do not expect a priori that all dioecious clades will be larger than expected. some could be smaller as well due to other factors influencing the diversification rate. the sister clade test of k€ afer & mousset (2014) resamples the given species richness of both clades within a pair, only swapping the character state within a clade pair in a randomized manner, and is thus conditioned on the observed values of both m and d. the test per clade pair calculates the expected value of d given m and is thus only

tree with a constant diversification rate, an assumption that the sister clade test does not need (the null hypothesis of the sister clade test is that the diversification rate is equal within a clade pair). consequently, in the test per clade pair, differences between the expected and the observed relative species richness could be due to any deviation from this assumption. life-history and ecological traits to test whether certain life-history or ecological traits could influence the diversification rate of the dioecious groups, data on seed dispersal (biotic vs. abiotic), mode of pollination (biotic vs. abiotic) and geographical distribution (tropical vs. nontropical)

dwarf shrubs (< 10 cm) were included in the category 'herbs'. when multiple characters were represented in a clade, it was discarded from this analysis. the character states of the nondioecious sister clades were not taken into account. for each of the four traits, we calculated the average observed relative species richness of the dioecious clade \001 \003 ( md ) per character state (e.g. abiotic vs. biotic pollination). the character states were compared using a wilcoxon signed-rank test to test for differences between character states in this relative species richness. also, an anova was performed on the observed relative species

each of the four traits, we calculated the average observed relative species richness of the dioecious clade \001 \003 ( md ) per character state (e.g. abiotic vs. biotic pollination). the character states were compared using a wilcoxon signed-rank test to test for differences between character states in this relative species richness. also, an anova was performed on the observed relative species richness, combining the four traits as fixed effects, to test for interactions between these traits. results identification of dioecious clades and their sisters our update of susanne renner's list of 994 dioecious genera with 14 600 species yielded a

species richness of the dioecious clade \001 \003 ( md ) per character state (e.g. abiotic vs. biotic pollination). the character states were compared using a wilcoxon signed-rank test to test for differences between character states in this relative species richness. also, an anova was performed on the observed relative species richness, combining the four traits as fixed effects, to test for interactions between these traits. results identification of dioecious clades and their sisters our update of susanne renner's list of 994 dioecious genera with 14 600 species yielded a list of 1018 genera containing 16 161 species. this increase is

this relative species richness. also, an anova was performed on the observed relative species richness, combining the four traits as fixed effects, to test for interactions between these traits. results identification of dioecious clades and their sisters our update of susanne renner's list of 994 dioecious genera with 14 600 species yielded a list of 1018 genera containing 16 161 species. this increase is mainly due to taxonomic revisions modifying the number of species ª 2014 the authors. j. evol. biol. 27 (2014) 1478 - 1490 journal of evolutionary biology ª 2014 european society for evolutionary biology 1482 € er et

the observed relative species richness, combining the four traits as fixed effects, to test for interactions between these traits. results identification of dioecious clades and their sisters our update of susanne renner's list of 994 dioecious genera with 14 600 species yielded a list of 1018 genera containing 16 161 species. this increase is mainly due to taxonomic revisions modifying the number of species ª 2014 the authors. j. evol. biol. 27 (2014) 1478 - 1490 journal of evolutionary biology ª 2014 european society for evolutionary biology 1482 € er et al. j . k af in certain genera, but we

test for interactions between these traits. results identification of dioecious clades and their sisters our update of susanne renner's list of 994 dioecious genera with 14 600 species yielded a list of 1018 genera containing 16 161 species. this increase is mainly due to taxonomic revisions modifying the number of species ª 2014 the authors. j. evol. biol. 27 (2014) 1478 - 1490 journal of evolutionary biology ª 2014 european society for evolutionary biology 1482 € er et al. j . k af in certain genera, but we also identified some dioecious species in closely related genera that were not included

due to taxonomic revisions modifying the number of species ª 2014 the authors. j. evol. biol. 27 (2014) 1478 - 1490 journal of evolutionary biology ª 2014 european society for evolutionary biology 1482 € er et al. j . k af in certain genera, but we also identified some dioecious species in closely related genera that were not included in the original list. we identified 115 sister clade pairs that most likely have a nondioecious ancestor, based on a literature survey and an additional ancestral state reconstruction using maximum parsimony for the malpighiales. the data set was split into two according

studies that were consulted for the retained sister clade pairs. of the 89 sister clade pairs from heilbuth (2000) that were also used by k€afer & mousset (2014), 48 pairs are present in the current data set. four of those are completely identical, and the remaining 44 have seen their species counts updated, either through small adjustments of the counts themselves, due to revisions of the phylogenetic relations leading to the inclusion or exclusion of some taxa in the sister groups (13 pairs), or to largely or completely different sister groups (12 pairs). forty-one of the clade pairs used by heilbuth

phylogenies confirming the sister clades heilbuth used, also leading us to discard these dioecious clades. the net effect is that of the 115 dioecious clades included in our extended data set, 67 have not been included in sister clade comparisons before. table s3 shows the taxonomic distribution of the dioecious species taken into account in this study (see also fig. 2). the data set of sister clade pairs includes 8301 dioecious species (strict data set; 9021 in the extended data set), of the 16 161 known to us, that is, 51 - 56%. in comparison, heilbuth's data set that was analysed

the 115 dioecious clades included in our extended data set, 67 have not been included in sister clade comparisons before. table s3 shows the taxonomic distribution of the dioecious species taken into account in this study (see also fig. 2). the data set of sister clade pairs includes 8301 dioecious species (strict data set; 9021 in the extended data set), of the 16 161 known to us, that is, 51 - 56%. in comparison, heilbuth's data set that was analysed by k€afer & mousset contained 7558 dioecious species, but according to heilbuth, this count systematically included hermaphroditic (reverted) species in the

also fig. 2). the data set of sister clade pairs includes 8301 dioecious species (strict data set; 9021 in the extended data set), of the 16 161 known to us, that is, 51 - 56%. in comparison, heilbuth's data set that was analysed by k€afer & mousset contained 7558 dioecious species, but according to heilbuth, this count systematically included hermaphroditic (reverted) species in the dioecious groups and is thus an overestimation. of the 51 orders containing dioecious species, 39 are repre- sented in our strict data set, 40 in the extended data set. only one of the nonrepresented orders had more

8301 dioecious species (strict data set; 9021 in the extended data set), of the 16 161 known to us, that is, 51 - 56%. in comparison, heilbuth's data set that was analysed by k€afer & mousset contained 7558 dioecious species, but according to heilbuth, this count systematically included hermaphroditic (reverted) species in the dioecious groups and is thus an overestimation. of the 51 orders containing dioecious species, 39 are repre- sented in our strict data set, 40 in the extended data set. only one of the nonrepresented orders had more than 50 dioecious species: the apiales, for which ancestral dioecy has

known to us, that is, 51 - 56%. in comparison, heilbuth's data set that was analysed by k€afer & mousset contained 7558 dioecious species, but according to heilbuth, this count systematically included hermaphroditic (reverted) species in the dioecious groups and is thus an overestimation. of the 51 orders containing dioecious species, 39 are repre- sented in our strict data set, 40 in the extended data set. only one of the nonrepresented orders had more than 50 dioecious species: the apiales, for which ancestral dioecy has been found (schlessmann, 2010). sister clade comparison given the ambiguities in the data, we performed the

heilbuth, this count systematically included hermaphroditic (reverted) species in the dioecious groups and is thus an overestimation. of the 51 orders containing dioecious species, 39 are repre- sented in our strict data set, 40 in the extended data set. only one of the nonrepresented orders had more than 50 dioecious species: the apiales, for which ancestral dioecy has been found (schlessmann, 2010). sister clade comparison given the ambiguities in the data, we performed the statistical test for sister clade comparisons (k€ afer & mousset, 2014) eight times: on the strict or extended data sets, using the maximum (mmax) or minimum (mmin)

using m* and d* is a violation of the assumptions of the test. tests using m* and d* were nevertheless performed for completeness. for the strict data set, the null hypothesis of equal diversification is rejected in all cases, in favour of the alternative hypothesis that dioecious clades are more species rich than expected. using the extended data set, the comparisons of the dioecious clade with the largest possible sister clades yielded a p-value above 5%. it is, however, very unlikely that in all ambiguous cases the largest sister clade is the correct one, and the comparisons using the smallest possible

with the largest possible sister clades yielded a p-value above 5%. it is, however, very unlikely that in all ambiguous cases the largest sister clade is the correct one, and the comparisons using the smallest possible sister clades yielded pvalues well below 5%. the p-values for the comparisons that excluded species that originated after secondary transitions or reversions were slightly higher than the comparisons that included them. this is counterintuitive at a first glance, because if dioecy increases diversification, leaving out dioecious groups in the sister clade and reverted groups in the dioecious clade should accentuate the differences. however, reducing the

after secondary transitions or reversions were slightly higher than the comparisons that included them. this is counterintuitive at a first glance, because if dioecy increases diversification, leaving out dioecious groups in the sister clade and reverted groups in the dioecious clade should accentuate the differences. however, reducing the number of species in the clades might simply affect the power of the statistical test. analysis of individual clade pairs for each clade pair, we compared the observed relative species richness of the dioecious clade with the expected species richness, as calculated using eqn [5] in data s2. moreover, this equation allows us

out dioecious groups in the sister clade and reverted groups in the dioecious clade should accentuate the differences. however, reducing the number of species in the clades might simply affect the power of the statistical test. analysis of individual clade pairs for each clade pair, we compared the observed relative species richness of the dioecious clade with the expected species richness, as calculated using eqn [5] in data s2. moreover, this equation allows us to calculate a p-value for this difference; observed and expected species richness per clade pair, as well as the associated p-value, are given in table s4 and

groups in the dioecious clade should accentuate the differences. however, reducing the number of species in the clades might simply affect the power of the statistical test. analysis of individual clade pairs for each clade pair, we compared the observed relative species richness of the dioecious clade with the expected species richness, as calculated using eqn [5] in data s2. moreover, this equation allows us to calculate a p-value for this difference; observed and expected species richness per clade pair, as well as the associated p-value, are given in table s4 and represented graphically in fig. 2. on average, the observed

statistical test. analysis of individual clade pairs for each clade pair, we compared the observed relative species richness of the dioecious clade with the expected species richness, as calculated using eqn [5] in data s2. moreover, this equation allows us to calculate a p-value for this difference; observed and expected species richness per clade pair, as well as the associated p-value, are given in table s4 and represented graphically in fig. 2. on average, the observed relative species richness is below the expected value (the average observed - expected value is \0020.025 for the extended and \0020.0098 for the strict data

as calculated using eqn [5] in data s2. moreover, this equation allows us to calculate a p-value for this difference; observed and expected species richness per clade pair, as well as the associated p-value, are given in table s4 and represented graphically in fig. 2. on average, the observed relative species richness is below the expected value (the average observed - expected value is \0020.025 for the extended and \0020.0098 for the strict data set). note that these expected values were obtained assuming that clade pairs were drawn from a tree with a constant diversification rate (see data and methods, analysis

1478 - 1490 journal of evolutionary biology ª 2014 european society for evolutionary biology dioecy and diversification in angiosperms 1483 fig. 2 representation of the analysis of individual clade pairs per angiosperm order. each point corresponds to a clade pair: a point left of the straight line indicates a lower-than-expected species richness of the dioecious clade; right of the straight line, the species richness of the dioecious clade is higher than expected. the expected relative species richness for each clade pair is calculated using eqn [5] in data s2. the distance of each point from the straight line is the inverse

evolutionary biology dioecy and diversification in angiosperms 1483 fig. 2 representation of the analysis of individual clade pairs per angiosperm order. each point corresponds to a clade pair: a point left of the straight line indicates a lower-than-expected species richness of the dioecious clade; right of the straight line, the species richness of the dioecious clade is higher than expected. the expected relative species richness for each clade pair is calculated using eqn [5] in data s2. the distance of each point from the straight line is the inverse of the log-transformed pvalue; dotted vertical lines correspond to p = 0.05.

analysis of individual clade pairs per angiosperm order. each point corresponds to a clade pair: a point left of the straight line indicates a lower-than-expected species richness of the dioecious clade; right of the straight line, the species richness of the dioecious clade is higher than expected. the expected relative species richness for each clade pair is calculated using eqn [5] in data s2. the distance of each point from the straight line is the inverse of the log-transformed pvalue; dotted vertical lines correspond to p = 0.05. the angiosperm phylogeny was obtained from the angiosperm phylogeny website (stevens, 2001 onwards;

000 iterations using a one-sided test (alternative hypothesis: dioecious clades are larger than expected). s2). the fact that the dioecious clades are mostly smaller than expected in this analysis, whereas the combined sister clade test found they were larger than expected, indicates that dioecy has a tendency to evolve in species-poor clades, a phenomenon due to the selection bias. as the probability for a branch to harbour a transition to dioecy depends on its length, dioecious clades will have on average longer stem branches (and less species) than randomly drawn clades. in a tree in which the diversification rates vary, clades

they were larger than expected, indicates that dioecy has a tendency to evolve in species-poor clades, a phenomenon due to the selection bias. as the probability for a branch to harbour a transition to dioecy depends on its length, dioecious clades will have on average longer stem branches (and less species) than randomly drawn clades. in a tree in which the diversification rates vary, clades with low diversification rates will have longer branches and thus more often present transitions to dioecy, without dioecy being the cause of the low diversification rate. this result does thus not contradict, but rather support the

the p-value for the pair including amborella is below this critical value. low p-values (< 0.01, and thus not significant given the bonferroni correction) for decreased diversification were obtained for simmondsia (simmondsiaceae, caryophyllales), gynerium (poaceae), symmeria (polygonaceae) and the lepidobotryaceae (celastrales). these dioecious clades consist of 1 or 2 (lepidobotryaceae) species being sister to a clade with more than 1000 species. four clade pairs in which the dioecious clade is larger than expected had p-values < 0.01 (but not significant): the clade formed by the four cucurbitales families, the putranjivaceae, hydrocharitaceae and the clade consisting of the aquifoliaceae and helwingiaceae. dioecy

critical value. low p-values (< 0.01, and thus not significant given the bonferroni correction) for decreased diversification were obtained for simmondsia (simmondsiaceae, caryophyllales), gynerium (poaceae), symmeria (polygonaceae) and the lepidobotryaceae (celastrales). these dioecious clades consist of 1 or 2 (lepidobotryaceae) species being sister to a clade with more than 1000 species. four clade pairs in which the dioecious clade is larger than expected had p-values < 0.01 (but not significant): the clade formed by the four cucurbitales families, the putranjivaceae, hydrocharitaceae and the clade consisting of the aquifoliaceae and helwingiaceae. dioecy and other traits to test whether the growth form, the

expected had p-values < 0.01 (but not significant): the clade formed by the four cucurbitales families, the putranjivaceae, hydrocharitaceae and the clade consisting of the aquifoliaceae and helwingiaceae. dioecy and other traits to test whether the growth form, the distribution and the modes of dispersal and pollination affected the relative species richness of the dioecious clades, we used the extended data set containing 115 clade pairs, as no hypothesis regarding the ancestral state is needed for table 2 life-history and ecological traits and the proportion of dioecious species in the extended data set, using the maximum estimated values of m (mmax)

form, the distribution and the modes of dispersal and pollination affected the relative species richness of the dioecious clades, we used the extended data set containing 115 clade pairs, as no hypothesis regarding the ancestral state is needed for table 2 life-history and ecological traits and the proportion of dioecious species in the extended data set, using the maximum estimated values of m (mmax) and the minimal estimates of m (mmin, in parentheses). p-values are calculated with wilcoxon's signed-rank test. dispersal biotic abiotic pollination biotic abiotic distribution tropical nontropical growth form herb vine or liana shrub tree number of clades \001d\003

23 6 23 55 0.442 0.413 0.169 0.343 0.114 (0.243)* 0.0700 (0.138)† 0.583 (0.647)‡ m (0.456) (0.482) (0.212) (0.374) *herbs compared to vines, shrubs and trees. †herbs and vines compared to shrubs and trees. ‡herbs, vines and shrubs compared to trees. such a comparison. the characters and the observed relative species richness of each of the dioecious clades are given in table s4. as shown in table 2, none of these characters significantly influenced the relative species richness of the dioecious clade. however, several trends can be detected: clades that consist of herbaceous or viney dioecious plants seem to harbour higher

†herbs and vines compared to shrubs and trees. ‡herbs, vines and shrubs compared to trees. such a comparison. the characters and the observed relative species richness of each of the dioecious clades are given in table s4. as shown in table 2, none of these characters significantly influenced the relative species richness of the dioecious clade. however, several trends can be detected: clades that consist of herbaceous or viney dioecious plants seem to harbour higher diversification rates than shrubby or arborescent dioecious clades. furthermore, clades with abiotic modes of dispersal have a higher d/m ratio than clades with biotic dispersal, a

diversification rates than shrubby or arborescent dioecious clades. furthermore, clades with abiotic modes of dispersal have a higher d/m ratio than clades with biotic dispersal, a result that is close to significance. a factorial anova (table s5) yielded no evidence for an effect of the interaction between the traits on species richness in dioecious clades. discussion dioecy increases diversification k€ afer & mousset (2014) showed that it is important to distinguish between derived and ancestral states of a trait in sister clade comparisons; ignoring this distinction leads to high type i error rates in statistical tests and could lead one to

discussion dioecy increases diversification k€ afer & mousset (2014) showed that it is important to distinguish between derived and ancestral states of a trait in sister clade comparisons; ignoring this distinction leads to high type i error rates in statistical tests and could lead one to wrongly attribute differences in species richness to differences in diversification rates. they proposed a new test for sister clade comparisons to study the effect of derived character states. in the current study, we present a data set of dioecious clades and their nondioecious sister clades, tailored for the requirements of this test, that is, including

is, including only clade pairs for which the ancestral state could be assigned as nondioecious with sufficient confidence. using the resampling test k€ afer & mousset proposed, we confirm their conclusion that dioecy seems to increase diversification moderately. thus, the hypothesis that dioecy is an evolutionary dead end, because dioecious species would suffer from higher extinction and/or lower speciation rates (heilbuth, 2000; barrett, 2002; vamosi et al., 2003), is not supported by our study. nevertheless, our results suggest that dioecy increases diversification rates only moderately, and it cannot be considered a 'key innovation' (hunter, 1998). questions have been raised as to

residues in the female ª 2014 the authors. j. evol. biol. 27 (2014) 1478 - 1490 journal of evolutionary biology ª 2014 european society for evolutionary biology dioecy and diversification in angiosperms flowers (endress & doyle, 2009) and included it in the sister clade data set. this clade is extremely species poor (one species) compared with its sister clade (the other angiosperms) and thus has a very low relative species richness (cf. fig. 2). removing it would have strengthened our conclusion that dioecy increases species richness. our sister clade data set includes more than 50% of the known dioecious species across

female ª 2014 the authors. j. evol. biol. 27 (2014) 1478 - 1490 journal of evolutionary biology ª 2014 european society for evolutionary biology dioecy and diversification in angiosperms flowers (endress & doyle, 2009) and included it in the sister clade data set. this clade is extremely species poor (one species) compared with its sister clade (the other angiosperms) and thus has a very low relative species richness (cf. fig. 2). removing it would have strengthened our conclusion that dioecy increases species richness. our sister clade data set includes more than 50% of the known dioecious species across the angiosperm orders.

biology ª 2014 european society for evolutionary biology dioecy and diversification in angiosperms flowers (endress & doyle, 2009) and included it in the sister clade data set. this clade is extremely species poor (one species) compared with its sister clade (the other angiosperms) and thus has a very low relative species richness (cf. fig. 2). removing it would have strengthened our conclusion that dioecy increases species richness. our sister clade data set includes more than 50% of the known dioecious species across the angiosperm orders. we did not include transitions to dioecy that occurred within a genus, excluding many recent transitions

& doyle, 2009) and included it in the sister clade data set. this clade is extremely species poor (one species) compared with its sister clade (the other angiosperms) and thus has a very low relative species richness (cf. fig. 2). removing it would have strengthened our conclusion that dioecy increases species richness. our sister clade data set includes more than 50% of the known dioecious species across the angiosperm orders. we did not include transitions to dioecy that occurred within a genus, excluding many recent transitions to dioecy. the focus of our data set is thus on dioecious clades that have

extremely species poor (one species) compared with its sister clade (the other angiosperms) and thus has a very low relative species richness (cf. fig. 2). removing it would have strengthened our conclusion that dioecy increases species richness. our sister clade data set includes more than 50% of the known dioecious species across the angiosperm orders. we did not include transitions to dioecy that occurred within a genus, excluding many recent transitions to dioecy. the focus of our data set is thus on dioecious clades that have survived for some time, and it could be biased if these recent clades are short-lived

of our data set is thus on dioecious clades that have survived for some time, and it could be biased if these recent clades are short-lived dioecious clades with low diversification rates. however, it is difficult to imagine that dioecy as a trait that managed to establish itself in a species population has such severe negative effects on diversification that it would drive the clade to extinction immediately upon its establishment, although this remains to be tested. this sister clade comparison test relies on the assumption that the transition to dioecy did not coincide with the split into two sister clades.

1978; maurice et al., 1994), or when viewing dioecy as an adaptation to specific conditions (e.g. bawa, 1980). however, it is possible that dioecy, as it concerns the reproductive organs of plants, could be cladogenetic, that is, coincides with a speciation that gives rise to one dioecious and one nondioecious species due to reproductive isolation. in such a case, the mathematics used by k€afer & mousset (2014) would not correctly describe the null hypothesis. however, it should be noted that extinctions are probably common (cf. ricklefs, 2007; rabosky, 2009), and it is thus very likely that the nondioecious species that originated

one nondioecious species due to reproductive isolation. in such a case, the mathematics used by k€afer & mousset (2014) would not correctly describe the null hypothesis. however, it should be noted that extinctions are probably common (cf. ricklefs, 2007; rabosky, 2009), and it is thus very likely that the nondioecious species that originated in such a cladogenetic transition has no extant representatives. these extinctions tend to place the transitions on the branches (see fig. 1) rather than on the nodes of a tree, and the anagenetic model might still be a better approximation than a purely cladogenetic one that does not

steps, this assumption seems justified. secondary transitions to dioecy after the breakdown of ancestral dioecy seem to have a higher transition rate (e.g. cucurbitaceae and euphorbiaceae, see below), but those secondary transitions were not included as clade pairs in the data set. when we calculated deviations from the expected relative species richness for each clade pair individu- 1485 ally (fig. 2 and table s4), we found that the observed species richness of a majority of the dioecious clades was lower than expected. at first sight, this seems to contradict the results from the combined sister clade comparison, where we found that

higher transition rate (e.g. cucurbitaceae and euphorbiaceae, see below), but those secondary transitions were not included as clade pairs in the data set. when we calculated deviations from the expected relative species richness for each clade pair individu- 1485 ally (fig. 2 and table s4), we found that the observed species richness of a majority of the dioecious clades was lower than expected. at first sight, this seems to contradict the results from the combined sister clade comparison, where we found that dioecy increases diversification. however, as k€ afer & mousset (2014) have argued, a selection of sister clades from a

k€ afer & mousset (2014) that transitions to dioecy are 'attracted' to long branches merely by chance, and illustrates the utility of resampling statistics that make no prior assumption on clade size distributions. for each clade pair, we calculated a p-value permitting assessment of whether or not the observed relative species richness could have been obtained with equal diversification rates. first, the underlying distribution for species richness asymmetries is uniform, and extreme asymmetries can be obtained quite easily (farris, 1976). basing a test of species richness on one single clade pair is therefore not a very powerful method to detect differences

by chance, and illustrates the utility of resampling statistics that make no prior assumption on clade size distributions. for each clade pair, we calculated a p-value permitting assessment of whether or not the observed relative species richness could have been obtained with equal diversification rates. first, the underlying distribution for species richness asymmetries is uniform, and extreme asymmetries can be obtained quite easily (farris, 1976). basing a test of species richness on one single clade pair is therefore not a very powerful method to detect differences in diversification rates. in our data set, after correcting for multiple testing, only one clade

each clade pair, we calculated a p-value permitting assessment of whether or not the observed relative species richness could have been obtained with equal diversification rates. first, the underlying distribution for species richness asymmetries is uniform, and extreme asymmetries can be obtained quite easily (farris, 1976). basing a test of species richness on one single clade pair is therefore not a very powerful method to detect differences in diversification rates. in our data set, after correcting for multiple testing, only one clade pair could be considered as having a significant difference in species richness, which is the pair consisting of amborella

quite easily (farris, 1976). basing a test of species richness on one single clade pair is therefore not a very powerful method to detect differences in diversification rates. in our data set, after correcting for multiple testing, only one clade pair could be considered as having a significant difference in species richness, which is the pair consisting of amborella and the rest of the angiosperms. if dioecy indeed positively influences the evolutionary success of a group, as our overall results suggest, it might also have contributed to the survival of 'living fossils' such as amborella by diminishing their risk of extinction.

the phylogeny by xi et al. (2012) showed that this was also the case for the ancestor of the euphorbiaceae, peraceae and rafflesiaceae. furthermore, we identified 38 families for which dioecy most probably was ancestral (see tables s1 and s2). some of these families have only one to a few species, so dioecy could be recent as well, but moderately sized families with ancestral dioecy are not rare (e.g. pandanaceae, restionaceae and menispermaceae have over 500 species). the fact that dioecy is observed 'mostly at the tips of phylogenetic trees' (heilbuth, 2000) was considered as an indication that dioecious clades are

38 families for which dioecy most probably was ancestral (see tables s1 and s2). some of these families have only one to a few species, so dioecy could be recent as well, but moderately sized families with ancestral dioecy are not rare (e.g. pandanaceae, restionaceae and menispermaceae have over 500 species). the fact that dioecy is observed 'mostly at the tips of phylogenetic trees' (heilbuth, 2000) was considered as an indication that dioecious clades are short-lived, but it might just be a characteristic of a trait evolving with a low transition rate (for a similar observation, see scotland & sanderson, 2004).

of the gymnosperms and only 6% of the angiosperms (ming et al., 2011), which could be explained by a low transition rate, although other explanations are possible. interestingly, dioecy is not necessarily the major breeding system in the groups in which it is ancestral. the apiales contain only 139 dioecious species (2.5%), which is below the angiosperm average. although the clades formed by the cucurbitaceae, begoniaceae, datiscaceae and tetramelaceae; the euphorbiaceae, peraceae and rafflesiaceae; and the moraceae and urticaceae have higher-than-average proportions of dioecious species, they are not the majority (11%, 27% and 43%, respectively). in the cucurbitales (zhang et al.,

breeding system in the groups in which it is ancestral. the apiales contain only 139 dioecious species (2.5%), which is below the angiosperm average. although the clades formed by the cucurbitaceae, begoniaceae, datiscaceae and tetramelaceae; the euphorbiaceae, peraceae and rafflesiaceae; and the moraceae and urticaceae have higher-than-average proportions of dioecious species, they are not the majority (11%, 27% and 43%, respectively). in the cucurbitales (zhang et al., 2006; schaefer & renner, 2010; de boer et al., 2012) and the moraceae (datwyler & weiblen, 2004), dioecy seems to have been lost and regained quite frequently, and genera having both dioecious and monoecious

they are not the majority (11%, 27% and 43%, respectively). in the cucurbitales (zhang et al., 2006; schaefer & renner, 2010; de boer et al., 2012) and the moraceae (datwyler & weiblen, 2004), dioecy seems to have been lost and regained quite frequently, and genera having both dioecious and monoecious species are common. the euphorbiaceae are less well studied, but the occurrence of both dioecious and monoecious species in a large number of genera seems to indicate the same trend. the frequent reversions from dioecy might also contribute to explain why dioecy is rare, despite the increase in diversification that we

schaefer & renner, 2010; de boer et al., 2012) and the moraceae (datwyler & weiblen, 2004), dioecy seems to have been lost and regained quite frequently, and genera having both dioecious and monoecious species are common. the euphorbiaceae are less well studied, but the occurrence of both dioecious and monoecious species in a large number of genera seems to indicate the same trend. the frequent reversions from dioecy might also contribute to explain why dioecy is rare, despite the increase in diversification that we identified. the idea that dioecy is a stable terminal state and maybe an evo- lutionary 'dead end'

dioecy, secondary transitions to dioecy will occur from monoecious ancestors (e.g. weiblen, 2000; volz & renner, 2008), but the question remains unanswered regarding the pathway through which dioecy originally evolved from a hermaphroditic ancestor (cf. barrett, 2002). another question that will merit attention is whether dioecy can be lost in species with sex chromosomes (cf. ming et al., 2011). the traits associated with dioecy have no effect on diversification we used this sister clade data set to investigate whether there was a link between the increase or decrease in diversification in dioecious clades and certain traits that are reported to be

dioecy is seen as a handicap, the ecological and life-history traits to which it is correlated are considered as ways to overcome or avoid this handicap. this is the case for the 'seed shadow handicap', that is, only half of the individuals contribute to the dispersal of seeds in dioecious species (heilbuth et al., 2001). as dispersal through animals leads to different, more efficient dispersal dynamics, this would reduce the effect of the handicap (vamosi et al., 2007) and thus decrease the extinction risk (vamosi & vamosi, 2004). however, and although the result is only marginally significant, we found that dioecious

divided over the two sexes (charnov et al., 1976; givnish, 1980). ª 2014 the authors. j. evol. biol. 27 (2014) 1478 - 1490 journal of evolutionary biology ª 2014 european society for evolutionary biology dioecy and diversification in angiosperms a second trait showing a marginally significant effect on the relative species richness of the dioecious clades is the growth form: dioecious clades consisting of herbs, vines or lianas tended to have higher diversification rates than those consisting of shrubs and trees. dioecy is thought to evolve more easily in trees than in species with smaller plant sizes, as a way to

showing a marginally significant effect on the relative species richness of the dioecious clades is the growth form: dioecious clades consisting of herbs, vines or lianas tended to have higher diversification rates than those consisting of shrubs and trees. dioecy is thought to evolve more easily in trees than in species with smaller plant sizes, as a way to avoid geitonogamy in self-compatible species (maynard-smith, 1978), or to allow more efficient pollen dispersal in outcrossing species (bawa, 1980). again, these hypotheses make no direct prediction on diversification. our results do not show an unambiguous effect of plant size on the species

clades is the growth form: dioecious clades consisting of herbs, vines or lianas tended to have higher diversification rates than those consisting of shrubs and trees. dioecy is thought to evolve more easily in trees than in species with smaller plant sizes, as a way to avoid geitonogamy in self-compatible species (maynard-smith, 1978), or to allow more efficient pollen dispersal in outcrossing species (bawa, 1980). again, these hypotheses make no direct prediction on diversification. our results do not show an unambiguous effect of plant size on the species richness of dioecious clades, as the dioecious clades with the lowest relative species

lianas tended to have higher diversification rates than those consisting of shrubs and trees. dioecy is thought to evolve more easily in trees than in species with smaller plant sizes, as a way to avoid geitonogamy in self-compatible species (maynard-smith, 1978), or to allow more efficient pollen dispersal in outcrossing species (bawa, 1980). again, these hypotheses make no direct prediction on diversification. our results do not show an unambiguous effect of plant size on the species richness of dioecious clades, as the dioecious clades with the lowest relative species richness are shrubs, that is, those of intermediate size. we did not

species with smaller plant sizes, as a way to avoid geitonogamy in self-compatible species (maynard-smith, 1978), or to allow more efficient pollen dispersal in outcrossing species (bawa, 1980). again, these hypotheses make no direct prediction on diversification. our results do not show an unambiguous effect of plant size on the species richness of dioecious clades, as the dioecious clades with the lowest relative species richness are shrubs, that is, those of intermediate size. we did not find any effects of the mode of pollination, a trait that was found to be correlated with dioecy (renner & ricklefs, 1995; vamosi et al.,

species (maynard-smith, 1978), or to allow more efficient pollen dispersal in outcrossing species (bawa, 1980). again, these hypotheses make no direct prediction on diversification. our results do not show an unambiguous effect of plant size on the species richness of dioecious clades, as the dioecious clades with the lowest relative species richness are shrubs, that is, those of intermediate size. we did not find any effects of the mode of pollination, a trait that was found to be correlated with dioecy (renner & ricklefs, 1995; vamosi et al., 2003), on the relative species richness of dioecious clades. mostly, dioecy is considered

as the dioecious clades with the lowest relative species richness are shrubs, that is, those of intermediate size. we did not find any effects of the mode of pollination, a trait that was found to be correlated with dioecy (renner & ricklefs, 1995; vamosi et al., 2003), on the relative species richness of dioecious clades. mostly, dioecy is considered more favourable for wind-pollinated species than for insect-pollinated species, for several reasons. either dioecy more often evolves in wind-pollinated species because selfing is more likely in those species (lloyd & yates, 1982; cox & grubb, 1991), or wind pollination more often evolves

is, those of intermediate size. we did not find any effects of the mode of pollination, a trait that was found to be correlated with dioecy (renner & ricklefs, 1995; vamosi et al., 2003), on the relative species richness of dioecious clades. mostly, dioecy is considered more favourable for wind-pollinated species than for insect-pollinated species, for several reasons. either dioecy more often evolves in wind-pollinated species because selfing is more likely in those species (lloyd & yates, 1982; cox & grubb, 1991), or wind pollination more often evolves in dioecious groups due to pollinator scarcity and because selfing is not possible

size. we did not find any effects of the mode of pollination, a trait that was found to be correlated with dioecy (renner & ricklefs, 1995; vamosi et al., 2003), on the relative species richness of dioecious clades. mostly, dioecy is considered more favourable for wind-pollinated species than for insect-pollinated species, for several reasons. either dioecy more often evolves in wind-pollinated species because selfing is more likely in those species (lloyd & yates, 1982; cox & grubb, 1991), or wind pollination more often evolves in dioecious groups due to pollinator scarcity and because selfing is not possible (friedman & barrett, 2009).

pollination, a trait that was found to be correlated with dioecy (renner & ricklefs, 1995; vamosi et al., 2003), on the relative species richness of dioecious clades. mostly, dioecy is considered more favourable for wind-pollinated species than for insect-pollinated species, for several reasons. either dioecy more often evolves in wind-pollinated species because selfing is more likely in those species (lloyd & yates, 1982; cox & grubb, 1991), or wind pollination more often evolves in dioecious groups due to pollinator scarcity and because selfing is not possible (friedman & barrett, 2009). another hypothesis supposed a negative effect of dioecy on diversification in

correlated with dioecy (renner & ricklefs, 1995; vamosi et al., 2003), on the relative species richness of dioecious clades. mostly, dioecy is considered more favourable for wind-pollinated species than for insect-pollinated species, for several reasons. either dioecy more often evolves in wind-pollinated species because selfing is more likely in those species (lloyd & yates, 1982; cox & grubb, 1991), or wind pollination more often evolves in dioecious groups due to pollinator scarcity and because selfing is not possible (friedman & barrett, 2009). another hypothesis supposed a negative effect of dioecy on diversification in insect-pollinated species and explained this as a side

selfing is more likely in those species (lloyd & yates, 1982; cox & grubb, 1991), or wind pollination more often evolves in dioecious groups due to pollinator scarcity and because selfing is not possible (friedman & barrett, 2009). another hypothesis supposed a negative effect of dioecy on diversification in insect-pollinated species and explained this as a side effect of competition for pollinators between the sexes (vamosi & otto, 2002), which would also explain why dioecious species often have inconspicuous flowers (but see bawa, 1980; renner & ricklefs, 1995). again, our results support the hypotheses that are not based on differences in

to pollinator scarcity and because selfing is not possible (friedman & barrett, 2009). another hypothesis supposed a negative effect of dioecy on diversification in insect-pollinated species and explained this as a side effect of competition for pollinators between the sexes (vamosi & otto, 2002), which would also explain why dioecious species often have inconspicuous flowers (but see bawa, 1980; renner & ricklefs, 1995). again, our results support the hypotheses that are not based on differences in diversification. similarly, in the light of our results, it seems difficult to explain the association between dioecy and a tropical distribution by the reduction in

derived traits might end up on the same branch merely by chance, and an angiosperm-wide analysis could detect such an effect. 1487 sister clade comparisons are (still) a useful tool these past years have seen the development of several methods for studying the evolution of traits and their effect on species diversification on the phylogeny. these were first limited to anagenetic binary traits (bisse - binary state speciation and extinction model, maddison et al., 2007; fitzjohn et al., 2009) and were rapidly extended to multistate traits (musse), quantitative traits (quasse, fitzjohn, 2010), incorporating cladogenetic transitions (classe, goldberg & igi\002c, 2012; and

systems and other traits on plant diversification (goldberg et al., 2010; johnson et al., 2011; mayrose et al., 2011; drummond et al., 2012b; goldberg & igi\002c, 2012; igic & busch, 2013). thus, they could in principle provide important data on the evolution of dioecy and the long-term survival of dioecious species, being a more powerful alternative than the sister clade comparisons we used here. however, a recent investigation by davis et al. (2013) sheds light on the limits of these methods, by showing that the accuracy and precision of parameter estimation of bisse is greatly reduced when one character represents <

< 10% of the taxa at the tips. the main problem thus seems to be intrinsically related to the rareness of dioecy in angiosperms (~6%). we made similar observations in our attempts to use bisse to study the effect of dioecy in the genus silene (caryophyllaceae), a genus with 700 species of which fewer than 20 are dioecious (fig. s2). sister clade comparisons, while they undeniably lack some of the advantages of these methods that are explicitly phylogeny based, are a more robust alternative. the selection of only those groups that present the character of interest, if correctly handled (k€ afer

methods that are explicitly phylogeny based, are a more robust alternative. the selection of only those groups that present the character of interest, if correctly handled (k€ afer & mousset, 2014), enables an increase in statistical power with only a moderate increase in computation time. furthermore, only relative differences in species richness are considered, so that inherent variations in diversification rates of widely divergent clades are automatically corrected for. the statistical inference is parameter- and, to a large extent, distribution free. this, of course, diminishes the statistical power if the correct underlying model can be identified, but provides a better safeguard

### NA
rieseberg,1,4 sarah p. otto1 olyploidy (or whole-genome duplication) is a widespread feature of plant genomes, but its importance to evolution has long been debated. polyploids have been postulated to be evolutionary dead ends because of the inefficiency of selection when genes are masked by multiple copies (1). however, most plant species have experienced at least one genome doubling early in their history (2), suggesting that rather than being an evolutionary dead end, polyploidy is a route to evolutionary success. a recent study (3) confirmed the ubiquity of polyploidy, with about 35% of vascular plant species being recent polyploids (\"neopolyploids,\" having formed

multiple copies (1). however, most plant species have experienced at least one genome doubling early in their history (2), suggesting that rather than being an evolutionary dead end, polyploidy is a route to evolutionary success. a recent study (3) confirmed the ubiquity of polyploidy, with about 35% of vascular plant species being recent polyploids (\"neopolyploids,\" having formed since their genus arose), representing 15% of speciation events in flowering plants and 31% in ferns. it remains unknown, however, whether the abundance of polyploids is a consequence of higher diversification rates following polyploidy or of frequent polyploid formation. we estimated diversification rates of

were inferred with a probabilistic model of chromosome number evolution that accounts for aneuploid and polyploid transitions but not diversification rate differences (5). this allowed us to label all descendants of a polyploidization event as neopolyploids, even when lacking chromosome data. p likelihood analyses indicated that 33% of the examined species are neopolyploids (609/2043 for angiosperms and 209/458 for sfvps), matching earlier estimates (1, 3). polyploidization events were not distributed uniformly across phylogenies but were disproportionately represented on the tips of the tree of life [c21 = 90.5 (all data); 48.2 (angiosperms); 45.1 (sfvps); p << 0.01 (4)], suggesting that newly

numbers were collected from the original studies and retrieved from genbank (7). in cases where the original studies were based on multiple data partitions we created a combined data matrix (if such combined data were supported by the original study), as long as the combined analysis reduced the number of species by less than 20%. otherwise, we selected the data matrix containing the largest number of species and verified that the reconstructed phylogeny (see below) was congruent with the tree presented in the original study. multiple sequence alignments were created using muscle (8), unless the alignment from the original study could

original studies were based on multiple data partitions we created a combined data matrix (if such combined data were supported by the original study), as long as the combined analysis reduced the number of species by less than 20%. otherwise, we selected the data matrix containing the largest number of species and verified that the reconstructed phylogeny (see below) was congruent with the tree presented in the original study. multiple sequence alignments were created using muscle (8), unless the alignment from the original study could be retrieved through treebase, in which case we used the latter. the findmodel server (http://www.hiv.lanl.gov/content/sequence/findmodel/findmodel.html) was

four markov chains (with heating according to default settings), run for 225,000 steps and sampled every 2000 steps. the first 125,000 steps were considered as burn-in and discarded from the analyses. the resulting mrbayes trees are already clock-like, and so ultrametrization is unnecessary. a few datasets contained multiple accessions per species. special care was taken to ensure that the choice of the included accession did not affect the results. in cases where the multiple accessions formed a monophyletic group in the resulting ml tree, a random representative accession was chosen. if a paraphyletic group for a single species was observed, we

multiple accessions per species. special care was taken to ensure that the choice of the included accession did not affect the results. in cases where the multiple accessions formed a monophyletic group in the resulting ml tree, a random representative accession was chosen. if a paraphyletic group for a single species was observed, we chose a sequence from the largest monophyletic group. in cases where no such group was observed but the accession choice did not change the categorization (polyploid versus diploid) of species in the tree (see below), a random accession was chosen. finally, the study was excluded if the

the resulting ml tree, a random representative accession was chosen. if a paraphyletic group for a single species was observed, we chose a sequence from the largest monophyletic group. in cases where no such group was observed but the accession choice did not change the categorization (polyploid versus diploid) of species in the tree (see below), a random accession was chosen. finally, the study was excluded if the accession choice resulted in different inferences of species as diploids or polyploids. chromosome counts were based on the original phylogenetic study, if reported. additional counts were obtained from the index to plant chromosome

the largest monophyletic group. in cases where no such group was observed but the accession choice did not change the categorization (polyploid versus diploid) of species in the tree (see below), a random accession was chosen. finally, the study was excluded if the accession choice resulted in different inferences of species as diploids or polyploids. chromosome counts were based on the original phylogenetic study, if reported. additional counts were obtained from the index to plant chromosome numbers database (http://www.tropicos.org/project/ipcn; (14)), the plant dna c-values database (http://data.kew.org/cvalues) and surveys of older compendiums (15, 16). where there were discrepancies between sources, we chose

counts for each genus, used in the diversification analysis (see below), were taken from the original study if reported. additional estimates were obtained from mabberley (17) and other sources (listed in table s1). 1 datasets were eliminated if they fell into any of the following categories: (1) less than ten species had sequence data (the exception is leavenworthia, a genus with only eight species but all have both sequence and chromosome number data); (2) chromosome counts were not available from enough species (either fewer than six species with count data or count data available for less than 35% of the sampled

from the original study if reported. additional estimates were obtained from mabberley (17) and other sources (listed in table s1). 1 datasets were eliminated if they fell into any of the following categories: (1) less than ten species had sequence data (the exception is leavenworthia, a genus with only eight species but all have both sequence and chromosome number data); (2) chromosome counts were not available from enough species (either fewer than six species with count data or count data available for less than 35% of the sampled species in the group); (3) chromosome counts in the dataset formed an aneuploid

table s1). 1 datasets were eliminated if they fell into any of the following categories: (1) less than ten species had sequence data (the exception is leavenworthia, a genus with only eight species but all have both sequence and chromosome number data); (2) chromosome counts were not available from enough species (either fewer than six species with count data or count data available for less than 35% of the sampled species in the group); (3) chromosome counts in the dataset formed an aneuploid series (e.g., 3, 4, 5, 6, 7, 11) that prevented a reliable designation of diploid and polyploid states;

eliminated if they fell into any of the following categories: (1) less than ten species had sequence data (the exception is leavenworthia, a genus with only eight species but all have both sequence and chromosome number data); (2) chromosome counts were not available from enough species (either fewer than six species with count data or count data available for less than 35% of the sampled species in the group); (3) chromosome counts in the dataset formed an aneuploid series (e.g., 3, 4, 5, 6, 7, 11) that prevented a reliable designation of diploid and polyploid states; or (4) no polyploidy transitions

had sequence data (the exception is leavenworthia, a genus with only eight species but all have both sequence and chromosome number data); (2) chromosome counts were not available from enough species (either fewer than six species with count data or count data available for less than 35% of the sampled species in the group); (3) chromosome counts in the dataset formed an aneuploid series (e.g., 3, 4, 5, 6, 7, 11) that prevented a reliable designation of diploid and polyploid states; or (4) no polyploidy transitions were inferred (see below). the final database included 63 phylogenies with both angiosperm (n =

(e.g., 3, 4, 5, 6, 7, 11) that prevented a reliable designation of diploid and polyploid states; or (4) no polyploidy transitions were inferred (see below). the final database included 63 phylogenies with both angiosperm (n = 49) and sfvp (n = 14) genera represented. the average number of sampled species per genus is 39, for an average coverage of 65% of the species in the generic-level groups. table s1 lists all phylogenetic studies and associated data used in our analyses as well as the type of loci used in our analyses and the life history characteristics of the various groups.

diploid and polyploid states; or (4) no polyploidy transitions were inferred (see below). the final database included 63 phylogenies with both angiosperm (n = 49) and sfvp (n = 14) genera represented. the average number of sampled species per genus is 39, for an average coverage of 65% of the species in the generic-level groups. table s1 lists all phylogenetic studies and associated data used in our analyses as well as the type of loci used in our analyses and the life history characteristics of the various groups. to get a sense of the age of these groups, we also provide

of each phylogeny as diploid. thus, polyploids are defined here as those lineages that underwent a polyploidization event since the divergence from the common ancestor of the group examined; our definition of polyploidy thus includes only neopolyploids. this procedure does not allow us to differentiate between genome duplications within a species (autopolyploidy) and those resulting from hybridization (allopolyploidy). determining the origin of large numbers of polyploid species is currently not feasible for the scale of this study, but future databases that distinguish allo- and autopolyploid species could be analyzed using similar methods to determine their relative effects on diversification. this methodology

polyploidization event since the divergence from the common ancestor of the group examined; our definition of polyploidy thus includes only neopolyploids. this procedure does not allow us to differentiate between genome duplications within a species (autopolyploidy) and those resulting from hybridization (allopolyploidy). determining the origin of large numbers of polyploid species is currently not feasible for the scale of this study, but future databases that distinguish allo- and autopolyploid species could be analyzed using similar methods to determine their relative effects on diversification. this methodology allowed us to categorize an extant species as polyploid or diploid regardless of whether chromosome number

only neopolyploids. this procedure does not allow us to differentiate between genome duplications within a species (autopolyploidy) and those resulting from hybridization (allopolyploidy). determining the origin of large numbers of polyploid species is currently not feasible for the scale of this study, but future databases that distinguish allo- and autopolyploid species could be analyzed using similar methods to determine their relative effects on diversification. this methodology allowed us to categorize an extant species as polyploid or diploid regardless of whether chromosome number data were available for that specific taxon. the ploidy inferences were examined manually and compared to those reported in

(allopolyploidy). determining the origin of large numbers of polyploid species is currently not feasible for the scale of this study, but future databases that distinguish allo- and autopolyploid species could be analyzed using similar methods to determine their relative effects on diversification. this methodology allowed us to categorize an extant species as polyploid or diploid regardless of whether chromosome number data were available for that specific taxon. the ploidy inferences were examined manually and compared to those reported in the original phylogenetic study, if available. table s1 lists the percentage of polyploid species in each dataset as inferred by chromevol and

this methodology allowed us to categorize an extant species as polyploid or diploid regardless of whether chromosome number data were available for that specific taxon. the ploidy inferences were examined manually and compared to those reported in the original phylogenetic study, if available. table s1 lists the percentage of polyploid species in each dataset as inferred by chromevol and figure s3 presents several representative phylogenies with the inferred polyploidization events. in a number of cases, the ploidy inferences seemed questionable for sub-clades with high proportions of missing 2 chromosome numbers. to be conservative, we changed the ploidy states for these species

species in each dataset as inferred by chromevol and figure s3 presents several representative phylogenies with the inferred polyploidization events. in a number of cases, the ploidy inferences seemed questionable for sub-clades with high proportions of missing 2 chromosome numbers. to be conservative, we changed the ploidy states for these species to \"not available\" (na in the diversitree r package) in our diversification analyses. we note that our methodology for assigning ploidy levels to extant taxa does not account for different diversification rates of diploids and polyploids. this may bias our estimates of ploidy levels among extant taxa, particularly for those

rates of diploids and polyploids. this may bias our estimates of ploidy levels among extant taxa, particularly for those taxa with missing chromosome counts. to verify that this potential bias did not substantively affect our conclusions, we repeated the diversification analysis (detailed below) with ploidy levels assigned only to those species with available cytological data, labeling the missing chromosome count data as na. this procedure yielded very similar results to those reported in the main text. in addition, very similar results were obtained when ploidy-level assignments were not based on the chromevol reconstructions but instead, following stebbins (1), regarding all species

species with available cytological data, labeling the missing chromosome count data as na. this procedure yielded very similar results to those reported in the main text. in addition, very similar results were obtained when ploidy-level assignments were not based on the chromevol reconstructions but instead, following stebbins (1), regarding all species with more than twice the minimum chromosome number for the genus as polyploids and labeling the missing chromosome count data as na. diversification analysis as a first indication of ploidy-dependent diversification rates we evaluated whether polyploidization events occur more often than expected along external branches (those leading directly to extant

with more than twice the minimum chromosome number for the genus as polyploids and labeling the missing chromosome count data as na. diversification analysis as a first indication of ploidy-dependent diversification rates we evaluated whether polyploidization events occur more often than expected along external branches (those leading directly to extant species) or internal branches of the tree. for each phylogeny, the expected number of polyploidization events along each branch, as inferred by chromevol, was calculated as the relative contribution of the branch to the total tree length (i.e., the branch length divided by the sum of all branch lengths of the

bisse co-estimates six parameters: speciation rates of lineages in state p (polyploid) and d (diploid) (λp and λd, respectively); extinction rates of lineages in state p and d (µp and µd); and transition rates from p to d (qpd) and d to p (qdp). because we defined as polyploid those species that underwent a polyploidization event sometime since divergence from their generic ancestor, we fixed qpd to zero (but see below). this constraint is also compatible with the common assumption that polyploidy is largely an irreversible process, especially over a short time interval (22). our analyses were performed using the \"skeletal\"

(but see below). this constraint is also compatible with the common assumption that polyploidy is largely an irreversible process, especially over a short time interval (22). our analyses were performed using the \"skeletal\" tree approach (23) implemented in the r package diversitree (http://www.zoology.ubc.ca/prog/diversitree/), which accounts for the sampling fraction of species in the given phylogeny out of the total number of species in that clade, using the null hypothesis that the unsampled species did not differ in proportion of diploid versus polyploid species from the sampled species. results obtained using the complete sampling assumption were nearly identical (results not shown). in

assumption that polyploidy is largely an irreversible process, especially over a short time interval (22). our analyses were performed using the \"skeletal\" tree approach (23) implemented in the r package diversitree (http://www.zoology.ubc.ca/prog/diversitree/), which accounts for the sampling fraction of species in the given phylogeny out of the total number of species in that clade, using the null hypothesis that the unsampled species did not differ in proportion of diploid versus polyploid species from the sampled species. results obtained using the complete sampling assumption were nearly identical (results not shown). in cases where diversity counts were unavailable (for example, when the focal

short time interval (22). our analyses were performed using the \"skeletal\" tree approach (23) implemented in the r package diversitree (http://www.zoology.ubc.ca/prog/diversitree/), which accounts for the sampling fraction of species in the given phylogeny out of the total number of species in that clade, using the null hypothesis that the unsampled species did not differ in proportion of diploid versus polyploid species from the sampled species. results obtained using the complete sampling assumption were nearly identical (results not shown). in cases where diversity counts were unavailable (for example, when the focal group represented a subgenus without a specified species richness), we ran

\"skeletal\" tree approach (23) implemented in the r package diversitree (http://www.zoology.ubc.ca/prog/diversitree/), which accounts for the sampling fraction of species in the given phylogeny out of the total number of species in that clade, using the null hypothesis that the unsampled species did not differ in proportion of diploid versus polyploid species from the sampled species. results obtained using the complete sampling assumption were nearly identical (results not shown). in cases where diversity counts were unavailable (for example, when the focal group represented a subgenus without a specified species richness), we ran diversitree under the complete sampling assumption. the markov chain monte

implemented in the r package diversitree (http://www.zoology.ubc.ca/prog/diversitree/), which accounts for the sampling fraction of species in the given phylogeny out of the total number of species in that clade, using the null hypothesis that the unsampled species did not differ in proportion of diploid versus polyploid species from the sampled species. results obtained using the complete sampling assumption were nearly identical (results not shown). in cases where diversity counts were unavailable (for example, when the focal group represented a subgenus without a specified species richness), we ran diversitree under the complete sampling assumption. the markov chain monte carlo (mcmc) approach described

that the unsampled species did not differ in proportion of diploid versus polyploid species from the sampled species. results obtained using the complete sampling assumption were nearly identical (results not shown). in cases where diversity counts were unavailable (for example, when the focal group represented a subgenus without a specified species richness), we ran diversitree under the complete sampling assumption. the markov chain monte carlo (mcmc) approach described in fitzjohn et al. (23) was applied to estimate the probability distributions for each of the five parameters (λp, λd, µp, µd, qdp), accounting for uncertainty in parameter estimation and incomplete sampling. in

in two daughter lineages that survive to the present, similar modifications were made to allow simultaneous speciation with state change along the branches, given that one of the lineages resulting from such speciation events must go extinct before the present and the other lineage must give rise to the descendant species, as observed. the above modifications resulted in the addition of four parameters to form the bisse-ness model: p0c, p0a, p1c, p1a (here state 0 being diploid and state 1 polyploid). here, however, we excluded polyploid to diploid transitions (p1c = 0 and p1a = 0) since in our generic phylogenies

in the bisse or bisse-ness models would imply that it is possible for a taxon to experience a near-instantaneous halving of its genome (i.e., polyhaploidy), which is not widely accepted. instead, diploidization is thought to be a gradual process involving the loss and differentiation of genetic material. in addition, polyploid species were defined in our studies as those lineages that underwent a polyploidization event since divergence from the common ancestor of the group examined. thus, a lineage may be labeled polyploid even if it has a chromosome number and meiotic behavior similar to that inferred for the base of the group

were similar between the two ploidal states (p > 0.1). we thus conclude that only under unrealistically high reversion rates from polyploidy to diploidy would polyploids be likely to diversify at rates that are as high as that inferred for diploids. simulating the number of ancient polyploidization events within angiosperm species we used a two-state (diploid and polyploid) birth and death process to simulate the distribution of the number of genome duplications expected in the evolutionary history of an angiosperm species since its divergence from the most recent common ancestor of angiosperms. in these simulations we assumed that variation in ploidy

at rates that are as high as that inferred for diploids. simulating the number of ancient polyploidization events within angiosperm species we used a two-state (diploid and polyploid) birth and death process to simulate the distribution of the number of genome duplications expected in the evolutionary history of an angiosperm species since its divergence from the most recent common ancestor of angiosperms. in these simulations we assumed that variation in ploidy does not affect speciation and extinction rates. the ratio of extinction to speciation was set to 0.6, as estimated by bokma (24) for a diverse set of angiosperm taxa. we

then recorded the total number of polyploidization events that occurred in the history of each extant taxon, assuming that a certain fraction of speciation events, fhet, involve polyploidization. we ran this process 100 times, for a given fhet value, starting from a common diploid ancestor until the total number of species reached 300,000, a somewhat conservative estimate of the total number of angiosperm species (25); other estimates ranging from 250,000 to 400,000 gave similar results). under the extent of heteroploid speciation estimated in our study (fhet = 29.7% for angiosperms), our simulations indicated that if polyploids and diploids were diversifying at

of each extant taxon, assuming that a certain fraction of speciation events, fhet, involve polyploidization. we ran this process 100 times, for a given fhet value, starting from a common diploid ancestor until the total number of species reached 300,000, a somewhat conservative estimate of the total number of angiosperm species (25); other estimates ranging from 250,000 to 400,000 gave similar results). under the extent of heteroploid speciation estimated in our study (fhet = 29.7% for angiosperms), our simulations indicated that if polyploids and diploids were diversifying at equal rates we would find the traces of even more paleopolyploidization events than

similar results). under the extent of heteroploid speciation estimated in our study (fhet = 29.7% for angiosperms), our simulations indicated that if polyploids and diploids were diversifying at equal rates we would find the traces of even more paleopolyploidization events than the 1 - 4 duplications observed in extant angiosperm species (>94% of the species having 5 or more; fig. s2a). the distributions of the number of paleopolyploidization events 6 obtained assuming lower rates of heteroploid speciation were also shifted to the right (fig s2b-c). in these simulations we did not aim to account for the complex dynamics of diversification rates

extent of heteroploid speciation estimated in our study (fhet = 29.7% for angiosperms), our simulations indicated that if polyploids and diploids were diversifying at equal rates we would find the traces of even more paleopolyploidization events than the 1 - 4 duplications observed in extant angiosperm species (>94% of the species having 5 or more; fig. s2a). the distributions of the number of paleopolyploidization events 6 obtained assuming lower rates of heteroploid speciation were also shifted to the right (fig s2b-c). in these simulations we did not aim to account for the complex dynamics of diversification rates through time, but rather

rates include only homoploid speciation. for each dataset, mcmc bisse-ness results were pooled across 50 trees sampled by mrbayes. 8 0 2 4 6 8 10 13 16 number of duplications 19 50000 c 30000 30000 40000 10000 0 0 10000 10000 20000 20000 30000 40000 b 0 number of species a 0 2 4 6 8 10 13 16 number of duplications 19 0 2 4 6 8 10 13 16 19 number of duplications figure s2|simulated distribution of the total number of genome duplications that have occurred within extant angiosperm species. a birth and death process was used to

20000 20000 30000 40000 b 0 number of species a 0 2 4 6 8 10 13 16 number of duplications 19 0 2 4 6 8 10 13 16 19 number of duplications figure s2|simulated distribution of the total number of genome duplications that have occurred within extant angiosperm species. a birth and death process was used to simulate the distribution of the number of genome duplications assuming equal net diversification rates of diploids and polyploids and polyploidizations occurring at speciation events with a probability, fhet, of (a) 30% (the average estimate of our angiosperm datasets), (b) 20%, and (c)

diploids and polyploids and polyploidizations occurring at speciation events with a probability, fhet, of (a) 30% (the average estimate of our angiosperm datasets), (b) 20%, and (c) 15%, resulting in an average number of past polyploidization events of 8.9, 6.0, and 4.6, respectively. in all cases the number of angiosperm species was assumed to be 300,000 (25), with a ratio of extinction to speciation of 0.6 (24). 9 a c taenitis_blechnoides-44 jamesonia_canescens-x anogramma_osteniana-29 eriosorus_flexuosus-87 cosentinia_vellea-29 anogramma_lorentzii-x anogramma_leptophylla_vcmexico-29 anogramma_guatemalensis-x anogramma_leptophylla_turkey-29 anogramma_caespitosa-x anogramma_leptophylla_nzealand-26 anogramma_novoglaciana-x anogramma_chaerophylla_crica-29 anogramma_chaerophylla_brazil-29 pityrogramma_trifoliata-60 pityrogramma_calomelanos-120 gaura_macrocarpa-7 gaura_parviflora-7 gaura_boquillensis-7 gaura_coccinea-14 gaura_filipes-7 gaura_villosa-7 gaura_sinuata-14 gaura_hexandra-7 gaura_angustifolia-7 gaura_biennis-7 gaura_longiflora-7 gaura_lindheimeri-7 gaura_demareei-7 gaura_triangulata-7 gaura_suffulta-7

betula_fruticosa-14 betula_pubescens-28 betula_humilis-14 betula_ermanii-28 stenosiphon_linifolius-7 gaura_mutabilis-7 figure s3| ploidy transitions for several representative groups. (a) anogramma, (b) tiquilia, (c) gaura/stenosiphon, (d) betula. branches of the tree where polyploidization events were inferred to occur using the chromevol methodology (5) are colored in blue. chromosome counts appear to the right of the species name following a hyphen, where 'x' indicates unavailable cytological data. 10 supporting tables table s1|datasets used in this study. ss = number of species sampled with sequence data, si = number of species recognized in the ingroup, sc = number of sampled species with cytological data, %pp = percentage of

the tree where polyploidization events were inferred to occur using the chromevol methodology (5) are colored in blue. chromosome counts appear to the right of the species name following a hyphen, where 'x' indicates unavailable cytological data. 10 supporting tables table s1|datasets used in this study. ss = number of species sampled with sequence data, si = number of species recognized in the ingroup, sc = number of sampled species with cytological data, %pp = percentage of neopolyploids as estimated by chromevol, th = tree height in terms of average number of substitutions per sitea, lh = life history attributesb, na

using the chromevol methodology (5) are colored in blue. chromosome counts appear to the right of the species name following a hyphen, where 'x' indicates unavailable cytological data. 10 supporting tables table s1|datasets used in this study. ss = number of species sampled with sequence data, si = number of species recognized in the ingroup, sc = number of sampled species with cytological data, %pp = percentage of neopolyploids as estimated by chromevol, th = tree height in terms of average number of substitutions per sitea, lh = life history attributesb, na = diversity counts not available. sc %pp tha lhb

counts appear to the right of the species name following a hyphen, where 'x' indicates unavailable cytological data. 10 supporting tables table s1|datasets used in this study. ss = number of species sampled with sequence data, si = number of species recognized in the ingroup, sc = number of sampled species with cytological data, %pp = percentage of neopolyploids as estimated by chromevol, th = tree height in terms of average number of substitutions per sitea, lh = life history attributesb, na = diversity counts not available. sc %pp tha lhb group focal group ss si eudicots sium s.l. 14 17

ml tree obtained using phyml by traversing the tree from the tips to the root. for each internal node, its height was calculated as the average height of the lineages descending from that node, finally reaching the root node. b life history characteristics are given based on the majority of species in the group as follows: pr = perennials; an = annuals or biennials; hb = herbaceous; wd = woody (trees/shrubs). data were obtained through eflora (http://www.efloras.org/), mabberley (17), or the original phylogenetic study. c nr: nuclear; cp: chloroplast; mt: mitochondrial; its: the nuclear internal transcribed spacer d diversity and cytology",

### NA
force on diversiﬁcation (anacker and strauss, 2014). the geographical extent of a clade has a signiﬁcant impact on its diversity, and it is shaped by several factors including geological processes such as continental drift, glaciation, and mountain formation; the clade's ecological niche; and dispersal events (wiens and donoghue, 2004). when species are exposed to diﬀerent environmental factors upon a range shift, they often become subject to diﬀerent selective forces, favoring a diﬀerent set of characters than the ones in their previous environment (sexton et al., 2009). previous studies have found many factors may increase diversiﬁcation rates in a lineage upon dispersal

diﬀerent set of characters than the ones in their previous environment (sexton et al., 2009). previous studies have found many factors may increase diversiﬁcation rates in a lineage upon dispersal into a new region (moore and donoghue, 2007). the biotic factors for a plant lineage include the exposure of the species to new ecological niches (linder, 2008), or simply the lack of competitors in the new region (sexton et al., 2009). abiotic factors include a more favorable climate decreasing the environmental stress on the species, thus reducing the risk of extinction caused by harsh environments with over 300,000 species, angiosperms form

(moore and donoghue, 2007). the biotic factors for a plant lineage include the exposure of the species to new ecological niches (linder, 2008), or simply the lack of competitors in the new region (sexton et al., 2009). abiotic factors include a more favorable climate decreasing the environmental stress on the species, thus reducing the risk of extinction caused by harsh environments with over 300,000 species, angiosperms form an extraordinarily species-rich group. this exceptional diversity has generated a wealth of explanatory mechanisms, many of which are not mutually exclusive (dodd et al., 1999; chase et al., 2010; vamosi and vamosi, 2011; armbruster,

of the species to new ecological niches (linder, 2008), or simply the lack of competitors in the new region (sexton et al., 2009). abiotic factors include a more favorable climate decreasing the environmental stress on the species, thus reducing the risk of extinction caused by harsh environments with over 300,000 species, angiosperms form an extraordinarily species-rich group. this exceptional diversity has generated a wealth of explanatory mechanisms, many of which are not mutually exclusive (dodd et al., 1999; chase et al., 2010; vamosi and vamosi, 2011; armbruster, 2014; tank et al., 2015). early theories of diversiﬁcation of angiosperms posited that reproductive

ecological niches (linder, 2008), or simply the lack of competitors in the new region (sexton et al., 2009). abiotic factors include a more favorable climate decreasing the environmental stress on the species, thus reducing the risk of extinction caused by harsh environments with over 300,000 species, angiosperms form an extraordinarily species-rich group. this exceptional diversity has generated a wealth of explanatory mechanisms, many of which are not mutually exclusive (dodd et al., 1999; chase et al., 2010; vamosi and vamosi, 2011; armbruster, 2014; tank et al., 2015). early theories of diversiﬁcation of angiosperms posited that reproductive features involved with seed production

shifts (forest et al., 2014). it was previously assumed that the allopatric speciation events represent the dominant speciation mode (futuyma and mayer, 1980), but more studies show non-allopatric speciation modes are also common, especially in plants (savolainen et al., 2006; grossenbacher and whittall, 2011). reproductive isolation is necessary for sympatric species to generate and maintain enough divergence to prevent gene ﬂow large enough to lapse back into a single species (rieseberg and willis, 2007). in many cases, closely-related species are isolated by multiple reproductive barriers (chari and wilson, 2001). nonrandom mating due to pollinator p", 
" is a common reproductive isolation

(futuyma and mayer, 1980), but more studies show non-allopatric speciation modes are also common, especially in plants (savolainen et al., 2006; grossenbacher and whittall, 2011). reproductive isolation is necessary for sympatric species to generate and maintain enough divergence to prevent gene ﬂow large enough to lapse back into a single species (rieseberg and willis, 2007). in many cases, closely-related species are isolated by multiple reproductive barriers (chari and wilson, 2001). nonrandom mating due to pollinator p", 
" is a common reproductive isolation mechanism that can facilitate sympatric speciation in ﬂowering plants (kiester et al., 1984; sexton et al., 2013; forest et

speciation modes are also common, especially in plants (savolainen et al., 2006; grossenbacher and whittall, 2011). reproductive isolation is necessary for sympatric species to generate and maintain enough divergence to prevent gene ﬂow large enough to lapse back into a single species (rieseberg and willis, 2007). in many cases, closely-related species are isolated by multiple reproductive barriers (chari and wilson, 2001). nonrandom mating due to pollinator p", 
" is a common reproductive isolation mechanism that can facilitate sympatric speciation in ﬂowering plants (kiester et al., 1984; sexton et al., 2013; forest et al., 2014; ferris et al., 2014; van der niet

wilson, 2001). nonrandom mating due to pollinator p", 
" is a common reproductive isolation mechanism that can facilitate sympatric speciation in ﬂowering plants (kiester et al., 1984; sexton et al., 2013; forest et al., 2014; ferris et al., 2014; van der niet et al., 2014). with 27 genera representing 240 species (the plant list; http:// www.theplantlist.org), antirrhineae dumort. is one of the larger tribes under the family plantaginaceae juss (albach et al., 2005). antirrhineae has a cosmopolitan distribution, and the tribe is represented in the old world and the new world (fig. 1). as one of the biggest 2. materials and

cosmopolitan distribution, and the tribe is represented in the old world and the new world (fig. 1). as one of the biggest 2. materials and methods 2.1. taxon sampling and phylogenetic analyses we recently generated an extensive phylogeny of antirrhineae (supplementary figure s1; ogutcen and vamosi, 2016). out of 240 species from 28 genera within the tribe (the plant list; http://www. theplantlist.org), we sampled 146 species (61% coverage) from 24 genera. node ages were inferred using an uncorrelated lognormal relaxed clock, and the tree was calibrated using the divergence between cheanorhinum and linaria (mean of 23 million years ago with the

(fig. 1). as one of the biggest 2. materials and methods 2.1. taxon sampling and phylogenetic analyses we recently generated an extensive phylogeny of antirrhineae (supplementary figure s1; ogutcen and vamosi, 2016). out of 240 species from 28 genera within the tribe (the plant list; http://www. theplantlist.org), we sampled 146 species (61% coverage) from 24 genera. node ages were inferred using an uncorrelated lognormal relaxed clock, and the tree was calibrated using the divergence between cheanorhinum and linaria (mean of 23 million years ago with the standard deviation of 4 million years) estimated in a previous study based on fossil data

age-range correlations we collected occurrence data from the global biodiversity information facility (gbif) database (www.gbif.org). we excluded duplicate records, records with missing coordinates, and those having coordinates with less than two signiﬁcant digits after the decimal. we also removed the data points that were outside the native range of a species. after these quality control procedures, we had with 104,318 geographic coordinate data points for 80 out of the 146 species represented in the current phylogeny (55% coverage). data points were visualized on a world map using the r package ggmap (kahle and wickham, 2013). we calculated age-range correlations with the

with missing coordinates, and those having coordinates with less than two signiﬁcant digits after the decimal. we also removed the data points that were outside the native range of a species. after these quality control procedures, we had with 104,318 geographic coordinate data points for 80 out of the 146 species represented in the current phylogeny (55% coverage). data points were visualized on a world map using the r package ggmap (kahle and wickham, 2013). we calculated age-range correlations with the fitzpatrick-turelli algorithm (fitzpatrick and turelli, 2006) using an r script provided by dena grossenbacher (university of minnesota; turelli et al.,

pollinator community within the studied region (see appendix a; proctor and yeo, 1973; elisens, 1985, 1986; elisens and freeman, 1988; sutton, 1988; kampny, 1995; ghebrehiwet et al., 2000; fernández-mazuecos et al., 2013). bee pollination is the more common pollination mode in the tribe, and bees also visit many \"hummingbird pollinated\" species. for simplicity, we categorized pollination data into two groups: bee pollination and hummingbird pollination. the bee pollination category represents species that are we used the bayesian analysis of macroevolutionary mixtures (bamm; rabosky, 2014) program to examine the rates of evolution, and to determine whether the evolutionary rates vary among the

sutton, 1988; kampny, 1995; ghebrehiwet et al., 2000; fernández-mazuecos et al., 2013). bee pollination is the more common pollination mode in the tribe, and bees also visit many \"hummingbird pollinated\" species. for simplicity, we categorized pollination data into two groups: bee pollination and hummingbird pollination. the bee pollination category represents species that are we used the bayesian analysis of macroevolutionary mixtures (bamm; rabosky, 2014) program to examine the rates of evolution, and to determine whether the evolutionary rates vary among the clades. in order to account for incomplete taxon sampling, we set the global sampling fraction to 0.61 (146 out of

are we used the bayesian analysis of macroevolutionary mixtures (bamm; rabosky, 2014) program to examine the rates of evolution, and to determine whether the evolutionary rates vary among the clades. in order to account for incomplete taxon sampling, we set the global sampling fraction to 0.61 (146 out of 240 species in antirrhineae). we executed the speciation/extinction analysis using mcmc simulation with 10,000,1000 generations, and used the r package bammtools (rabosky et al., 2014) to visualize and analyze the bamm data. we used the r package coda (plummer et al., 2006) to conﬁrm the eﬀective 41 perspectives in plant ecology, evolution

of the sister taxa and the age (in million years) for each node in the phylogeny. taxon sampling corrections based on diﬀerent assumptions: (iii) in the \"realistic\" setting, pollination mode was kept consistent throughout a genus (i.e., if a single pollination mode is observed in the genus, then all the species within the genus were assumed to have the same pollination mode. alternatively, if there are both bee and hummingbird pollination within a genus, then the ratio of bee:hummingbird pollination was kept constant); (iv) in the \"more bee\" setting, all the unknown pollinators in the new world were assumed to be

constant); (iv) in the \"more bee\" setting, all the unknown pollinators in the new world were assumed to be bee, (v) in the \"less bee\" setting, all the unknown pollinators in the new world were assumed to be hummingbird; and (vi) in the \"all nw bird\" setting, all new world species are assumed to be hummingbird pollinated, even if they are observed to be bee pollinated. two more assumptions were made for all the state-dependent settings: (i) all old world species are bee pollinated; and (ii) two species that were not in the phylogeny (mabrya coccineae and mabrya geniculata) are hummingbird

in the new world were assumed to be hummingbird; and (vi) in the \"all nw bird\" setting, all new world species are assumed to be hummingbird pollinated, even if they are observed to be bee pollinated. two more assumptions were made for all the state-dependent settings: (i) all old world species are bee pollinated; and (ii) two species that were not in the phylogeny (mabrya coccineae and mabrya geniculata) are hummingbird pollinated (assumption based on elisens, 1986). visited only by bees, whereas the hummingbird pollination category represents species that are visited by hummingbirds and also by bees to some extent. in

be hummingbird; and (vi) in the \"all nw bird\" setting, all new world species are assumed to be hummingbird pollinated, even if they are observed to be bee pollinated. two more assumptions were made for all the state-dependent settings: (i) all old world species are bee pollinated; and (ii) two species that were not in the phylogeny (mabrya coccineae and mabrya geniculata) are hummingbird pollinated (assumption based on elisens, 1986). visited only by bees, whereas the hummingbird pollination category represents species that are visited by hummingbirds and also by bees to some extent. in order to characterize the pollination history of

more assumptions were made for all the state-dependent settings: (i) all old world species are bee pollinated; and (ii) two species that were not in the phylogeny (mabrya coccineae and mabrya geniculata) are hummingbird pollinated (assumption based on elisens, 1986). visited only by bees, whereas the hummingbird pollination category represents species that are visited by hummingbirds and also by bees to some extent. in order to characterize the pollination history of antirrhineae, we performed ancestral state reconstructions in mesquite 3.03 (maddison and maddison, 2014). we performed both unordered maximum parsimony (mp) and maximum likelihood (ml) reconstruction. for the ml reconstruction with

hummingbird pollination) as 0. 2.5. trait-dependent diversiﬁcation analyses in order to determine the diﬀerences in speciation and extinction rates within the old world and the new world, we used the \"geosse\" function (geographic state speciation and extinction; goldberg et al., 2011) within the r package diversitree (fitzjohn, 2012). we listed species under two categories: new world distributed (a) and old world distributed (b). the parameters were: speciation within new world (sa), speciation within old world (sb), between-region speciation (sab), extinction from new world (xa), extinction from new world (xb), dispersal from new world to new world (da), and dispersal from old

world (sa), speciation within old world (sb), between-region speciation (sab), extinction from new world (xa), extinction from new world (xb), dispersal from new world to new world (da), and dispersal from old world to new world (db). our sampling was unbiased in terms of the ratio of the old world species to the new world species. old world and new world species represent 78% and 22% of our phylogeny respectively. this ratio is comparable to the ratio of the whole tribe (77% old world species, 23% new world species). next, we examined how pollination mode correlates with diversiﬁcation rates in antirrhineae

world (sb), between-region speciation (sab), extinction from new world (xa), extinction from new world (xb), dispersal from new world to new world (da), and dispersal from old world to new world (db). our sampling was unbiased in terms of the ratio of the old world species to the new world species. old world and new world species represent 78% and 22% of our phylogeny respectively. this ratio is comparable to the ratio of the whole tribe (77% old world species, 23% new world species). next, we examined how pollination mode correlates with diversiﬁcation rates in antirrhineae using the \"bisse\" function (binary

from new world (xa), extinction from new world (xb), dispersal from new world to new world (da), and dispersal from old world to new world (db). our sampling was unbiased in terms of the ratio of the old world species to the new world species. old world and new world species represent 78% and 22% of our phylogeny respectively. this ratio is comparable to the ratio of the whole tribe (77% old world species, 23% new world species). next, we examined how pollination mode correlates with diversiﬁcation rates in antirrhineae using the \"bisse\" function (binary state speciation and extinction; maddison et

new world (db). our sampling was unbiased in terms of the ratio of the old world species to the new world species. old world and new world species represent 78% and 22% of our phylogeny respectively. this ratio is comparable to the ratio of the whole tribe (77% old world species, 23% new world species). next, we examined how pollination mode correlates with diversiﬁcation rates in antirrhineae using the \"bisse\" function (binary state speciation and extinction; maddison et al., 2007) within the r package diversitree (fitzjohn, 2012). we coded bee pollination as 0, and hummingbird pollination as 1. we estimated speciation

sampling was unbiased in terms of the ratio of the old world species to the new world species. old world and new world species represent 78% and 22% of our phylogeny respectively. this ratio is comparable to the ratio of the whole tribe (77% old world species, 23% new world species). next, we examined how pollination mode correlates with diversiﬁcation rates in antirrhineae using the \"bisse\" function (binary state speciation and extinction; maddison et al., 2007) within the r package diversitree (fitzjohn, 2012). we coded bee pollination as 0, and hummingbird pollination as 1. we estimated speciation rates in taxa with

one (f = 0.72), followed by the conﬁguration where lafuentea had decreased speciation rates (f = 0.15) and the third conﬁguration where the rest of the tribe (with lafuentea as an outgroup) had increased speciation rate (f = 0.071). 3.3. geographical distribution and pollination mode in antirrhineae out of 146 species in the phylogeny, we retrieved pollinator data for 42 perspectives in plant ecology, evolution and systematics 26 (2017) 39 - 52 e. ogutcen et al. table 1 age-range correlation analysis results. (a) regression slopes and p values for the age-range correlations. ow: old world, nw: new world, all: old world

rss 4.65 4.90 4.53 0.744 0.623 0.0352 f p 0.376 1.267 1.513 0.995 0.173 0.0998 ﬂowers (fig. 6). in gambelia-galvezia lineage, a switch from purple to red ﬂowers was observed along with a bee to hummingbird transition at the ancestral node. the splitting of bee-pollinated schweinfurthia papilionaceae from its sister species gambelia speciosa was coupled with a red to yellow transition. lophospermum-mabrya lineage was observed to have similar transitions; a purple to red transition at the base, and a red to yellow transition at the node splitting mabrya acerifolia (the only hummingbird pollinated species with yellow ﬂowers) from its sister species

of bee-pollinated schweinfurthia papilionaceae from its sister species gambelia speciosa was coupled with a red to yellow transition. lophospermum-mabrya lineage was observed to have similar transitions; a purple to red transition at the base, and a red to yellow transition at the node splitting mabrya acerifolia (the only hummingbird pollinated species with yellow ﬂowers) from its sister species lophospermum scandens. in sairocarpus-galvezia lineage, however, purple remained the ﬂoral color for the hummingbird-pollinated sairocarpus species, but galvezia juncea had a switch from purple to red. 97 of them (supplementary table s1). 85 species are bee pollinated (58%), 12 are hummingbird pollinated (8%),

species gambelia speciosa was coupled with a red to yellow transition. lophospermum-mabrya lineage was observed to have similar transitions; a purple to red transition at the base, and a red to yellow transition at the node splitting mabrya acerifolia (the only hummingbird pollinated species with yellow ﬂowers) from its sister species lophospermum scandens. in sairocarpus-galvezia lineage, however, purple remained the ﬂoral color for the hummingbird-pollinated sairocarpus species, but galvezia juncea had a switch from purple to red. 97 of them (supplementary table s1). 85 species are bee pollinated (58%), 12 are hummingbird pollinated (8%), and the rest of the taxa had

have similar transitions; a purple to red transition at the base, and a red to yellow transition at the node splitting mabrya acerifolia (the only hummingbird pollinated species with yellow ﬂowers) from its sister species lophospermum scandens. in sairocarpus-galvezia lineage, however, purple remained the ﬂoral color for the hummingbird-pollinated sairocarpus species, but galvezia juncea had a switch from purple to red. 97 of them (supplementary table s1). 85 species are bee pollinated (58%), 12 are hummingbird pollinated (8%), and the rest of the taxa had no pollination data available (34%; fig. 4). in terms of geographic distribution, 112 of the 146

the node splitting mabrya acerifolia (the only hummingbird pollinated species with yellow ﬂowers) from its sister species lophospermum scandens. in sairocarpus-galvezia lineage, however, purple remained the ﬂoral color for the hummingbird-pollinated sairocarpus species, but galvezia juncea had a switch from purple to red. 97 of them (supplementary table s1). 85 species are bee pollinated (58%), 12 are hummingbird pollinated (8%), and the rest of the taxa had no pollination data available (34%; fig. 4). in terms of geographic distribution, 112 of the 146 species were distributed in the old world (78%), whereas 22 species were distributed in the new world (22%).

but galvezia juncea had a switch from purple to red. 97 of them (supplementary table s1). 85 species are bee pollinated (58%), 12 are hummingbird pollinated (8%), and the rest of the taxa had no pollination data available (34%; fig. 4). in terms of geographic distribution, 112 of the 146 species were distributed in the old world (78%), whereas 22 species were distributed in the new world (22%). in examining the distribution of predominant pollinators in the two geographic regions, we ﬁnd that bee pollination was observed in 100% of the old world species, and hummingbird pollination was observed in 44.44%

97 of them (supplementary table s1). 85 species are bee pollinated (58%), 12 are hummingbird pollinated (8%), and the rest of the taxa had no pollination data available (34%; fig. 4). in terms of geographic distribution, 112 of the 146 species were distributed in the old world (78%), whereas 22 species were distributed in the new world (22%). in examining the distribution of predominant pollinators in the two geographic regions, we ﬁnd that bee pollination was observed in 100% of the old world species, and hummingbird pollination was observed in 44.44% of the new world species. 3.4. ancestral pollination mode in

of geographic distribution, 112 of the 146 species were distributed in the old world (78%), whereas 22 species were distributed in the new world (22%). in examining the distribution of predominant pollinators in the two geographic regions, we ﬁnd that bee pollination was observed in 100% of the old world species, and hummingbird pollination was observed in 44.44% of the new world species. 3.4. ancestral pollination mode in antirrhineae the ancestral pollination mode was inferred to be bee pollination in both mp and ml ancestral state reconstruction analyses (bootstrap value: bs = 93.68%; fig. 5) antirrhineae experienced three switches from bee

old world (78%), whereas 22 species were distributed in the new world (22%). in examining the distribution of predominant pollinators in the two geographic regions, we ﬁnd that bee pollination was observed in 100% of the old world species, and hummingbird pollination was observed in 44.44% of the new world species. 3.4. ancestral pollination mode in antirrhineae the ancestral pollination mode was inferred to be bee pollination in both mp and ml ancestral state reconstruction analyses (bootstrap value: bs = 93.68%; fig. 5) antirrhineae experienced three switches from bee to hummingbird pollination (fig. 5): (i) gambelia-galvezia lineage (bs = 53.67%) between

(i) gambelia-galvezia lineage (bs = 53.67%) between 24.69 and 12.35 mya, (ii) lophospermum-mabrya lineage (bs = 99.44%) between 11.71 and 7.29 mya, and (iii) sairocarpus-galvezia lineage (bs = 97.04%) between 9.77 and 5.98 mya. one hummingbird to bee pollination transition was observed within the gambelia-galvezia lineage, when the bee pollinated species schweinfurthia papilionaceae split from its sister species gambelia speciosa 8 mya (fig. 5). the ancestor of these two sister species were inferred to be hummingbird pollinated (bs = 53.49%). when the ﬂoral color transitions are examined, the switches from bee to hummingbird pollination were mostly accompanied by changes from purple

24.69 and 12.35 mya, (ii) lophospermum-mabrya lineage (bs = 99.44%) between 11.71 and 7.29 mya, and (iii) sairocarpus-galvezia lineage (bs = 97.04%) between 9.77 and 5.98 mya. one hummingbird to bee pollination transition was observed within the gambelia-galvezia lineage, when the bee pollinated species schweinfurthia papilionaceae split from its sister species gambelia speciosa 8 mya (fig. 5). the ancestor of these two sister species were inferred to be hummingbird pollinated (bs = 53.49%). when the ﬂoral color transitions are examined, the switches from bee to hummingbird pollination were mostly accompanied by changes from purple to red ﬂoral color (data not shown).

7.29 mya, and (iii) sairocarpus-galvezia lineage (bs = 97.04%) between 9.77 and 5.98 mya. one hummingbird to bee pollination transition was observed within the gambelia-galvezia lineage, when the bee pollinated species schweinfurthia papilionaceae split from its sister species gambelia speciosa 8 mya (fig. 5). the ancestor of these two sister species were inferred to be hummingbird pollinated (bs = 53.49%). when the ﬂoral color transitions are examined, the switches from bee to hummingbird pollination were mostly accompanied by changes from purple to red ﬂoral color (data not shown). out of 12 hummingbird-pollinated species, nine of them have red-dominant (crimson, red, magenta;

(fig. 5). the ancestor of these two sister species were inferred to be hummingbird pollinated (bs = 53.49%). when the ﬂoral color transitions are examined, the switches from bee to hummingbird pollination were mostly accompanied by changes from purple to red ﬂoral color (data not shown). out of 12 hummingbird-pollinated species, nine of them have red-dominant (crimson, red, magenta; collectively called red from now on) ﬂowers, one has yellowdominant (cream, yellow, brown, orange; collectively called yellow from now on) ﬂowers, and two have purple-dominant (pink, mauve, lilac, violet, purple, blue; collectively called purple from now on) 3.5. trait-dependent diversiﬁcation patterns in

in a variety of taxa including ants (savolainen and vepsäläinen, 2003), birds (sorenson et al., 2003), palms (savolainen et al., 2006), and monkeyﬂowers (grossenbacher and whittall, 2011). however, because geographic ranges can shift over time, the geographic speciation mode can be obscure (anacker and strauss, 2014), and making once sympatric species currently allopatric (graham and lundelius, 1984), or vice versa (berlocher, 1998). throughout its evolutionary history, antirrhineae has experienced four independent cross-continent dispersal events (ogutcen and vamosi, 2016), which might inﬂuence the correlation between time since divergence and range overlap. these ancient splits that have zero range overlap will skew the

remained a controversial concept (mayr, 1942; coyne and orr, 2004). it recently became clear that geographic speciation is not bimodal, but a contin44 perspectives in plant ecology, evolution and systematics 26 (2017) 39 - 52 e. ogutcen et al. fig. 4. the pollination modes and the geographic distributions of the species in the antirrhineae phylogeny. left panel: for pollination mode, 34% of the species had no available data. bee pollination and hummingbird pollination percentages are shown on the top pie chart; old world distribution and new world distribution percentages are shown on the bottom pie chart. right panel: pollination and distribution

clear that geographic speciation is not bimodal, but a contin44 perspectives in plant ecology, evolution and systematics 26 (2017) 39 - 52 e. ogutcen et al. fig. 4. the pollination modes and the geographic distributions of the species in the antirrhineae phylogeny. left panel: for pollination mode, 34% of the species had no available data. bee pollination and hummingbird pollination percentages are shown on the top pie chart; old world distribution and new world distribution percentages are shown on the bottom pie chart. right panel: pollination and distribution states for each tip on the antirrhineae phylogeny. large contributors to antirrhineae diversiﬁcation.

is the major factor in the plantaginaceae diversiﬁcation, whereas regional diﬀerences in diversiﬁcation are not signiﬁcant. may also skew the age-range correlation within the tribe, masking the true speciation mode in the region. in addition to these factors explained above, undersampling can be another source of error. \"absence\" of a species in a particular region can either mean the area is not within the species' range, or simply the species has not been sampled in that area. finally, as previously discussed by other authors (phillimore et al., 2008; hodge et al., 2013). the age-range analysis is accurate only when one mode

not signiﬁcant. may also skew the age-range correlation within the tribe, masking the true speciation mode in the region. in addition to these factors explained above, undersampling can be another source of error. \"absence\" of a species in a particular region can either mean the area is not within the species' range, or simply the species has not been sampled in that area. finally, as previously discussed by other authors (phillimore et al., 2008; hodge et al., 2013). the age-range analysis is accurate only when one mode of speciation is dominant, but most clades experience a mixture of diﬀerent geographic speciation

the age-range correlation within the tribe, masking the true speciation mode in the region. in addition to these factors explained above, undersampling can be another source of error. \"absence\" of a species in a particular region can either mean the area is not within the species' range, or simply the species has not been sampled in that area. finally, as previously discussed by other authors (phillimore et al., 2008; hodge et al., 2013). the age-range analysis is accurate only when one mode of speciation is dominant, but most clades experience a mixture of diﬀerent geographic speciation modes throughout their evolutionary history,

to the new world and dispersal from the new world to the old world respectively. bottom panel: regional diﬀerences in the parameters used. s.diﬀ: sa-sb, x.diﬀ: xa-xb, and d.diﬀ: da-db. fig. 6. floral color transitions in the hummingbird-pollinated lineages in antirrhineae. the branch colors represent the ﬂoral color of the species, and the outgroup represents the sister clade for the hummingbird-pollinated lineage on each tree. support value (bs = 54%), which could be due to the presence of a beepollinated species (schweinfurthia papilionacea) within the lineage. s. papilionacea experienced a transition from hummingbird to bee pollination, which was in conjunction with

fig. 6. floral color transitions in the hummingbird-pollinated lineages in antirrhineae. the branch colors represent the ﬂoral color of the species, and the outgroup represents the sister clade for the hummingbird-pollinated lineage on each tree. support value (bs = 54%), which could be due to the presence of a beepollinated species (schweinfurthia papilionacea) within the lineage. s. papilionacea experienced a transition from hummingbird to bee pollination, which was in conjunction with the long-distance dispersal event from the new world to the old world. therefore, our ﬁnding supports that transition from bee to hummingbird pollination is not an irreversible process (tripp and

virtually irreversible, because restoring purple pigment production would involve simultaneous gain-of-function mutation of several genes in the pathway (zufall and rausher, 2003). previous studies have shown correlations between red ﬂowers and hummingbird pollination (schemske and bradshaw, 1999; rausher, 2008). considering our ﬁnding that red color is not universal in hummingbird-pollinated species, and that all hummingbird pollinated species with red ﬂowers are nested within purple-ﬂowered clades, a pattern that has been shown to be the case in many other clades (zufall and rausher, 2003; rausher, 2008; thomson and wilson, 2008), we posit that hummingbird pollination in antirrhineae likely table 2 geosse analysis

production would involve simultaneous gain-of-function mutation of several genes in the pathway (zufall and rausher, 2003). previous studies have shown correlations between red ﬂowers and hummingbird pollination (schemske and bradshaw, 1999; rausher, 2008). considering our ﬁnding that red color is not universal in hummingbird-pollinated species, and that all hummingbird pollinated species with red ﬂowers are nested within purple-ﬂowered clades, a pattern that has been shown to be the case in many other clades (zufall and rausher, 2003; rausher, 2008; thomson and wilson, 2008), we posit that hummingbird pollination in antirrhineae likely table 2 geosse analysis results. (a) parameter values are shown

color and pollination modes, because some of the pollination data retrieved for this study were inferred from pollination syndromes (elisens, 1985, 1986; elisens and freeman, 1988), which would give false positive correlation between the two traits. it is also important to note that whereas most of the hummingbird pollinated antirrhineae species have red ﬂowers, this is not always the case. some hummingbird-pollinated species are purple (e.g.: sairocarpus multiﬂorus, sairocarpus virga), and some red ﬂowered species are not pollinated by hummingbirds (e.g.: linaria amoi). it is important to note that bees are able to see red, and that red ﬂoral color does

this study were inferred from pollination syndromes (elisens, 1985, 1986; elisens and freeman, 1988), which would give false positive correlation between the two traits. it is also important to note that whereas most of the hummingbird pollinated antirrhineae species have red ﬂowers, this is not always the case. some hummingbird-pollinated species are purple (e.g.: sairocarpus multiﬂorus, sairocarpus virga), and some red ﬂowered species are not pollinated by hummingbirds (e.g.: linaria amoi). it is important to note that bees are able to see red, and that red ﬂoral color does not exclude bees as visitors (chittka and waser, 1997). these observations suggest

freeman, 1988), which would give false positive correlation between the two traits. it is also important to note that whereas most of the hummingbird pollinated antirrhineae species have red ﬂowers, this is not always the case. some hummingbird-pollinated species are purple (e.g.: sairocarpus multiﬂorus, sairocarpus virga), and some red ﬂowered species are not pollinated by hummingbirds (e.g.: linaria amoi). it is important to note that bees are able to see red, and that red ﬂoral color does not exclude bees as visitors (chittka and waser, 1997). these observations suggest that the pollination modes inferred solely by pollination syndromes should be approached

with spatial and environmental heterogeneity, unique geological properties, and inﬂuences from its surrounding ﬂoras, the mediterranean region hosts rich plant diversity (thompson, 2005). therefore, dispersing to a diﬀerent environment may not be advantageous for the mediterranean clades, because they have traits that are likely adapted for mediterranean conditions. most antirrhineae species have ﬂoral traits that are often linked to bee-pollination. the features of typical antirrhineae ﬂowers include zygomorphy, tubular corollas enclosed by a lip, and nectar containing spurs (ghebrehiwet et al., 2000; guzmán et al., 2015). considering the high frequency of these bee-pollination related traits within the tribe, hummingbird pollination may

hummingbird pollination may not be advantageous to a clade that migrated from a bee-dominant region. the emergence of hummingbird pollination in the new world from bee pollinated old world ancestors in the mediterranean was relatively recent in the tribe. therefore, instead of specializing in hummingbird pollination, the new world antirrhineae species may simply incorporate hummingbirds into their pollinator suite and become more generalist. depending on the strength and eﬃciency of reproductive barriers, sympatric speciation may take a long time, and pollinator p", 
" may be an insuﬃcient mechanism to establish and maintain reproductive isolation between the two sympatric populations (van der

2014). it was previously shown that pollinator p", " alone is usually not strong enough to maintain reproductive isolation, and that additional mechanisms may be necessary to form an eﬀective reproductive barrier between the two diverging populations (gegear and burns, 2007; armbruster, 2014). it would be interesting to investigate hummingbird-pollinated species at population levels to see whether they show any signatures of population structure hinting at early stages of speciation, and if they do, research should focus on whether there are other modes of reproductive isolation along with pollinator p", 
" to facilitate speciation. 4.4. trait-speciﬁc diversiﬁcation rates in antirrhineae: adding

new world biodiversity hotspot (bleiweiss, 1998; rymer et al., 2010). on the other hand, antirrhineae is predominantly an old world tribe, and they are densely distributed in the mediterranean basin, which is another biodiversity hotspot. during the last glacial period, the mediterranean basin served as a refugium for many plant species, and the landmasses in the area have split up and merged several times throughout the history (thompson, 5. conclusion in this study, we showed that (i) the evolutionary rate was homogeneous within antirrhineae; (ii) range overlap between sister taxa decreases over time, suggesting a signiﬁcant impact of sympatric speciation on

eﬀect of any character states on diversiﬁcation rates was important to avoid false funding this work was supported by an nserc dg to jcv. acknowledgements the authors thank d. grossenbacher for providing an r script and instructions to calculate age-range correlations. appendix a geographic distribution and pollination data for the species used in the study. ow: old world, nw: new world. the question mark (?) indicates pollination data was not available for the species. note that the hummingbird-pollinated species are also visited by bees, but not vice versa. species distribution pollination acanthorrhinum ramosissimum albraunia foveopilosa anarrhinum bellidifolium anarrhinum corsicum anarrhinum fruticosum

jcv. acknowledgements the authors thank d. grossenbacher for providing an r script and instructions to calculate age-range correlations. appendix a geographic distribution and pollination data for the species used in the study. ow: old world, nw: new world. the question mark (?) indicates pollination data was not available for the species. note that the hummingbird-pollinated species are also visited by bees, but not vice versa. species distribution pollination acanthorrhinum ramosissimum albraunia foveopilosa anarrhinum bellidifolium anarrhinum corsicum anarrhinum fruticosum antirrhinum australe antirrhinum barrelieri antirrhinum braun-blanquetii antirrhinum charidemi antirrhinum graniticum antirrhinum grosii antirrhinum hispanicum antirrhinum latifolium antirrhinum majus antirrhinum meonanthum antirrhinum microphyllum antirrhinum

d. grossenbacher for providing an r script and instructions to calculate age-range correlations. appendix a geographic distribution and pollination data for the species used in the study. ow: old world, nw: new world. the question mark (?) indicates pollination data was not available for the species. note that the hummingbird-pollinated species are also visited by bees, but not vice versa. species distribution pollination acanthorrhinum ramosissimum albraunia foveopilosa anarrhinum bellidifolium anarrhinum corsicum anarrhinum fruticosum antirrhinum australe antirrhinum barrelieri antirrhinum braun-blanquetii antirrhinum charidemi antirrhinum graniticum antirrhinum grosii antirrhinum hispanicum antirrhinum latifolium antirrhinum majus antirrhinum meonanthum antirrhinum microphyllum antirrhinum molle antirrhinum pertegasii antirrhinum pulverulentum

calculate age-range correlations. appendix a geographic distribution and pollination data for the species used in the study. ow: old world, nw: new world. the question mark (?) indicates pollination data was not available for the species. note that the hummingbird-pollinated species are also visited by bees, but not vice versa. species distribution pollination acanthorrhinum ramosissimum albraunia foveopilosa anarrhinum bellidifolium anarrhinum corsicum anarrhinum fruticosum antirrhinum australe antirrhinum barrelieri antirrhinum braun-blanquetii antirrhinum charidemi antirrhinum graniticum antirrhinum grosii antirrhinum hispanicum antirrhinum latifolium antirrhinum majus antirrhinum meonanthum antirrhinum microphyllum antirrhinum molle antirrhinum pertegasii antirrhinum pulverulentum antirrhinum sempervirens antirrhinum siculum antirrhinum valentinum asarina procumbens chaenorhinum cryptarum

j. exp. bot. 59, 715 - 727. danforth, b.n., sipes, s., fang, j., brady, s.g., 2006. the history of early bee diversiﬁcation based on ﬁve genes plus morphology. proc. natl. acad. sci. u.s.a. 103, 15118 - 15123. dodd, m.e., silvertown, j., chase, m.w., 1999. phylogenetic analysis of trait evolution and species diversity variation among angiosperm families. evolution 53, 732 - 744. elisens, w.j., 1985. monograph of the maurandyinae (scrophulariaceae-antirrhineae). syst. bot. monogr. 5, 1 - 97. elisens, w.j., 1986. pollen morphology and systematic relationships among new world species in tribe antirrhineae (scrophulariaceae). am. j. bot. 73, 1298 - 1311. elisens, w.j.,

dodd, m.e., silvertown, j., chase, m.w., 1999. phylogenetic analysis of trait evolution and species diversity variation among angiosperm families. evolution 53, 732 - 744. elisens, w.j., 1985. monograph of the maurandyinae (scrophulariaceae-antirrhineae). syst. bot. monogr. 5, 1 - 97. elisens, w.j., 1986. pollen morphology and systematic relationships among new world species in tribe antirrhineae (scrophulariaceae). am. j. bot. 73, 1298 - 1311. elisens, w.j., freeman, c.e., 1988. floral nectar sugar composition and pollinator type among new world genera in tribe antirrhineae (scrophulariaceae). am. j. bot. 75, 971 - 978. etienne, r.s., apol, m.e.f., 2008. estimating speciation and extinction rates from diversity

### NA
greater specialization for sex-specific functions and can be advantageous under various ecological and environmental conditions. however, dioecy is rare among flowering plants. previous studies identified contradictory trends regarding the relative diversification rates of dioecious lineages vs their nondioecious counterparts, depending on the methods and data used. \017 we gathered detailed species-level data for dozens of genera that contain both dioecious and nondioecious species. we then applied a probabilistic approach that accounts for differential speciation, extinction, and transition rates between states to examine whether there is an association between dioecy and lineage diversification. \017 we found a bimodal distribution, whereby dioecious lineages

and environmental conditions. however, dioecy is rare among flowering plants. previous studies identified contradictory trends regarding the relative diversification rates of dioecious lineages vs their nondioecious counterparts, depending on the methods and data used. \017 we gathered detailed species-level data for dozens of genera that contain both dioecious and nondioecious species. we then applied a probabilistic approach that accounts for differential speciation, extinction, and transition rates between states to examine whether there is an association between dioecy and lineage diversification. \017 we found a bimodal distribution, whereby dioecious lineages exhibited higher diversification in certain genera but lower diversification in others. additional

brake on diversification nor is a strong driver. introduction sexual reproduction entails the union of a male gamete (sperm) and a female gamete (egg), but in angiosperms the universality of separate sexes at the gametophytic stage is coupled with a diverse array of sexual systems at the sporophytic stage. some species have separate male and female sexes (dioecy), many species are entirely hermaphroditic, and virtually every variant in between is found (supporting information table s1). because of the high degree of variability in sexual systems, flowering plants provide an ideal group in which to conduct comparative evolutionary analyses to understand the

sexual reproduction entails the union of a male gamete (sperm) and a female gamete (egg), but in angiosperms the universality of separate sexes at the gametophytic stage is coupled with a diverse array of sexual systems at the sporophytic stage. some species have separate male and female sexes (dioecy), many species are entirely hermaphroditic, and virtually every variant in between is found (supporting information table s1). because of the high degree of variability in sexual systems, flowering plants provide an ideal group in which to conduct comparative evolutionary analyses to understand the evolution of sexual systems. in particular, there is a

in sexual systems, flowering plants provide an ideal group in which to conduct comparative evolutionary analyses to understand the evolution of sexual systems. in particular, there is a long-standing interest in understanding why dioecy is such a rare sexual system in flowering plants, present in only c. 6% of the species (renner, 2014) compared with c. 95% in animals (jarne & auld, 2006). the rarity of a trait (here dioecy) can be explained in a number of ways: it may slow speciation, hasten extinction, be slow to emerge from an ancestral state, or disappear faster than it appears (thomson & brunet,

might be positively selected because it dampens trade-offs between male and female functions. dioecy may also be selectively favored under certain environmental conditions. for example, muenchow (1987) argued that dioecy may be favored in understory habitats where nonspecific pollinators, which are often associated with dioecy, tend to be found. dioecious species have also flourished on the hawaiian islands, perhaps because dispersal among islands favors long-lived tree species with fleshy fruits, both traits that are associated with dioecy (schwander & crespi, 2009). despite these potential advantages, species richness of dioecious angiosperm clades is often lower than that of hermaphroditic sister groups (heilbuth,

be selectively favored under certain environmental conditions. for example, muenchow (1987) argued that dioecy may be favored in understory habitats where nonspecific pollinators, which are often associated with dioecy, tend to be found. dioecious species have also flourished on the hawaiian islands, perhaps because dispersal among islands favors long-lived tree species with fleshy fruits, both traits that are associated with dioecy (schwander & crespi, 2009). despite these potential advantages, species richness of dioecious angiosperm clades is often lower than that of hermaphroditic sister groups (heilbuth, 2000; kay et al., 2006). dioecious lineages may suffer a lower diversification rate for a number

habitats where nonspecific pollinators, which are often associated with dioecy, tend to be found. dioecious species have also flourished on the hawaiian islands, perhaps because dispersal among islands favors long-lived tree species with fleshy fruits, both traits that are associated with dioecy (schwander & crespi, 2009). despite these potential advantages, species richness of dioecious angiosperm clades is often lower than that of hermaphroditic sister groups (heilbuth, 2000; kay et al., 2006). dioecious lineages may suffer a lower diversification rate for a number of reasons. species ó 2015 the authors new phytologist ó 2015 new phytologist trust new phytologist that are dioecious

fruits, both traits that are associated with dioecy (schwander & crespi, 2009). despite these potential advantages, species richness of dioecious angiosperm clades is often lower than that of hermaphroditic sister groups (heilbuth, 2000; kay et al., 2006). dioecious lineages may suffer a lower diversification rate for a number of reasons. species ó 2015 the authors new phytologist ó 2015 new phytologist trust new phytologist that are dioecious may be less able to colonize new sites because of the need for a mating partner (baker, 1955; pannell & barrett, 1998) or because seed production is restricted to the female half of a

(barrett, 2003). alternatively, sexual dimorphism driven by stronger sexual selection acting on males to attract pollinators can place dioecious lineages at greater risk of extinction, particularly when pollinator availability is fluctuating (vamosi & otto, 2002). indeed, a survey of the international union for conservation of nature red list of threatened species revealed that dioecious families are enriched for species at risk (vamosi & vamosi, 2005), thus hinting that dioecy is associated with elevated extinction risk, at least under present conditions. furthermore, dioecious lineages tend to exhibit a suite of correlated traits, including abiotic pollination, biotic dispersal, woody growth form, tropical distribution

sexual selection acting on males to attract pollinators can place dioecious lineages at greater risk of extinction, particularly when pollinator availability is fluctuating (vamosi & otto, 2002). indeed, a survey of the international union for conservation of nature red list of threatened species revealed that dioecious families are enriched for species at risk (vamosi & vamosi, 2005), thus hinting that dioecy is associated with elevated extinction risk, at least under present conditions. furthermore, dioecious lineages tend to exhibit a suite of correlated traits, including abiotic pollination, biotic dispersal, woody growth form, tropical distribution (renner & ricklefs, 1995; vamosi et al., 2003;

underlying mechanism. in addition, sister-clade contrasts discard phylogenetic information both internal and external to the sister clades and assume only a single trait transition differentiating the two sisters (but see heilbuth, 2000). furthermore, a recent analysis demonstrated that sister-clade comparisons can be strongly biased when the derived state has fewer species because the waiting time until the derived state first appears reduces opportunities for diversification in that state (k€afer & mousset, 2014). notably, using a resampling test that should correct this problem, k€afer & mousset (2014) reanalyzed the data of heilbuth (2000) and concluded that dioecy does not lead to lower

associations, rabosky & goldberg (2015) suggested performing comparative analyses on multiple data sets, drawing conclusions based on a preponderance of evidence (mayrose et al., 2011; rolland et al., 2014). to this end, we gathered data on sexual systems in many taxonomic groups (largely genera) that contain both dioecious and nondioecious species (tree of sex consortium, 2014). in this study, we assembled molecular phylogenies for dozens of genera for which we had detailed sexual system information. this allowed us to investigate the consequences of dioecy for rates of speciation and extinction using the binary state speciation and extinction (bisse) likelihood method (maddison

moreover, we examined whether the diversification rate of dioecious lineages was significantly correlated with the form of the nondioecious taxa (primarily hermaphroditic or primarily monoecious), given that some of the advantages of dioecy may be new phytologist (2016) 209: 1290 - 1300 www.newphytologist.com new phytologist 1292 research shared by monoecious species (reduced inbreeding and an elimination of trade-offs at the flower level between male and female function; barrett, 2002, 2003). finally, we conducted extensive simulations to reveal the distribution of the relative diversification rates of dioecious lineages expected for a 'neutral' trait, given the observed trees and transition rates. we close

observed patterns, including limitations of the current data sets. materials and methods database a broad data set of angiosperm sexual systems was assembled as part of a working group sponsored by the national evolutionary synthesis center, with a focus on 77 genera exhibiting both the presence and absence of dioecious species (tree of sex consortium, 2014). the genera used in the current study were chosen largely from the list of angiosperm genera that contain dioecious species (renner & ricklefs, 1995) and supplemented with genera analyzed by miller & venable (2000), focusing on groups with sufficient genetic data in genbank (benson et

of a working group sponsored by the national evolutionary synthesis center, with a focus on 77 genera exhibiting both the presence and absence of dioecious species (tree of sex consortium, 2014). the genera used in the current study were chosen largely from the list of angiosperm genera that contain dioecious species (renner & ricklefs, 1995) and supplemented with genera analyzed by miller & venable (2000), focusing on groups with sufficient genetic data in genbank (benson et al., 2013). while ideally all sexual systems would be considered separately, doing so in a diversification analysis leads to a large number of parameters (speciation

systems with some unisexual plants (including dioecy, gynodioecy, androdioecy, and polygamodioecy), in contrast to the 'bisexual' state where both sexes are found on every plant (hermaphrodites and all forms of monoecy). in addition, the resulting data set was filtered to genera for which the reconstructed phylogeny included at least 10 species and at least two species in each state (dioecy and nondioecy). taxonomic name resolution was largely performed using the plant list (v.1.1; http://www.theplantlist.org/). we also employed synonymy provided by experts in particular groups (notably solanaceae source; http://solanaceaesource.org). the resulting data set included 38 genera in the 'strict' classification and 41

(including dioecy, gynodioecy, androdioecy, and polygamodioecy), in contrast to the 'bisexual' state where both sexes are found on every plant (hermaphrodites and all forms of monoecy). in addition, the resulting data set was filtered to genera for which the reconstructed phylogeny included at least 10 species and at least two species in each state (dioecy and nondioecy). taxonomic name resolution was largely performed using the plant list (v.1.1; http://www.theplantlist.org/). we also employed synonymy provided by experts in particular groups (notably solanaceae source; http://solanaceaesource.org). the resulting data set included 38 genera in the 'strict' classification and 41 in the 'broad' classification (table

source; http://solanaceaesource.org). the resulting data set included 38 genera in the 'strict' classification and 41 in the 'broad' classification (table s2). phylogenetic reconstruction the phylogeny for each genus was reconstructed using the following general procedures. (a detailed description of the steps is provided in methods s1). sequence data for all species in the genus were automatically retrieved from genbank (benson et al., 2013) using national center for biotechnology information taxonomy. this was followed by a name resolution procedure that matched species names as they appear in genbank to accepted taxonomic new phytologist (2016) 209: 1290 - 1300 www.newphytologist.com names (according to

using the following general procedures. (a detailed description of the steps is provided in methods s1). sequence data for all species in the genus were automatically retrieved from genbank (benson et al., 2013) using national center for biotechnology information taxonomy. this was followed by a name resolution procedure that matched species names as they appear in genbank to accepted taxonomic new phytologist (2016) 209: 1290 - 1300 www.newphytologist.com names (according to the plant list and solanaceae source). sequences were then clustered to orthology groups using orthomcl (li et al., 2003). an appropriate outgroup was chosen and then added to each cluster.

and accounts for correlations between speciation and extinction. nevertheless, we complemented each diversification analysis with one in which the bisse model is constrained to enforce equal extinction in both states. our analyses were performed using the 'skeletal' tree approach (fitzjohn et al., 2009), which accounts for the sampling fraction of species in the given phylogeny out of the total number of species in the clade. sampling fraction estimates were calculated based on the number of species in a genus as obtained through a literature search (see", 
"s in table s2). to assess the sensitivity of our results to the assumed fraction

each diversification analysis with one in which the bisse model is constrained to enforce equal extinction in both states. our analyses were performed using the 'skeletal' tree approach (fitzjohn et al., 2009), which accounts for the sampling fraction of species in the given phylogeny out of the total number of species in the clade. sampling fraction estimates were calculated based on the number of species in a genus as obtained through a literature search (see", 
"s in table s2). to assess the sensitivity of our results to the assumed fraction of sampled species, we also used the total number of accepted

equal extinction in both states. our analyses were performed using the 'skeletal' tree approach (fitzjohn et al., 2009), which accounts for the sampling fraction of species in the given phylogeny out of the total number of species in the clade. sampling fraction estimates were calculated based on the number of species in a genus as obtained through a literature search (see", 
"s in table s2). to assess the sensitivity of our results to the assumed fraction of sampled species, we also used the total number of accepted and unresolved species from the plant list (v.1.1; http://www.theplantlist.org/), which on average yields twice

given phylogeny out of the total number of species in the clade. sampling fraction estimates were calculated based on the number of species in a genus as obtained through a literature search (see", 
"s in table s2). to assess the sensitivity of our results to the assumed fraction of sampled species, we also used the total number of accepted and unresolved species from the plant list (v.1.1; http://www.theplantlist.org/), which on average yields twice the number of species (table s2); this second analysis generated extremely similar results, so only the results using the number of species in the genus reported in the

clade. sampling fraction estimates were calculated based on the number of species in a genus as obtained through a literature search (see", 
"s in table s2). to assess the sensitivity of our results to the assumed fraction of sampled species, we also used the total number of accepted and unresolved species from the plant list (v.1.1; http://www.theplantlist.org/), which on average yields twice the number of species (table s2); this second analysis generated extremely similar results, so only the results using the number of species in the genus reported in the literature are given here. a markov chain monte carlo (mcmc) sampling

as obtained through a literature search (see", 
"s in table s2). to assess the sensitivity of our results to the assumed fraction of sampled species, we also used the total number of accepted and unresolved species from the plant list (v.1.1; http://www.theplantlist.org/), which on average yields twice the number of species (table s2); this second analysis generated extremely similar results, so only the results using the number of species in the genus reported in the literature are given here. a markov chain monte carlo (mcmc) sampling approach described in fitzjohn et al. (2009) was used to estimate the posterior probability distributions

to the assumed fraction of sampled species, we also used the total number of accepted and unresolved species from the plant list (v.1.1; http://www.theplantlist.org/), which on average yields twice the number of species (table s2); this second analysis generated extremely similar results, so only the results using the number of species in the genus reported in the literature are given here. a markov chain monte carlo (mcmc) sampling approach described in fitzjohn et al. (2009) was used to estimate the posterior probability distributions for each of the six parameters, accounting for uncertainty in parameter estimation, incomplete sampling, and phylogeny. an exponential

abiotic), dispersal mode (biotic vs abiotic), and geographical distribution (tropical vs temperate). these other traits were coded based on the genus level categorization of renner (2014). for growth form, renner (2014) only specified presence/absence of trees, shrubs, herbs, or climbers, and many of the genera contained both woody and herbaceous species. to determine whether the ó 2015 the authors new phytologist ó 2015 new phytologist trust research 1293 genus was predominantly woody or herbaceous, we gathered species-level life form data from the literature (tree of sex consortium, 2014), supplemented with data from zanne et al. (2014) and data collected from eflora

for growth form, renner (2014) only specified presence/absence of trees, shrubs, herbs, or climbers, and many of the genera contained both woody and herbaceous species. to determine whether the ó 2015 the authors new phytologist ó 2015 new phytologist trust research 1293 genus was predominantly woody or herbaceous, we gathered species-level life form data from the literature (tree of sex consortium, 2014), supplemented with data from zanne et al. (2014) and data collected from eflora (brach & song, 2006). genera that are mostly (≥ 70%) woody or herbaceous were defined as such. genera exhibiting a mixture of the growth forms (with

geographical distribution, the database of renner (2014) specified whether the genus is distributed in tropical and/or temperate regions. again, genera distributed in both tropical and temperate regions were excluded from the particular comparison. in addition to the life history traits described earlier, we also examined whether the types of nondioecious species present in a genus are associated with the inferred diversification rates. we categorized genera in which the nondioecious state is mostly (≥ 70%) hermaphrodite as 'mostly h'; genera with mostly (≥ 70%) monoecy were labeled as 'mostly m'; all other genera were dropped from this analysis. for each categorization, we

extinction rates. of the 41 genera considered within strict dioecy (31) 3.0 broad dioecy (36) 0 to date, the most comprehensive database detailing the taxonomic distribution of dioecy was first compiled by renner & ricklefs (1995) and recently updated (renner, 2014). the latest database lists 987 genera in which dioecious species are known to exist, c. 1/3 of which vary in sexual system within the genus. this database is not suitable for detailed comparative phylogenetic analysis because sexual system assignments are given at the genus rather than species level. to this end, we summarized the tree of sex database of species-level

recently updated (renner, 2014). the latest database lists 987 genera in which dioecious species are known to exist, c. 1/3 of which vary in sexual system within the genus. this database is not suitable for detailed comparative phylogenetic analysis because sexual system assignments are given at the genus rather than species level. to this end, we summarized the tree of sex database of species-level sexual system assignments for angiosperm genera with variation in sexual system: 38 genera were included in the 'strict' classification and 41 in the 'broad' classifications. the 'strict' classification includes 475 dioecious and 1535 nondioecious species, whereas the

species are known to exist, c. 1/3 of which vary in sexual system within the genus. this database is not suitable for detailed comparative phylogenetic analysis because sexual system assignments are given at the genus rather than species level. to this end, we summarized the tree of sex database of species-level sexual system assignments for angiosperm genera with variation in sexual system: 38 genera were included in the 'strict' classification and 41 in the 'broad' classifications. the 'strict' classification includes 475 dioecious and 1535 nondioecious species, whereas the 'broad' classification includes 663 species with unisexual plants and 1676 consisting of bisexual

rather than species level. to this end, we summarized the tree of sex database of species-level sexual system assignments for angiosperm genera with variation in sexual system: 38 genera were included in the 'strict' classification and 41 in the 'broad' classifications. the 'strict' classification includes 475 dioecious and 1535 nondioecious species, whereas the 'broad' classification includes 663 species with unisexual plants and 1676 consisting of bisexual plants only (table s2). in total, the reconstructed phylogenies of these genera include 3380 species and cover on average 42% of the recognized species in these groups. 2 sexual system database frequency results the 'broad'

we summarized the tree of sex database of species-level sexual system assignments for angiosperm genera with variation in sexual system: 38 genera were included in the 'strict' classification and 41 in the 'broad' classifications. the 'strict' classification includes 475 dioecious and 1535 nondioecious species, whereas the 'broad' classification includes 663 species with unisexual plants and 1676 consisting of bisexual plants only (table s2). in total, the reconstructed phylogenies of these genera include 3380 species and cover on average 42% of the recognized species in these groups. 2 sexual system database frequency results the 'broad' classification, preliminary bisse runs indicated that the

included in the 'strict' classification and 41 in the 'broad' classifications. the 'strict' classification includes 475 dioecious and 1535 nondioecious species, whereas the 'broad' classification includes 663 species with unisexual plants and 1676 consisting of bisexual plants only (table s2). in total, the reconstructed phylogenies of these genera include 3380 species and cover on average 42% of the recognized species in these groups. 2 sexual system database frequency results the 'broad' classification, preliminary bisse runs indicated that the results for five data sets were highly dependent on the exact prior choice and were thus deemed unreliable. dropping these genera, our final

'broad' classifications. the 'strict' classification includes 475 dioecious and 1535 nondioecious species, whereas the 'broad' classification includes 663 species with unisexual plants and 1676 consisting of bisexual plants only (table s2). in total, the reconstructed phylogenies of these genera include 3380 species and cover on average 42% of the recognized species in these groups. 2 sexual system database frequency results the 'broad' classification, preliminary bisse runs indicated that the results for five data sets were highly dependent on the exact prior choice and were thus deemed unreliable. dropping these genera, our final data set consisted of 36 genera, whose reconstructed phylogenies

groups. 2 sexual system database frequency results the 'broad' classification, preliminary bisse runs indicated that the results for five data sets were highly dependent on the exact prior choice and were thus deemed unreliable. dropping these genera, our final data set consisted of 36 genera, whose reconstructed phylogenies encompassed 2702 species (table s2). fig. 1 presents the distribution of pp(rn > rd) values with each genus represented by the value inferred across the sample of 100 bayesian trees. (the per-tree pp(rn > rd) values, as well as the raw distributions of rn and rd, for each genus are shown in fig.

we denote these two approaches as 'free tip frequency' and 'matched tip frequency'. 20 40 60 80 pp(r n > rd ) 100 0 20 40 60 80 pp(r n > rd ) 100 fig. 1 the proportion of the posterior probability distribution supporting a higher diversification rate for nondioecious species than dioecious species, pp(rn > rd), using (a) 'broad' and (b) 'strict' definitions of dioecy. with either classification, the distribution of pp(rn > rd) values did not deviate from 50% (p = 0.13; one-sample wilcoxon rank-sum test). in the 'broad' classification, the distribution showed significant evidence of multimodality (p =

two approaches as 'free tip frequency' and 'matched tip frequency'. 20 40 60 80 pp(r n > rd ) 100 0 20 40 60 80 pp(r n > rd ) 100 fig. 1 the proportion of the posterior probability distribution supporting a higher diversification rate for nondioecious species than dioecious species, pp(rn > rd), using (a) 'broad' and (b) 'strict' definitions of dioecy. with either classification, the distribution of pp(rn > rd) values did not deviate from 50% (p = 0.13; one-sample wilcoxon rank-sum test). in the 'broad' classification, the distribution showed significant evidence of multimodality (p = 0.048; hartigans' dip

six parameters strict broad strict broad 38 41 38 41 31 36 30 34 0.13 0.17 0.34 0.31 0.1838 0.0482 0.1710 0.0374 0.9 (0) 0.93 (0) 1 (0) 1 (0) 0.11 (0.55) 0.07 (0.70) nag nag constrained extinction (ln = ld) a the number of genera with at least 10 species total and two species in each state. the number of genera remaining after removing genera that exhibit sensitivity to parameters. c p-value for testing if the distribution of pp(rn > rd) deviates from 50% (one-sample wilcoxon rank-sum test). d p-value for testing if the distribution of pp(rn > rd) deviates

strict broad 38 41 38 41 31 36 30 34 0.13 0.17 0.34 0.31 0.1838 0.0482 0.1710 0.0374 0.9 (0) 0.93 (0) 1 (0) 1 (0) 0.11 (0.55) 0.07 (0.70) nag nag constrained extinction (ln = ld) a the number of genera with at least 10 species total and two species in each state. the number of genera remaining after removing genera that exhibit sensitivity to parameters. c p-value for testing if the distribution of pp(rn > rd) deviates from 50% (one-sample wilcoxon rank-sum test). d p-value for testing if the distribution of pp(rn > rd) deviates from a uniform or

equal (ln and ld), suggesting that the known difficulty in accurately estimating extinction rates (rabosky, 2010) does not influence the conclusions using either 'strict' or 'broad' classifications (table 1; fig. s1). in all four analyses ('strict' or 'broad' classifications, with and without constraining the extinction rates), genera with more nondioecious species tended to display higher pp(rn > rd) values (fig. s5), as expected whether or not sexual system truly influences diversification rates (e.g. when by chance dioecy happens to be overrepresented in a clade, higher rd will be inferred and vice versa). note that several clades have very skewed tip ratios

trait). specifically, we subdivided the genera according to whether they were predominantly herbaceous or woody, temperate or tropical, animal dispersed or abiotically dispersed, animal pollinated or abiotically pollinated, or primarily hermaphroditic or monoecious when considering the ó 2015 the authors new phytologist ó 2015 new phytologist trust research 1295 nondioecious species. as shown in fig. 2, we did not find any significant associations between the relative diversification rates of dioecious lineages and the analyzed traits (all p-values above 0.27). simulations and sampling issues to examine whether our results could be explained solely by elevated false positives from the bisse methodology (fitzjohn,

ost 3) m(1 m broad ly ost m h(1 4) ly ost 6) m(1 m strict fig. 2 the distribution of pp(rn > rd) values for genera classified according to (a) growth form, (b) geographical distribution, (c) mode of fruit dispersal, (d) mode of pollination, and (e) whether the nondioecious species in the genus were predominantly hermaphroditic ('mostly h') vs monoecious ('mostly m'). each panel presents both 'broad' (left side) and 'strict' (right side) classifications. horizontal and vertical lines mark the sample means and \006 se, respectively. for each particular trait, we tested whether the mean of the pp(rn > rd)

we had accounted for uncertainty in the transition rates within the simulated data sets. however, poor sampling may have also reduced our power to detect departures from neutrally simulated characters. in line with this idea, there was a significantly negative correlation between pp(rn > rd) values and the percentage of species within a genus included in the tree (r = 0.53; p = 0.001). indeed, the new phytologist (2016) 209: 1290 - 1300 www.newphytologist.com taxa with the most complete sampling (sidalcea, allocasuarina, fraxinus, momordica, rhus, dodonaea, fragaria, and lepechinia, each with > 60% sampling on the phylogeny) tended to support higher

suggests that there might indeed be some clades where dioecy significantly impacts diversification and that increased sampling would improve our power to detect these clades. discussion here, we asked whether dioecy affects diversification rates by applying probabilistic phylogenetic methods to a large set of genera with both dioecious and nondioecious species. we found no ó 2015 the authors new phytologist ó 2015 new phytologist trust new phytologist research 1297 (a) (c) 8 6 4 frequency 15 80 100 frequency 20 40 60 pp(r n > r d) 80 100 0.10 0.15 0.20 0.15 0.20 4 2 5 0 0 0 0

other studies that used similar methods in other plant groups and did not detect a strong effect of sexual system on diversification rates (leslie et al., 2013; mcdaniel et al., 2013; villarreal & renner, 2013). villarreal & renner (2013) found no difference between the diversification rates of monoicous and dioicous species applying bisse to a 98 species (out of 200) phylogeny of hornworts. using a large conifer phylogeny and the bisse model, leslie et al. (2013) found no effect of sexual system (monoecy or dioecy), cone type (dry or fleshy), or their combination on diversification rates. mcdaniel et al. (2013) used

in other plant groups and did not detect a strong effect of sexual system on diversification rates (leslie et al., 2013; mcdaniel et al., 2013; villarreal & renner, 2013). villarreal & renner (2013) found no difference between the diversification rates of monoicous and dioicous species applying bisse to a 98 species (out of 200) phylogeny of hornworts. using a large conifer phylogeny and the bisse model, leslie et al. (2013) found no effect of sexual system (monoecy or dioecy), cone type (dry or fleshy), or their combination on diversification rates. mcdaniel et al. (2013) used a genus-level phylogeny of 493 moss

model, leslie et al. (2013) found no effect of sexual system (monoecy or dioecy), cone type (dry or fleshy), or their combination on diversification rates. mcdaniel et al. (2013) used a genus-level phylogeny of 493 moss genera and found only a nonsignificant trend towards higher diversification for hermaphrodites over dioecious species. our analysis extends these previous studies by examining multiple clades simultaneously and conducting comprehensive simulations to evaluate the significance of the results. ó 2015 the authors new phytologist ó 2015 new phytologist trust while low power may have limited the fraction of clades with extreme diversification rate asymmetry (in the

that the effect of dioecy on diversification is not strong enough or consistent enough to have imparted a signal in the data analyzed. we emphasize that several aspects of the data may have limited the power of our analyses and/or potentially biased our conclusions. first, on average only 42% of species within each genus were included (between 7 and 100% per genus). although simulation studies showed that such coverage allows detection of differential diversification rates with only modest reductions in power (fitzjohn et al., 2009), greater sampling within genera would have improved our ability to detect the effects of dioecy. second,

diversification rates with only modest reductions in power (fitzjohn et al., 2009), greater sampling within genera would have improved our ability to detect the effects of dioecy. second, we assumed that the sampling fraction was the same for dioecious and nondioecious taxa, which might well be incorrect (e.g. if dioecious species are considered unusual and hence more often investigated and sequenced). however, for each genus, the fraction of dioecious species included on the trees (i.e. with genbank data) was highly correlated with the fraction of dioecious species in the tree of sex database (tree of sex consortium, 2014) (r = 0.94;

our ability to detect the effects of dioecy. second, we assumed that the sampling fraction was the same for dioecious and nondioecious taxa, which might well be incorrect (e.g. if dioecious species are considered unusual and hence more often investigated and sequenced). however, for each genus, the fraction of dioecious species included on the trees (i.e. with genbank data) was highly correlated with the fraction of dioecious species in the tree of sex database (tree of sex consortium, 2014) (r = 0.94; pearson correlation coefficient; fig. s8). that said, species whose sexual system was not specified in our search of the

same for dioecious and nondioecious taxa, which might well be incorrect (e.g. if dioecious species are considered unusual and hence more often investigated and sequenced). however, for each genus, the fraction of dioecious species included on the trees (i.e. with genbank data) was highly correlated with the fraction of dioecious species in the tree of sex database (tree of sex consortium, 2014) (r = 0.94; pearson correlation coefficient; fig. s8). that said, species whose sexual system was not specified in our search of the literature would be missing from both data sets. in particular, many authors might not mention if a

investigated and sequenced). however, for each genus, the fraction of dioecious species included on the trees (i.e. with genbank data) was highly correlated with the fraction of dioecious species in the tree of sex database (tree of sex consortium, 2014) (r = 0.94; pearson correlation coefficient; fig. s8). that said, species whose sexual system was not specified in our search of the literature would be missing from both data sets. in particular, many authors might not mention if a species is hermaphroditic, given how common this sexual system is among flowering plants. underreporting of hermaphroditism might artificially bias our results towards

in the tree of sex database (tree of sex consortium, 2014) (r = 0.94; pearson correlation coefficient; fig. s8). that said, species whose sexual system was not specified in our search of the literature would be missing from both data sets. in particular, many authors might not mention if a species is hermaphroditic, given how common this sexual system is among flowering plants. underreporting of hermaphroditism might artificially bias our results towards higher diversification of dioecious lineages. third, we used only genera that contain at least two species of each sexual system. we may thus have preselected genera with higher than

from both data sets. in particular, many authors might not mention if a species is hermaphroditic, given how common this sexual system is among flowering plants. underreporting of hermaphroditism might artificially bias our results towards higher diversification of dioecious lineages. third, we used only genera that contain at least two species of each sexual system. we may thus have preselected genera with higher than average origination rates and/or diversification rates of the derived state (dioecy for most of our genera). moreover, by including only genera with both dioecious and nondioecious taxa (as required by bisse), a large number of clades that

than average origination rates and/or diversification rates of the derived state (dioecy for most of our genera). moreover, by including only genera with both dioecious and nondioecious taxa (as required by bisse), a large number of clades that are entirely dioecious were ignored (e.g. nearly half of all dioecious angiosperm species are found within 34 clades that are entirely dioecious; renner, 2014). ideally, one should use the complete angiosperm phylogeny (or multiple new phytologist (2016) 209: 1290 - 1300 www.newphytologist.com large-scale phylogenies) in which the sexual system distribution is representative of angiosperms as a whole. this, however, requires much higher coverage

demands and biological heterogeneity of very large phylogenies. fourth, our results may have been affected by the somewhat artificial nature of the binary classifications that we made. that said, we observed very similar results for the 'strict' and 'broad' classifications, even though the 'broad' classification contains c. 30% more dioecy species than the 'strict' classification. considering all of the potential biases discussed earlier, our result that dioecy is not consistently associated with either increased or decreased diversification must be considered preliminary and deserving of further research. we hope that that the issues we have highlighted in obtaining these results will aid

### NA
evolutionary biology (2017) 17:174 doi 10.1186/s12862-017-1016-x research article open access hostplant change and paleoclimatic events explain diversification shifts in skipper butterflies (family: hesperiidae) ranjit kumar sahoo1* , andrew d. warren2, steve c. collins3 and ullasa kodandaramaiah1 abstract background: skippers (family: hesperiidae) are a large group of butterflies with ca. 4000 species under 567 genera. the lack of a time-calibrated higher-level phylogeny of the group has precluded understanding of its evolutionary past. we here use a 10-gene dataset to reconstruct the most comprehensive time-calibrated phylogeny of the group, and explore factors that affected the diversification of these butterflies. results: ancestral state reconstructions

extrinsic limitation for niche exploration. the shifts in diversification rate coincided with paleoclimatic events during which grasses and forest monocots were diversified. keywords: k-pg, ecological opportunity, diversification, paleocene-eocene thermal maximum, grass-feeding, monocot feeding, insect-hostplant, coevolution background the remarkable diversity of life is often attributed to either the gradual accumulation of species over long periods of time [1, 2] or to dramatic changes in diversification rates across lineages and time [3, 4]. however, paleontological evidence [3] and phylogenetic comparisons [4] across the tree of life predict the latter scenario as the major cause, where the fluctuations in speciation and extinction rates explain

([7 - 9]; but see [10, 11]). of particular note is the evolution of herbivory, i.e., shifts from carnivory to herbivory, which has repeatedly elevated diversification rates [12, 13], and this is especially true in the case of insects [13]. indeed, herbivorous insects comprise about half of all terrestrial eukaryotic species [13], exemplifying the importance of insect-plant interactions in © the author(s). 2017 open access this article is distributed under the terms of the creative commons attribution 4.0 international license (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and

otherwise stated. sahoo et al. bmc evolutionary biology (2017) 17:174 generating the diversity of life [14 - 16]. among butterflies, major shifts in hostplant use have led to bursts in diversification rates [17, 18]. for instance, satyrini butterflies (subfamily satyrinae: tribe satyrini), which specialise on grasses and include ca. 2200 species, diversified simultaneously with the expansion of grasslands [19]. therefore, feeding on grasses appears to be closely associated with increased diversification rate in satyrini. paleontological records and comparative phylogenetic analyses indicate that diversification of a taxon is often associated with the extinction of others [20 - 22]. for instance, nymphalid butterflies

possibility is not explored in detail for any group of organisms. we here investigate the pattern of appearance of intrinsic and extrinsic factors of eo and their effect on diversification in a highly speciose but hitherto ignored group of butterflies - the skippers (family hesperiidae). the skippers comprise ca. 4000 species distributed among ca. 567 genera [34]. about 50% of skippers feed on monocots during larval stages. we compiled a 10gene dataset of 7726 bp (base pair) from 290 genera representing all the known major clades. based on estimated times of divergence, we checked whether skippers have experienced rapid shifts in

one mitochondrial gene for the divergence time estimation. our dataset was built upon the previous work [34], to which we added 34 specimens, resulting in a dataset of 290 genera that accounts for nearly 60% of known skipper genera. many skipper genera have highly restricted distributions and comprise very few species, making them logistically very challenging to sample. therefore, despite the massive field effort of several collaborators worldwide spanning more than a decade, we were unable to achieve a more comprehensive sampling of the taxa. however, we believe we have included representatives of all the major clades. a previous study [34]

of divergence for nodes. we then calculated the distribution of relative age estimates of the deeper clades from these ultrametric trees. we used the software beast v2.4.2 [36] on the cipres science gateway [37] to simultaneously estimate phylogenetic relationships and times of divergences for our current dataset. sequences of two species from hedylidae, which is the sister family to hesperiidae [38], were acquired from genbank and treated as outgroups. we estimated the most appropriate partitioning scheme for the data matrix using tiger v1.02 (tree independent generation of evolutionary rates) [39] and the nucleotide substitution models for the partitions by partitionfinder v1.1.1

of trees as burnin. the tree files, after discarding burnin, were combined using logcombiner v2.4.5 and the parameter values were annotated to the maximum clade credibility (mcc) tree using treeannotater v2.4.5 (a part of beast v2.4.5 package; [36]). diversification analysis for subsequent analyses, we retained only one randomly chosen representative species for each genus if multiple species for that genus were available. we estimated gamma (γ) statistic [43] from the mcc tree using the r (v3.2.3; [44]) package laser v2.4.1 [45]. the gamma statistic indicates whether internal nodes are closer to the root (γ < 0) or to the tips (γ

files, after discarding burnin, were combined using logcombiner v2.4.5 and the parameter values were annotated to the maximum clade credibility (mcc) tree using treeannotater v2.4.5 (a part of beast v2.4.5 package; [36]). diversification analysis for subsequent analyses, we retained only one randomly chosen representative species for each genus if multiple species for that genus were available. we estimated gamma (γ) statistic [43] from the mcc tree using the r (v3.2.3; [44]) package laser v2.4.1 [45]. the gamma statistic indicates whether internal nodes are closer to the root (γ < 0) or to the tips (γ > 0) of the tree than

information criteria (aic) in the r package laser v2.4.1 [45]. we calculated the diversification rates of higher level clades following the method-of-moments estimator for stem-group ages [48] and checked using phylogenetic generalized least squares (pgls) [49] in the r package ape v3.5 [50] whether clade age or diversification rate predicts species richness. pgls accounts for phylogenetic non-independence of clades while modeling regression between the parameters. we employed the program bamm v2.5.0 [51] to model the diversification rate shifts. this is a bayesian page 3 of 9 approach that describes the number and locations of rate shifts as posterior distributions. we specified

while modeling regression between the parameters. we employed the program bamm v2.5.0 [51] to model the diversification rate shifts. this is a bayesian page 3 of 9 approach that describes the number and locations of rate shifts as posterior distributions. we specified the sampling probability as the proportion of representative species in our dataset out of total known species in each tribe or subfamily. to check for the prior sensitivity of bamm to the number of rate shifts detected, we performed multiple bamm analyses with varying priors (1, 5 and 10) for expected rate shifts. we ran the analyses for 5

the program bamm v2.5.0 [51] to model the diversification rate shifts. this is a bayesian page 3 of 9 approach that describes the number and locations of rate shifts as posterior distributions. we specified the sampling probability as the proportion of representative species in our dataset out of total known species in each tribe or subfamily. to check for the prior sensitivity of bamm to the number of rate shifts detected, we performed multiple bamm analyses with varying priors (1, 5 and 10) for expected rate shifts. we ran the analyses for 5 million generations each and sampled every 5000 generations.

sampling (fig. 1d), indicating heterogeneity in diversification rate through time. moreover, a model of diversification specifying change in rate best fits the lineage accumulation on the mcc tree (additional file 1: table s1). the pattern of variation in diversification rates was also reflected in the bamm analysis accounting for extant species richness. there was a slow and continuous increase in the net diversification rate up to 50 mya (fig. 1e; black curve), after which the slope of the curve increased steadily indicating an increase in the rate. after ~40 mya, the diversification rate gradually declined. page 4 of 9 this disparity

in the net diversification rate up to 50 mya (fig. 1e; black curve), after which the slope of the curve increased steadily indicating an increase in the rate. after ~40 mya, the diversification rate gradually declined. page 4 of 9 this disparity in diversification rates across the phylogeny explains the species richness of higher-level clades (pgls: r = 0.87; p < 0.0001). the trait-independent analysis in bamm estimated two major shifts in diversification rate over the phylogeny (fig. 1a) (basal rate: 0.10, rate at 1 st shift = 0.48 and 2nd shift = 0.24), both within the subfamily hesperiinae. we note

this correlation (monocot feeding ~ diversification) to fundamentally different approaches of analyses (bamm, and bisse; see fig. 1b, c) lends further support to the higher diversification of monocot-feeding group. although the monocot-feeding lineage represented <20% of the lineages at the k-pg boundary (~65 mya), it includes ca. 50% of extant species. however, model comparisons indicated a possible influence of unknown factors on the estimated diversification pattern suggesting that the association between monocot-feeding and diversification rate must be influenced by certain unmeasured factors. interestingly, we found that the change to monocot feeding did not increase diversification immediately - the rate shift, a

number of tips is low, and when the tip-ratio bias (ratio of numbers of tips with different character states) is high [92]. davis et al. [92] concluded that bisse results should be inferred with extreme caution when the phylogeny included fewer than 300 terminals and/or when fewer than 10% of species are of one character state. although tip-ratio bias is unlikely to be a problem in our dataset, we acknowledge that the number of tips is a concern. however, the main result from bisse, i.e., increased diversification of monocot feeders compared to that of dicot feeders, is also corroborated sahoo et

### NA
main, germany; ∗ correspondence to be sent to: biodiversity and climate research centre (bik-f), senckenberganlage 25, 60325 frankfurt am main, germany; e-mail: jan.schnitzler@senckenberg.de. 2 jodrell abstract. - the cape region of south africa is one of the most remarkable hotspots of biodiversity with a flora comprising more than 9000 plant species, almost 70% of which are endemic, within an area of only ± 90,000 km2 . much of the diversity is due to an exceptionally large contribution of just a few clades that radiated substantially within this region, but little is known about the causes of these radiations. here, we present

of only ± 90,000 km2 . much of the diversity is due to an exceptionally large contribution of just a few clades that radiated substantially within this region, but little is known about the causes of these radiations. here, we present a comprehensive analysis of plant diversification, using near complete species-level phylogenies of four major cape clades (more than 470 species): the genus protea, a tribe of legumes (podalyrieae) and two speciose genera within the iris family (babiana and moraea), representing three of the seven largest plant families in this biodiversity hotspot. combining these molecular phylogenetic data with ecological and biogeographical

is due to an exceptionally large contribution of just a few clades that radiated substantially within this region, but little is known about the causes of these radiations. here, we present a comprehensive analysis of plant diversification, using near complete species-level phylogenies of four major cape clades (more than 470 species): the genus protea, a tribe of legumes (podalyrieae) and two speciose genera within the iris family (babiana and moraea), representing three of the seven largest plant families in this biodiversity hotspot. combining these molecular phylogenetic data with ecological and biogeographical information, we tested key hypotheses that have been proposed to

data with ecological and biogeographical information, we tested key hypotheses that have been proposed to explain the radiation of the cape flora. our results show that the radiations started throughout the oligocene and miocene and that net diversification rates have remained constant through time at globally moderate rates. furthermore, using sister-species comparisons to assess the impact of different factors on speciation, we identified soil type shifts as the most important cause of speciation in babiana, moraea, and protea, whereas shifts in fire-survival strategy is the most important factor for podalyrieae. contrary to previous findings in other groups, such as orchids, pollination

together with relative climatic stability promoted high speciation and/or low extinction rates as the most likely scenario leading to present-day patterns of hyperdiversity in the cape. [biodiversity hotspots; cape floristic region; diversification; flowering plants; phylogenetics; speciation.] the flora of the southwestern tip of africa is characterized by unique levels of species richness and endemism, reflected in the region's inclusion of 2 of the 34 hotspots of biodiversity (mittermeier et al. 2004): the cape floristic region (cfr) and the succulent karoo. containing more than 9000 plant species, almost 70% of which are endemic within an area of only ± 90,000 km2 (goldblatt

speciation.] the flora of the southwestern tip of africa is characterized by unique levels of species richness and endemism, reflected in the region's inclusion of 2 of the 34 hotspots of biodiversity (mittermeier et al. 2004): the cape floristic region (cfr) and the succulent karoo. containing more than 9000 plant species, almost 70% of which are endemic within an area of only ± 90,000 km2 (goldblatt and manning 2000; goldblatt et al. 2005), the cfr represents one of the most diverse temperate floras of the world and is substantially richer than other mediterranean-type climate regions (cowling et al. 1996). extending along

2000; goldblatt et al. 2005), the cfr represents one of the most diverse temperate floras of the world and is substantially richer than other mediterranean-type climate regions (cowling et al. 1996). extending along the south african west coast into southern namibia, the succulent karoo also harbors exceptional numbers of plant species (driver et al. 2003; mittermeier et al. 2004), including 30% of the world's 10,000 succulents. close floristic affinities between both regions led to proposals to unify them as the greater cape floristic region (gcfr; jürgens 1997; born et al. 2007). given its remarkable diversity and circumscribed area, this region provides

dated molecular phylogenies - indicate that this might not be a general feature of the cape flora, but that the radiation of several plant lineages had started well before these climatic changes took place (linder and hardy 2004; linder 2005; verboom et al. 2009). furthermore, global analyses of vascular plant species richness (kreft and jetz 2007) and of the plant family iridaceae (davies et al. 2005) have shown that plant diversity in the cape is significantly higher than expected given its contemporary environmental conditions. this suggests that, despite the strong influence of 343 downloaded from https://academic.oup.com/sysbio/article-abstract/60/3/343/1666085 by guest on 02 march

from https://academic.oup.com/sysbio/article-abstract/60/3/343/1666085 by guest on 02 march 2020 received 24 june 2009; reviews returned 23 november 2009; accepted 13 january 2011 associate editor: mark fishbein 344 vol. 60 systematic biology analyses, however, include phylogenies with a high proportion of missing taxa, which introduces uncertainty regarding the correct assignment of sister species. furthermore, the study did not control for the number of realized states in each trait, which eventually precludes a direct comparison of shift frequencies. the present study on the other hand is not only based on a more thorough taxonomic sampling but also accounts for phylogenetic uncertainty and includes statistical

or the adaptation to different soil types (fine et al. 2005). we are however unaware of any rigorous test of different drivers of diversification for other regions. here, we present results of one of the most comprehensive analyses of plant diversification in the cape, using data for more than 470 species from four major cape clades for which we were able to generate near complete, multigene species-level phylogenetic trees: the genus protea (proteaceae), the tribe podalyrieae (fabaceae), and the genera babiana and moraea (iridaceae), representing three of the seven largest plant families in the cfr (goldblatt et al. 2005). combining phylogenetic,

any rigorous test of different drivers of diversification for other regions. here, we present results of one of the most comprehensive analyses of plant diversification in the cape, using data for more than 470 species from four major cape clades for which we were able to generate near complete, multigene species-level phylogenetic trees: the genus protea (proteaceae), the tribe podalyrieae (fabaceae), and the genera babiana and moraea (iridaceae), representing three of the seven largest plant families in the cfr (goldblatt et al. 2005). combining phylogenetic, ecological, and biogeographical information, we evaluate the competing hypotheses of plant diversification. specifically, we use two

character states are optimized on the phylogenetic trees, and links between diversification rate shifts and shifts in biological and ecological traits are evaluated. shifts in diversification rates are expected to coincide with shifts in traits related to proposed drivers of diversification (e.g., fire-survival strategy). second, differences in traits between sister species are compared, based on the assumption that sister species can be expected to differ in traits that diverge during speciation (kurzweil et al. 1991; linder 2003; barraclough 2006). in this case, avoiding the potential bias of underestimating the number of shifts in labile characters on interior nodes of the phylogenetic

links between diversification rate shifts and shifts in biological and ecological traits are evaluated. shifts in diversification rates are expected to coincide with shifts in traits related to proposed drivers of diversification (e.g., fire-survival strategy). second, differences in traits between sister species are compared, based on the assumption that sister species can be expected to differ in traits that diverge during speciation (kurzweil et al. 1991; linder 2003; barraclough 2006). in this case, avoiding the potential bias of underestimating the number of shifts in labile characters on interior nodes of the phylogenetic tree, we test whether sister species differ more often

assumption that sister species can be expected to differ in traits that diverge during speciation (kurzweil et al. 1991; linder 2003; barraclough 2006). in this case, avoiding the potential bias of underestimating the number of shifts in labile characters on interior nodes of the phylogenetic tree, we test whether sister species differ more often than expected under a null model according to which traits are distributed randomly. m aterial and m ethods taxon sampling the genus babiana (iridaceae: crocoideae) consists of 92 species (lewis 1959; goldblatt and manning 2007b, 2010; goldblatt et al. 2008), which have radiated extensively in southern africa,

the number of shifts in labile characters on interior nodes of the phylogenetic tree, we test whether sister species differ more often than expected under a null model according to which traits are distributed randomly. m aterial and m ethods taxon sampling the genus babiana (iridaceae: crocoideae) consists of 92 species (lewis 1959; goldblatt and manning 2007b, 2010; goldblatt et al. 2008), which have radiated extensively in southern africa, with the vast majority (97%) downloaded from https://academic.oup.com/sysbio/article-abstract/60/3/343/1666085 by guest on 02 march 2020 climatic conditions on plant diversification, it is unlikely that climate alone is responsible for the high levels of

selection pressures and thus promote speciation (linder 2003; barraclough 2006). here, we consider five commonly discussed factors: first, the cape is topographically complex and, therefore, geographical isolation by physical barriers, or divergent selection caused by sharp altitudinal gradients might cause speciation (cowling et al. 2009). this hypothesis predicts that sister species should tend to be isolated geographically. in addition, lineages inhabiting more topographically complex regions should tend to diversify into more species than those in less complex regions given the availability of larger areas and more complex habitats. furthermore, it has been suggested that mountain regions have experienced relatively stable climatic

topographically complex and, therefore, geographical isolation by physical barriers, or divergent selection caused by sharp altitudinal gradients might cause speciation (cowling et al. 2009). this hypothesis predicts that sister species should tend to be isolated geographically. in addition, lineages inhabiting more topographically complex regions should tend to diversify into more species than those in less complex regions given the availability of larger areas and more complex habitats. furthermore, it has been suggested that mountain regions have experienced relatively stable climatic conditions and promoted the long-term persistence of lineages (cowling and lombard 2002; linder 2008). this hypothesis predicts that mountain regions should

harbor a mixture of old and recently derived lineages, whereas lowland regions should contain more derived lineages linder (2008). the second factor, edaphic heterogeneity, might be important by providing a mosaic of divergent selection pressures promoting divergence and speciation (rourke 1972; linder 2003). this hypothesis predicts that recently diverged sister species should tend to occur in different edaphic environments. third and fourth, pollinator specialization and phenological shifts might promote speciation by causing reproductive isolation among populations (johnson 1996; linder 2003). if these are frequent causes of speciation, we predict that recently diverged sister species should tend to have different pollinators or

this hypothesis predicts that recently diverged sister species should tend to occur in different edaphic environments. third and fourth, pollinator specialization and phenological shifts might promote speciation by causing reproductive isolation among populations (johnson 1996; linder 2003). if these are frequent causes of speciation, we predict that recently diverged sister species should tend to have different pollinators or flowering times. finally, in fire-prone environments, such as the cape, two different fire-survival strategies have evolved - species either sprout from underground roots and stems (resprouters) or regenerate only from seeds (reseeders; schutte et al. 1995). these strategies are coupled with different life

promote speciation by causing reproductive isolation among populations (johnson 1996; linder 2003). if these are frequent causes of speciation, we predict that recently diverged sister species should tend to have different pollinators or flowering times. finally, in fire-prone environments, such as the cape, two different fire-survival strategies have evolved - species either sprout from underground roots and stems (resprouters) or regenerate only from seeds (reseeders; schutte et al. 1995). these strategies are coupled with different life histories, which could result in a reduction of gene flow between populations and the avoidance of competition for resources (linder 2003). according to this hypothesis,

sprout from underground roots and stems (resprouters) or regenerate only from seeds (reseeders; schutte et al. 1995). these strategies are coupled with different life histories, which could result in a reduction of gene flow between populations and the avoidance of competition for resources (linder 2003). according to this hypothesis, sister species should frequently have contrasting fire survival strategies. a second hypothesis concerning fire survival is the suggestion that reseeding lineages have diversified more than species that resprout. the mechanism is unclear, but could involve shorter generation times resulting in higher rates of molecular evolution in reseeders compared with resprouters (cowling 1987;

histories, which could result in a reduction of gene flow between populations and the avoidance of competition for resources (linder 2003). according to this hypothesis, sister species should frequently have contrasting fire survival strategies. a second hypothesis concerning fire survival is the suggestion that reseeding lineages have diversified more than species that resprout. the mechanism is unclear, but could involve shorter generation times resulting in higher rates of molecular evolution in reseeders compared with resprouters (cowling 1987; but see verdú et al. 2007). this hypothesis predicts that clades of reseeders should tend to have more species than clades of resprouters. in

lineages have diversified more than species that resprout. the mechanism is unclear, but could involve shorter generation times resulting in higher rates of molecular evolution in reseeders compared with resprouters (cowling 1987; but see verdú et al. 2007). this hypothesis predicts that clades of reseeders should tend to have more species than clades of resprouters. in a recent analysis, van der niet and johnson (2009) found that sister species in the cape frequently differ in general habitat, pollinators, and fire-survival strategy. their 2011 schnitzler et al. - diversification of the cape flora and manning 2000; goldblatt and manning 2002, 2004, 2007a,

resulting in higher rates of molecular evolution in reseeders compared with resprouters (cowling 1987; but see verdú et al. 2007). this hypothesis predicts that clades of reseeders should tend to have more species than clades of resprouters. in a recent analysis, van der niet and johnson (2009) found that sister species in the cape frequently differ in general habitat, pollinators, and fire-survival strategy. their 2011 schnitzler et al. - diversification of the cape flora and manning 2000; goldblatt and manning 2002, 2004, 2007a, 2007b, 2009, 2010; rebelo 2001; additional data on soil types and pollinators were contributed by p. goldblatt, j.s.

al. 1997), 71-f and 1661-r for rpl16 (jordan et al. 1996), \"c\" and \"f\" for trnl-f (taberlet et al. 1991; shaw et al. 2007), rpl32-f and trnl(uag) -r for rpl32-trnl (shaw et al. 2007) and downloaded from https://academic.oup.com/sysbio/article-abstract/60/3/343/1666085 by guest on 02 march 2020 being endemic to the gcfr. all species are small to medium-sized geophytes and display a considerable amount of floral variation. pollination studies have recently led to the description of six largely nonoverlapping pollination systems (goldblatt and manning 2007a). here, we present a near complete species-level phylogeny representing 87 species based on one nuclear and several plastid markers.

by guest on 02 march 2020 being endemic to the gcfr. all species are small to medium-sized geophytes and display a considerable amount of floral variation. pollination studies have recently led to the description of six largely nonoverlapping pollination systems (goldblatt and manning 2007a). here, we present a near complete species-level phylogeny representing 87 species based on one nuclear and several plastid markers. comprising some 200 species of herbaceous geophytes, the genus moraea (iridaceae: iridoideae) is widely distributed in sub-saharan africa, the mediterranean basin, and the middle east (goldblatt et al. 2008) but has the majority of its species (>75%) in

march 2020 being endemic to the gcfr. all species are small to medium-sized geophytes and display a considerable amount of floral variation. pollination studies have recently led to the description of six largely nonoverlapping pollination systems (goldblatt and manning 2007a). here, we present a near complete species-level phylogeny representing 87 species based on one nuclear and several plastid markers. comprising some 200 species of herbaceous geophytes, the genus moraea (iridaceae: iridoideae) is widely distributed in sub-saharan africa, the mediterranean basin, and the middle east (goldblatt et al. 2008) but has the majority of its species (>75%) in the cape of south

medium-sized geophytes and display a considerable amount of floral variation. pollination studies have recently led to the description of six largely nonoverlapping pollination systems (goldblatt and manning 2007a). here, we present a near complete species-level phylogeny representing 87 species based on one nuclear and several plastid markers. comprising some 200 species of herbaceous geophytes, the genus moraea (iridaceae: iridoideae) is widely distributed in sub-saharan africa, the mediterranean basin, and the middle east (goldblatt et al. 2008) but has the majority of its species (>75%) in the cape of south africa. similar to babiana, floral diversity in moraea is remarkable with five

near complete species-level phylogeny representing 87 species based on one nuclear and several plastid markers. comprising some 200 species of herbaceous geophytes, the genus moraea (iridaceae: iridoideae) is widely distributed in sub-saharan africa, the mediterranean basin, and the middle east (goldblatt et al. 2008) but has the majority of its species (>75%) in the cape of south africa. similar to babiana, floral diversity in moraea is remarkable with five distinct pollination systems (goldblatt et al. 2005). the data set from a previous phylogenetic analysis of 73 species by goldblatt et al. (2002) was substantially extended to 162 species of the genus,

and the middle east (goldblatt et al. 2008) but has the majority of its species (>75%) in the cape of south africa. similar to babiana, floral diversity in moraea is remarkable with five distinct pollination systems (goldblatt et al. 2005). the data set from a previous phylogenetic analysis of 73 species by goldblatt et al. (2002) was substantially extended to 162 species of the genus, including 90% of species that occur in the cfr. the tribe podalyrieae consists of eight genera of papilionoid legumes (schutte and van wyk 1998) which - except for six species - are endemic to the cfr,

majority of its species (>75%) in the cape of south africa. similar to babiana, floral diversity in moraea is remarkable with five distinct pollination systems (goldblatt et al. 2005). the data set from a previous phylogenetic analysis of 73 species by goldblatt et al. (2002) was substantially extended to 162 species of the genus, including 90% of species that occur in the cfr. the tribe podalyrieae consists of eight genera of papilionoid legumes (schutte and van wyk 1998) which - except for six species - are endemic to the cfr, with growth forms ranging from shrublets to tall upright trees. species

cape of south africa. similar to babiana, floral diversity in moraea is remarkable with five distinct pollination systems (goldblatt et al. 2005). the data set from a previous phylogenetic analysis of 73 species by goldblatt et al. (2002) was substantially extended to 162 species of the genus, including 90% of species that occur in the cfr. the tribe podalyrieae consists of eight genera of papilionoid legumes (schutte and van wyk 1998) which - except for six species - are endemic to the cfr, with growth forms ranging from shrublets to tall upright trees. species display two different life histories in response

a previous phylogenetic analysis of 73 species by goldblatt et al. (2002) was substantially extended to 162 species of the genus, including 90% of species that occur in the cfr. the tribe podalyrieae consists of eight genera of papilionoid legumes (schutte and van wyk 1998) which - except for six species - are endemic to the cfr, with growth forms ranging from shrublets to tall upright trees. species display two different life histories in response to fire, with slightly less than half of the species regenerating only from seeds. with very few exceptions, species in this tribe are adapted to pollination

species of the genus, including 90% of species that occur in the cfr. the tribe podalyrieae consists of eight genera of papilionoid legumes (schutte and van wyk 1998) which - except for six species - are endemic to the cfr, with growth forms ranging from shrublets to tall upright trees. species display two different life histories in response to fire, with slightly less than half of the species regenerating only from seeds. with very few exceptions, species in this tribe are adapted to pollination by carpenter bees (schutte and van wyk 1998). here, we use data from a phylogenetic study for

of eight genera of papilionoid legumes (schutte and van wyk 1998) which - except for six species - are endemic to the cfr, with growth forms ranging from shrublets to tall upright trees. species display two different life histories in response to fire, with slightly less than half of the species regenerating only from seeds. with very few exceptions, species in this tribe are adapted to pollination by carpenter bees (schutte and van wyk 1998). here, we use data from a phylogenetic study for 107 of a total of about 128 species in the tribe (boatwright et al. 2008). finally, the

wyk 1998) which - except for six species - are endemic to the cfr, with growth forms ranging from shrublets to tall upright trees. species display two different life histories in response to fire, with slightly less than half of the species regenerating only from seeds. with very few exceptions, species in this tribe are adapted to pollination by carpenter bees (schutte and van wyk 1998). here, we use data from a phylogenetic study for 107 of a total of about 128 species in the tribe (boatwright et al. 2008). finally, the genus protea is the largest and most widely distributed

to fire, with slightly less than half of the species regenerating only from seeds. with very few exceptions, species in this tribe are adapted to pollination by carpenter bees (schutte and van wyk 1998). here, we use data from a phylogenetic study for 107 of a total of about 128 species in the tribe (boatwright et al. 2008). finally, the genus protea is the largest and most widely distributed genus of proteaceae in africa, comprising about 115 species (goldblatt and manning 2000), with about 60% of these being endemic to the cfr. all species are woody shrubs or trees and, like

carpenter bees (schutte and van wyk 1998). here, we use data from a phylogenetic study for 107 of a total of about 128 species in the tribe (boatwright et al. 2008). finally, the genus protea is the largest and most widely distributed genus of proteaceae in africa, comprising about 115 species (goldblatt and manning 2000), with about 60% of these being endemic to the cfr. all species are woody shrubs or trees and, like in podalyrieae, two different adaptations to fire regimes (resprouters and reseeders) can be found. again, this genus shows a variety of pollination syndromes, including bird, arthropod, and

107 of a total of about 128 species in the tribe (boatwright et al. 2008). finally, the genus protea is the largest and most widely distributed genus of proteaceae in africa, comprising about 115 species (goldblatt and manning 2000), with about 60% of these being endemic to the cfr. all species are woody shrubs or trees and, like in podalyrieae, two different adaptations to fire regimes (resprouters and reseeders) can be found. again, this genus shows a variety of pollination syndromes, including bird, arthropod, and rodent pollination (collins and rebelo 1987). we use a species-level molecular phylogeny of 90 taxa (reeves

being endemic to the cfr. all species are woody shrubs or trees and, like in podalyrieae, two different adaptations to fire regimes (resprouters and reseeders) can be found. again, this genus shows a variety of pollination syndromes, including bird, arthropod, and rodent pollination (collins and rebelo 1987). we use a species-level molecular phylogeny of 90 taxa (reeves 2001; valente et al. 2010a) including all 70 cape species. all species distributions were recorded as presence/ absence data in grid cells with an edge length of a quarter degree (quarter degree square [qds]). data for babiana were based on collection localities of herbarium

two different adaptations to fire regimes (resprouters and reseeders) can be found. again, this genus shows a variety of pollination syndromes, including bird, arthropod, and rodent pollination (collins and rebelo 1987). we use a species-level molecular phylogeny of 90 taxa (reeves 2001; valente et al. 2010a) including all 70 cape species. all species distributions were recorded as presence/ absence data in grid cells with an edge length of a quarter degree (quarter degree square [qds]). data for babiana were based on collection localities of herbarium accessions from various herbaria (pre, nbg, sam, bol, wind, k). for podalyrieae, data were taken from

adaptations to fire regimes (resprouters and reseeders) can be found. again, this genus shows a variety of pollination syndromes, including bird, arthropod, and rodent pollination (collins and rebelo 1987). we use a species-level molecular phylogeny of 90 taxa (reeves 2001; valente et al. 2010a) including all 70 cape species. all species distributions were recorded as presence/ absence data in grid cells with an edge length of a quarter degree (quarter degree square [qds]). data for babiana were based on collection localities of herbarium accessions from various herbaria (pre, nbg, sam, bol, wind, k). for podalyrieae, data were taken from schutte (1995)

regions. topographic complexity was calculated as the standard deviation of all grid altitude values at 1 × 1 km within a qds grid following thuiller et al. (2006) using a digital elevation model obtained from worldclim (v 1.4; hijmans et al. 2005). to test if topographically complex areas are more species rich, phylogenetically independent contrasts as implemented in macrocaic (agapow and isaac 2002) were calculated using the relative rate difference measure to evaluate the correlation between per qds species richness and topographic complexity. ancestral habitats were reconstructed in an mcmc framework, using the multistate option as implemented in bayestraits (pagel et

using a digital elevation model obtained from worldclim (v 1.4; hijmans et al. 2005). to test if topographically complex areas are more species rich, phylogenetically independent contrasts as implemented in macrocaic (agapow and isaac 2002) were calculated using the relative rate difference measure to evaluate the correlation between per qds species richness and topographic complexity. ancestral habitats were reconstructed in an mcmc framework, using the multistate option as implemented in bayestraits (pagel et al. 2004) with the wwf terrestrial ecoregions (olson et al. 2001) as discrete biogeographic units to identify the ecoregion that most likely constitutes the origin of each clade.

between biogeographic regions) were set equal using a uniform prior, and an estimate of the probability of each state at the root node was derived by combining the particular posterior probabilities. mean topographic complexity was calculated for all terrestrial ecoregions in subsaharan africa occupied by at least one of the species in our study. novel approaches (e.g., lagrange; ree and smith 2008) offer promising improvements, such as explicit models of dispersal and local extinction based on multiple area concepts. the current implementation downloaded from https://academic.oup.com/sysbio/article-abstract/60/3/343/1666085 by guest on 02 march 2020 trnv(uac) x2-f and ndhc-r for trnv-ndhc (shaw et al. 2007).

the cape flora diversification rates net diversification rates of all four groups were calculated following equation 7 of magallón and sanderson (2001) for crown groups under the assumptions of 1) no extinction (ε = 0) and 2) a high relative extinction rate (ε = 0.9) to evaluate the tempo of species diversification. furthermore, we used both temporal and topological approaches to test for shifts in diversification rates using branching times and tree topologies, respectively. first, a maximum likelihood approach (rabosky 2006b) was utilized to test whether diversification rates have changed over time, contrasting the likelihoods of the data under models with

a high probability of a type i error. thus, to obtain an estimate of the null distribution of the test statistic (δaic), we generated 1000 trees for each study group using phylogen (rambaut 2002). this simulation accommodates incomplete taxon sampling by first generating phylogenetic trees consisting of the number of species described for each group following a yule process, and subsequently sampling these trees to reconstruct phylogenies containing the same number of taxa as included in our data sets. the observed δaic value is then compared with the same statistic for the simulated trees, which provide the null distribution of the

generating 1,000,000 trees of the same size as the study groups under the equal rates markov model. to avoid a potential bias caused by incomplete sampling, missing taxa were added to our phylogenies using a perl script (james cotton, unpublished data). for each group, 100 trees were created, adding missing species to the phylogeny at random points along branches within clades identified according to current taxonomy, subsequently calculating the δ1 statistic for each tree. only diversification rate shifts at nodes with a posterior probability of 0.85 or higher were taken into consideration for further analysis. to identify synchronized shifts in diversification

phylogeny at random points along branches within clades identified according to current taxonomy, subsequently calculating the δ1 statistic for each tree. only diversification rate shifts at nodes with a posterior probability of 0.85 or higher were taken into consideration for further analysis. to identify synchronized shifts in diversification rates and species traits (fire survival, edaphic conditions, and pollinators), ancestral character states were reconstructed on a random sample of 400 trees from the post - burn-in distribution of trees from the beast analysis using a maximum likelihood approach (mk1 model) implemented in mesquite (maddison and maddison 2008). modes of speciation to test

analysis (diva; ronquist 1997) reconstructs ancestral ranges using a multiple area concept. however, diva does not incorporate branch length, and, as clark et al. (2008) have shown, generally reconstructs broad ancestral areas, which contradicts the narrow distribution found in many extant lineages in the cape. given the large proportion of species within the gcfr that are endemic to a single ecoregion (species with wider ranges are predominantly found outside of the gcfr), we consider the approach implemented in bayestraits as appropriate to reconstruct ancestral habitats. to account for phylogenetic uncertainty, we used a sample of 400 randomly selected trees from the

concept. however, diva does not incorporate branch length, and, as clark et al. (2008) have shown, generally reconstructs broad ancestral areas, which contradicts the narrow distribution found in many extant lineages in the cape. given the large proportion of species within the gcfr that are endemic to a single ecoregion (species with wider ranges are predominantly found outside of the gcfr), we consider the approach implemented in bayestraits as appropriate to reconstruct ancestral habitats. to account for phylogenetic uncertainty, we used a sample of 400 randomly selected trees from the post - burn-in distribution of the beast analysis. furthermore, we reconstructed

individually for its influence on rates of speciation and extinction. significant differences in speciation and extinction rates in relation to trait changes were assessed using likelihood ratio tests between unconstrained (six parameters) and constrained (five parameters, with either speciation or extinction rates set equal) models. geographical range overlap for all sister-species pairs was calculated as described in the previous section. finally, we calculated the degree of temporal overlap in flowering times between sister species by dividing the number of months of phenological overlap by the flowering time of the species with the shorter flowering period. the average phenological overlap indicates whether

assessed using likelihood ratio tests between unconstrained (six parameters) and constrained (five parameters, with either speciation or extinction rates set equal) models. geographical range overlap for all sister-species pairs was calculated as described in the previous section. finally, we calculated the degree of temporal overlap in flowering times between sister species by dividing the number of months of phenological overlap by the flowering time of the species with the shorter flowering period. the average phenological overlap indicates whether sister species shows a pattern of co-flowering (index near or equal to 1) or flower at different times (close or equal to 0).

or extinction rates set equal) models. geographical range overlap for all sister-species pairs was calculated as described in the previous section. finally, we calculated the degree of temporal overlap in flowering times between sister species by dividing the number of months of phenological overlap by the flowering time of the species with the shorter flowering period. the average phenological overlap indicates whether sister species shows a pattern of co-flowering (index near or equal to 1) or flower at different times (close or equal to 0). to assess the significance of the observed differences between sister species, we performed 1000 random associations

was calculated as described in the previous section. finally, we calculated the degree of temporal overlap in flowering times between sister species by dividing the number of months of phenological overlap by the flowering time of the species with the shorter flowering period. the average phenological overlap indicates whether sister species shows a pattern of co-flowering (index near or equal to 1) or flower at different times (close or equal to 0). to assess the significance of the observed differences between sister species, we performed 1000 random associations of sister-species pairs. the significance of sister-species differences was determined by calculating whether

the flowering time of the species with the shorter flowering period. the average phenological overlap indicates whether sister species shows a pattern of co-flowering (index near or equal to 1) or flower at different times (close or equal to 0). to assess the significance of the observed differences between sister species, we performed 1000 random associations of sister-species pairs. the significance of sister-species differences was determined by calculating whether the observed values fall outside the 95% confidence interval of the randomization tests. to be conservative, if data for one species of a sister-species pair was missing or if the main state

the shorter flowering period. the average phenological overlap indicates whether sister species shows a pattern of co-flowering (index near or equal to 1) or flower at different times (close or equal to 0). to assess the significance of the observed differences between sister species, we performed 1000 random associations of sister-species pairs. the significance of sister-species differences was determined by calculating whether the observed values fall outside the 95% confidence interval of the randomization tests. to be conservative, if data for one species of a sister-species pair was missing or if the main state could not be assigned with confidence, the

average phenological overlap indicates whether sister species shows a pattern of co-flowering (index near or equal to 1) or flower at different times (close or equal to 0). to assess the significance of the observed differences between sister species, we performed 1000 random associations of sister-species pairs. the significance of sister-species differences was determined by calculating whether the observed values fall outside the 95% confidence interval of the randomization tests. to be conservative, if data for one species of a sister-species pair was missing or if the main state could not be assigned with confidence, the pair was scored as not

0). to assess the significance of the observed differences between sister species, we performed 1000 random associations of sister-species pairs. the significance of sister-species differences was determined by calculating whether the observed values fall outside the 95% confidence interval of the randomization tests. to be conservative, if data for one species of a sister-species pair was missing or if the main state could not be assigned with confidence, the pair was scored as not differing in the trait under consideration, thus avoiding an artificial inflation of sister-species differences. in addition, only sister-species pairs with a posterior probability of 0.7 or higher

the significance of the observed differences between sister species, we performed 1000 random associations of sister-species pairs. the significance of sister-species differences was determined by calculating whether the observed values fall outside the 95% confidence interval of the randomization tests. to be conservative, if data for one species of a sister-species pair was missing or if the main state could not be assigned with confidence, the pair was scored as not differing in the trait under consideration, thus avoiding an artificial inflation of sister-species differences. in addition, only sister-species pairs with a posterior probability of 0.7 or higher were included in

95% confidence interval of the randomization tests. to be conservative, if data for one species of a sister-species pair was missing or if the main state could not be assigned with confidence, the pair was scored as not differing in the trait under consideration, thus avoiding an artificial inflation of sister-species differences. in addition, only sister-species pairs with a posterior probability of 0.7 or higher were included in the analysis. r esults phylogenetic analysis and timing of divergence postrun analysis of the mcmc log files from beast indicated parameter convergence and adequate sampling (ess values were all above 200). the topologies

randomization tests. to be conservative, if data for one species of a sister-species pair was missing or if the main state could not be assigned with confidence, the pair was scored as not differing in the trait under consideration, thus avoiding an artificial inflation of sister-species differences. in addition, only sister-species pairs with a posterior probability of 0.7 or higher were included in the analysis. r esults phylogenetic analysis and timing of divergence postrun analysis of the mcmc log files from beast indicated parameter convergence and adequate sampling (ess values were all above 200). the topologies of the chronograms (figs s1

using alternative dating methods (such as nonparametric rate smoothing or penalized likelihood) would not be appropriate in our case. divergence time estimations revealed stem node ages of the four groups are spread from the early oligocene to the mid-miocene, ranging from downloaded from https://academic.oup.com/sysbio/article-abstract/60/3/343/1666085 by guest on 02 march 2020 sister-species comparisons: jordan index differences in traits between sister species were analyzed using the jordan index (jsis ) as proposed by fitzpatrick and turelli (2006). excluding deeper nodes in the phylogeny in the sister-species comparisons avoids the uncertainty associated with the reconstruction of ancestral character states and follows the rationale that

or penalized likelihood) would not be appropriate in our case. divergence time estimations revealed stem node ages of the four groups are spread from the early oligocene to the mid-miocene, ranging from downloaded from https://academic.oup.com/sysbio/article-abstract/60/3/343/1666085 by guest on 02 march 2020 sister-species comparisons: jordan index differences in traits between sister species were analyzed using the jordan index (jsis ) as proposed by fitzpatrick and turelli (2006). excluding deeper nodes in the phylogeny in the sister-species comparisons avoids the uncertainty associated with the reconstruction of ancestral character states and follows the rationale that observed differences between sister species might reflect the differences

the early oligocene to the mid-miocene, ranging from downloaded from https://academic.oup.com/sysbio/article-abstract/60/3/343/1666085 by guest on 02 march 2020 sister-species comparisons: jordan index differences in traits between sister species were analyzed using the jordan index (jsis ) as proposed by fitzpatrick and turelli (2006). excluding deeper nodes in the phylogeny in the sister-species comparisons avoids the uncertainty associated with the reconstruction of ancestral character states and follows the rationale that observed differences between sister species might reflect the differences that led to speciation in the first place (linder 2003). thus, if two sister species differ in a given trait (e.g., having different pollinators),

in traits between sister species were analyzed using the jordan index (jsis ) as proposed by fitzpatrick and turelli (2006). excluding deeper nodes in the phylogeny in the sister-species comparisons avoids the uncertainty associated with the reconstruction of ancestral character states and follows the rationale that observed differences between sister species might reflect the differences that led to speciation in the first place (linder 2003). thus, if two sister species differ in a given trait (e.g., having different pollinators), this factor might be assumed to be key for the divergence of the two lineages. if, on the other hand, sister species

(2006). excluding deeper nodes in the phylogeny in the sister-species comparisons avoids the uncertainty associated with the reconstruction of ancestral character states and follows the rationale that observed differences between sister species might reflect the differences that led to speciation in the first place (linder 2003). thus, if two sister species differ in a given trait (e.g., having different pollinators), this factor might be assumed to be key for the divergence of the two lineages. if, on the other hand, sister species do not differ, the trait in question is unlikely to have played an important role in speciation. the index

species might reflect the differences that led to speciation in the first place (linder 2003). thus, if two sister species differ in a given trait (e.g., having different pollinators), this factor might be assumed to be key for the divergence of the two lineages. if, on the other hand, sister species do not differ, the trait in question is unlikely to have played an important role in speciation. the index was calculated for the following traits: fire-survival strategy (for podalyrieae and protea only, not applicable for babiana and moraea), edaphic conditions, and pollinators with the index taking the value of either

the trait in question is unlikely to have played an important role in speciation. the index was calculated for the following traits: fire-survival strategy (for podalyrieae and protea only, not applicable for babiana and moraea), edaphic conditions, and pollinators with the index taking the value of either 0 (indicating sister species do not differ) or 1 (indicating that sister species differ in a given trait) for each pairwise sister-species comparison. averaged over all sister-species pairs, the index provides a measure of the proportion of species pairs that differ in the trait in question. accordingly, a high frequency of sister-species differences would

an important role in speciation. the index was calculated for the following traits: fire-survival strategy (for podalyrieae and protea only, not applicable for babiana and moraea), edaphic conditions, and pollinators with the index taking the value of either 0 (indicating sister species do not differ) or 1 (indicating that sister species differ in a given trait) for each pairwise sister-species comparison. averaged over all sister-species pairs, the index provides a measure of the proportion of species pairs that differ in the trait in question. accordingly, a high frequency of sister-species differences would be expected for traits that drive speciation. calculations of

for the following traits: fire-survival strategy (for podalyrieae and protea only, not applicable for babiana and moraea), edaphic conditions, and pollinators with the index taking the value of either 0 (indicating sister species do not differ) or 1 (indicating that sister species differ in a given trait) for each pairwise sister-species comparison. averaged over all sister-species pairs, the index provides a measure of the proportion of species pairs that differ in the trait in question. accordingly, a high frequency of sister-species differences would be expected for traits that drive speciation. calculations of the jordan index were based on the predominant states

strategy (for podalyrieae and protea only, not applicable for babiana and moraea), edaphic conditions, and pollinators with the index taking the value of either 0 (indicating sister species do not differ) or 1 (indicating that sister species differ in a given trait) for each pairwise sister-species comparison. averaged over all sister-species pairs, the index provides a measure of the proportion of species pairs that differ in the trait in question. accordingly, a high frequency of sister-species differences would be expected for traits that drive speciation. calculations of the jordan index were based on the predominant states of each trait (e.g., the

moraea), edaphic conditions, and pollinators with the index taking the value of either 0 (indicating sister species do not differ) or 1 (indicating that sister species differ in a given trait) for each pairwise sister-species comparison. averaged over all sister-species pairs, the index provides a measure of the proportion of species pairs that differ in the trait in question. accordingly, a high frequency of sister-species differences would be expected for traits that drive speciation. calculations of the jordan index were based on the predominant states of each trait (e.g., the soil type that species were most commonly found on). however, to

(indicating sister species do not differ) or 1 (indicating that sister species differ in a given trait) for each pairwise sister-species comparison. averaged over all sister-species pairs, the index provides a measure of the proportion of species pairs that differ in the trait in question. accordingly, a high frequency of sister-species differences would be expected for traits that drive speciation. calculations of the jordan index were based on the predominant states of each trait (e.g., the soil type that species were most commonly found on). however, to incorporate the variability in species traits (in particular with regard to the edaphic conditions),

provides a measure of the proportion of species pairs that differ in the trait in question. accordingly, a high frequency of sister-species differences would be expected for traits that drive speciation. calculations of the jordan index were based on the predominant states of each trait (e.g., the soil type that species were most commonly found on). however, to incorporate the variability in species traits (in particular with regard to the edaphic conditions), we recalculated the jordan index, this time coding polymorphic species such as to minimize the number of sister-species differences. this provides a more conservative estimate of the variability (i.e.,

the trait in question. accordingly, a high frequency of sister-species differences would be expected for traits that drive speciation. calculations of the jordan index were based on the predominant states of each trait (e.g., the soil type that species were most commonly found on). however, to incorporate the variability in species traits (in particular with regard to the edaphic conditions), we recalculated the jordan index, this time coding polymorphic species such as to minimize the number of sister-species differences. this provides a more conservative estimate of the variability (i.e., the minimum number of shifts) for each trait and allowed us to

calculations of the jordan index were based on the predominant states of each trait (e.g., the soil type that species were most commonly found on). however, to incorporate the variability in species traits (in particular with regard to the edaphic conditions), we recalculated the jordan index, this time coding polymorphic species such as to minimize the number of sister-species differences. this provides a more conservative estimate of the variability (i.e., the minimum number of shifts) for each trait and allowed us to assess the robustness of the observed differences. phylogenetic clustering on the other hand would be expected not only for

the predominant states of each trait (e.g., the soil type that species were most commonly found on). however, to incorporate the variability in species traits (in particular with regard to the edaphic conditions), we recalculated the jordan index, this time coding polymorphic species such as to minimize the number of sister-species differences. this provides a more conservative estimate of the variability (i.e., the minimum number of shifts) for each trait and allowed us to assess the robustness of the observed differences. phylogenetic clustering on the other hand would be expected not only for traits with no impact on divergence but also

significantly decreased diversification rates toward the present with shifts occurring between 1.67 ma (protea) and 0.382 ma (moraea; table 1; the full output of the maximum likelihood analysis is provided in table s2). such a decrease could however also be an artefact caused by incomplete taxon sampling (missing or cryptic species; pybus and harvey 2000). thus, to assess whether these decreases in diversification rates are likely to be attributable to incomplete sampling due figure 1. lineage-through-time plots for babiana, moraea, podalyrieae, and protea based on the mean node ages of the maximum clade credibility trees using an ucln relaxed clock in

lineage-through-time plots for babiana, moraea, podalyrieae, and protea based on the mean node ages of the maximum clade credibility trees using an ucln relaxed clock in beast. downloaded from https://academic.oup.com/sysbio/article-abstract/60/3/343/1666085 by guest on 02 march 2020 reconstruction of ancestral habitats analyses of phylogenetically independent contrasts showed no significant correlation between species richness and topographic complexity (babiana: t = 0.786, df = 57, p = 0.446; moraea: t = −0.153, df = 111, p = 0.879; podalyrieae: t = −0.93, df = 70, p = 0.355; protea: t = −0.225, df = 57, p = 0.823). reconstruction of ancestral habitats following an

88) of all nodes show no range overlap, whereas 16% (14 nodes) are sympatric, including some of the most recent speciation events (fig. 3). these include the split between babiana vanzijliae and b. papyracea, the latter being a narrow endemic known only from two populations on the bokkeveld plateau. further sister-species pairs with sympatric ranges include b. teretifolia and b. hirsuta, both occurring along the west coast of south africa, as well as b. karooica and b. radiata, which are known only from a narrow region in the little karoo. finally, the range of b. regia is embedded within that of

sympatric ranges include b. teretifolia and b. hirsuta, both occurring along the west coast of south africa, as well as b. karooica and b. radiata, which are known only from a narrow region in the little karoo. finally, the range of b. regia is embedded within that of its sister species b. odorata, which is common in swartland region of southwestern south africa. the slope of the regression line is slightly negative (−0.004), indicating almost no range movements. the low observed intercept together with the positive slope of the regression (0.047) in moraea suggests predominantly allopatric speciation with range movements occurring

than expected at random, together with the positive slope of the regression (0.039), suggests once more that speciation is predominantly allopatric. however, although 16% (14 of 84) of the nodes have no range overlap, 19% (16 nodes) are fully sympatric, including several recent splits (fig. 3). in most cases, these sisters-species pairs consist of one range-restricted species (protea pudens, p. stokoei, and p. scabriusucula) whose range is encompassed entirely by their sister species (p. longifolia, p. speciosa, and p. scolopendriifolia respectively). in another case, both p. susannae and p. obtusifolia are common on the coastal flats of the southwestern cape, on

the positive slope of the regression (0.039), suggests once more that speciation is predominantly allopatric. however, although 16% (14 of 84) of the nodes have no range overlap, 19% (16 nodes) are fully sympatric, including several recent splits (fig. 3). in most cases, these sisters-species pairs consist of one range-restricted species (protea pudens, p. stokoei, and p. scabriusucula) whose range is encompassed entirely by their sister species (p. longifolia, p. speciosa, and p. scolopendriifolia respectively). in another case, both p. susannae and p. obtusifolia are common on the coastal flats of the southwestern cape, on neutral sand and limestone, respectively. sister-species

although 16% (14 of 84) of the nodes have no range overlap, 19% (16 nodes) are fully sympatric, including several recent splits (fig. 3). in most cases, these sisters-species pairs consist of one range-restricted species (protea pudens, p. stokoei, and p. scabriusucula) whose range is encompassed entirely by their sister species (p. longifolia, p. speciosa, and p. scolopendriifolia respectively). in another case, both p. susannae and p. obtusifolia are common on the coastal flats of the southwestern cape, on neutral sand and limestone, respectively. sister-species analyses geographical range overlap between sister species was found to be relatively low (babiana: 0.275, moraea:

species (protea pudens, p. stokoei, and p. scabriusucula) whose range is encompassed entirely by their sister species (p. longifolia, p. speciosa, and p. scolopendriifolia respectively). in another case, both p. susannae and p. obtusifolia are common on the coastal flats of the southwestern cape, on neutral sand and limestone, respectively. sister-species analyses geographical range overlap between sister species was found to be relatively low (babiana: 0.275, moraea: 0.173, podalyrieae: 0.205, protea: 0.409), consistent with the predominantly allopatric mode of speciation found in the arc analysis. the proportion of sister species differing in traits was found to vary substantially between groups as

scabriusucula) whose range is encompassed entirely by their sister species (p. longifolia, p. speciosa, and p. scolopendriifolia respectively). in another case, both p. susannae and p. obtusifolia are common on the coastal flats of the southwestern cape, on neutral sand and limestone, respectively. sister-species analyses geographical range overlap between sister species was found to be relatively low (babiana: 0.275, moraea: 0.173, podalyrieae: 0.205, protea: 0.409), consistent with the predominantly allopatric mode of speciation found in the arc analysis. the proportion of sister species differing in traits was found to vary substantially between groups as well as for different traits. all ecological

flats of the southwestern cape, on neutral sand and limestone, respectively. sister-species analyses geographical range overlap between sister species was found to be relatively low (babiana: 0.275, moraea: 0.173, podalyrieae: 0.205, protea: 0.409), consistent with the predominantly allopatric mode of speciation found in the arc analysis. the proportion of sister species differing in traits was found to vary substantially between groups as well as for different traits. all ecological traits were found to be phylogenetically clustered in a least some of the groups studied (table 3 and fig. 2). within podalyrieae, about half of the sister-species pairs show differences in their

analysis. the proportion of sister species differing in traits was found to vary substantially between groups as well as for different traits. all ecological traits were found to be phylogenetically clustered in a least some of the groups studied (table 3 and fig. 2). within podalyrieae, about half of the sister-species pairs show differences in their fire-survival strategy (jsis 0.55; table 3); whereas in protea different fire-survival strategies were found in less than 20% of sister-species pairs (table 3). differences in edaphic conditions were very high, especially in babiana, moraea, and podalyrieae, where between 71% and 92% of sister-species pairs occur

were found to be phylogenetically clustered in a least some of the groups studied (table 3 and fig. 2). within podalyrieae, about half of the sister-species pairs show differences in their fire-survival strategy (jsis 0.55; table 3); whereas in protea different fire-survival strategies were found in less than 20% of sister-species pairs (table 3). differences in edaphic conditions were very high, especially in babiana, moraea, and podalyrieae, where between 71% and 92% of sister-species pairs occur in different edaphic environments (table 3 and fig. 2). differences in protea on the other hand are much lower (jsis 0.334), showing significantly less variation

of the sister-species pairs show differences in their fire-survival strategy (jsis 0.55; table 3); whereas in protea different fire-survival strategies were found in less than 20% of sister-species pairs (table 3). differences in edaphic conditions were very high, especially in babiana, moraea, and podalyrieae, where between 71% and 92% of sister-species pairs occur in different edaphic environments (table 3 and fig. 2). differences in protea on the other hand are much lower (jsis 0.334), showing significantly less variation between sister species than expected at random. to further scrutinize the effect of edaphic conditions, we also calculated the jordan index separately for

3). differences in edaphic conditions were very high, especially in babiana, moraea, and podalyrieae, where between 71% and 92% of sister-species pairs occur in different edaphic environments (table 3 and fig. 2). differences in protea on the other hand are much lower (jsis 0.334), showing significantly less variation between sister species than expected at random. to further scrutinize the effect of edaphic conditions, we also calculated the jordan index separately for lithology and soil types (table s3), which despite yielding a pattern similar to the combined factor, did not show a significant clustering. with the exception of babiana (jsis 0.471; table

further scrutinize the effect of edaphic conditions, we also calculated the jordan index separately for lithology and soil types (table s3), which despite yielding a pattern similar to the combined factor, did not show a significant clustering. with the exception of babiana (jsis 0.471; table 3), pollinator shifts between sister species occur significantly less frequently than based on our randomizations, revealing again a pattern of phylogenetic clustering, with a low frequency of shifts between sister species (moraea 0.25, p < 0.001; podalyrieae 0, p < 0.001; protea 0.231, p < 0.001; table 3). the bisse ln likelihood test yielded no significant

pattern similar to the combined factor, did not show a significant clustering. with the exception of babiana (jsis 0.471; table 3), pollinator shifts between sister species occur significantly less frequently than based on our randomizations, revealing again a pattern of phylogenetic clustering, with a low frequency of shifts between sister species (moraea 0.25, p < 0.001; podalyrieae 0, p < 0.001; protea 0.231, p < 0.001; table 3). the bisse ln likelihood test yielded no significant differences in speciation or extinction rates for any of the traits analyzed (table s4). as direct comparisons of the proportion of sisterspecies differences (jsis )

links between diversification rate shifts and shifts in the habitat or traits included in our analysis (fig. 2, data for fire survival not shown). vol. 60 2011 schnitzler et al. - diversification of the cape flora 351 downloaded from https://academic.oup.com/sysbio/article-abstract/60/3/343/1666085 by guest on 02 march 2020 figure 2. variability of species traits. maximum clade credibility trees of the beast analysis for a) babiana, b) moraea, c) podalyrieae, and d) protea. branches are colored according to maximum likelihood reconstructions of soil types (blue - rocky outcrops; red - gravel, yellow - sand; greenloam; cyan - clay; black - marshy soil). unknown states

traits is restricted by the variable number of states, the shift frequency (jsis ) was conditioned by the number of realized states, thus obtaining a measure of the relative variability of each trait. the results show that for babiana, moraea, and protea, soil types exhibit the highest variability between sister species (table 4), whereas changes in fire-survival strategy show the highest degree of variability in podalyrieae (table 4). calculating the jordan index under a conservative scenario of sisterspecies differences (minimum number of shifts for polymorphic characters) resulted in slightly lower estimates of sister-species differences, but had little impact on the overall

soil types exhibit the highest variability between sister species (table 4), whereas changes in fire-survival strategy show the highest degree of variability in podalyrieae (table 4). calculating the jordan index under a conservative scenario of sisterspecies differences (minimum number of shifts for polymorphic characters) resulted in slightly lower estimates of sister-species differences, but had little impact on the overall patterns. most importantly, soil types still displayed the highest variability in babiana (0.111), moraea (0.12), and protea (0.097). thus, even assuming the most conservative pattern of trait shifts between sister species, soil types remain the most variable factor. the analysis of flowering

of shifts for polymorphic characters) resulted in slightly lower estimates of sister-species differences, but had little impact on the overall patterns. most importantly, soil types still displayed the highest variability in babiana (0.111), moraea (0.12), and protea (0.097). thus, even assuming the most conservative pattern of trait shifts between sister species, soil types remain the most variable factor. the analysis of flowering times showed that sister species exhibit a high degree of phenological overlap (babiana 0.569, p = 0.42; moraea 0.762, p < 0.001; poda- lyrieae 0.628, p = 0.08; protea 0.744, p < 0.001), which in the case of moraea

impact on the overall patterns. most importantly, soil types still displayed the highest variability in babiana (0.111), moraea (0.12), and protea (0.097). thus, even assuming the most conservative pattern of trait shifts between sister species, soil types remain the most variable factor. the analysis of flowering times showed that sister species exhibit a high degree of phenological overlap (babiana 0.569, p = 0.42; moraea 0.762, p < 0.001; poda- lyrieae 0.628, p = 0.08; protea 0.744, p < 0.001), which in the case of moraea and protea is significantly higher than expected based on the randomization of flowering times. d iscussion

are provided in table 2. downloaded from https://academic.oup.com/sysbio/article-abstract/60/3/343/1666085 by guest on 02 march 2020 notes: values for range overlap were arcsine transformed prior to the regression analysis. ranges were randomly shuffled among tips (n = 1000) to test whether intercepts were significantly different from those obtained under random distribution of species. ns = not significant. 2011 353 schnitzler et al. - diversification of the cape flora table 3. jordan index (jsis ) indicating the proportion of observed pairwise species differences fire survival observed babiana moraea podalyrieae protea observed 95% confidence interval - - 0.28 - 0.73 0.23 - 0.77 0.824 0.923

were randomly shuffled among tips (n = 1000) to test whether intercepts were significantly different from those obtained under random distribution of species. ns = not significant. 2011 353 schnitzler et al. - diversification of the cape flora table 3. jordan index (jsis ) indicating the proportion of observed pairwise species differences fire survival observed babiana moraea podalyrieae protea observed 95% confidence interval - - 0.28 - 0.73 0.23 - 0.77 0.824 0.923 0.714 0.334∗ 0.74 - 1 0.8 - 1 0.67 - 1 0.36 - 0.8 - - 0.56 0.16∗∗ pollinators edaphic conditions 95% confidence interval observed 0.471 0.25∗∗∗ 0∗∗∗

0.67 - 1 0.36 - 0.8 - - 0.56 0.16∗∗ pollinators edaphic conditions 95% confidence interval observed 0.471 0.25∗∗∗ 0∗∗∗ 0.231∗∗∗ 95% confidence interval 0.41 - 0.82 0.46 - 0.79 0 - 0.158 0.46 - 0.93 notes: significance was assessed by creating 1000 random associations of the respective number of sister-species pairs: babiana = 17, moraea = 29, podalyrieae = 19, protea = 15 (*p = 0.05, **p < 0.01, ***p < 0.001). differential fire-survival strategies are not present in babiana and moraea. table 4. relative variability of species traits babiana moraea podalyrieae protea fire survival edaphic conditions pollinators lithology soil

was assessed by creating 1000 random associations of the respective number of sister-species pairs: babiana = 17, moraea = 29, podalyrieae = 19, protea = 15 (*p = 0.05, **p < 0.01, ***p < 0.001). differential fire-survival strategies are not present in babiana and moraea. table 4. relative variability of species traits babiana moraea podalyrieae protea fire survival edaphic conditions pollinators lithology soil type - - 0.28 0.08 0.037 0.038 0.054 0.033 0.078 0.05 0 0.077 0.078 0.092 0.1 0.04 0.118 0.152 0.106 0.1 notes: proportion of observed pairwise species differences conditioned by the number of states in each category. traits

not present in babiana and moraea. table 4. relative variability of species traits babiana moraea podalyrieae protea fire survival edaphic conditions pollinators lithology soil type - - 0.28 0.08 0.037 0.038 0.054 0.033 0.078 0.05 0 0.077 0.078 0.092 0.1 0.04 0.118 0.152 0.106 0.1 notes: proportion of observed pairwise species differences conditioned by the number of states in each category. traits with the highest variability for each lineage are in bold. it is important to note though that although the maximum likelihood method to detect temporal shifts in diversification rates assumes constant rates across clades (rabosky 2006b), and therefore is

act throughout the geographic range of a lineage), a node-by-node examination of tree imbalances can identify diversification rate shifts that occur in only one or a few clades (\"concealed\" shifts). the whole-tree test for diversification rate shifts identified eight potential shifts, none of which were directly linked to shifts in species traits, suggesting that trait shifts and diversification rate shifts seem to be largely decoupled, indicating that other factors not included in this study might be driving these rate shifts. the test, however, cannot distinguish between increases and decreases in diversification rates, as decreases (which can be caused by either a

loss of the corresponding phylogenetic information throughout the evolutionary history of the lineage (moore et al. 2004), and thus, rate shifts are interpreted as increases along the branch leading to the more diverse clade. nevertheless, we remain cautious about the interpretation of the diversification rate shifts, especially in cases where species-poor lineages are nested within more diverse clades. spatial patterns of speciation our analysis of the geographical mode of speciation showed that all groups apart from podalyrieae have intercepts significantly different from those obtained under the range randomization test. thus, present-day distributions might still contain information on the mode of speciation.

the mode of speciation. using the correlation between node ages and degree of sympatry, we infer that the geographic mode of speciation is predominantly allopatric, although extensive range movements have occurred as apparent by the wide scatter of overlap values. however, in a few cases, we found recently diverged sister species occurring in sympatry, a pattern that might reflect sympatric speciation occurring at least at low frequencies. alternatively, the high degree of range overlap observed might be due to the spatial resolution of the distribution data used in this study. the qds grid cells have an area of approximately 625 km2

diversification ancestral range reconstructions identified the montane fynbos and renosterveld ecoregion as the most likely ancestral habitat for the three of the clades studied here, with only babiana having its ancestral range in the succulent karoo. these findings suggest that high topographical complexity of the cape mountains could be promoting species diversity by 1) providing opportunities for habitat differentiation, 2) restricting gene flow between geographically isolated populations, or 3) increasing the persistence of species. a test for habitat differentiation reveals only a weak negative correlation between geographical and altitudinal range overlap (b = −0.15, r = −0.09; n = 17, sister

studied here, with only babiana having its ancestral range in the succulent karoo. these findings suggest that high topographical complexity of the cape mountains could be promoting species diversity by 1) providing opportunities for habitat differentiation, 2) restricting gene flow between geographically isolated populations, or 3) increasing the persistence of species. a test for habitat differentiation reveals only a weak negative correlation between geographical and altitudinal range overlap (b = −0.15, r = −0.09; n = 17, sister species with geographical range overlap >0.5). the predominantly allopatric mode of speciation (table 2) on the other hand, together with evidence that dispersal

diversity by 1) providing opportunities for habitat differentiation, 2) restricting gene flow between geographically isolated populations, or 3) increasing the persistence of species. a test for habitat differentiation reveals only a weak negative correlation between geographical and altitudinal range overlap (b = −0.15, r = −0.09; n = 17, sister species with geographical range overlap >0.5). the predominantly allopatric mode of speciation (table 2) on the other hand, together with evidence that dispersal distances in the cape are generally short (goldblatt and manning 2000; latimer et al. 2005), suggests that the second scenario provides a more plausible explanation. finally, our results

range overlap >0.5). the predominantly allopatric mode of speciation (table 2) on the other hand, together with evidence that dispersal distances in the cape are generally short (goldblatt and manning 2000; latimer et al. 2005), suggests that the second scenario provides a more plausible explanation. finally, our results show that species diversity is not significantly different between areas of high and low topographic complexity, suggesting that lineages have probably also diversified extensively in habitats with reduced topographic complexity. as the potential for clades to diversify also depends on the area of the geographical region (losos and schluter 2000; davies et al.

fynbos and renosterveld ecoregions (approximately 1.3 times larger). together with the finding that ancestral habitats are predominantly topographically complex areas, this provides evidence for a higher persistence of lineages these regions, suggesting that, in part, low levels of extinction in mountainous regions have contributed to present-day patterns of diversity. although sister-species analyses avoid uncertainties associated with the reconstruction of character changes on the phylogeny, this approach relies on a number of assumptions. first, the identification of traits driving speciation is based on the concept of competition for limiting factors; and second, species ranges and ecologies are assumed to reflect conditions immediately

regions have contributed to present-day patterns of diversity. although sister-species analyses avoid uncertainties associated with the reconstruction of character changes on the phylogeny, this approach relies on a number of assumptions. first, the identification of traits driving speciation is based on the concept of competition for limiting factors; and second, species ranges and ecologies are assumed to reflect conditions immediately following speciation, rather than resulting from changes occurring since speciation. despite a high probability that some ecological changes and range movements have occurred after sister species have become separated, the large sample size and wide taxonomic range allows us to identify

traits driving speciation is based on the concept of competition for limiting factors; and second, species ranges and ecologies are assumed to reflect conditions immediately following speciation, rather than resulting from changes occurring since speciation. despite a high probability that some ecological changes and range movements have occurred after sister species have become separated, the large sample size and wide taxonomic range allows us to identify patterns of plausible drivers of speciation even in the presence of these potentially confounding effects. the factor with the highest variability between sister species in three of four groups (babiana, moraea, and protea) is soil

that some ecological changes and range movements have occurred after sister species have become separated, the large sample size and wide taxonomic range allows us to identify patterns of plausible drivers of speciation even in the presence of these potentially confounding effects. the factor with the highest variability between sister species in three of four groups (babiana, moraea, and protea) is soil type, consistent with the idea that adaptation to different soil types could be a major driver of plant diversification in southern africa. this is supported by the high diversity of soil types found in the southwestern cape (fig. s6),

could be a major driver of plant diversification in southern africa. this is supported by the high diversity of soil types found in the southwestern cape (fig. s6), which has gradually increased during the late cenozoic (cowling et al. 2009). furthermore, these results are consistent with studies of neotropical plant species (fine et al. 2004, 2005), which highlighted the potential importance of edaphic heterogeneity in plant speciation. in podalyrieae, firesurvival strategy is more variable than other traits and thus should be considered a potential driver of diversification in this group. this is highlighted by the fact that fire-survival strategy is often

plant speciation. in podalyrieae, firesurvival strategy is more variable than other traits and thus should be considered a potential driver of diversification in this group. this is highlighted by the fact that fire-survival strategy is often the most important distinguishing character between closely related - and morphologically almost identical - species (schutte et al. 1995). these findings are robust with regard to sampling effort, which could potentially affect the correct identification of sister species. considering only clades where taxon sampling is above 90% (the genera liparia, podalyria, stirtonanthus, virgilia, and xiphotheca), we find the same level of variation and thus conclude

this group. this is highlighted by the fact that fire-survival strategy is often the most important distinguishing character between closely related - and morphologically almost identical - species (schutte et al. 1995). these findings are robust with regard to sampling effort, which could potentially affect the correct identification of sister species. considering only clades where taxon sampling is above 90% (the genera liparia, podalyria, stirtonanthus, virgilia, and xiphotheca), we find the same level of variation and thus conclude that the observed differences are unlikely to be a sampling artefact. shifts in fire-survival strategy have previously received little attention and the frequent

where taxon sampling is above 90% (the genera liparia, podalyria, stirtonanthus, virgilia, and xiphotheca), we find the same level of variation and thus conclude that the observed differences are unlikely to be a sampling artefact. shifts in fire-survival strategy have previously received little attention and the frequent shifts between sister species are somewhat surprising, especially as species with a mixed fire biology are rare (linder 2003), but the adaptations involved in a switch between different firesurvival strategies may not be as complex as previously thought (verdaguer and ojeda 2005). furthermore, maximum likelihood reconstructions of fire-survival strategies in podalyrieae yielded largely equivocal

(the genera liparia, podalyria, stirtonanthus, virgilia, and xiphotheca), we find the same level of variation and thus conclude that the observed differences are unlikely to be a sampling artefact. shifts in fire-survival strategy have previously received little attention and the frequent shifts between sister species are somewhat surprising, especially as species with a mixed fire biology are rare (linder 2003), but the adaptations involved in a switch between different firesurvival strategies may not be as complex as previously thought (verdaguer and ojeda 2005). furthermore, maximum likelihood reconstructions of fire-survival strategies in podalyrieae yielded largely equivocal states for deeper nodes in the

a period when summer droughts in the cape were already in place and just before the suggested establishment of regular fires within the region (bytebier et al. 2011), providing further support for the importance of fire as a driver of diversification in podalyrieae. finally, the high phenological overlap between sister species indicates that shifts in flowering times do not act as a driving force in the creation and/or maintenance of reproductive barriers in these clades, although evidence in other clades suggest a potential role of phenology in the reduction of gene flow (linder 2001). downloaded from https://academic.oup.com/sysbio/article-abstract/60/3/343/1666085 by guest on 02

in these clades, although evidence in other clades suggest a potential role of phenology in the reduction of gene flow (linder 2001). downloaded from https://academic.oup.com/sysbio/article-abstract/60/3/343/1666085 by guest on 02 march 2020 high-resolution distribution data for protea (1-km grid) allowed us to evaluate the extent of range overlap of putative sympatric species at different spatial scales, which, as expected, is substantially lower than at the qds scale (mean difference in range overlap: 59%), with no sister-species pair still being fully sympatric (table s5). thus, the relatively coarse scale of this study clearly inflates the degree of range overlap, and although fine-scale distribution

https://academic.oup.com/sysbio/article-abstract/60/3/343/1666085 by guest on 02 march 2020 high-resolution distribution data for protea (1-km grid) allowed us to evaluate the extent of range overlap of putative sympatric species at different spatial scales, which, as expected, is substantially lower than at the qds scale (mean difference in range overlap: 59%), with no sister-species pair still being fully sympatric (table s5). thus, the relatively coarse scale of this study clearly inflates the degree of range overlap, and although fine-scale distribution data are currently not available for the other groups, we can assume that at least some but possibly all of the cases identified as

of nature and environmental conservation are thanked for the provision of collecting permits. parts of the analyses were run on the clusters at the swiss institute of bioinformatics (vital-it) and the computational biology service unit (cbsu) at cornell university. r eferences agapow p.-m., isaac n.j.b. 2002. macrocaic: revealing correlates of species richness by comparative analysis. divers. distrib. 8:41 - 43. armbruster w.s., muchhala n. 2009. associations between floral specialisation and species diversity: cause, effect, or correlation? evol. ecol. 23:159 - 179. baldwin b.g., sanderson m.j. 1998. age and rate of diversification of the hawaiian silversword alliance (compositae). proc. natl. acad. sci.

the clusters at the swiss institute of bioinformatics (vital-it) and the computational biology service unit (cbsu) at cornell university. r eferences agapow p.-m., isaac n.j.b. 2002. macrocaic: revealing correlates of species richness by comparative analysis. divers. distrib. 8:41 - 43. armbruster w.s., muchhala n. 2009. associations between floral specialisation and species diversity: cause, effect, or correlation? evol. ecol. 23:159 - 179. baldwin b.g., sanderson m.j. 1998. age and rate of diversification of the hawaiian silversword alliance (compositae). proc. natl. acad. sci. u.s.a. 95:9402 - 9406. barraclough t.g. 2006. what can phylogenetics tell us about speciation in the cape flora? divers. distrib.

age and rate of diversification of the hawaiian silversword alliance (compositae). proc. natl. acad. sci. u.s.a. 95:9402 - 9406. barraclough t.g. 2006. what can phylogenetics tell us about speciation in the cape flora? divers. distrib. 12:21 - 26. barraclough t.g., vogler a.p. 2000. detecting the geographical pattern of speciation from species-level phylogenies. am. nat. 155:419 - 434. beaumont a.j., beckett r.p., edwards t.j., stirton c.h. 1999. revision of the genus calpurnia (sophoreae: leguminosae). bothalia. 29: 5 - 23. boatwright j.s., savolainen v., van wyk b.-e., schutte-vlok a.l., forest f., van der bank m. 2008. systematic position of the anomalous genus cadia

j.r., ree r.h., alfaro m.e., king m.g., wagner w.l., roalson e.h. 2008. a comparative study in ancestral range reconstruction methods: retracing the uncertain histories of insular lineages. syst. biol. 57:693 - 707. downloaded from https://academic.oup.com/sysbio/article-abstract/60/3/343/1666085 by guest on 02 march 2020 one constraint of the jordan index used here for sister-species comparisons is the limited power to distinguish traits with a causal role in speciation from those that show a high degree of variability (as in the null model). however, we feel that in this case using random associations as a null hypothesis is more appropriate than the one that assumes

power to distinguish traits with a causal role in speciation from those that show a high degree of variability (as in the null model). however, we feel that in this case using random associations as a null hypothesis is more appropriate than the one that assumes no difference between sister species (due to phylogenetic conservatism), in particular given that losos (2008) and wiens (2008) pointed out that these patterns should be tested rather than assumed a priori. thus, the test statistic might have limitations to detect a positive effect of a trait on diversification but is powerful (albeit conservative) to reject

rather than assumed a priori. thus, the test statistic might have limitations to detect a positive effect of a trait on diversification but is powerful (albeit conservative) to reject hypotheses of potential drivers of plant diversification. this is the case when traits 1) are found to rarely differ between sister species, and 2) lack the signature of a key innovation. for example, apart from the genus babiana, pollination systems show a high degree of phylogenetic conservatism, and the absence of a significant effect on speciation and/or extinction rates indicates that shifts in pollination systems are unlikely to be a main driver

of realized states in each trait, no definitive conclusions can be drawn regarding the causes of speciation. potential explanations for the high frequency of pollinator shifts observed in babiana could be that pollinator shifts are the result of a direct selection for reproductive isolation in secondary contact zones between sister species adapted to different soil types (goldblatt and manning 1996; van der niet et al. 2006), or that the diversity of pollination systems might in fact be the result, not the cause, of high species richness (armbruster and muchhala 2009). in conclusion, our study shows that the remarkable plant diversity in

are the result of a direct selection for reproductive isolation in secondary contact zones between sister species adapted to different soil types (goldblatt and manning 1996; van der niet et al. 2006), or that the diversity of pollination systems might in fact be the result, not the cause, of high species richness (armbruster and muchhala 2009). in conclusion, our study shows that the remarkable plant diversity in the cape of southern africa is not the result of a recent and rapid radiation triggered by climatic changes, but that diversification took place over an extended period of time. furthermore, the relative climatic

will reveal whether the patterns described here provide a global scenario for the evolution of hyperdiverse floras. 355 356 systematic biology goldblatt p., manning j.c. 2007b. floral biology of babiana (iridaceae: crocoideae): adaptive floral radiation and pollination. ann. mo. bot. gard. 94:709 - 733. goldblatt p., manning j.c. 2009. new species of moraea (iridaceae: iridoideae), with range extensions and miscellaneous notes for southern african species. bothalia. 39:1 - 10. goldblatt p., manning j.c. 2010. new taxa of babiana (iridaceae: crocoideae) from coastal western cape, south africa. bothalia. 40: 47 - 53. goldblatt p., manning j.c., gereau r. 2008. two new species

of hyperdiverse floras. 355 356 systematic biology goldblatt p., manning j.c. 2007b. floral biology of babiana (iridaceae: crocoideae): adaptive floral radiation and pollination. ann. mo. bot. gard. 94:709 - 733. goldblatt p., manning j.c. 2009. new species of moraea (iridaceae: iridoideae), with range extensions and miscellaneous notes for southern african species. bothalia. 39:1 - 10. goldblatt p., manning j.c. 2010. new taxa of babiana (iridaceae: crocoideae) from coastal western cape, south africa. bothalia. 40: 47 - 53. goldblatt p., manning j.c., gereau r. 2008. two new species of babiana (iridaceae: crocoideae) from western south africa, new names for b. longiflora and

species of moraea (iridaceae: iridoideae), with range extensions and miscellaneous notes for southern african species. bothalia. 39:1 - 10. goldblatt p., manning j.c. 2010. new taxa of babiana (iridaceae: crocoideae) from coastal western cape, south africa. bothalia. 40: 47 - 53. goldblatt p., manning j.c., gereau r. 2008. two new species of babiana (iridaceae: crocoideae) from western south africa, new names for b. longiflora and b. thunbergii, and comments on the original publication of the genus. bothalia. 38:49 - 55. goldblatt p., manning j.c., snijman d. 2005. cape plants: corrections and additions to the flora. 1. bothalia. 35:35 - 46. goldblatt

and rapid speciation in a biodiversity hot spot. science. 309:1722 - 1725. levyns m.r. 1964. migrations and the origins of the cape flora. trans. r. soc. s. afr. 37:85 - 107. lewis g.j. 1959. the genus babiana. s. afr. j. bot. 50(suppl. 3):365 - 379. linder h.p. 2001. two new species of ceratocaryum (restionaceae). kew. bull. 56:465 - 477. linder h.p. 2003. the radiation of the cape flora, southern africa. biol. rev. camb. philos. soc. 78:597 - 638. linder h.p. 2005. evolution of diversity: the cape flora. trends plant sci. 10:536 - 541. linder h.p. 2008. plant species radiations: where, when,

2001. two new species of ceratocaryum (restionaceae). kew. bull. 56:465 - 477. linder h.p. 2003. the radiation of the cape flora, southern africa. biol. rev. camb. philos. soc. 78:597 - 638. linder h.p. 2005. evolution of diversity: the cape flora. trends plant sci. 10:536 - 541. linder h.p. 2008. plant species radiations: where, when, why? philos. trans. r. soc. lond. b. biol. sci. 363:3097 - 3105. linder h.p., hardy c.r. 2004. evolution of the species-rich cape flora. philos. trans. r. soc. lond. b. biol. sci. 359:1623 - 1632. linder h.p., meadows m.e., cowling r.m. 1992. history of the cape flora. in:

rev. camb. philos. soc. 78:597 - 638. linder h.p. 2005. evolution of diversity: the cape flora. trends plant sci. 10:536 - 541. linder h.p. 2008. plant species radiations: where, when, why? philos. trans. r. soc. lond. b. biol. sci. 363:3097 - 3105. linder h.p., hardy c.r. 2004. evolution of the species-rich cape flora. philos. trans. r. soc. lond. b. biol. sci. 359:1623 - 1632. linder h.p., meadows m.e., cowling r.m. 1992. history of the cape flora. in: cowling r.m., editor. the ecology of fynbos: nutrients, fire and diversity. oxford university press. p. 113 - 134. losos j.b. 2008. phylogenetic niche conservatism,

1632. linder h.p., meadows m.e., cowling r.m. 1992. history of the cape flora. in: cowling r.m., editor. the ecology of fynbos: nutrients, fire and diversity. oxford university press. p. 113 - 134. losos j.b. 2008. phylogenetic niche conservatism, phylogenetic signal and the relationship between phylogenetic relatedness and ecological similarity among species. ecol. lett. 11:995 - 1003. losos j.b., schluter d. 2000. analysis of an evolutionary species-area relationship. nature. 408:847 - 850. maddison w.p., maddison d.r. 2008. mesquite: a modular system for evolutionary analysis. version 2.5. available from: http://mesquiteproject.org. downloaded from https://academic.oup.com/sysbio/article-abstract/60/3/343/1666085 by guest on 02 march 2020 collins b., rebelo t.

r.m., editor. the ecology of fynbos: nutrients, fire and diversity. oxford university press. p. 113 - 134. losos j.b. 2008. phylogenetic niche conservatism, phylogenetic signal and the relationship between phylogenetic relatedness and ecological similarity among species. ecol. lett. 11:995 - 1003. losos j.b., schluter d. 2000. analysis of an evolutionary species-area relationship. nature. 408:847 - 850. maddison w.p., maddison d.r. 2008. mesquite: a modular system for evolutionary analysis. version 2.5. available from: http://mesquiteproject.org. downloaded from https://academic.oup.com/sysbio/article-abstract/60/3/343/1666085 by guest on 02 march 2020 collins b., rebelo t. 1987. pollination biology of the proteaceae in australia and southern africa. aust. j. ecol. 12:387

csiba l., powell m.p. 2006. dna extraction protocols. in: savolainen v., powell m.p., davis k., editors. dna and tissue banking for biodiversity conservation: theory, practice and uses. richmond (uk): royal botanic gardens, kew. davies t.j., savolainen v., chase m.w., goldblatt p., barraclough t.g. 2005. environment, area, and diversification in the species-rich flowering plant family iridaceae. am. nat. 166:418 - 425. doyle j.j., doyle j.l. 1987. a rapid dna isolation procedure for small quantities of fresh leaf tissue. phytochem. bull. 19:11 - 15. driver a., desmet p.g., rouget m., cowling r.m., maze k.e. 2003. succulent karoo ecosystem plan. biodiversity component, technical report.

by trees in amazonian forests. science. 305: 663 - 665. fitzpatrick b.m., turelli m. 2006. the geography of mammalian speciation: mixed signals from phylogenies and range maps. evolution. 60:601 - 615. goldblatt p. 1986. the moraeas of southern africa. cape town (south africa): national botanic gardens. goldblatt p. 1992. new species, chromosome cytology and notes on the southern african iridaceae-irideae - moraea, roggeveldia and homeria. s. afr. j. bot. 58:209 - 214. goldblatt p. 1998. reduction of barnardiella, galaxia, gynandriris, hexaglottis, homeria, and roggeveldia in moraea (iridaceae: irideae). novon. 8:371 - 377. goldblatt p., bernhardt p., manning j.c. 2005. pollination mechanisms

goldblatt p., manning j. 2000. cape plants - a conspectus of the cape flora of south africa. pretoria (south africa): national botanical institute. goldblatt p., manning j.c. 1996. phylogeny and speciation in lapeirousia subgenus lapeirousia (iridaceae: ixioideae). ann. mo. bot. gard. 83:346 - 361. goldblatt p., manning j.c. 2000. new species of moraea (iridaceaeiridoideae) from southern africa. novon. 10:14 - 21. goldblatt p., manning j.c. 2002. notes and new species of moraea (iridaceae: iridoideae) from the southern african winter-rainfall zone. novon. 12:352 - 359. goldblatt p., manning j.c. 2004. new species of ixia (crocoideae) and moraea (iridoideae), and taxonomic notes on

africa): national botanical institute. goldblatt p., manning j.c. 1996. phylogeny and speciation in lapeirousia subgenus lapeirousia (iridaceae: ixioideae). ann. mo. bot. gard. 83:346 - 361. goldblatt p., manning j.c. 2000. new species of moraea (iridaceaeiridoideae) from southern africa. novon. 10:14 - 21. goldblatt p., manning j.c. 2002. notes and new species of moraea (iridaceae: iridoideae) from the southern african winter-rainfall zone. novon. 12:352 - 359. goldblatt p., manning j.c. 2004. new species of ixia (crocoideae) and moraea (iridoideae), and taxonomic notes on some other african iridaceae. novon. 14:288 - 298. goldblatt p., manning j.c. 2007a. a revision of the southern african

gard. 83:346 - 361. goldblatt p., manning j.c. 2000. new species of moraea (iridaceaeiridoideae) from southern africa. novon. 10:14 - 21. goldblatt p., manning j.c. 2002. notes and new species of moraea (iridaceae: iridoideae) from the southern african winter-rainfall zone. novon. 12:352 - 359. goldblatt p., manning j.c. 2004. new species of ixia (crocoideae) and moraea (iridoideae), and taxonomic notes on some other african iridaceae. novon. 14:288 - 298. goldblatt p., manning j.c. 2007a. a revision of the southern african genus babiana, iridaceae: crocoideae. pretoria (south africa): south african national biodiversity institute. vol. 60 2011 schnitzler et al. - diversification of

parsimony (*and other methods) v. 4.0b10. sunderland (ma): sinauer associates. taberlet p., gielly l., pautou g., bouvet j. 1991. universal primers for amplification of three non-coding regions of the chloroplast dna. plant mol. biol. 17:1105 - 1109. thuiller w., midgley g.f., rouget m., cowling r.m. 2006. predicting patterns of plant species richness in megadiverse south africa. ecography. 29:733 - 744. tripp e.a., manos p.s. 2008. is floral specialisation an evolutionary dead-end? pollination system transitions in ruella (acanthaceae). evolution. 62:1712 - 1737. valente l., reeves g., schnitzler j., pizer mazon i., fay m., rebelo t., chase m., barraclough t. 2010a. diversification of

l., reeves g., schnitzler j., pizer mazon i., fay m., rebelo t., chase m., barraclough t. 2010a. diversification of the african genus protea in the cape biodiversity hotspot and beyond: equal rates but different spatial scales. evolution. 64:745 - 760. valente l.m., savolainen v., vargas p. 2010b. unparalleled rates of species diversification in europe. proc. r. soc. lond. b. biol. sci. 277:1489 - 1496. van der niet t., johnson s.d. 2009. patterns of plant speciation in the cape floristic region. mol. phylogenet. evol. 51:85 - 93. van der niet t., johnson s.d., linder h.p. 2006. macroevolutionary data suggest a role for

- 1601. verboom g.a., archibald j.k., bakker f.t., bellstedt d.u., conrad f., dreyer l.l., forest f., galley c., goldblatt p., henning j.f., mummenhoff k., linder h.p., muasya a.m., oberlander k.c., savolainen v., snijman d.a., van der niet t., nowell t.l. 2009. origin and diversification of the greater cape flora: ancient species repository, hot-bed of recent radiation, or both? mol. phylogenet. evol. 51:44 - 53. verdaguer d., ojeda f. 2005. evolutionary transition from resprouter to seeder life history in two erica (ericaceae) species: insights from seedling axillary buds. ann. bot. 95:593 - 599. verdú m., pausas j.g., segarra-moragues j.g., ojeda f. 2007.

v., snijman d.a., van der niet t., nowell t.l. 2009. origin and diversification of the greater cape flora: ancient species repository, hot-bed of recent radiation, or both? mol. phylogenet. evol. 51:44 - 53. verdaguer d., ojeda f. 2005. evolutionary transition from resprouter to seeder life history in two erica (ericaceae) species: insights from seedling axillary buds. ann. bot. 95:593 - 599. verdú m., pausas j.g., segarra-moragues j.g., ojeda f. 2007. burning phylogenies: fire, molecular evolutionary rates, and diversification. evolution. 61:2195 - 2204. whittall j.b., hodges s.a. 2007. pollinator shifts drive increasingly long nectar spurs in columbine flowers. nature. 447:706 - 710.

range evolution by dispersal, local extinction, and cladogenesis. syst. biol. 57:4 - 14. reeves g. 2001. radiation and macroevolutionary ecology of the african genus protea l. london: university of london. richardson j.e., weitz f.m., fay m.f., cronk q.c.b., linder h.p., reeves g., chase m.w. 2001. rapid and recent origin of species richness in the cape flora of south africa. nature. 412:181 - 183. 357 ;.;test_pdfs;na;schnitzler_et_al_2011_causes_of_plant_diversification_in_the_cape_biodiv.txt"
) NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA NA

### NA
by increasing opportunity for speciation through an expanded realized niche. nonetheless, the hypothesis that defense mutualisms consistently enhance plant diversification across lineages has not been well tested due to a lack of phenotypic and phylogenetic information. using a global analysis, we show that the >100 vascular plant families in which species have evolved extrafloral nectaries (efns), sugar-secreting organs that recruit arthropod mutualists, have twofold higher diversification rates than families that lack species with efns. zooming in on six distantly related plant clades, traitdependent diversification models confirmed the tendency for lineages with efns to display increased rates of diversification. these results were

lineages has not been well tested due to a lack of phenotypic and phylogenetic information. using a global analysis, we show that the >100 vascular plant families in which species have evolved extrafloral nectaries (efns), sugar-secreting organs that recruit arthropod mutualists, have twofold higher diversification rates than families that lack species with efns. zooming in on six distantly related plant clades, traitdependent diversification models confirmed the tendency for lineages with efns to display increased rates of diversification. these results were consistent across methodological approaches. inference using reversible-jump markov chain monte carlo (mcmc) to model the placement and number of rate shifts

plant - insect interactions lineage diversification rates plant defense | | downloaded by guest on march 2, 2020 e ver since the key innovation hypothesis was first proposed in the 1940s (1, 2), the origination of novel traits has been a popular yet controversial explanation for the exceptional disparity in species richness observed across clades in the tree of life. despite decades of research linking traits to diversification, we have remarkably few examples of traits that have been convincingly demonstrated to spur diversification repeatedly across independent, distantly related groups. notable exceptions include a number of ecologically important traits mediating interactions between

food rewards to arthropod bodyguards in return for protection against natural enemies (7), increase the evolutionary diversification rate of the plant lineages that participate in them. the morphological traits that mediate defense mutualisms represent well-studied examples of characters hypothesized to expand a plant's niche via interactions with mutualists and influence species success in various environmental contexts (8). although the costs and benefits of participating in defense mutualisms are well studied (9), the hypothesis that the ecological impact of defense mutualisms leaves a predictable macroevolutionary signature, increasing lineage diversification within and among clades of plants, has only been examined in a single

the indirect effect of significance plants that provide food and housing to animals in return for defense against enemies are classic examples of mutualistic partnerships in nature. here, we show that the evolution of such plant - animal mutualisms also can lead to a trajectory of accelerated accumulation of plant species in the lineages that participate in these cooperative interactions. we found that the evolution of plant organs (extrafloral nectaries) that facilitate mutualisms with animal defenders was repeatedly followed by increased rates of diversification across distantly related plant lineages. these results suggest that by enabling ecological interactions with animals, the convergent

well studied ecologically, and their only known function is defense against herbivores and microbial pathogens by attracting natural enemies (17). such features have evolved hundreds of times and occur in about a quarter of all vascular plant families (18). here, we first ask whether, across all vascular plants, families containing species with efns are associated with higher diversification rates than families without efns. we then focus in on the phylogenetic history and evolution of efns in six distantly related plant clades to evaluate whether efns are linked, directly or indirectly, to increased lineage diversification rates. as such, this study represents a

test of the macroevolutionary consequences of a convergently evolved and ecologically important mutualistic trait. results and discussion in a global analysis of vascular plant families, we combined published records of efn occurrence (18) with fossil calibrated megatrees (19 - 21) to compare net diversification rates across families with and without species with efns. overall, net diversification rates were more than twofold higher among the 108 weber and agrawal families that contain instances of species with efn compared with the ∼300 families without species with efns (fig. 2 and tables s1 and s2). because our current knowledge likely underestimates the number of

plant families, we combined published records of efn occurrence (18) with fossil calibrated megatrees (19 - 21) to compare net diversification rates across families with and without species with efns. overall, net diversification rates were more than twofold higher among the 108 weber and agrawal families that contain instances of species with efn compared with the ∼300 families without species with efns (fig. 2 and tables s1 and s2). because our current knowledge likely underestimates the number of families with efns by ∼7% (18), we repeated this analysis with randomized inclusion of efns in otherwise non-efn families and found the initial

(18) with fossil calibrated megatrees (19 - 21) to compare net diversification rates across families with and without species with efns. overall, net diversification rates were more than twofold higher among the 108 weber and agrawal families that contain instances of species with efn compared with the ∼300 families without species with efns (fig. 2 and tables s1 and s2). because our current knowledge likely underestimates the number of families with efns by ∼7% (18), we repeated this analysis with randomized inclusion of efns in otherwise non-efn families and found the initial result to be robust to missing information (fig. s1).

this analysis with randomized inclusion of efns in otherwise non-efn families and found the initial result to be robust to missing information (fig. s1). additionally, we found no evidence that efn-bearing clades were older than non-efn clades, suggesting they did not have more time to accumulate their greater number of species than clades without efns (fig. s1). finally, phylogenetic nonindependence did not confound estimates of the relationship between efns and species richness, because the evolution of efns was significantly unstructured compared with the null expectation of brownian motion evolution [angiosperm phylogeny group megatree (apgiii): d = 0.75, p < 0.001; zanne

missing information (fig. s1). additionally, we found no evidence that efn-bearing clades were older than non-efn clades, suggesting they did not have more time to accumulate their greater number of species than clades without efns (fig. s1). finally, phylogenetic nonindependence did not confound estimates of the relationship between efns and species richness, because the evolution of efns was significantly unstructured compared with the null expectation of brownian motion evolution [angiosperm phylogeny group megatree (apgiii): d = 0.75, p < 0.001; zanne et al. megatree (21): d = 0.64, p = 0.006]. results from our global analysis are consistent with a pattern

brownian motion evolution [angiosperm phylogeny group megatree (apgiii): d = 0.75, p < 0.001; zanne et al. megatree (21): d = 0.64, p = 0.006]. results from our global analysis are consistent with a pattern in which there is a net positive effect of this mutualistic trait on rates of species diversification across the hundreds of independent origins of efns. nonetheless, these results should be interpreted with caution due to the scale of the analysis. in particular, at this broad level, it is not possible to link shifts in diversification directly with the origin and loss of efns. additionally, cases of

(yellow stars in fig. 2): byttneria (malvales), senna (fabales), turnera (malpighaiales), viburnum (dipsacales), polygoneae (caryophyllales), and pleopeltis (polypodiales). we selected these evolution efns originate direct effect diversification rate 0.00 0.10 0.20 0.30 no effect efns absent efns present fig. 2. phylogeny of vascular plant families (apgiii) (19), with families containing species with efns colored red. outer bars correspond to the agestandardized number of species [i.e., (log number of species)/(age of plant family in millions of years)]. yellow stars mark the families of the six clades analyzed subsequently in this study. (inset) mean diversification rate (r) ± se of families with and

polygoneae (caryophyllales), and pleopeltis (polypodiales). we selected these evolution efns originate direct effect diversification rate 0.00 0.10 0.20 0.30 no effect efns absent efns present fig. 2. phylogeny of vascular plant families (apgiii) (19), with families containing species with efns colored red. outer bars correspond to the agestandardized number of species [i.e., (log number of species)/(age of plant family in millions of years)]. yellow stars mark the families of the six clades analyzed subsequently in this study. (inset) mean diversification rate (r) ± se of families with and without species with efns calculated according to the method of magallon and sanderson

we selected these evolution efns originate direct effect diversification rate 0.00 0.10 0.20 0.30 no effect efns absent efns present fig. 2. phylogeny of vascular plant families (apgiii) (19), with families containing species with efns colored red. outer bars correspond to the agestandardized number of species [i.e., (log number of species)/(age of plant family in millions of years)]. yellow stars mark the families of the six clades analyzed subsequently in this study. (inset) mean diversification rate (r) ± se of families with and without species with efns calculated according to the method of magallon and sanderson (41) assuming no extinction. see

efns colored red. outer bars correspond to the agestandardized number of species [i.e., (log number of species)/(age of plant family in millions of years)]. yellow stars mark the families of the six clades analyzed subsequently in this study. (inset) mean diversification rate (r) ± se of families with and without species with efns calculated according to the method of magallon and sanderson (41) assuming no extinction. see tables s1 and s2 for f and p statistics and calculations with additional extinction fractions for both megatrees. pnas | november 18, 2014 | vol. 111 | no. 46 | 16443 0.04 0.04 400

0 0 turnera senna −50 −40 0.02 r1- r0 0.00 100 200 300 400 50 100 150 0.000 0 0 r1- r0 0.00 −400 0.003 density 0.000 polygoneae 0.06 pleopeltis 0.006 byttneria 100 300 200 500 net diversification rate clades because they have recently published phylogenies, are known to contain species with and without efns (10, 18, 22 - 27), and are distantly related to one another. for senna, which was previously investigated for a link between efn and diversification rates (10), we added recently published records on efns in an additional clade (28). together, these six plant groups encompass over

playing a role, allowing us to disentangle the complex ways in which traits interact with each other and the environment to influence diversification. for efns, our work within and between clades suggests that over deep time, these important defensive traits have enhanced diversification rates and, ultimately, the diversity of plant species. methods vascular plant family analysis. to test for a global association between defense mutualism and plant diversification, we compared net diversification rates of vascular plant families with and without species with efns. families were scored as either containing or not containing accounts of at least one species with efns based

clades suggests that over deep time, these important defensive traits have enhanced diversification rates and, ultimately, the diversity of plant species. methods vascular plant family analysis. to test for a global association between defense mutualism and plant diversification, we compared net diversification rates of vascular plant families with and without species with efns. families were scored as either containing or not containing accounts of at least one species with efns based on previous work (18). we used two published megatree phylogenies to account for phylogenetic uncertainty. the first was the apgiii megatree from the angiosperm phylogeny group (19) acquired from phylomatic

diversity of plant species. methods vascular plant family analysis. to test for a global association between defense mutualism and plant diversification, we compared net diversification rates of vascular plant families with and without species with efns. families were scored as either containing or not containing accounts of at least one species with efns based on previous work (18). we used two published megatree phylogenies to account for phylogenetic uncertainty. the first was the apgiii megatree from the angiosperm phylogeny group (19) acquired from phylomatic (r20120829) (37) using one representative tip per family. the apgiii tree is a compilation of previously published

zanne et al. (21). pnas | november 18, 2014 | vol. 111 | no. 46 | 16445 evolution senna s0 s1 probability 0.0 0.2 0.4 s0 0 we calculated the net diversification rate using the method of magallón and sanderson (41) implemented in geiger (42). we calculated rates based on species richness and clade ages estimated as terminal branch lengths from both the apgiii tree and the tree published by zanne et al. (21). we repeated calculations for four values of e, the extinction rate, expressed as a fraction of the speciation rate: 0, 0.1, 0.5, and 0.9. the number of

richness and clade ages estimated as terminal branch lengths from both the apgiii tree and the tree published by zanne et al. (21). we repeated calculations for four values of e, the extinction rate, expressed as a fraction of the speciation rate: 0, 0.1, 0.5, and 0.9. the number of species in each family was taken from stevens (43). because a range of species counts for a family was reported in some cases, we repeated all analyses across maximum and minimum species estimates. the difference in the mean net diversification rate of families with and without efns was then analyzed by

tree and the tree published by zanne et al. (21). we repeated calculations for four values of e, the extinction rate, expressed as a fraction of the speciation rate: 0, 0.1, 0.5, and 0.9. the number of species in each family was taken from stevens (43). because a range of species counts for a family was reported in some cases, we repeated all analyses across maximum and minimum species estimates. the difference in the mean net diversification rate of families with and without efns was then analyzed by a two-way anova. because our current knowledge likely underestimates the number of families

the extinction rate, expressed as a fraction of the speciation rate: 0, 0.1, 0.5, and 0.9. the number of species in each family was taken from stevens (43). because a range of species counts for a family was reported in some cases, we repeated all analyses across maximum and minimum species estimates. the difference in the mean net diversification rate of families with and without efns was then analyzed by a two-way anova. because our current knowledge likely underestimates the number of families with efns by 4 - 9% (18), we repeated this analysis 10,000 times, each time converting 10 (an

with efns by 4 - 9% (18), we repeated this analysis 10,000 times, each time converting 10 (an additional ∼9% of the current total) randomly selected efn-absent families to efn-present families to simulate conservatively the discovery of new families with efns. we tested for phylogenetic signal in the presence of species with efns in a family via the estimation of fritz and purvis' d for binary traits, which is a measure of sister-clade differences in a discrete character state for a given phylogeny (44). an estimated d of 1 implies a distribution that is random with respect to the phylogeny, whereas

of families, as well as the number of families assigned to each character state, although it varies the distribution of character states across the tree. clade-level analyses. we selected six vascular plant clades for phylogenetic comparative analyses that (i) had sequence representation in genbank and (ii) were known to contain species with and without efns based on descriptions in the literature: byttneria (malvaceae), pleopeltis (polypodiaceae), polygoneae (polygoneaceae), senna (fabaceae), turnera (passifloraceae), and viburnum (adoxaceae). each of these clades represents an independent evolutionary origin of efns, and, together, they span a large portion of the angiosperm tree of life. downloaded by guest

represents an independent evolutionary origin of efns, and, together, they span a large portion of the angiosperm tree of life. downloaded by guest on march 2, 2020 phylogenetic inference. we reconstructed a distribution of time-calibrated phylogenies separately for each genus using bayesian methods to include the highest possible number of species because the most recently published phylogenies of our clades of interest were frequently not ultrametric. sequence availability for each group was evaluated, and sequences were obtained from the genbank using the phylota browser (release 1.5) (46). molecular markers were chosen for inclusion in phylogenetic analyses if they were sampled for

published phylogenies of our clades of interest were frequently not ultrametric. sequence availability for each group was evaluated, and sequences were obtained from the genbank using the phylota browser (release 1.5) (46). molecular markers were chosen for inclusion in phylogenetic analyses if they were sampled for over 30% of the species available in the genbank (accession numbers have been deposited in datadryad; dx.doi.org/10.5061/ dryad.17fj8). outgroup taxa were selected based on the most recent published phylogeny or from the parent cluster in phylota based on overlapping sequence coverage with in-group taxa. nucleotide sequences were aligned using the l-ins-i strategy in mafft (version

in the jstor global plants database (plants.jstor.org). the world list of plants with efns database can be accessed at www.extrafloralnectaries.org (56). lineage diversification analyses. we evaluated whether net diversification rates in each clade were dependent on efn state using bisse (29) and bamm (32). outgroup taxa and multiple individuals per species were pruned from the mcc tree for each analysis so that resulting phylogenies contained only one sample per species. we used the bisse (29) state-dependent speciation and extinction model to estimate net diversification (speciation − extinction) rates in lineages with and without efns. we implemented the bayesian mcmc bisse algorithm

www.extrafloralnectaries.org (56). lineage diversification analyses. we evaluated whether net diversification rates in each clade were dependent on efn state using bisse (29) and bamm (32). outgroup taxa and multiple individuals per species were pruned from the mcc tree for each analysis so that resulting phylogenies contained only one sample per species. we used the bisse (29) state-dependent speciation and extinction model to estimate net diversification (speciation − extinction) rates in lineages with and without efns. we implemented the bayesian mcmc bisse algorithm in the diversitree package in r (31). because some species are missing from our phylogenies for each of our

so that resulting phylogenies contained only one sample per species. we used the bisse (29) state-dependent speciation and extinction model to estimate net diversification (speciation − extinction) rates in lineages with and without efns. we implemented the bayesian mcmc bisse algorithm in the diversitree package in r (31). because some species are missing from our phylogenies for each of our six groups, we accounted for missing taxa in these analyses by including information on the proportions of taxa (included and missing) assigned to each character state (30) using species descriptions and clade estimates from previous publications (10, 23 - 26, 57).

mcmc bisse algorithm in the diversitree package in r (31). because some species are missing from our phylogenies for each of our six groups, we accounted for missing taxa in these analyses by including information on the proportions of taxa (included and missing) assigned to each character state (30) using species descriptions and clade estimates from previous publications (10, 23 - 26, 57). in cases where the character state of missing species was unknown, we assumed the proportions of character states in our known samples were representative (table s6). to test whether efn and non-efn lineages had different diversification rates, we

our six groups, we accounted for missing taxa in these analyses by including information on the proportions of taxa (included and missing) assigned to each character state (30) using species descriptions and clade estimates from previous publications (10, 23 - 26, 57). in cases where the character state of missing species was unknown, we assumed the proportions of character states in our known samples were representative (table s6). to test whether efn and non-efn lineages had different diversification rates, we used bayesian mcmc bisse analyses on the mcc tree for each clade. we used exponential priors for all parameters estimated using

analyses across the same topologies using simulated trait data. efn tip states were simulated as discrete characters 100 times for each topology using an mk2 model with parameter estimates fit using the observed trait data with the fitdiscrete function in the r package geiger (41). simulations with fewer than five species in each state were rejected to condition on having more than a small number of species in one state. root states for simulations were determined using a random draw from a binomial distribution with a probability of successfully drawing a state proportional to the marginal probability of that state at

characters 100 times for each topology using an mk2 model with parameter estimates fit using the observed trait data with the fitdiscrete function in the r package geiger (41). simulations with fewer than five species in each state were rejected to condition on having more than a small number of species in one state. root states for simulations were determined using a random draw from a binomial distribution with a probability of successfully drawing a state proportional to the marginal probability of that state at the root in the ancestral reconstruction using the asr.bisse function in diversitree (31). we used a

bayesfactorbranches function. this method corrects for differences in branch length or biases introduced by the prior on the number of diversification rates. we estimated the number of shifts in sister clades with and without efns for groups that had sister groups with and without efns that included at least two species each. we used the subtreebamm function to extract sister clades from the original bamm objects. the shift density for each extracted sister subclade was calculated by dividing each sample from the subclade's posterior by the sum of branch lengths for that subclade. data availability. phylogenies, character states, genbank accession numbers,

- 1092. 32. rabosky dl (2014) automatic detection of key innovations, rate shifts, and diversitydependence on phylogenetic trees. plos one 9(2):e89543. 33. bronstein jl, alarcón r, geber m (2006) the evolution of plant-insect mutualisms. new phytol 172(3):412 - 428. 34. spriggs el, christin p-a, edwards ej (2014) c4 photosynthesis promoted species diversification during the miocene grassland expansion. plos one 9(5):e97722. 35. bininda-emonds or, et al. (2007) the delayed rise of present-day mammals. nature 446(7135):507 - 512. 36. nel a, roques p, nel p, prokop j, steyer js (2007) the earliest holometabolous insect from the carboniferous: a "crucial" innovation with delayed success

### NA
arizona, tucson, az 85721, usa summary authors for correspondence: xue-jun ge tel: +86 20 37252551 email: xjge@scbg.ac.cn john j. wiens tel: +1 520 6210337 email: wiensj@email.arizona.edu received: 24 december 2017 accepted: 5 march 2018 new phytologist (2018) 219: 436 - 448 doi: 10.1111/nph.15144 key words: biogeography, diversification rate, lysimachieae, phylogeny, species richness.  what causes the disparity in biodiversity among regions is a fundamental question in biogeography, ecology, and evolutionary biology. evolutionary and biogeographic processes (speciation, extinction, dispersal) directly determine species richness patterns, and can be studied using integrative phylogenetic approaches. however, the strikingly high richness of east asia relative to

accepted: 5 march 2018 new phytologist (2018) 219: 436 - 448 doi: 10.1111/nph.15144 key words: biogeography, diversification rate, lysimachieae, phylogeny, species richness.  what causes the disparity in biodiversity among regions is a fundamental question in biogeography, ecology, and evolutionary biology. evolutionary and biogeographic processes (speciation, extinction, dispersal) directly determine species richness patterns, and can be studied using integrative phylogenetic approaches. however, the strikingly high richness of east asia relative to other northern hemisphere regions remains poorly understood from this perspective. here, for the first time, we test two general hypotheses (older colonization time, faster diversification rate) to explain this pattern,

hemisphere regions remains poorly understood from this perspective. here, for the first time, we test two general hypotheses (older colonization time, faster diversification rate) to explain this pattern, using the plant tribe lysimachieae (primulaceae) as a model system.  we generated a new time-calibrated phylogeny for lysimachieae (13 genes, 126 species), to estimate colonization times and diversification rates for each region and to test the relative importance of these two factors for explaining regional richness patterns.  we find that neither time nor diversification rates alone explain richness patterns among regions in lysimachieae. instead, a new index that combines both factors

the group and their high east asian biodiversity.  based on our results from lysimachieae, we suggest that the high richness of plants in east asia may be explained by a combination of older colonization times and faster diversification rates in this region. introduction why do some regions have higher species diversity than others? explaining richness patterns is a major goal of ecology, biogeography, and evolutionary biology. for example, 25 biodiversity hotspots cover c. 1.4% of earth's land surface, but include c. 44% of vascular plant species and c. 35% of species in four vertebrate groups (myers et al., 2000). however,

and faster diversification rates in this region. introduction why do some regions have higher species diversity than others? explaining richness patterns is a major goal of ecology, biogeography, and evolutionary biology. for example, 25 biodiversity hotspots cover c. 1.4% of earth's land surface, but include c. 44% of vascular plant species and c. 35% of species in four vertebrate groups (myers et al., 2000). however, the underlying causes of many diversity patterns remain unclear. for example, despite many studies on the latitudinal diversity gradient (e.g. willig et al., 2003; mittelbach et al., 2007) and elevational richness gradients (e.g. hutter et al.,

this region. introduction why do some regions have higher species diversity than others? explaining richness patterns is a major goal of ecology, biogeography, and evolutionary biology. for example, 25 biodiversity hotspots cover c. 1.4% of earth's land surface, but include c. 44% of vascular plant species and c. 35% of species in four vertebrate groups (myers et al., 2000). however, the underlying causes of many diversity patterns remain unclear. for example, despite many studies on the latitudinal diversity gradient (e.g. willig et al., 2003; mittelbach et al., 2007) and elevational richness gradients (e.g. hutter et al., 2013), other richness patterns remain

underlying causes of many diversity patterns remain unclear. for example, despite many studies on the latitudinal diversity gradient (e.g. willig et al., 2003; mittelbach et al., 2007) and elevational richness gradients (e.g. hutter et al., 2013), other richness patterns remain insufficiently studied and poorly understood. many factors may ultimately impact species richness, but only three processes can directly change species numbers in a region: speciation, extinction, and dispersal (ricklefs, 1987). given this perspective, two major hypotheses can explain why certain regions have more species than others (review in wiens, 2011): (1) more time for richness to accumulate through speciation in regions

example, despite many studies on the latitudinal diversity gradient (e.g. willig et al., 2003; mittelbach et al., 2007) and elevational richness gradients (e.g. hutter et al., 2013), other richness patterns remain insufficiently studied and poorly understood. many factors may ultimately impact species richness, but only three processes can directly change species numbers in a region: speciation, extinction, and dispersal (ricklefs, 1987). given this perspective, two major hypotheses can explain why certain regions have more species than others (review in wiens, 2011): (1) more time for richness to accumulate through speciation in regions that have been colonized longer (e.g. willis, 1922); and/

et al., 2013), other richness patterns remain insufficiently studied and poorly understood. many factors may ultimately impact species richness, but only three processes can directly change species numbers in a region: speciation, extinction, and dispersal (ricklefs, 1987). given this perspective, two major hypotheses can explain why certain regions have more species than others (review in wiens, 2011): (1) more time for richness to accumulate through speciation in regions that have been colonized longer (e.g. willis, 1922); and/ or (2) faster rates of net diversification (speciation - extinction) in 436 new phytologist (2018) 219: 436 - 448 www.newphytologist.com lineages in high-diversity regions

reduce extinction, or both. these hypotheses can be tested using phylogenetic approaches for estimating biogeographic history and diversification rates. we do not treat 'carrying capacity' as a separate hypothesis since it can influence both colonization times and diversification rates (pontarp & wiens, 2017), but (like climate) does not directly change species numbers itself. a striking but still unexplained richness pattern involves the remarkable species diversity of plants in east asia (china, japan, and korea) relative to similar latitudes in north america and europe (latham & ricklefs, 1993; qian & ricklefs, 2000; qian, 2001, 2002; adams, 2009). for example, china is among

estimating biogeographic history and diversification rates. we do not treat 'carrying capacity' as a separate hypothesis since it can influence both colonization times and diversification rates (pontarp & wiens, 2017), but (like climate) does not directly change species numbers itself. a striking but still unexplained richness pattern involves the remarkable species diversity of plants in east asia (china, japan, and korea) relative to similar latitudes in north america and europe (latham & ricklefs, 1993; qian & ricklefs, 2000; qian, 2001, 2002; adams, 2009). for example, china is among the world's most megadiverse countries (> 32 000 vascular plant species, c. 1.5

the remarkable species diversity of plants in east asia (china, japan, and korea) relative to similar latitudes in north america and europe (latham & ricklefs, 1993; qian & ricklefs, 2000; qian, 2001, 2002; adams, 2009). for example, china is among the world's most megadiverse countries (> 32 000 vascular plant species, c. 1.5 times more than the usa and canada combined; hong & blackmore, 2013). several explanations have been proposed to explain this diversity anomaly in terms of speciation, extinction, and colonization times. these hypotheses suggest that lineages in east asia have lower extinction rates (i.e. during quaternary ice ages; adams,

1996; ó 2018 the authors new phytologist ó 2018 new phytologist trust new phytologist research 437 (e.g. banfi et al., 2005; manns & anderberg, 2009). we followed the traditional classification, but our results also support expansion of lysimachia. we therefore place these genera in quotation marks. in total, 126 lysimachieae species were sampled in the phylogeny, including c. 50% of the 250 described species (ray, 1956; hu & kelso, 1996; cholewa et al., 2009; manns & anderberg, 2009). our sampling includes all genera, all infrageneric taxa, and all major biogeographic regions. five outgroup taxa (from myrsinaceae) were also included (myrsine semiserrata,

phytologist research 437 (e.g. banfi et al., 2005; manns & anderberg, 2009). we followed the traditional classification, but our results also support expansion of lysimachia. we therefore place these genera in quotation marks. in total, 126 lysimachieae species were sampled in the phylogeny, including c. 50% of the 250 described species (ray, 1956; hu & kelso, 1996; cholewa et al., 2009; manns & anderberg, 2009). our sampling includes all genera, all infrageneric taxa, and all major biogeographic regions. five outgroup taxa (from myrsinaceae) were also included (myrsine semiserrata, myrsine faberi, myrsine seguinii, ardisia verbascifolia, and embelia ribes). these taxa are all

genera, all infrageneric taxa, and all major biogeographic regions. five outgroup taxa (from myrsinaceae) were also included (myrsine semiserrata, myrsine faberi, myrsine seguinii, ardisia verbascifolia, and embelia ribes). these taxa are all closely related to lysimachieae (e.g. k€allersj€o et al., 2000; yesson et al., 2009). details on taxon sampling and species distributions are provided in table s2. species sampled per region are given in table 1. there was a very strong relationship between the total richness of each region and the richness sampled in the tree from each region (r2 = 0.99; p < 0.0001). among regions, the proportion of species

biogeographic regions. five outgroup taxa (from myrsinaceae) were also included (myrsine semiserrata, myrsine faberi, myrsine seguinii, ardisia verbascifolia, and embelia ribes). these taxa are all closely related to lysimachieae (e.g. k€allersj€o et al., 2000; yesson et al., 2009). details on taxon sampling and species distributions are provided in table s2. species sampled per region are given in table 1. there was a very strong relationship between the total richness of each region and the richness sampled in the tree from each region (r2 = 0.99; p < 0.0001). among regions, the proportion of species sampled ranged from 0.42 to 1.00 (mean:

species distributions are provided in table s2. species sampled per region are given in table 1. there was a very strong relationship between the total richness of each region and the richness sampled in the tree from each region (r2 = 0.99; p < 0.0001). among regions, the proportion of species sampled ranged from 0.42 to 1.00 (mean: 0.73). details on estimating regional richness (and delimiting regions) are provided later. there was also a very strong relationship between each clade's overall richness and its richness in our tree (r2 = 0.94; p < 0.0001; table s3). qian & ricklefs, 2000), or

qian & ricklefs, 2000), or were present in east asia longer than in other regions (latham & ricklefs, 1993). however, few studies have used a phylogenetic approach to directly test these hypotheses. xiang et al. (2004) inferred higher speciation rates in east asia than in eastern north america by comparing species richness and substitution rates between 10 sister clades with species in both regions. nevertheless, to our knowledge, no previous studies have directly tested both the diversification-rate and time hypotheses to explain east asia's exceptional species richness. here, we use the tribe lysimachieae (primulaceae) to test whether high richness in east

longer than in other regions (latham & ricklefs, 1993). however, few studies have used a phylogenetic approach to directly test these hypotheses. xiang et al. (2004) inferred higher speciation rates in east asia than in eastern north america by comparing species richness and substitution rates between 10 sister clades with species in both regions. nevertheless, to our knowledge, no previous studies have directly tested both the diversification-rate and time hypotheses to explain east asia's exceptional species richness. here, we use the tribe lysimachieae (primulaceae) to test whether high richness in east asia is explained by older colonization times, faster diversification rates,

(2004) inferred higher speciation rates in east asia than in eastern north america by comparing species richness and substitution rates between 10 sister clades with species in both regions. nevertheless, to our knowledge, no previous studies have directly tested both the diversification-rate and time hypotheses to explain east asia's exceptional species richness. here, we use the tribe lysimachieae (primulaceae) to test whether high richness in east asia is explained by older colonization times, faster diversification rates, or both. lysimachieae offers an excellent model system. they are distributed globally, but most species occur in east asia (171 spp.), with fewer species in

the diversification-rate and time hypotheses to explain east asia's exceptional species richness. here, we use the tribe lysimachieae (primulaceae) to test whether high richness in east asia is explained by older colonization times, faster diversification rates, or both. lysimachieae offers an excellent model system. they are distributed globally, but most species occur in east asia (171 spp.), with fewer species in other regions (< 30 each; table 1). we test the time and diversification-rate hypotheses using phylogenetic approaches. we first combine new molecular data with previously published data to estimate a time-calibrated phylogeny for lysimachieae. we next perform biogeographic analyses to

exceptional species richness. here, we use the tribe lysimachieae (primulaceae) to test whether high richness in east asia is explained by older colonization times, faster diversification rates, or both. lysimachieae offers an excellent model system. they are distributed globally, but most species occur in east asia (171 spp.), with fewer species in other regions (< 30 each; table 1). we test the time and diversification-rate hypotheses using phylogenetic approaches. we first combine new molecular data with previously published data to estimate a time-calibrated phylogeny for lysimachieae. we next perform biogeographic analyses to estimate their time of colonization in each region. we

rates of lysimachieae clades in each region and test for relationships between overall diversification rates of regions and their current richness. we also develop a new index that combines the time and diversification-rate hypotheses. phylogenetic analysis a total of 1111 new sequences were generated (84.62% of 13 genes for 126 species). sequences included 10 plastid loci (i.e. matk, rbcl, trnl-f, trnh-psba, rpl20-rps12, atpf-atph, atpbrbcl, rps16, trns-trng, rpl32-trnl) and one nuclear locus (internal transcribed spacer region of nuclear ribosomal dna; its, including its1, 5.8s ribosomal rna gene, and its2). total genomic dna was isolated from silica-gel-dried fresh material or herbarium materials using

its2). total genomic dna was isolated from silica-gel-dried fresh material or herbarium materials using a modified cetyltrimethylammonium bromide protocol (doyle & doyle, 1987). primers for amplification and sequencing are described in table s4. pcr amplification followed zhang et al. (2012). new sequences were deposited in genbank (table s5). to increase species sampling, we included sequences from 14 'anagallis' species (manns & anderberg, 2005, 2011) and seven materials and methods taxonomy we generally follow the taxonomy established by pax & knuth (1905) and followed in recent studies (e.g. hu, 1994; st ahl & anderberg, 2004). thus, lysimachieae includes six genera: lysimachia (211

fresh material or herbarium materials using a modified cetyltrimethylammonium bromide protocol (doyle & doyle, 1987). primers for amplification and sequencing are described in table s4. pcr amplification followed zhang et al. (2012). new sequences were deposited in genbank (table s5). to increase species sampling, we included sequences from 14 'anagallis' species (manns & anderberg, 2005, 2011) and seven materials and methods taxonomy we generally follow the taxonomy established by pax & knuth (1905) and followed in recent studies (e.g. hu, 1994; st ahl & anderberg, 2004). thus, lysimachieae includes six genera: lysimachia (211 species), anagallis (31), trientalis (3), glaux (1), asterolinon

sampling, we included sequences from 14 'anagallis' species (manns & anderberg, 2005, 2011) and seven materials and methods taxonomy we generally follow the taxonomy established by pax & knuth (1905) and followed in recent studies (e.g. hu, 1994; st ahl & anderberg, 2004). thus, lysimachieae includes six genera: lysimachia (211 species), anagallis (31), trientalis (3), glaux (1), asterolinon (2), and pelletiera (2). species and their regional distributions (based on previous literature) are listed in supporting information table s1. however, some traditionally recognized genera have been merged into lysimachia in recent studies table 1 summary of species richness, timing of colonization and

2011) and seven materials and methods taxonomy we generally follow the taxonomy established by pax & knuth (1905) and followed in recent studies (e.g. hu, 1994; st ahl & anderberg, 2004). thus, lysimachieae includes six genera: lysimachia (211 species), anagallis (31), trientalis (3), glaux (1), asterolinon (2), and pelletiera (2). species and their regional distributions (based on previous literature) are listed in supporting information table s1. however, some traditionally recognized genera have been merged into lysimachia in recent studies table 1 summary of species richness, timing of colonization and weighted diversification rate of lysimachieae in each region first colonization time 9

includes six genera: lysimachia (211 species), anagallis (31), trientalis (3), glaux (1), asterolinon (2), and pelletiera (2). species and their regional distributions (based on previous literature) are listed in supporting information table s1. however, some traditionally recognized genera have been merged into lysimachia in recent studies table 1 summary of species richness, timing of colonization and weighted diversification rate of lysimachieae in each region first colonization time 9 diversification rate species richness (sampled species) first colonization time (myr) summed ages (myr) ɛ=0 ɛ = 0.45 ɛ = 0.90 ɛ=0 ɛ = 0.45 ɛ = 0.90 summed ages 9 diversification rate (ɛ

regional distributions (based on previous literature) are listed in supporting information table s1. however, some traditionally recognized genera have been merged into lysimachia in recent studies table 1 summary of species richness, timing of colonization and weighted diversification rate of lysimachieae in each region first colonization time 9 diversification rate species richness (sampled species) first colonization time (myr) summed ages (myr) ɛ=0 ɛ = 0.45 ɛ = 0.90 ɛ=0 ɛ = 0.45 ɛ = 0.90 summed ages 9 diversification rate (ɛ = 0.45) 171 (74) 22 (18) 28 (17) 19 (13) 10 (8) 22.2562 28.9201 9.5613 26.8263 9.9528 27.9167 49.3975 18.4416

on previous literature) are listed in supporting information table s1. however, some traditionally recognized genera have been merged into lysimachia in recent studies table 1 summary of species richness, timing of colonization and weighted diversification rate of lysimachieae in each region first colonization time 9 diversification rate species richness (sampled species) first colonization time (myr) summed ages (myr) ɛ=0 ɛ = 0.45 ɛ = 0.90 ɛ=0 ɛ = 0.45 ɛ = 0.90 summed ages 9 diversification rate (ɛ = 0.45) 171 (74) 22 (18) 28 (17) 19 (13) 10 (8) 22.2562 28.9201 9.5613 26.8263 9.9528 27.9167 49.3975 18.4416 44.4665 16.3157 0.2517

0.3171 0 0.2544 0 0.1048 0 0.5901 0 0.4735 0 0.1950 0 1.0106 0 diversification rate regions east asia (a) europe (b) africa (c) north america (d) central - south america (e) hawaii (f) australasia (g) ɛ, the relative extinction fraction (e = speciation rate/extinction rate); diversification rates are in species per millions of years (myr). ó 2018 the authors new phytologist ó 2018 new phytologist trust new phytologist (2018) 219: 436 - 448 www.newphytologist.com 438 research from lysimachia subgenus lysimachiopsis (oh et al., 2013). we used data from genbank for all 11 genes sequenced here (with available data), and two

from lysimachia subgenus lysimachiopsis (oh et al., 2013). we used data from genbank for all 11 genes sequenced here (with available data), and two additional plastid loci (rpl16 ndhf; table s5). these new taxa lacked data for some markers. however, the limited missing data in our matrix (mean: 25.58% per species for 131 species) should have little impact on topology estimation or divergence dating, based on simulations and empirical analyses (e.g. wiens & morrill, 2011; zheng & wiens, 2015). furthermore, the placements of these incomplete taxa in our tree and in previous studies (manns & anderberg, 2005, 2011; oh et al.,

lysimachiopsis (oh et al., 2013). we used data from genbank for all 11 genes sequenced here (with available data), and two additional plastid loci (rpl16 ndhf; table s5). these new taxa lacked data for some markers. however, the limited missing data in our matrix (mean: 25.58% per species for 131 species) should have little impact on topology estimation or divergence dating, based on simulations and empirical analyses (e.g. wiens & morrill, 2011; zheng & wiens, 2015). furthermore, the placements of these incomplete taxa in our tree and in previous studies (manns & anderberg, 2005, 2011; oh et al., 2013) were congruent.

calibration points (from other dating analyses). fossil seeds similar to lysimachia vulgaris are known from the latest part of the middle miocene (12 - 16 ma; friis, 1985). oh et al. (2008) found that seeds of l. vulgaris and its close relatives (lysimachia terrestris, lysimachia thyrsiflora) are very similar (all species in clade iv in our phylogeny). these species all have a poroid - alveolate seed-coat surface, a sponge-like outer layer, and columnar cells in the outer seed-coat layer (oh et al., 2008). we therefore new phytologist (2018) 219: 436 - 448 www.newphytologist.com new phytologist used this seed fossil to set

similar to lysimachia vulgaris are known from the latest part of the middle miocene (12 - 16 ma; friis, 1985). oh et al. (2008) found that seeds of l. vulgaris and its close relatives (lysimachia terrestris, lysimachia thyrsiflora) are very similar (all species in clade iv in our phylogeny). these species all have a poroid - alveolate seed-coat surface, a sponge-like outer layer, and columnar cells in the outer seed-coat layer (oh et al., 2008). we therefore new phytologist (2018) 219: 436 - 448 www.newphytologist.com new phytologist used this seed fossil to set the age prior for clade iv. some species

species all have a poroid - alveolate seed-coat surface, a sponge-like outer layer, and columnar cells in the outer seed-coat layer (oh et al., 2008). we therefore new phytologist (2018) 219: 436 - 448 www.newphytologist.com new phytologist used this seed fossil to set the age prior for clade iv. some species in the clade presently lack data on seed morphology, but the clade is strongly supported by molecular results (hao et al., 2004; this study), pollen type (bennell & hu, 1983), and chromosome number (n = 21; l€ove & l€ove, 1982; probatova et al., 2006). we assigned this fossil to the

similar clade ages overall (see the results section; tables s8 - s10), and subsequent analyses used the intermediate calibration point. importantly, biogeographic and diversification analyses were also conducted on these alternative trees. the data matrix and all mcc trees files generated here are available as supporting information (notes s1). incomplete species sampling might affect dating results (stadler, 2009). we used two approaches to address these effects. first, we used the 'birth - death skyline contemporary' prior in beast v.2.3.2 to account for incomplete sampling, following stadler et al. (2013). however, the major clades consistently estimated by other analyses were not recovered.

to address these effects. first, we used the 'birth - death skyline contemporary' prior in beast v.2.3.2 to account for incomplete sampling, following stadler et al. (2013). however, the major clades consistently estimated by other analyses were not recovered. therefore, we did not consider these results further. second, we subsampled species ó 2018 the authors new phytologist ó 2018 new phytologist trust new phytologist from our tree, to see if age estimates were robust to incomplete sampling (if they are, it would suggest that our original subsampling should not strongly impact age estimates). we randomly sampled species from each major clade

further. second, we subsampled species ó 2018 the authors new phytologist ó 2018 new phytologist trust new phytologist from our tree, to see if age estimates were robust to incomplete sampling (if they are, it would suggest that our original subsampling should not strongly impact age estimates). we randomly sampled species from each major clade (using r 3.2.3; r core team, 2015) and performed beast analyses with the calibrations and tree priors described earlier. we randomly selected two or three species per large clade (≥ 10 species sampled) using r. for smaller clades, we included a single, randomly selected species. we

sampling (if they are, it would suggest that our original subsampling should not strongly impact age estimates). we randomly sampled species from each major clade (using r 3.2.3; r core team, 2015) and performed beast analyses with the calibrations and tree priors described earlier. we randomly selected two or three species per large clade (≥ 10 species sampled) using r. for smaller clades, we included a single, randomly selected species. we included 26 species in total, including one outgroup, representing 10% of all lysimachieae species. we then performed a regression between the stem ages of the 11 major clades from the

suggest that our original subsampling should not strongly impact age estimates). we randomly sampled species from each major clade (using r 3.2.3; r core team, 2015) and performed beast analyses with the calibrations and tree priors described earlier. we randomly selected two or three species per large clade (≥ 10 species sampled) using r. for smaller clades, we included a single, randomly selected species. we included 26 species in total, including one outgroup, representing 10% of all lysimachieae species. we then performed a regression between the stem ages of the 11 major clades from the subsampled analyses and the full analyses.

sampled species from each major clade (using r 3.2.3; r core team, 2015) and performed beast analyses with the calibrations and tree priors described earlier. we randomly selected two or three species per large clade (≥ 10 species sampled) using r. for smaller clades, we included a single, randomly selected species. we included 26 species in total, including one outgroup, representing 10% of all lysimachieae species. we then performed a regression between the stem ages of the 11 major clades from the subsampled analyses and the full analyses. biogeographic analyses we compiled distributional data (table s1) for all species described using

major clade (using r 3.2.3; r core team, 2015) and performed beast analyses with the calibrations and tree priors described earlier. we randomly selected two or three species per large clade (≥ 10 species sampled) using r. for smaller clades, we included a single, randomly selected species. we included 26 species in total, including one outgroup, representing 10% of all lysimachieae species. we then performed a regression between the stem ages of the 11 major clades from the subsampled analyses and the full analyses. biogeographic analyses we compiled distributional data (table s1) for all species described using literature data, assigning each

beast analyses with the calibrations and tree priors described earlier. we randomly selected two or three species per large clade (≥ 10 species sampled) using r. for smaller clades, we included a single, randomly selected species. we included 26 species in total, including one outgroup, representing 10% of all lysimachieae species. we then performed a regression between the stem ages of the 11 major clades from the subsampled analyses and the full analyses. biogeographic analyses we compiled distributional data (table s1) for all species described using literature data, assigning each species to one or more geographic regions. we generally used standard

randomly selected species. we included 26 species in total, including one outgroup, representing 10% of all lysimachieae species. we then performed a regression between the stem ages of the 11 major clades from the subsampled analyses and the full analyses. biogeographic analyses we compiled distributional data (table s1) for all species described using literature data, assigning each species to one or more geographic regions. we generally used standard geographic definitions of continental boundaries to delineate regions, but with some differences based on patterns of endemism in lysimachieae. specifically, we used seven regions (fig. 1): (a) east asia, (b) mediterranean region of

in total, including one outgroup, representing 10% of all lysimachieae species. we then performed a regression between the stem ages of the 11 major clades from the subsampled analyses and the full analyses. biogeographic analyses we compiled distributional data (table s1) for all species described using literature data, assigning each species to one or more geographic regions. we generally used standard geographic definitions of continental boundaries to delineate regions, but with some differences based on patterns of endemism in lysimachieae. specifically, we used seven regions (fig. 1): (a) east asia, (b) mediterranean region of europe, (c) africa, including madagascar but excluding

in lysimachieae. specifically, we used seven regions (fig. 1): (a) east asia, (b) mediterranean region of europe, (c) africa, including madagascar but excluding north africa, (d) north america, (e) central - south america, (f) hawaii, and (g) australasia. detailed limits of regions (and their justifications) are given in methods s2. species could be assigned to more than one region using the biogeographic method used here (see below). colonization events were inferred using the dispersal - extinction - cladogenesis (dec) model in rasp v.3.2 (yu et al., 2015) based on lagrange (ree & smith, 2008). analyses were performed using the mcc tree

matrix (q-matrix) to reflect changes in paleogeographic events (land bridges between continents) or dispersal ability between those areas (e.g. stepping stone dispersal or long-distance dispersal across oceans) in different time slices (table s11). we obtained effectively identical results without these assumptions (table s12). we found that 90.48% (114/126) of ingroup species are restricted to one region, whereas only 12 species occur in two to five regions (table s2). therefore, we initially ran rasp allowing ancestral species to occur in a maximum of two areas. we also used likelihood ratio, v2, and akaike information criterion tests to compare the fit of different

bridges between continents) or dispersal ability between those areas (e.g. stepping stone dispersal or long-distance dispersal across oceans) in different time slices (table s11). we obtained effectively identical results without these assumptions (table s12). we found that 90.48% (114/126) of ingroup species are restricted to one region, whereas only 12 species occur in two to five regions (table s2). therefore, we initially ran rasp allowing ancestral species to occur in a maximum of two areas. we also used likelihood ratio, v2, and akaike information criterion tests to compare the fit of different assumptions about the maximum number of regions. aic weights

across oceans) in different time slices (table s11). we obtained effectively identical results without these assumptions (table s12). we found that 90.48% (114/126) of ingroup species are restricted to one region, whereas only 12 species occur in two to five regions (table s2). therefore, we initially ran rasp allowing ancestral species to occur in a maximum of two areas. we also used likelihood ratio, v2, and akaike information criterion tests to compare the fit of different assumptions about the maximum number of regions. aic weights (aicw) for each main clade (its crown and stem node) were interpreted as the relative probabilities

all possible ancestral states (yu et al., 2015). the most likely state for each node (region with highest proportional likelihood) was used to test the time hypothesis. the analyses were also performed using all alternative trees from beast (except the subsampling analyses). time hypothesis we examined the relationship between regional species richness and the oldest colonization time for each region using leastsquares linear regression (following stephens & wiens, 2003). we did not correct for phylogeny in these analyses because they ó 2018 the authors new phytologist ó 2018 new phytologist trust research 439 were based on regions (which lack a phylogeny).

here and most branches are short. some clades had two ancestral areas (e.g. ab for clades i - v; fig. 1), indicating that the ancestor occurred in two regions simultaneously. in these cases, we used the clade's age as the colonization time for both regions. in other cases, an extant species represented a region's oldest colonization. we calculated the colonization time as half the species' age in these cases. the primary analyses were based on the oldest colonization time for each region. however, multiple invasions of the same region occurred in some cases. therefore, we examined the relationship between species richness

for clades i - v; fig. 1), indicating that the ancestor occurred in two regions simultaneously. in these cases, we used the clade's age as the colonization time for both regions. in other cases, an extant species represented a region's oldest colonization. we calculated the colonization time as half the species' age in these cases. the primary analyses were based on the oldest colonization time for each region. however, multiple invasions of the same region occurred in some cases. therefore, we examined the relationship between species richness and summed ages of colonization of each region (e.g. stephens & wiens, 2003; hutter

extant species represented a region's oldest colonization. we calculated the colonization time as half the species' age in these cases. the primary analyses were based on the oldest colonization time for each region. however, multiple invasions of the same region occurred in some cases. therefore, we examined the relationship between species richness and summed ages of colonization of each region (e.g. stephens & wiens, 2003; hutter et al., 2013). thus, regions that have been colonized many times (and less recently) were given added weight. we included widespread, extant species in these calculations also. finally, the species in our phylogeny do not

same region occurred in some cases. therefore, we examined the relationship between species richness and summed ages of colonization of each region (e.g. stephens & wiens, 2003; hutter et al., 2013). thus, regions that have been colonized many times (and less recently) were given added weight. we included widespread, extant species in these calculations also. finally, the species in our phylogeny do not encompass all lysimachieae species, nor all species in all regions (table 1). even though our species sampling in each region is proportional to its total richness (see above), and all regions have > 40% sampling (table 1), some

we examined the relationship between species richness and summed ages of colonization of each region (e.g. stephens & wiens, 2003; hutter et al., 2013). thus, regions that have been colonized many times (and less recently) were given added weight. we included widespread, extant species in these calculations also. finally, the species in our phylogeny do not encompass all lysimachieae species, nor all species in all regions (table 1). even though our species sampling in each region is proportional to its total richness (see above), and all regions have > 40% sampling (table 1), some colonizations may still have been missed. however,

ages of colonization of each region (e.g. stephens & wiens, 2003; hutter et al., 2013). thus, regions that have been colonized many times (and less recently) were given added weight. we included widespread, extant species in these calculations also. finally, the species in our phylogeny do not encompass all lysimachieae species, nor all species in all regions (table 1). even though our species sampling in each region is proportional to its total richness (see above), and all regions have > 40% sampling (table 1), some colonizations may still have been missed. however, unsampled colonizations may be particularly unlikely to have high

of each region (e.g. stephens & wiens, 2003; hutter et al., 2013). thus, regions that have been colonized many times (and less recently) were given added weight. we included widespread, extant species in these calculations also. finally, the species in our phylogeny do not encompass all lysimachieae species, nor all species in all regions (table 1). even though our species sampling in each region is proportional to its total richness (see above), and all regions have > 40% sampling (table 1), some colonizations may still have been missed. however, unsampled colonizations may be particularly unlikely to have high richness in a

et al., 2013). thus, regions that have been colonized many times (and less recently) were given added weight. we included widespread, extant species in these calculations also. finally, the species in our phylogeny do not encompass all lysimachieae species, nor all species in all regions (table 1). even though our species sampling in each region is proportional to its total richness (see above), and all regions have > 40% sampling (table 1), some colonizations may still have been missed. however, unsampled colonizations may be particularly unlikely to have high richness in a region (all other things being equal, more species-rich clades

our species sampling in each region is proportional to its total richness (see above), and all regions have > 40% sampling (table 1), some colonizations may still have been missed. however, unsampled colonizations may be particularly unlikely to have high richness in a region (all other things being equal, more species-rich clades are more likely to be sampled). therefore, unsampled colonizations seem unlikely to be the oldest or most influential in a region. furthermore, our primary analyses used the oldest colonization time, so failure to include later colonization events will have no impact on the results. diversification hypothesis we examined the

more likely to be sampled). therefore, unsampled colonizations seem unlikely to be the oldest or most influential in a region. furthermore, our primary analyses used the oldest colonization time, so failure to include later colonization events will have no impact on the results. diversification hypothesis we examined the relationship between species richness of each region and the diversification rates of clades that occurred there. we first divided the species in each region into previously recognized, nonoverlapping, endemic higher taxa (e.g. genera, subgenera, species groups). we selected higher taxa to represent the largest monophyletic groups in each region (table s13). if a

a region. furthermore, our primary analyses used the oldest colonization time, so failure to include later colonization events will have no impact on the results. diversification hypothesis we examined the relationship between species richness of each region and the diversification rates of clades that occurred there. we first divided the species in each region into previously recognized, nonoverlapping, endemic higher taxa (e.g. genera, subgenera, species groups). we selected higher taxa to represent the largest monophyletic groups in each region (table s13). if a clade had only one species in the region (regardless of its global richness), we assigned the clade a

include later colonization events will have no impact on the results. diversification hypothesis we examined the relationship between species richness of each region and the diversification rates of clades that occurred there. we first divided the species in each region into previously recognized, nonoverlapping, endemic higher taxa (e.g. genera, subgenera, species groups). we selected higher taxa to represent the largest monophyletic groups in each region (table s13). if a clade had only one species in the region (regardless of its global richness), we assigned the clade a rate of zero. our concern is only with the clade's richness in that region

and the diversification rates of clades that occurred there. we first divided the species in each region into previously recognized, nonoverlapping, endemic higher taxa (e.g. genera, subgenera, species groups). we selected higher taxa to represent the largest monophyletic groups in each region (table s13). if a clade had only one species in the region (regardless of its global richness), we assigned the clade a rate of zero. our concern is only with the clade's richness in that region (and alternative approaches should have very little impact, given our weighting scheme). we then calculated the species richness for each named clade, assuming

if a clade had only one species in the region (regardless of its global richness), we assigned the clade a rate of zero. our concern is only with the clade's richness in that region (and alternative approaches should have very little impact, given our weighting scheme). we then calculated the species richness for each named clade, assuming that species unsampled in our tree also belonged to these higher taxa, given their apparent monophyly based on sampled species. we estimated net diversification rates for each clade using the method-of-moments estimator for stem-group ages (magallon & sanderson, 2001) using geiger version 2.0.6 (harmon

the region (regardless of its global richness), we assigned the clade a rate of zero. our concern is only with the clade's richness in that region (and alternative approaches should have very little impact, given our weighting scheme). we then calculated the species richness for each named clade, assuming that species unsampled in our tree also belonged to these higher taxa, given their apparent monophyly based on sampled species. we estimated net diversification rates for each clade using the method-of-moments estimator for stem-group ages (magallon & sanderson, 2001) using geiger version 2.0.6 (harmon et al., 2008). the stem-group estimator is more

only with the clade's richness in that region (and alternative approaches should have very little impact, given our weighting scheme). we then calculated the species richness for each named clade, assuming that species unsampled in our tree also belonged to these higher taxa, given their apparent monophyly based on sampled species. we estimated net diversification rates for each clade using the method-of-moments estimator for stem-group ages (magallon & sanderson, 2001) using geiger version 2.0.6 (harmon et al., 2008). the stem-group estimator is more accurate than the crown-group estimator, especially when taxon new phytologist (2018) 219: 436 - 448 www.newphytologist.com new phytologist

the inset map indicates the geographic regions used in the biogeographic analyses. colors indicate regions only. ancestral areas were inferred by the dispersal - extinction - cladogenesis (dec) model (in rasp 3.2). colored branches and squares on nodes indicate ancestral areas. colored squares at tips indicate current distributions of each species (supporting information table s2). main clades were represented with latin numbers that are also shown in fig. s1. sampling is incomplete (meyer & wiens, 2018). this method uses clade ages, species richness, and a correction for failing to sample extinct clades (relative extinction fraction ɛ). we initially used an intermediate

3.2). colored branches and squares on nodes indicate ancestral areas. colored squares at tips indicate current distributions of each species (supporting information table s2). main clades were represented with latin numbers that are also shown in fig. s1. sampling is incomplete (meyer & wiens, 2018). this method uses clade ages, species richness, and a correction for failing to sample extinct clades (relative extinction fraction ɛ). we initially used an intermediate ɛ of 0.45, but found that very low and high values (ɛ = 0, ɛ = 0.90) gave similar results. the diversification rate for australasia was treated as zero since there

extinct clades (relative extinction fraction ɛ). we initially used an intermediate ɛ of 0.45, but found that very low and high values (ɛ = 0, ɛ = 0.90) gave similar results. the diversification rate for australasia was treated as zero since there was no in situ diversification there (no endemic species or sister species). importantly, the approach used here to estimate diversification rates incorporates all species in each clade (included in the phylogeny or not), and only requires sampling one species per clade. to estimate a single diversification rate for regions with multiple clades, we weighted the diversification rate for each

extinction fraction ɛ). we initially used an intermediate ɛ of 0.45, but found that very low and high values (ɛ = 0, ɛ = 0.90) gave similar results. the diversification rate for australasia was treated as zero since there was no in situ diversification there (no endemic species or sister species). importantly, the approach used here to estimate diversification rates incorporates all species in each clade (included in the phylogeny or not), and only requires sampling one species per clade. to estimate a single diversification rate for regions with multiple clades, we weighted the diversification rate for each clade by multiplying

found that very low and high values (ɛ = 0, ɛ = 0.90) gave similar results. the diversification rate for australasia was treated as zero since there was no in situ diversification there (no endemic species or sister species). importantly, the approach used here to estimate diversification rates incorporates all species in each clade (included in the phylogeny or not), and only requires sampling one species per clade. to estimate a single diversification rate for regions with multiple clades, we weighted the diversification rate for each clade by multiplying it by its proportional richness in the region (the clade's richness in

results. the diversification rate for australasia was treated as zero since there was no in situ diversification there (no endemic species or sister species). importantly, the approach used here to estimate diversification rates incorporates all species in each clade (included in the phylogeny or not), and only requires sampling one species per clade. to estimate a single diversification rate for regions with multiple clades, we weighted the diversification rate for each clade by multiplying it by its proportional richness in the region (the clade's richness in that region divided by the region's total richness). thus, clades with more species in a

sampling one species per clade. to estimate a single diversification rate for regions with multiple clades, we weighted the diversification rate for each clade by multiplying it by its proportional richness in the region (the clade's richness in that region divided by the region's total richness). thus, clades with more species in a region had more weight than clades with fewer species in that region. these weighted rates were then summed across clades to yield the weighted diversification rate for the region. finally, we tested the relationship between the current richness of each region and its weighted diversification rate using linear

for regions with multiple clades, we weighted the diversification rate for each clade by multiplying it by its proportional richness in the region (the clade's richness in that region divided by the region's total richness). thus, clades with more species in a region had more weight than clades with fewer species in that region. these weighted rates were then summed across clades to yield the weighted diversification rate for the region. finally, we tested the relationship between the current richness of each region and its weighted diversification rate using linear regression in r. again, we did not correct for phylogeny in

accurate regardless of the relationship between clade age and richness (kozak & wiens, 2016) and regardless of whether rates are homogeneous or heterogeneous within the clade (meyer & wiens, 2018). furthermore, the net diversification rate depends only on the age and richness of clades. thus, a young clade with many species will have a high net diversification rate (and an older clade with few species a low rate), regardless of variation in instantaneous rates within the clade (either over time or among subclades). simulations that include diversity dependence (i.e. rates changing over time) show that diversification rates still strongly predict richness

and regardless of whether rates are homogeneous or heterogeneous within the clade (meyer & wiens, 2018). furthermore, the net diversification rate depends only on the age and richness of clades. thus, a young clade with many species will have a high net diversification rate (and an older clade with few species a low rate), regardless of variation in instantaneous rates within the clade (either over time or among subclades). simulations that include diversity dependence (i.e. rates changing over time) show that diversification rates still strongly predict richness patterns (pontarp & wiens, 2017). nevertheless, the method-ofmoments estimator does not estimate separate speciation

speciation extinction (musse) model implemented in the r package diversitree (fitzjohn, 2012). we did not use geosse (goldberg et al., 2011) here because it only allows analysis of two regions, and we analyzed six here. we used one state to represent each region. australasia was excluded because it lacks endemic species. species present in more than one region were coded based on their inferred ancestral region. three parameters were included for each region (state): a speciation rate k, an extinction rate l, and a transition rate between different regions q. we compared the relative fit of the data with eight likelihood

extinction (musse) model implemented in the r package diversitree (fitzjohn, 2012). we did not use geosse (goldberg et al., 2011) here because it only allows analysis of two regions, and we analyzed six here. we used one state to represent each region. australasia was excluded because it lacks endemic species. species present in more than one region were coded based on their inferred ancestral region. three parameters were included for each region (state): a speciation rate k, an extinction rate l, and a transition rate between different regions q. we compared the relative fit of the data with eight likelihood models,

q free. we compared their relative fit using the size-corrected aic (aicc) (burnham & anderson, 2002). the chi-squared distribution (chisq) and their significance (pr) were calculated by comparison with the minimal model. we accounted for incomplete sampling in our tree for each region using a sampling fraction for each region (species in our tree in each region divided by the total number known in each region; table 2). we estimated the posterior density distribution of parameters for the best-fitting model with bayesian markov chain monte carlo analyses (100 000 steps) to estimate speciation, extinction, and dispersal rates. we also used these

the posterior density distribution of parameters for the best-fitting model with bayesian markov chain monte carlo analyses (100 000 steps) to estimate speciation, extinction, and dispersal rates. we also used these estimated speciation and extinction rates from musse as an alternative approach to estimating the overall table 2 summary of species richness and diversification rate (based on musse analyses) of lysimachieae in each region first colonization time 9 diversification rate diversification rate regions east asia (a) europe (b) africa (c) north america (d) central - south america (e) hawaii (f) australasia (g) species richness (sampled species) sampling fraction (%) musse model

to estimating the overall table 2 summary of species richness and diversification rate (based on musse analyses) of lysimachieae in each region first colonization time 9 diversification rate diversification rate regions east asia (a) europe (b) africa (c) north america (d) central - south america (e) hawaii (f) australasia (g) species richness (sampled species) sampling fraction (%) musse model 4 musse model 6 musse model 4 musse model 6 171 (74) 22 (18) 28 (17) 19 (13) 10 (8) 43.27 81.82 60.71 68.42 80.00 0.2866 0.0851 0.1987 0.1508 0.0678 0.2704 0.0656 0.1958 0.0832 0.0578 6.3793 2.4620 1.9000 4.0454 0.6749 6.0191 1.8960

overall table 2 summary of species richness and diversification rate (based on musse analyses) of lysimachieae in each region first colonization time 9 diversification rate diversification rate regions east asia (a) europe (b) africa (c) north america (d) central - south america (e) hawaii (f) australasia (g) species richness (sampled species) sampling fraction (%) musse model 4 musse model 6 musse model 4 musse model 6 171 (74) 22 (18) 28 (17) 19 (13) 10 (8) 43.27 81.82 60.71 68.42 80.00 0.2866 0.0851 0.1987 0.1508 0.0678 0.2704 0.0656 0.1958 0.0832 0.0578 6.3793 2.4620 1.9000 4.0454 0.6749 6.0191 1.8960 1.8722 2.2309 0.5751

2.2309 0.5751 18 (14) 6 (6) 77.78 100.00 0.8073 0 0.6691 0 1.5022 0 1.2450 0 ó 2018 the authors new phytologist ó 2018 new phytologist trust new phytologist (2018) 219: 436 - 448 www.newphytologist.com new phytologist 442 research diversification rate for each region. importantly, this did not require assigning species to clades, nor calculating a weighted index among clades within a region. australasia was assigned zero (see earlier). although some concerns have been raised about sse methods (e.g. maddison & fitzjohn, 2015; rabosky & goldberg, 2015), we show that our diversification rate estimates from musse are similar to those from

the tree with the intermediate root age and yule speciation tree prior. we compared the fit of different constraints on the maximum number of ancestral regions for each node using likelihood ratio, v2, and aic tests (table s15). the best-fitting model in the dec analysis (maximum two areas per ancestral species; table s15) was used in the main analyses. biogeographic inferences using a highly constrained transition matrix (table s14) were basically identical to those without these assumptions (table s12). however, the model utilizing these constraints had somewhat better fit (loge l = 123.774 vs loge l = 126.619; tables s12, s14).

= 126.619; tables s12, s14). biogeographic inferences are illustrated in fig. 1. these analyses suggest that the ancestor of lysimachieae most likely occurred in both north america and europe, and then dispersed to other regions (i.e. africa, east asia and central - south america). phylogenetic relationships time, diversification rate, and species richness the combined matrix consisted of 13 462 aligned base pairs for 131 species (including five outgroups). the likelihood, bayesian, and beast topologies were similar and are summarized in figs 1 and s1 - s3. we identified 11 major clades in lysimachieae with strong support (fig. 1), which were concordant

suggest that the ancestor of lysimachieae most likely occurred in both north america and europe, and then dispersed to other regions (i.e. africa, east asia and central - south america). phylogenetic relationships time, diversification rate, and species richness the combined matrix consisted of 13 462 aligned base pairs for 131 species (including five outgroups). the likelihood, bayesian, and beast topologies were similar and are summarized in figs 1 and s1 - s3. we identified 11 major clades in lysimachieae with strong support (fig. 1), which were concordant with major clades found in other recent studies (hao et al., 2004; anderberg et

i - vi ('glaux' and all lyismachia except the endemic lysimachia group in the new world), but with only moderate support. results here are based on the tree using the intermediate root age and yule speciation tree prior. other trees yielded similar results (table s16). for each region, the current species richness, oldest colonization time, and summed colonization times are listed in table 1. there was no significant relationship between current species richness and the oldest colonization time among regions (r2 = 0.121, p = 0.446; fig. 2a), nor between richness and summed colonization ages (r2 = 0.022, p = 0.750;

results here are based on the tree using the intermediate root age and yule speciation tree prior. other trees yielded similar results (table s16). for each region, the current species richness, oldest colonization time, and summed colonization times are listed in table 1. there was no significant relationship between current species richness and the oldest colonization time among regions (r2 = 0.121, p = 0.446; fig. 2a), nor between richness and summed colonization ages (r2 = 0.022, p = 0.750; fig. 2b). importantly, the ancestral regions (europe and north america) for lysimachieae have only 22 and 19 species respectively (table 1).

relationship between current species richness and the oldest colonization time among regions (r2 = 0.121, p = 0.446; fig. 2a), nor between richness and summed colonization ages (r2 = 0.022, p = 0.750; fig. 2b). importantly, the ancestral regions (europe and north america) for lysimachieae have only 22 and 19 species respectively (table 1). other regions (africa, hawaii, south america, and australia) have younger colonization times and low species richness. by contrast, east asia has a relatively old colonization time and the highest species richness (table 1). therefore, europe and north america act as outliers in these analyses, weakening any positive

fig. 2a), nor between richness and summed colonization ages (r2 = 0.022, p = 0.750; fig. 2b). importantly, the ancestral regions (europe and north america) for lysimachieae have only 22 and 19 species respectively (table 1). other regions (africa, hawaii, south america, and australia) have younger colonization times and low species richness. by contrast, east asia has a relatively old colonization time and the highest species richness (table 1). therefore, europe and north america act as outliers in these analyses, weakening any positive relationship between age and richness. the time - richness relationship is relatively strong when these two areas are

fig. 2b). importantly, the ancestral regions (europe and north america) for lysimachieae have only 22 and 19 species respectively (table 1). other regions (africa, hawaii, south america, and australia) have younger colonization times and low species richness. by contrast, east asia has a relatively old colonization time and the highest species richness (table 1). therefore, europe and north america act as outliers in these analyses, weakening any positive relationship between age and richness. the time - richness relationship is relatively strong when these two areas are removed (r2 = 0.809, p = 0.038, using oldest colonization times), but not significant using

age and richness. the time - richness relationship is relatively strong when these two areas are removed (r2 = 0.809, p = 0.038, using oldest colonization times), but not significant using summed colonization ages (r2 = 0.637, p = 0.106; fig. s9). thus, time seems to contribute to the high species richness of east asia, despite the lack of a time - richness relationship among regions overall. the relationship between the weighted diversification rate of each region (stem-group estimator, ɛ = 0.45) and its regional species richness was also nonsignificant (r2 = 0.207, p = 0.306, fig. 2c). the lack of

= 0.637, p = 0.106; fig. s9). thus, time seems to contribute to the high species richness of east asia, despite the lack of a time - richness relationship among regions overall. the relationship between the weighted diversification rate of each region (stem-group estimator, ɛ = 0.45) and its regional species richness was also nonsignificant (r2 = 0.207, p = 0.306, fig. 2c). the lack of a significant relationship may be explained by two regions (hawaii and africa) having low richness but divergence-time estimation similar time estimates and topologies were obtained using three different calibration ages and different tree priors for

estimation similar time estimates and topologies were obtained using three different calibration ages and different tree priors for the stem age of lysimachieae (figs 1, s2 - s8; tables s8, s14). we also tested the effect of incomplete sampling on divergence-time estimation using a tree with only 10% of all species included (26 species). new phytologist (2018) 219: 436 - 448 www.newphytologist.com ó 2018 the authors new phytologist ó 2018 new phytologist trust new phytologist research 443 r2 = 0.022 p = 0.750 50 100 species richness r2 = 0.121 p = 0.446 100 150 (b) 0 0 50 species richness

estimates and topologies were obtained using three different calibration ages and different tree priors for the stem age of lysimachieae (figs 1, s2 - s8; tables s8, s14). we also tested the effect of incomplete sampling on divergence-time estimation using a tree with only 10% of all species included (26 species). new phytologist (2018) 219: 436 - 448 www.newphytologist.com ó 2018 the authors new phytologist ó 2018 new phytologist trust new phytologist research 443 r2 = 0.022 p = 0.750 50 100 species richness r2 = 0.121 p = 0.446 100 150 (b) 0 0 50 species richness 150 (a) 5

effect of incomplete sampling on divergence-time estimation using a tree with only 10% of all species included (26 species). new phytologist (2018) 219: 436 - 448 www.newphytologist.com ó 2018 the authors new phytologist ó 2018 new phytologist trust new phytologist research 443 r2 = 0.022 p = 0.750 50 100 species richness r2 = 0.121 p = 0.446 100 150 (b) 0 0 50 species richness 150 (a) 5 10 15 20 25 30 10 first colonization time (ma) 40 50 150 0 0 50 50 r2 = 0.034 p = 0.693 100 species richness r2 = 0.207 p = 0.306

all species included (26 species). new phytologist (2018) 219: 436 - 448 www.newphytologist.com ó 2018 the authors new phytologist ó 2018 new phytologist trust new phytologist research 443 r2 = 0.022 p = 0.750 50 100 species richness r2 = 0.121 p = 0.446 100 150 (b) 0 0 50 species richness 150 (a) 5 10 15 20 25 30 10 first colonization time (ma) 40 50 150 0 0 50 50 r2 = 0.034 p = 0.693 100 species richness r2 = 0.207 p = 0.306 100 150 30 (d) (c) species richness 20 summed ages (ma) 0.00 0.05 0.10

= 0.022 p = 0.750 50 100 species richness r2 = 0.121 p = 0.446 100 150 (b) 0 0 50 species richness 150 (a) 5 10 15 20 25 30 10 first colonization time (ma) 40 50 150 0 0 50 50 r2 = 0.034 p = 0.693 100 species richness r2 = 0.207 p = 0.306 100 150 30 (d) (c) species richness 20 summed ages (ma) 0.00 0.05 0.10 0.15 0.20 0.25 net diversification rate (ε = 0.45) 0.00 0.05 0.10 relatively high diversification rates. for example, hawaii has only 18 lysimachieae species (mainly lysimachia subgenus lysimachiopsis) but

= 0.446 100 150 (b) 0 0 50 species richness 150 (a) 5 10 15 20 25 30 10 first colonization time (ma) 40 50 150 0 0 50 50 r2 = 0.034 p = 0.693 100 species richness r2 = 0.207 p = 0.306 100 150 30 (d) (c) species richness 20 summed ages (ma) 0.00 0.05 0.10 0.15 0.20 0.25 net diversification rate (ε = 0.45) 0.00 0.05 0.10 relatively high diversification rates. for example, hawaii has only 18 lysimachieae species (mainly lysimachia subgenus lysimachiopsis) but a high diversification rate (0.2544 species/ myr). the rate and richness are similar

0.034 p = 0.693 100 species richness r2 = 0.207 p = 0.306 100 150 30 (d) (c) species richness 20 summed ages (ma) 0.00 0.05 0.10 0.15 0.20 0.25 net diversification rate (ε = 0.45) 0.00 0.05 0.10 relatively high diversification rates. for example, hawaii has only 18 lysimachieae species (mainly lysimachia subgenus lysimachiopsis) but a high diversification rate (0.2544 species/ myr). the rate and richness are similar in the african clade (0.2322, table 1). a strong relationship between regional richness and diversification rate was supported after removing hawaii and africa (r2 = 0.960, p = 0.003; fig. s9). east

= 0.306 100 150 30 (d) (c) species richness 20 summed ages (ma) 0.00 0.05 0.10 0.15 0.20 0.25 net diversification rate (ε = 0.45) 0.00 0.05 0.10 relatively high diversification rates. for example, hawaii has only 18 lysimachieae species (mainly lysimachia subgenus lysimachiopsis) but a high diversification rate (0.2544 species/ myr). the rate and richness are similar in the african clade (0.2322, table 1). a strong relationship between regional richness and diversification rate was supported after removing hawaii and africa (r2 = 0.960, p = 0.003; fig. s9). east asia has a relatively high diversification rate (0.2147), which seemingly helps

and richness are similar in the african clade (0.2322, table 1). a strong relationship between regional richness and diversification rate was supported after removing hawaii and africa (r2 = 0.960, p = 0.003; fig. s9). east asia has a relatively high diversification rate (0.2147), which seemingly helps explain its high species richness. relationships were similar using diversification rates estimated with ɛ of 0 and 0.90 (fig. s10). we also tested the hypothesis that time and diversification rates act together to explain richness patterns among regions. to test this combined effect, we simply multiplied the weighted diversification rate (ɛ = 0.45) for

(fig. s10). we also tested the hypothesis that time and diversification rates act together to explain richness patterns among regions. to test this combined effect, we simply multiplied the weighted diversification rate (ɛ = 0.45) for each region by its oldest colonization time. we found a strong relationship between regional species richness and this combined effect (r2 = 0.855, p = 0.003, ó 2018 the authors new phytologist ó 2018 new phytologist trust 0.15 0.20 0.25 net diversification rate (model6) fig. 2 time and diversification separately fail to explain species richness of lysimachieae across seven regions. colored dots in each graph

its oldest colonization time. we found a strong relationship between regional species richness and this combined effect (r2 = 0.855, p = 0.003, ó 2018 the authors new phytologist ó 2018 new phytologist trust 0.15 0.20 0.25 net diversification rate (model6) fig. 2 time and diversification separately fail to explain species richness of lysimachieae across seven regions. colored dots in each graph correspond to the colored regions as shown in fig. 1. (a) relationship between the time of first colonization and species richness for each region (r2 = 0.121, p = 0.446). (b) relationship between the summed age and species richness

new phytologist trust 0.15 0.20 0.25 net diversification rate (model6) fig. 2 time and diversification separately fail to explain species richness of lysimachieae across seven regions. colored dots in each graph correspond to the colored regions as shown in fig. 1. (a) relationship between the time of first colonization and species richness for each region (r2 = 0.121, p = 0.446). (b) relationship between the summed age and species richness for each region (r2 = 0.022, p = 0.750). (c) relationship between the weighted diversification rate (based on e = 0.45) and species richness for each region (r2 = 0.207, p

explain species richness of lysimachieae across seven regions. colored dots in each graph correspond to the colored regions as shown in fig. 1. (a) relationship between the time of first colonization and species richness for each region (r2 = 0.121, p = 0.446). (b) relationship between the summed age and species richness for each region (r2 = 0.022, p = 0.750). (c) relationship between the weighted diversification rate (based on e = 0.45) and species richness for each region (r2 = 0.207, p = 0.306). (d) relationship between the diversification rate (based on the result of musse: model 6) and species

relationship between the time of first colonization and species richness for each region (r2 = 0.121, p = 0.446). (b) relationship between the summed age and species richness for each region (r2 = 0.022, p = 0.750). (c) relationship between the weighted diversification rate (based on e = 0.45) and species richness for each region (r2 = 0.207, p = 0.306). (d) relationship between the diversification rate (based on the result of musse: model 6) and species richness for each region (r2 = 0.034, p = 0.693). ma, millions of years ago. fig. 3a). this relationship remains strong when east asia

species richness for each region (r2 = 0.022, p = 0.750). (c) relationship between the weighted diversification rate (based on e = 0.45) and species richness for each region (r2 = 0.207, p = 0.306). (d) relationship between the diversification rate (based on the result of musse: model 6) and species richness for each region (r2 = 0.034, p = 0.693). ma, millions of years ago. fig. 3a). this relationship remains strong when east asia is removed (r2 = 0.809, p = 0.006). the best-fitting musse model supported different speciation rates but equal extinction and transition rates among regions (aicc =

anderson, 2002). speciation rates under the two models (best-fitting and second-best) are very similar (r2 = 0.979, p = 0.0002), but extinction rates vary under the second-best model. using this latter model new phytologist (2018) 219: 436 - 448 www.newphytologist.com new phytologist 444 research (b) 1 2 3 4 100 species richness 100 species richness 0 0 50 100 50 0 time by net diversification (ε = 0.45) r2 = 0.712 p = 0.017 150 r = 0.945 p = 0.001 150 r = 0.855 p = 0.003 (c) 2 50 2 0 species richness 150 (a) 0 1 2 3

rates under the two models (best-fitting and second-best) are very similar (r2 = 0.979, p = 0.0002), but extinction rates vary under the second-best model. using this latter model new phytologist (2018) 219: 436 - 448 www.newphytologist.com new phytologist 444 research (b) 1 2 3 4 100 species richness 100 species richness 0 0 50 100 50 0 time by net diversification (ε = 0.45) r2 = 0.712 p = 0.017 150 r = 0.945 p = 0.001 150 r = 0.855 p = 0.003 (c) 2 50 2 0 species richness 150 (a) 0 1 2 3 4 5 6

research (b) 1 2 3 4 100 species richness 100 species richness 0 0 50 100 50 0 time by net diversification (ε = 0.45) r2 = 0.712 p = 0.017 150 r = 0.945 p = 0.001 150 r = 0.855 p = 0.003 (c) 2 50 2 0 species richness 150 (a) 0 1 2 3 4 5 6 time by net diversification (model6) 0 1 2 3 4 5 6 time by net diversification (model4) fig. 3 time and diversification rates together explain species richness of lysimachieae across seven regions. colored dots in each graph show the colored

= 0.001 150 r = 0.855 p = 0.003 (c) 2 50 2 0 species richness 150 (a) 0 1 2 3 4 5 6 time by net diversification (model6) 0 1 2 3 4 5 6 time by net diversification (model4) fig. 3 time and diversification rates together explain species richness of lysimachieae across seven regions. colored dots in each graph show the colored regions in fig. 1. (a) strong relationship between the combined effect of time (first colonization of each region) and weighted net diversification rate (based on e = 0.45) and species richness of regions (r2 = 0.855,

time and diversification rates together explain species richness of lysimachieae across seven regions. colored dots in each graph show the colored regions in fig. 1. (a) strong relationship between the combined effect of time (first colonization of each region) and weighted net diversification rate (based on e = 0.45) and species richness of regions (r2 = 0.855, p = 0.003). (b) strong relationship between the combined effect of time (first colonization of each region) and diversification rate (based on musse: model 6) and species richness of regions (r2 = 0.945, p = 0.001). (c) strong relationship between the combined effect of

time (first colonization of each region) and weighted net diversification rate (based on e = 0.45) and species richness of regions (r2 = 0.855, p = 0.003). (b) strong relationship between the combined effect of time (first colonization of each region) and diversification rate (based on musse: model 6) and species richness of regions (r2 = 0.945, p = 0.001). (c) strong relationship between the combined effect of time (first colonization of each region) and diversification rate (based on musse: model 4) and species richness of regions (r2 = 0.712, p = 0.017). (table s19), europe (0.1515) and north america (0.2665)

effect of time (first colonization of each region) and diversification rate (based on musse: model 6) and species richness of regions (r2 = 0.945, p = 0.001). (c) strong relationship between the combined effect of time (first colonization of each region) and diversification rate (based on musse: model 4) and species richness of regions (r2 = 0.712, p = 0.017). (table s19), europe (0.1515) and north america (0.2665) have higher extinction rates than east asia (0.1228) and africa (0.0794). musse analyses with alternative trees showed model 4 (best-fitting model) and model 6 (second-best) had a similar fit (table s20). net diversification

net diversification rate using musse (0.6691) relative to the weighted net diversification rate (0.2544; tables 1, 2). when hawaii is removed, the relationship is very strong (r2 = 0.863, p = 0.022). there was no significant relationship between the net diversification rate for each region from musse and each region's species richness (r2 = 0.034, p = 0.693, fig. 2d). results were similar under the model in which only speciation rates differed (r2 = 0.001, p = 0.944). however, there was a significant relationship between the species richness of each region and the combined index of time and musse-estimated diversification rates

relationship between the net diversification rate for each region from musse and each region's species richness (r2 = 0.034, p = 0.693, fig. 2d). results were similar under the model in which only speciation rates differed (r2 = 0.001, p = 0.944). however, there was a significant relationship between the species richness of each region and the combined index of time and musse-estimated diversification rates (under the model with different speciation and extinction rates; r2 = 0.945, p = 0.001, fig. 3b). results were significant but weaker with east asia removed (r2 = 0.767, p = 0.022). results were also similar

= 0.001, fig. 3b). results were significant but weaker with east asia removed (r2 = 0.767, p = 0.022). results were also similar but somewhat weaker (r2 = 0.712, p = 0.017) using the model with only speciation rates that differ among regions (fig. 3c). discussion what explains the high species richness of plants in east asia relative to other temperate regions? this is a long-standing question, new phytologist (2018) 219: 436 - 448 www.newphytologist.com but no studies have tested both time and diversification rates as potential explanations. our results for lysimachieae suggest that this pattern is explained by the combination

that follow, we discuss the importance of this combined explanation. we then discuss future work, which should address two major questions: (1) what ecological factors explain the evolutionary and biogeographic patterns found? (2) do results for lysimachieae apply to other plant groups in east asia? our results suggest that high species richness in east asia in lysimachieae is explained by a combination of time and relatively high diversification rates. neither variable alone explained richness patterns among regions. specifically, even though east asia has been colonized for a relatively long period of time relative to most other regions, europe and north america

by a combination of time and relatively high diversification rates. neither variable alone explained richness patterns among regions. specifically, even though east asia has been colonized for a relatively long period of time relative to most other regions, europe and north america have been colonized longer and yet are relatively species poor. therefore, based on time alone, europe and north america should have the highest richness. similarly, even though east asia has a relatively high weighted net diversification rate (much higher than europe or north america), both hawaii and africa have higher rates. however, hawaii and africa were both colonized relatively

highest richness. similarly, even though east asia has a relatively high weighted net diversification rate (much higher than europe or north america), both hawaii and africa have higher rates. however, hawaii and africa were both colonized relatively recently, and therefore their high net diversification rates have not yet yielded high species richness. overall, neither time nor diversification rates explained richness patterns among regions. yet, a weighted measure that incorporates both time and diversification rates provides a strong explanation for richness patterns among ó 2018 the authors new phytologist ó 2018 new phytologist trust new phytologist regions, especially the high richness of

of time and diversification rates are not an inevitable explanation for richness patterns. specifically, if we were attempting to explain richness patterns among clades only (e.g. genera), then it is true that multiplying the age of the clade by its net diversification rate should yield that clade's observed number of species. however, this is not so straightforward for spatial richness patterns among regions. for example, the weighted net diversification rate for the region is based on the age of each named clade (not how long they have been present in each region), weighted by the number of clades in each region,

not so straightforward for spatial richness patterns among regions. for example, the weighted net diversification rate for the region is based on the age of each named clade (not how long they have been present in each region), weighted by the number of clades in each region, the number of species from each clade in the region, and the global diversification rate of each clade. conversely, the age of a clade in a region (i.e. oldest colonization time) is not necessarily the same as the age used to estimate the diversification rate. therefore, we strongly emphasize that the strong relationship observed

clade in the region, and the global diversification rate of each clade. conversely, the age of a clade in a region (i.e. oldest colonization time) is not necessarily the same as the age used to estimate the diversification rate. therefore, we strongly emphasize that the strong relationship observed here between species richness and our combined measure of colonization time and diversification rate is not inevitable. our results help identify the combination of evolutionary and biogeographic processes (i.e. speciation, extinction, colonization time) that explain high species richness in east asia (axelrod et al., 1996; xiang et al., 2004). we suggest that a

to estimate the diversification rate. therefore, we strongly emphasize that the strong relationship observed here between species richness and our combined measure of colonization time and diversification rate is not inevitable. our results help identify the combination of evolutionary and biogeographic processes (i.e. speciation, extinction, colonization time) that explain high species richness in east asia (axelrod et al., 1996; xiang et al., 2004). we suggest that a high priority for future research should be to identify the specific ecological factors that act on these processes (e.g. guo et al., 1998; qian & ricklefs, 2000; qian, 2002; wang et al., 2009, 2011).

xiang et al., 2004). we suggest that a high priority for future research should be to identify the specific ecological factors that act on these processes (e.g. guo et al., 1998; qian & ricklefs, 2000; qian, 2002; wang et al., 2009, 2011). for example, our results suggest that patterns of species richness in lysimachieae are explained by relatively high speciation rates in east asia, the highest of any region except hawaii (tables s10, s16). therefore, future work should seek to explain what causes high speciation rates in these regions. we suggest that the radiation of lysimachieae across diverse elevations and associated

to have the highest extinction rates of any region, but its very high speciation rate seems to compensate for the dampening effects of this high extinction on net diversification rates. our results also show that area and latitude may not be the most important factors driving richness patterns. in lysimacheae, species richness in east asia is higher than the combined richness of europe, north america, central - south america, and australasia (table 1), despite the much greater area of these combined regions. furthermore, these species-poor regions include low-latitude tropical regions (e.g. africa, central - south america). we suggest that area and

that area and latitude may not be the most important factors driving richness patterns. in lysimacheae, species richness in east asia is higher than the combined richness of europe, north america, central - south america, and australasia (table 1), despite the much greater area of these combined regions. furthermore, these species-poor regions include low-latitude tropical regions (e.g. africa, central - south america). we suggest that area and latitude may only impact richness if there is sufficient time to build up richness in each region. a second major question from our study is whether our results for lysimachieae are representative of all

goal is to understand why east asia has such high plant richness overall (e.g. qian & ricklefs, 2000). we addressed this question using lysimachieae as a model system. however, we recognize that this tribe may not be fully representative of all groups of plants that contribute to high east asian species richness. that being said, we do not know of any potentially relevant way in which lysimachieae is different from other species-rich plant groups in the region. for example, the tribe has been considered a typical group with high species richness in east asia (chen & hu, 1979; li, 1996; st

question using lysimachieae as a model system. however, we recognize that this tribe may not be fully representative of all groups of plants that contribute to high east asian species richness. that being said, we do not know of any potentially relevant way in which lysimachieae is different from other species-rich plant groups in the region. for example, the tribe has been considered a typical group with high species richness in east asia (chen & hu, 1979; li, 1996; st ahl & anderberg, 2004). in addition, the tribe has many characteristics that are widespread in other large chinese plant groups (e.g.

of all groups of plants that contribute to high east asian species richness. that being said, we do not know of any potentially relevant way in which lysimachieae is different from other species-rich plant groups in the region. for example, the tribe has been considered a typical group with high species richness in east asia (chen & hu, 1979; li, 1996; st ahl & anderberg, 2004). in addition, the tribe has many characteristics that are widespread in other large chinese plant groups (e.g. primula, gentiana, saxifraga, and pedicularis; li, 1996). these characteristics include herbaceous growth form, insect pollination, no long-distance dispersal

and pedicularis; li, 1996). these characteristics include herbaceous growth form, insect pollination, no long-distance dispersal ability, and mesic habitat (st ahl & anderberg, 2004). nevertheless, we think that the best way to address whether our results are representative or not is to do similar analyses in other groups with high species richness in east asia, and see whether they have patterns similar to those found here for lysimachieae. in summary, our study helps reveal the underlying causes of high plant species richness in east asia. in the tribe lysimachieae, this pattern is caused by both relatively long occurrence in the region

way to address whether our results are representative or not is to do similar analyses in other groups with high species richness in east asia, and see whether they have patterns similar to those found here for lysimachieae. in summary, our study helps reveal the underlying causes of high plant species richness in east asia. in the tribe lysimachieae, this pattern is caused by both relatively long occurrence in the region (time-for-speciation effect) and relatively high net diversification rates in the clades that occur there. importantly, neither variable alone fully explains this pattern. more broadly, our results demonstrate the importance of

high net diversification rates in the clades that occur there. importantly, neither variable alone fully explains this pattern. more broadly, our results demonstrate the importance of testing the combined effects of time and diversification, and the need to test both hypotheses (and possibly their interaction) to explain large-scale patterns of species richness. acknowledgements we thank the plant dna bank in korea, herbario nacional de mexico, and ki-joong kim, qing liu, and lian-ming gao for samples. we thank chi-ming hu for species identification. we thank cristian roman-palacios, shea lambert, elizabeth miller, and tania hernandez-hernandez for help with data analyses. new phytologist (2018)

of time and diversification, and the need to test both hypotheses (and possibly their interaction) to explain large-scale patterns of species richness. acknowledgements we thank the plant dna bank in korea, herbario nacional de mexico, and ki-joong kim, qing liu, and lian-ming gao for samples. we thank chi-ming hu for species identification. we thank cristian roman-palacios, shea lambert, elizabeth miller, and tania hernandez-hernandez for help with data analyses. new phytologist (2018) 219: 436 - 448 www.newphytologist.com 446 research this research was supported by the national natural science foundation of china (31570222), the national key basic research program of china (2014cb954100), and

### NA
followed by polyploidization. \001 comparing multiple stochastic diversification models that include complex trait interactions alongside hidden states enhances our understanding of the macroevolutionary patterns in plant phylogenies. among life history traits, reproductive characters that determine mating patterns are perhaps the most influential in governing macroevolution. barrett et al. (1996) introduction species accumulate across the tree of life at different rates. one possible explanation for this phenomenon is that various traits differentially affect rates of diversification. dramatic increases in phylogenetic and phenotypic data, along with methodological advances, have greatly accelerated the search for traits that influence diversification. nevertheless, identifying focal traits associated

2010). initial mutant individuals with pollen containing doubled haploid genomes are consequently capable of self-fertilisation, with exceedingly few exceptions (hauck et al., 2002; nunes et al., 2006). rnase-based si is regarded as ancestral in eudicots (igi\001c & kohn, 2001; steinbachs & holsinger, 2002), and it is expressed in all si species of solanaceae examined to date. the absence of si polyploids in this family yields a strong correlation between ploidy and breeding system (robertson et al., 2011). we address two macroevolutionary questions about the correlated evolution of ploidy and breeding system. first, we investigate their joint influence on rates of speciation

contrasted against the ploidy dataset from robertson et al. (2011), original", 
"s therein, and against ploidy data in the c-value dna dataset from bennett & leitch (2012). by comparing three different sources of information, we were able to code taxa as diploid, d or polyploid, p. for the majority of species, ploidy was assigned according to information from the original publications included in the cvalue dna dataset (bennett & leitch, 2012). for taxa without ploidy information but with information about chromosome number, we assigned ploidy based on the multiplicity of chromosomes within the genus/family, or based on si/sc classification. for example,

did not have information about ploidy level, but it has 2n = 24 chromosomes, and x = 12 is the base chromosome number of the genus solanum (olmstead & bohs, 2007), so we assigned s. betaceum as diploid. additionally, because of the absence of si polyploids (explained above and below), species known to be si could be scored as diploid. species with more than one ploidy level were assigned the most frequent ploidy level recorded or the smallest ploidy in case of frequency ties. breeding system states were scored as self-incompatible, i or self-compatible, c, based on results curated from the

2n = 24 chromosomes, and x = 12 is the base chromosome number of the genus solanum (olmstead & bohs, 2007), so we assigned s. betaceum as diploid. additionally, because of the absence of si polyploids (explained above and below), species known to be si could be scored as diploid. species with more than one ploidy level were assigned the most frequent ploidy level recorded or the smallest ploidy in case of frequency ties. breeding system states were scored as self-incompatible, i or self-compatible, c, based on results curated from the literature (as compiled in igi\001c et al., 2006; goldberg et

case of frequency ties. breeding system states were scored as self-incompatible, i or self-compatible, c, based on results curated from the literature (as compiled in igi\001c et al., 2006; goldberg et al., 2010; robertson et al., 2011; goldberg & igi\001c, 2012) and original experimental crosses (bi and eeg unpublished). most species could unambiguously be coded as either i or c (raduski et al., 2012). following previous work, we coded any species with a functional si system as i, even if sc or dioecy was also reported. dioecious species without functional si were coded as c. resolution of taxonomic synonymy followed solanaceae

the literature (as compiled in igi\001c et al., 2006; goldberg et al., 2010; robertson et al., 2011; goldberg & igi\001c, 2012) and original experimental crosses (bi and eeg unpublished). most species could unambiguously be coded as either i or c (raduski et al., 2012). following previous work, we coded any species with a functional si system as i, even if sc or dioecy was also reported. dioecious species without functional si were coded as c. resolution of taxonomic synonymy followed solanaceae source (pbi solanum project, 2012). hybrids and cultivars were excluded because ploidy and breeding system can be affected by artificial

goldberg & igi\001c, 2012) and original experimental crosses (bi and eeg unpublished). most species could unambiguously be coded as either i or c (raduski et al., 2012). following previous work, we coded any species with a functional si system as i, even if sc or dioecy was also reported. dioecious species without functional si were coded as c. resolution of taxonomic synonymy followed solanaceae source (pbi solanum project, 2012). hybrids and cultivars were excluded because ploidy and breeding system can be affected by artificial selection during domestication. following the reasoning outlined in robertson et al. (2011), we closely examined the few

without functional si were coded as c. resolution of taxonomic synonymy followed solanaceae source (pbi solanum project, 2012). hybrids and cultivars were excluded because ploidy and breeding system can be affected by artificial selection during domestication. following the reasoning outlined in robertson et al. (2011), we closely examined the few species for which the merged ploidy and breeding system data indicated the presence of self-incompatible polyploids. although si populations frequently contain some sc individuals, and diploid populations frequently contain some polyploid individuals, in no case did we find convincing data for a naturally occurring si polyploid population (discussed in robertson et

the functional explanation for polyploidy disabling gametophytic si systems with nonself recognition (see the introduction), we consider only three observed character states: self-incompatible diploids (id), self-compatible diploids (cd) and self-compatible polyploids (cp). matching our character state data to the largest time-calibrated phylogeny of solanaceae (s€arkinen et al., 2013) yielded 651 species with ploidy and/or breeding system information on the tree. of these, 368 had information for both states. the number of species in each combination of states is summarised in fig. 1(a) and in fig. 1(a) and in fig. s1. we retained all 651 species in new phytologist (2019) 224: 1252

states: self-incompatible diploids (id), self-compatible diploids (cd) and self-compatible polyploids (cp). matching our character state data to the largest time-calibrated phylogeny of solanaceae (s€arkinen et al., 2013) yielded 651 species with ploidy and/or breeding system information on the tree. of these, 368 had information for both states. the number of species in each combination of states is summarised in fig. 1(a) and in fig. 1(a) and in fig. s1. we retained all 651 species in new phytologist (2019) 224: 1252 - 1265 www.newphytologist.com each of the analyses below because pruning away tips lacking breeding system in the ploidy-only analyses (and vice

(s€arkinen et al., 2013) yielded 651 species with ploidy and/or breeding system information on the tree. of these, 368 had information for both states. the number of species in each combination of states is summarised in fig. 1(a) and in fig. 1(a) and in fig. s1. we retained all 651 species in new phytologist (2019) 224: 1252 - 1265 www.newphytologist.com each of the analyses below because pruning away tips lacking breeding system in the ploidy-only analyses (and vice versa) would discard data that could inform the diversification models. in total, 372 taxa without any information about breeding system or ploidy were

170 157 70 85 +23 ploidy-only models (m1 - m10) id i? ?d cd c? cp ?p breeding systemonly models (m11 - m15) id i? ?d cd c? cp ?p ploidy and breeding system models and lumped models m16 - m29 id i? ?d cd c? cp ?p number of species (b) = fig. 1 character states used in the models. (a) each species retained on the phylogeny belonged to one of five possible categories, depending on whether ploidy and/or breeding system were known. number of species in each category is indicated; for example, 70 species are selfcompatible with unknown ploidy.

cd c? cp ?p breeding systemonly models (m11 - m15) id i? ?d cd c? cp ?p ploidy and breeding system models and lumped models m16 - m29 id i? ?d cd c? cp ?p number of species (b) = fig. 1 character states used in the models. (a) each species retained on the phylogeny belonged to one of five possible categories, depending on whether ploidy and/or breeding system were known. number of species in each category is indicated; for example, 70 species are selfcompatible with unknown ploidy. character state abbreviations are: i for selfincompatible, c for selfcompatible, d for diploid,

lumped models m16 - m29 id i? ?d cd c? cp ?p number of species (b) = fig. 1 character states used in the models. (a) each species retained on the phylogeny belonged to one of five possible categories, depending on whether ploidy and/or breeding system were known. number of species in each category is indicated; for example, 70 species are selfcompatible with unknown ploidy. character state abbreviations are: i for selfincompatible, c for selfcompatible, d for diploid, p for polyploid, ? for unknown. because polyploidization breaks this form of self-incompatibility, selfincompatible species with unobserved ploidy (i ?) are taken to

c? cp ?p number of species (b) = fig. 1 character states used in the models. (a) each species retained on the phylogeny belonged to one of five possible categories, depending on whether ploidy and/or breeding system were known. number of species in each category is indicated; for example, 70 species are selfcompatible with unknown ploidy. character state abbreviations are: i for selfincompatible, c for selfcompatible, d for diploid, p for polyploid, ? for unknown. because polyploidization breaks this form of self-incompatibility, selfincompatible species with unobserved ploidy (i ?) are taken to be diploid (id), and polyploid species with unobserved breeding

ploidy and/or breeding system were known. number of species in each category is indicated; for example, 70 species are selfcompatible with unknown ploidy. character state abbreviations are: i for selfincompatible, c for selfcompatible, d for diploid, p for polyploid, ? for unknown. because polyploidization breaks this form of self-incompatibility, selfincompatible species with unobserved ploidy (i ?) are taken to be diploid (id), and polyploid species with unobserved breeding system (?p) are taken to be sc (cp). (b) category groupings into states for each model class. in the ploidy-only models (m1 - m10), states are coded as d & p when uncertain/consistent

for example, 70 species are selfcompatible with unknown ploidy. character state abbreviations are: i for selfincompatible, c for selfcompatible, d for diploid, p for polyploid, ? for unknown. because polyploidization breaks this form of self-incompatibility, selfincompatible species with unobserved ploidy (i ?) are taken to be diploid (id), and polyploid species with unobserved breeding system (?p) are taken to be sc (cp). (b) category groupings into states for each model class. in the ploidy-only models (m1 - m10), states are coded as d & p when uncertain/consistent with either state; in the breeding system-only models (m11 - m15) such states are

- m10), states are coded as d & p when uncertain/consistent with either state; in the breeding system-only models (m11 - m15) such states are coded as c; in the ploidy and breeding system models (m16 - m29), they are coded as cd & cp. in the hiddentrait models, all species could take on either of two 'hidden' character states. two species, lycium californicum and solanum bulbocastanum, are simultaneously id and cp, and by adding them the sample adds to the total of 651 taxa used for analyses. ó 2019 the authors new phytologist ó 2019 new phytologist trust new phytologist

with either state; in the breeding system-only models (m11 - m15) such states are coded as c; in the ploidy and breeding system models (m16 - m29), they are coded as cd & cp. in the hiddentrait models, all species could take on either of two 'hidden' character states. two species, lycium californicum and solanum bulbocastanum, are simultaneously id and cp, and by adding them the sample adds to the total of 651 taxa used for analyses. ó 2019 the authors new phytologist ó 2019 new phytologist trust new phytologist research 1255 rnase-based gametophytic self-incompatibility (shared even with other euasterid families;

are simultaneously id and cp, and by adding them the sample adds to the total of 651 taxa used for analyses. ó 2019 the authors new phytologist ó 2019 new phytologist trust new phytologist research 1255 rnase-based gametophytic self-incompatibility (shared even with other euasterid families; ramanauskas & igi\001c, 2017). furthermore, species that are distantly related within this family carry closely-related alleles, with deep trans-specific polymorphism at the locus that controls the si response (ioerger et al., 1990; igi\001c et al., 2006). thus, there is strong evidence in solanaceae that the i state is ancestral in the family, and that the si

the family, and that the si mechanism was not regained. for all breeding system models, we estimated a transition rate from i to c but not the reverse (qci = 0). ploidy and diversification we first employed a model (labeled m1), previously used by mayrose et al. (2011), with each species classified as diploid (d) or polyploid (p). although this model can be powerful in studies of trait evolution, it is prone to incorrectly reporting that a trait is associated with diversification differences (maddison & fitzjohn, 2015; rabosky & goldberg, 2015). we therefore define several models that incorporate additional forms of

system and diversification ploidy and breeding system might influence lineage diversification individually, but these two traits also have an intricate association (discussed in the introduction). therefore, we considered several multistate models that investigate the contribution of both traits and the allowable transitions between them. the simplest model (m16) classifies each species as either si diploid (id), sc diploid (cd) or sc polyploid (cp); recall that si polyploids do not occur. each of these states may again be associated with different rates of speciation and extinction, and the allowable transitions are loss of si within the diploid state (from id to cd),

to caetano et al., 2018; huang et al., 2018). we also fit the analogous models but allowing for diploidization (m21 - m25). breeding system and diversification we propose five breeding system models following the same logic as the ploidy models above. under the simplest breeding system and diversification model (m11), species are classified as self-incompatible (i) or self-compatible (c). this is the same model as in the analysis presented in goldberg et al. (2010) but with an updated phylogeny (s€arkinen et al., 2013) and a larger aggregated dataset. we then add models to allow diversification to be influenced by only a

by only a hidden trait (m12), or by both breeding system and a hidden trait (m13 - m15, with varying degrees of complexity in the hidden trait transitions table 2). similar models were used by freyman & h€ohna (2019) to study diversification in onagraceae. self-incompatibility is homologous in all solanaceae species in which s-alleles have been cloned and controlled crosses performed. all species sampled to date possess a nonself recognition, lumped models the models described so far allow us to assess the contributions of our two focal characters - ploidy and breeding system - to lineage diversification, but they do not

a hidden trait (m13 - m15, with varying degrees of complexity in the hidden trait transitions table 2). similar models were used by freyman & h€ohna (2019) to study diversification in onagraceae. self-incompatibility is homologous in all solanaceae species in which s-alleles have been cloned and controlled crosses performed. all species sampled to date possess a nonself recognition, lumped models the models described so far allow us to assess the contributions of our two focal characters - ploidy and breeding system - to lineage diversification, but they do not reveal whether it is valuable to include both characters in the analysis.

on a dated phylogeny and models that allow for state-dependent diversification. both of our methods are based on a propagation matrix that describes flow from id to cp, as in robertson et al. (2011). we insert an artificial division in the cp state, so that one substate contains the cp species that arrived via the one-step pathway and the other substate contains the cp species that arrived via the two-step pathway. we consider unidirectional change along each step of the pathway in order to separate them into clear alternatives, and because in this family there is no support for regain of

methods are based on a propagation matrix that describes flow from id to cp, as in robertson et al. (2011). we insert an artificial division in the cp state, so that one substate contains the cp species that arrived via the one-step pathway and the other substate contains the cp species that arrived via the two-step pathway. we consider unidirectional change along each step of the pathway in order to separate them into clear alternatives, and because in this family there is no support for regain of si, and no strong support for diploidization (as discussed in key question (5) in

each of the 29 models were coded as graphical models and bayesian statistical inference was performed with revbayes (h€ ohna et al., 2016). scripts for analyses and key results are available in dryad (doi: 10.5061/dryad.tb7055f). we accounted for incomplete sampling in all analyses by setting the probability of sampling a species at the present to 651/3000 (using the method of fitzjohn et al., 2009) since the solanaceae family has c. 3000 species (pbi solanum project, 2012). for all models, we assumed that speciation and extinction parameters had log-normal prior distributions with means equal to the expected net diversification rate (number of

al., 2016). scripts for analyses and key results are available in dryad (doi: 10.5061/dryad.tb7055f). we accounted for incomplete sampling in all analyses by setting the probability of sampling a species at the present to 651/3000 (using the method of fitzjohn et al., 2009) since the solanaceae family has c. 3000 species (pbi solanum project, 2012). for all models, we assumed that speciation and extinction parameters had log-normal prior distributions with means equal to the expected net diversification rate (number of taxa/(2 9 root age)) and standard deviation 0.5. priors for parameters defining trait changes were assumed to be gamma distributed with

the role of ploidy in net diversification is unclear, with marginal support for diploids having higher diversification rates, but rate differences perhaps better explained by another factor. breeding system only when considering breeding system alone (m11; fig. 2c), we found a larger net diversification rate for si than for sc species, in agreement with goldberg et al. (2010). when a hidden state is included, the large net diversification rate difference persists for one hidden state but is diminished for the other (m14; fig. 2d). in the statistical model comparisons, the best two supported models include diversification differences due to both breeding

diversification associated with breeding system persists even when ploidy is included in the model. the reverse is not true: the association of ploidy with net diversification in the simplest ploidy-only model (m1; figs 2a, s18a) appears to be driven by the subset of diploids that are si, while among sc species, net diversification rates for diploids and polyploids are similar. when a hidden state is included, the separation in net diversification rate of id vs cd and cp persists within one hidden state but is reduced in the other (m19, fig. 2f). the same general pattern remains when diploidization is included

elapsed, the one-step pathway is more likely because it only necessitates a single event to reach the sc polyploid state. when more time has elapsed, the two-step pathway is more likely because the rate of loss of si within diploids, qic, is greater than the rate of polyploidization for si species, qi (fig. s7). that is, an id lineage is more likely to begin its path to polyploidy with a transition to cd, but completing this path to cp takes longer. robertson et al. (2011) id one-step two-step cd cd 1.0 with diversification one−step two−step 0.0 0.01 0.2 1 0.4 0.6

id, means that relatively fewer lineages are available to complete the second step of the two-step pathway, ending in cp. consequently, even over long timescales, we find that the two-step pathway contributes less to the formation of polyploids (fig. 4, right panels) when considering diversification as well as transitions. discussion species are composed of vast assemblages of variable traits. many traits are both heritable and possibly affect the propensity of species to perish or multiply (lewontin, 1970). examining the effects of complex trait combinations on lineage diversification, however, remains challenging. focusing first on ploidy and then on breeding system, we found

consequently, even over long timescales, we find that the two-step pathway contributes less to the formation of polyploids (fig. 4, right panels) when considering diversification as well as transitions. discussion species are composed of vast assemblages of variable traits. many traits are both heritable and possibly affect the propensity of species to perish or multiply (lewontin, 1970). examining the effects of complex trait combinations on lineage diversification, however, remains challenging. focusing first on ploidy and then on breeding system, we found that considering each trait in isolation provides an incomplete story. considering them together, and in conjunction with another hidden factor,

in this family found that si may cause higher diversification rates, compared with sc (goldberg et al., 2010; fig. 2c). our analyses that include a hidden trait recover the same pattern, with one important difference. on the background of one hidden state, we recover a net diversification rate for sc species that is positive and greater than the diversification rate of si species on the background of the other hidden state (fig. 2d). therefore, sc may not be a 'dead end' when coupled with some unknown trait combinations or processes not modelled by these two traits. our analyses also reveal that

with sc (goldberg et al., 2010; fig. 2c). our analyses that include a hidden trait recover the same pattern, with one important difference. on the background of one hidden state, we recover a net diversification rate for sc species that is positive and greater than the diversification rate of si species on the background of the other hidden state (fig. 2d). therefore, sc may not be a 'dead end' when coupled with some unknown trait combinations or processes not modelled by these two traits. our analyses also reveal that models of joint evolution of ploidy and breeding system are statistically preferred,

highest net diversification rate is associated with si diploids, while sc diploids have a lower diversification rate that overlaps with the net diversification of sc polyploids (fig. 2e). thus, breeding system appears to provide a relatively better explanation of diversification rate differences, with ploidy providing a secondary effect within sc species. throughout our numerous model comparisons, we find that inclusion of hidden states provides a considerably better fit (table s4). this is consistent with the expectation that many processes, beyond those associated with the focal traits, can affect inference of speciation and extinction. it is, however, unclear exactly which processes are

versa (figs s15, s16). but the strong statistical support for processes not well explained by ploidy and breeding system (table s4) tempts one to interpret the remaining variation as the effect of other measurable traits. for example, our data appear to show a rapidly diversifying australasian clade of mostly sc species within solanum, which suggests that geography may play a role. nevertheless, it is also possible that the addition of hidden states instead explains variation stemming from any of a number of unrelated processes or methodological artifacts, as previously discussed by beaulieu & o'meara (2016). in the absence of additional information,

two pathways, but future work could instead rely on stochastic character mapping to estimate the numbers of each type of transition more directly. macroevolutionary transition rates represent a combination of time spent waiting for individuals with a new character state, and for that new state to become widespread within the species. for our traits, this consists of mutations that break si or generate polyploid individuals, and selective pressures that cause fixation (or loss) of these mutants. estimates of mutation rates are highly uncertain, but the chance of breakdown of si within diploids is perhaps 10\0035 per pollen grain; this includes breakdown

the evolution of gender dimorphism as a different mechanism of inbreeding avoidance (miller & venable, 2000). a further test of this hypothesis would additionally examine the propensity of polyploids generated through either pathway to become dioecious (robertson et al., 2011). second, we might wonder whether the propensity for a polyploid species to diversify depends on whether it arose via the one-step or two-step pathway. this could be tested with a different form of a hidden state model, in which the polyploid state is subdivided into parts, reflecting the arriving path taken to that state. such an estimate of path dependence could

we included a slate of models with a diploidization parameter, and show that our main conclusions are robust to this process (fig. s18). these models also suggest modest statistical support for diploidization (table s6), although our estimates of its rate were highly uncertain. furthermore, additional lines of evidence for classifying species as diploid or polyploid (beyond the genus-level chromosome multiplicity that we primarily relied on) are needed for more reliable conclusions. other lines of evidence about the prevalence of diploidization within solanaceae or its ancestors are mixed or even conflicting. on the one hand, polyploidy may have occurred before the origin

19 myr; tomato genome consortium, 2012) likely pre-dating the origin of the family (49 \004 3 myr; s€arkinen et al., 2013). on the other hand, there is little evidence for the occurrence of diploidization after the origin of solanaceae itself. studies comparing genetic map-based genome synteny within a number of species in this family find no evidence for diploidiztion (wu & tanksley, 2010). instead, simple genome re-arrangements appear sufficient to explain chromosomal evolution within species in the cytogenetically conserved 'x = 12' group, which includes tomato, potato, eggplant, pepper and tobacco. recovery of comparatively few rearrangements would require outstanding convergent loss

hand, there is little evidence for the occurrence of diploidization after the origin of solanaceae itself. studies comparing genetic map-based genome synteny within a number of species in this family find no evidence for diploidiztion (wu & tanksley, 2010). instead, simple genome re-arrangements appear sufficient to explain chromosomal evolution within species in the cytogenetically conserved 'x = 12' group, which includes tomato, potato, eggplant, pepper and tobacco. recovery of comparatively few rearrangements would require outstanding convergent loss of duplicated segments. furthermore, whole genome duplication in a eudicot lineage ancestral to solanaceae would clash with the evidence that the homologous mechanism of

